<HTML>
<HEAD>
    <TITLE>Selection</TITLE>
</HEAD>
<BODY BGCOLOR="white" BACKGROUND="images/backgrounds/WhiteLeather3.gif">

<IMG SRC="images/crystal_ball-tn.gif" ALT="help" ALIGN="LEFT">

<H1>Selecting a protein to model</H1>

<HR>

<P>It is always good to <STRONG>check our data</STRONG> before starting
any analysis.</P>

<P>The final goal of all our work is to understand the characteristics
and dynamic behaviour of a protein sequence. We need a 
structure to achieve this or, if none is available, we
will need to build a new model.</P>

<P>New structures are being reported in the scientific literature every
day at a continuously increasing rate. We must find out if <EM>the
structure for our molecule</EM> has already been solved. It might
have been just published, or even better, it might have not been
published yet but be already available on a databank (e.g. the manuscript may 
be in press, or in preparation with the data submitted to get an ID to
publish). The only way to know is searching the databases.</P>

<P>Remember that the definitive reference for any structure will always
be experimental data, and we may save a lot of work by looking it up
first.</P>

<H2>Is the protein structure already known?</H2>

<P>If that is the case, then it doesn't make sense to build a new model for
it, since experimental data is already available. To find out, we need to
check with existing structural databases.</P>

<P>There are two possibilities:

<UL>
  <LI>If we have the protein sequence then the most straightforward 
  approach is to look for a <STRONG>good match</STRONG> of your 
  sequence data
  with any structure published in the database. You can do it running a 
  FASTA or BLAST job against <EM>PDB sequences</EM></LI>
  
  <LI>If we don't have the sequence, but know the protein's name, 
  function or other associated data, then check out directly in PDB with 
  SRS.
  This will give us pointers to our target structure or other similar
  or related structures. We need to review the entries to find out
  what have we found. </LI>
</UL>

<P>We are looking for <STRONG>good matches</STRONG> between the
protein sequence (be it a whole protein, a domain or a protein 
fragment) we want to study and that of a reported structure.</P>

<P>If we find an existing structure for our exact protein sequence 
we may skip all the modelling procedure and 
jump directly to the <A HREF="analysis.html" TARGET="main">analysis</A> section.</P>

<P>Otherwise, we need to <A HREF="initial.html">build a model</A> for our protein,
even if it only differs in one or two amino acids from an existing
structure.</P>

<IMG SRC="images/dragon_fly.gif" ALT="Dragon" WIDTH="112" HEIGHT="75" ALIGN="RIGHT">

<P>We are looking for <STRONG>exact matches</STRONG> among the protein
(or fragment) we want to analyze and reported structures. Any difference,
no matter how minor it may seem, may have an impact in the overall
structure and properties of the protein, and will therefore require
a new model to be built. That includes <EM>sequence polymorphisms</EM> and
<EM>point mutations</EM>. We can't just substitute one (or more) amino acid 
by other and naïvely expect that everything will remain the same.</P>

<P>The final model must be coherent with existing knowledge about
the protein we want to study: that is why it was so important to start 
<A HREF="learning.html" TARGET="main">finding out more</A> details about the protein we 
want to model.</P>

<HR>

</BODY>

</HTML>
