<HTML>
<HEAD>
<TITLE>Title</TITLE>
<!--Created by Applixware HTML Author, Release 5.00 (1315) on Fri Jul 27 15:51:41 2001-->
<!--Ax:WP:DocVar:HTMLOriginalPath@:"/tmp/ex01693z.aw"-->
</HEAD>
<BODY BGCOLOR="#ffffff">
<CENTER>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P><FONT SIZE=7><FONT FACE="New Century Schoolbook">TINKER</FONT></FONT></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P><B><FONT SIZE=5><FONT FACE="New Century Schoolbook">Software Tools for
Molecular Design</FONT></FONT></B></P>
<P><B><FONT SIZE=5><FONT FACE="New Century Schoolbook">Version 3.9</FONT></FONT></B></P>
<P><B><FONT SIZE=5><FONT FACE="New Century Schoolbook">June 2001</FONT></FONT></B></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">Copyright &#169; 1990-2001
by Jay William Ponder</FONT></FONT></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">All Rights Reserved</FONT></FONT></P>
<P></P>
</CENTER>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P></P>
<P><FONT FACE="New Century Schoolbook">Copyright &#169; 1990-2001 by Jay
William Ponder</FONT></P>
<P><FONT FACE="New Century Schoolbook">All Rights Reserved</FONT></P>
<P></P>
<P></P>
<P><FONT FACE="New Century Schoolbook">User's Guide Cover Illustration by Jay Nelson</FONT></P>
<P><FONT FACE="New Century Schoolbook">Courtesy of Prof. R. T. Paine, Univ. of
New Mexico</FONT></P>
<P></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook">TINKER IS PROVIDED
&quot;AS IS&quot; AND WITHOUT ANY WARRANTY EXPRESS OR IMPLIED. THE USER
ASSUMES ALL RISKS OF USING THIS SOFTWARE. THERE IS NO CLAIM OF THE
MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE.</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook">YOU MAY MAKE COPIES OF
TINKER FOR YOUR OWN USE, AND MODIFY THOSE COPIES. YOU MAY NOT DISTRIBUTE ANY
MODIFIED SOURCE CODE OR DOCUMENTATION TO USERS AT ANY SITE OTHER THAN YOUR OWN.</FONT></FONT></P>
<P></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook">v3.9 6/01</FONT></FONT></P>
<CENTER>
<P></P>
<P></P>
<P><FONT SIZE=7><FONT FACE="New Century Schoolbook">TINKER</FONT></FONT></P>
<P></P>
<P><FONT SIZE=5><FONT FACE="New Century Schoolbook">Software Tools for
Molecular Design</FONT></FONT></P>
<P><FONT SIZE=5><FONT FACE="New Century Schoolbook">Version 3.9   June 2001</FONT></FONT></P>
<P></P>
<P></P>
</CENTER>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook"><B><U>Table of Contents</U></B>&#9;<B><U>Page</U></B></FONT></FONT></P>
<P></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">Introduction to the TINKER
Package&#9; 5</FONT></FONT></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">Installing TINKER on your
Computer&#9; 7</FONT></FONT></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">Types of Input &amp;
Output Files&#9; 9</FONT></FONT></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">Potential Energy
Programs&#9; 12</FONT></FONT></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">Structure Manipulation
Programs&#9; 18</FONT></FONT></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">Force Field Parameter
Sets&#9; 22</FONT></FONT></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">Use of the Keyword Control
File&#9; 29</FONT></FONT></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">Notes on Special Features
&amp; Methods&#9; 59</FONT></FONT></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">Descriptions of TINKER
Routines&#9; 65</FONT></FONT></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">Contents of Common Block
Variables&#9; 124</FONT></FONT></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">Index of Function &amp;
Subroutine Calls&#9; 150</FONT></FONT></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">Examples using the TINKER
Package&#9; 174</FONT></FONT></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">Benchmark Results&#9; 176</FONT></FONT></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">Collaborators &amp;
Acknowledgments&#9; 179</FONT></FONT></P>
<P><FONT SIZE=4><FONT FACE="New Century Schoolbook">References &amp; Suggested
Reading&#9; 181</FONT></FONT></P>
<P></P>
<TABLE WIDTH=450 CELLPADDING=0>
<TR VALIGN=TOP>
<TD COLSPAN=1 WIDTH="12%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">1.</FONT></FONT></U></B></P>
</TD>
<TD COLSPAN=1 WIDTH="88%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">Introduction to the
TINKER Package</FONT></FONT></U></B></P>
</TD></TR>
</TABLE>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Welcome
to the TINKER molecular modeling package!  TINKER is designed to be an easily
used and flexible system of programs and routines for molecular mechanics and
dynamics as well as other energy-based and structural manipulation
calculations. It is intended to be modular enough to enable development of new
computational methods and efficient enough to meet most production calculation
needs. Rather than incorporating all the functionality in one monolithic
program, TINKER provides a set of relatively small programs that interoperate
to perform complex computations. New programs can be easily added by modelers
with only limited programming experience. The series of major programs
included in the distribution system perform the following core tasks:</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;(1)&#9;build
protein and nucleic acid models from sequence</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;(2)&#9;energy
minimization and structural optimization</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;(3)&#9;analysis
of energy distribution within a structure</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;(4)&#9;molecular
dynamics and stochastic dynamics</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;(5)&#9;simulated
annealing with a choice of cooling schedules</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;(6)&#9;normal
modes and vibrational frequencies</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;(7)&#9;conformational
search and global optimization</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;(8)&#9;transition
state location and conformational pathways</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;(9)&#9;fitting
of energy parameters to crystal data</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;(10)&#9;distance
geometry with pairwise metrization</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;(11)&#9;molecular
volumes and surface areas</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;(12)&#9;free
energy changes for structural mutations</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;(13)&#9;advanced
algorithms based on potential smoothing</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Many of
the various energy minimization and molecular dynamics computations can be
performed on full or partial structures, over Cartesian, internal or rigid
body coordinates, and including a variety of boundary conditions and crystal
cell types. Other programs are available to generate timing data and allow
checking of potential function derivatives for coding errors. Special features
are available to facilitate input and output of protein and nucleic acid
structures. However, the basic core routines have no knowledge of biopolymer
structure and can be used for general molecular systems.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Due to
its emphasis on ease of modification, TINKER differs from many other currently
available molecular modeling packages in that the user is expected to be
willing to write simple ``front-end'' programs and make some alterations at
the source code level. The main programs provided should be considered as
templates for the users to change according to their wishes. All subroutines
are internally documented and structured programming practices are adhered to
throughout. The result, it is hoped, will be a calculational system which can
be tailored to local needs and desires.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The core
TINKER system consists of nearly 120,000 lines of source written entirely in a
portable Fortran77 superset. Use is made of only some very common extensions
that aid in writing highly structured code. The current version of the package
has been ported to a wide range of computers with no or extremely minimal
changes. Tested machines include: Compaq Alphas under Tru64 Unix or OpenVMS;
Hewlett-Packard, IBM RS/6000, Silicon Graphics and Sun workstations under the
vendor's Unix; Apple Macintosh; and Intel PCs under Windows9X/NT/2000 and
Linux. At present, our new code is written on Athlon Linux and Compaq Alpha
platforms, and occasionally tested for compatibility on various of the other
machine and OS combinations listed above. At present, we are in the process of
converting our primary development efforts from Fortran77 to a more modern
Fortran dialect. A machine-translated C version of TINKER is currently
available, and a hand-translated optimized C version of a previous TINKER
version is available for inspection. Conversion to C or C++ is under
consideration, but not being actively pursued at this time.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The
basic design of the energy function engine used by the TINKER system allows
usage of several different parameter sets. At present we are distributing
parameters that implement AMBER-95, CHARMM27, MM2, MM3, OPLS-AA, OPLS-UA and
our own TINKER parameters. In most cases, the source code separates the
geometric manipulations needed for energy derivatives from the actual form of
the energy function itself. Several other literature parameter sets are being
considered for future development (ENCAD, MMFF-94, MM4, UFF, <I>etc.)</I>, and
many of the alternative potential function forms reported in the literature
can be implemented directly or after minor code changes.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Much of
the software in the TINKER package has been heavily used and well tested, but
some modules are still in a fairly early stage of development. Further work on
the TINKER system is planned in three main areas: (1) extension and
improvement of the potential energy parameters including further development
of our polarizable multipole TINKER force field, (2) coding of new
computational algorithms including additional methods for free energy
determination, torsional Monte Carlo and molecular dynamics sampling, advanced
methods for long range interactions, better transition state location, and
further application of the potential smoothing paradigm, (3) a friendlier user
interface for protein/nucleic acid/polysaccharide computations including
direct input/output of Protein Data Bank files, and (4) a Java-based GUI
front-end to tie together the TINKER programs and provide for basic visualization.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Questions
and comments regarding the TINKER package, including suggestions for
improvements and changes should be made to the author:</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Professor
Jay William Ponder</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Biochemistry
&amp; Molecular Biophysics</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;North
Building, Room 2811, Box 8231</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Washington
University School of Medicine</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;660
South Euclid Avenue</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Saint
Louis, MO  63110  U.S.A.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;phone:&#9;(314)
362-4195</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;fax:&#9;(314)
362-7183</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;email:&#9;ponder@dasher.wustl.edu</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">In addition,
an Internet web site containing an online version of this User's Guide, the
most recent distribution version of the full TINKER package and other useful
information can be found at </FONT><B><TT><FONT FACE="courier, courier new">http://dasher.wustl.edu/tinker</FONT></TT></B><FONT FACE="New Century Schoolbook">,
the Home Page for the TINKER Molecular Modeling Package.</FONT></FONT></P>
<TABLE WIDTH=450 CELLPADDING=0>
<TR VALIGN=TOP>
<TD COLSPAN=1 WIDTH="12%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">2.</FONT></FONT></U></B></P>
</TD>
<TD COLSPAN=1 WIDTH="88%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">Installing TINKER on
your Computer</FONT></FONT></U></B></P>
</TD></TR>
</TABLE>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The
TINKER package is distributed on the Internet via either the web site or the
anonymous ftp account on </FONT><B><TT><FONT FACE="courier, courier new">dasher.wustl.edu</FONT></TT></B><FONT FACE="New Century Schoolbook">
with an IP number of 128.252.68.48. This node is an AlphaServer 4100 file
server running Compaq Tru64 Unix located in the Ponder lab at Washington
University School of Medicine. The package is available via the web and
standard browsers from the TINKER home page at </FONT><B><TT><FONT FACE="courier, courier new">http://dasher.wustl.edu/tinker</FONT></TT></B><FONT FACE="New Century Schoolbook">.
Alternatively TINKER can be downloaded by logging into <B>dasher.wustl.edu</B>
under anonymous ftp (Username: </FONT><B><TT><FONT FACE="courier, courier new">anonymous</FONT></TT></B><FONT FACE="New Century Schoolbook">,
Password: </FONT><FONT FACE="New Century Schoolbook"><B><FONT FACE="New Century Schoolbook">&quot;</FONT><TT><FONT FACE="courier, courier new">your
email address</FONT></TT>&quot;</B>) and downloading the software from the
/pub/tinker subdirectory. The complete TINKER distribution as well as
individual files can be downloaded from this site.</FONT></FONT></P>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;On
dasher.wustl.edu, the TINKER package is present as both a compressed Unix tar
archive and as a complete set of uncompressed source and data files. Binaries
are provided for Intel PCs running Windows 9X/ME/NT/2000, PCs running Linux,
and for Apple Power Macintosh running Mac OS 9 or earlier. All of these
executables are available in standard compressed formats as individual
programs or as complete sets of executables. It is expected that other Unix
users and PC users who need specially customized versions, will build binaries
for their specific system. Sites with access to the Unix tar, compress and
uncompress commands should simply obtain the archive file </FONT><B><TT><FONT FACE="courier, courier new">tinker.tar.Z</FONT></TT></B><FONT FACE="New Century Schoolbook">.
Alternatively, a file </FONT><B><TT><FONT FACE="courier, courier new">tinker.tar.gz</FONT></TT></B><FONT FACE="New Century Schoolbook">
containing a tar archive compressed with GNU gzip is also provided. If you
choose to download individual files, you will need at a minimum the contents
of the /doc, /source and /params subdirectories. Also required are the
compile/build scripts from the subdirectory named for your machine type. Other
areas contain test cases and examples, benchmark results, and
machine-translated C code. The entire TINKER package, after building the
executables, will require from about 40 to over 100 megabytes of disk space
depending on the components installed and the use of shared libraries in the executables.</FONT></FONT></P>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The
documentation for the TINKER programs, including the guide you are currently
reading, is located in the /pub/tinker/doc subdirectory. The documentation was
prepared using the Applixware Words and Graphics programs. Portable versions
of the documentation are provided as ascii text in </FONT><B><TT><FONT FACE="courier, courier new">.txt</FONT></TT></B><FONT FACE="New Century Schoolbook">
files and in </FONT><B><TT><FONT FACE="courier, courier new">.ps</FONT></TT></B><FONT FACE="New Century Schoolbook">
Postscript and </FONT><B><TT><FONT FACE="courier, courier new">.pdf</FONT></TT></B><FONT FACE="New Century Schoolbook">
Adobe Acrobat file formats. Please read and return by mail the TINKER license.
While the intent is to distribute the TINKER code to anyone who wants it, the
authors would like to keep track of the sites using the package. The returned
license forms also help us justify further development of TINKER. When new
modules and capabilities become available, and when the almost inevitable bugs
are uncovered, we will attempt to notify those who have returned a license
form. Finally, we remind you that this software is copyrighted, and ask that
it not be redistributed in any form.</FONT></FONT></P>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The
compilation and building of the TINKER executables should be easy for most of
the common workstation and PC class computers. We provide in the /make area a
Unix-style Makefile that with some modification can be used to build TINKER on
most Unix machines. As a simpler alternative to Makefiles for the Unix
versions, we also provide machine-specific directories with three separate
shell scripts to compile the source, build an object library, and link binary
executables. Three similar command files are provided for Open VMS, and for PC
and Macintosh systems. Compilation on Unix workstations should use the vendor
supplied Fortran compiler, if available. The public domain GNU g77 Fortran
compiler available from </FONT><B><TT><FONT FACE="courier, courier new">http://gcc.gnu.org</FONT></TT></B><FONT FACE="New Century Schoolbook">
is also capable of building TINKER on Unix machines and under Linux. The Linux
executables we provide are built with the either the Portland Group (PGI)
compiler or the Absoft ProFortran compiler, both of which generate somewhat
faster executables than g77. For the Macintosh we have favorable experience
with the Absoft ProFortran compiler running under the Macintosh Programmers'
Workbench (MPW). On PC's running Windows 9X/NT/2000, the distributed TINKER
executables are built on an Intel Pentium III CPU under the Compaq Visual
Fortran 6.5 compiler. While the CVF 6.5 compiler has Athlon-specific
optimizations, we have not yet investigated these options. The Microsoft
Fortran Power Station 4.0 and Watcom F77 compilers are also sufficient for
building TINKER, and we provide scripts for each of these PC compilers.
Alternative Windows compilers such as those from Lahey/Fujitsu and The
Portland Group should work as well, but we have not evaluated them yet. Please
see the README files in each of the machine-specific areas for further information.</FONT></FONT></P>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The
first step in building TINKER using the script files is to run the appropriate
``compile'' script. Next you must use the ``library'' script to create an
archive of object code modules. Finally, run the ``link'' script to produce
the complete set of TINKER executables. The executables can be renamed and
moved to wherever you like by editing and running the ``rename'' script.</FONT></FONT></P>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Regardless
of your target machine, only three small pieces of code can possibly require
attention prior to building. The first two are the system dependent time and
date routines found in </FONT><B><TT><FONT FACE="courier, courier new">clock.f</FONT></TT></B><FONT FACE="New Century Schoolbook">
and </FONT><B><TT><FONT FACE="courier, courier new">calendar.f</FONT></TT></B><FONT FACE="New Century Schoolbook">
respectively. Please uncomment the sections of these routines needed for your
computer type. The final set of source alterations are the master array
dimensions found in the include file </FONT><B><TT><FONT FACE="courier, courier new">sizes.i</FONT></TT></B><FONT FACE="New Century Schoolbook">.
The most basic limit is on the number of atoms allowed, ``maxatm''. This
parameter can be set to 10000 or more on most workstations. Personal computers
with minimal memory may need a lower limit, perhaps 1000 atoms, depending on
available memory, swap space and other resources. A description of the other
parameter values is contained in the header of the file. Note that in order to
keep the code completely transparent, TINKER does not implement any sort of
virtual memory or heap data structure. This requires that </FONT><B><TT><FONT FACE="courier, courier new">sizes.i</FONT></TT></B><FONT FACE="New Century Schoolbook">
dimensioning values be set at least as large as the biggest problem you intend
to run. Obviously, you should not set the array sizes to unnecessarily large
values, since this can tax your compute resources and may result in
performance degradation. The worst case we know of at present is for some of
the Compaq Alpha-based machines, where running a ``small'' problem with TINKER
executables dimensioned to ``large'' sizes can result in a 25-50% CPU time
penalty, especially if only the default compiler options are used.</FONT></FONT></P>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Specific
questions about the building or use of the TINKER package should be directed
to </FONT><B><TT><FONT FACE="courier, courier new">ponder@dasher.wustl.edu</FONT></TT></B><FONT FACE="New Century Schoolbook">.
TINKER related questions or comments of more general interest can be sent to
the Computational Chemistry List (</FONT><B><TT><FONT FACE="courier, courier new">http://www.ccl.net</FONT></TT></B><FONT FACE="New Century Schoolbook">)
run by Jan Labanowski of The Ohio Supercomputer Center. The TINKER developers
monitor this list and will respond to the list or the individual poster as
appropriate. </FONT></FONT></P>
<TABLE WIDTH=450 CELLPADDING=0>
<TR VALIGN=TOP>
<TD COLSPAN=1 WIDTH="12%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">3.</FONT></FONT></U></B></P>
</TD>
<TD COLSPAN=1 WIDTH="88%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">Types of Input &amp;
Output Files</FONT></FONT></U></B></P>
</TD></TR>
</TABLE>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;This
section describes the basic file types used by the TINKER package. Let's say
you wish to perform a calculation on a particular small organic molecule.
Assume that the file name chosen for our input and output files is </FONT><B><TT><FONT FACE="courier, courier new">sample</FONT></TT></B><FONT FACE="New Century Schoolbook">.
Then all of the TINKER files will reside on the computer under the name </FONT><B><TT><FONT FACE="courier, courier new">sample.xxx</FONT></TT></B><FONT FACE="New Century Schoolbook">
where </FONT><B><TT><FONT FACE="courier, courier new">.xxx</FONT></TT></B><FONT FACE="New Century Schoolbook">
is any of the several extension types to be described below.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SAMPLE.XYZ</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
file is the basic TINKER Cartesian coordinates file type. It contains a title
line followed by one line for each atom in the structure. Each line contains:
the sequential number within the structure, an atomic symbol or name, X-, Y-,
and Z-coordinates, the force field atom type number of the atom, and a list of
the atoms connected to the current atom. Except for programs whose basic
operation is in torsional space, all TINKER calculations are done from some
version of the </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
format.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SAMPLE.INT</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The </FONT><B><TT><FONT FACE="courier, courier new">.int</FONT></TT></B><FONT FACE="New Century Schoolbook">
file contains an internal coordinates representation of the molecular
structure. It consists of a title line followed by one line for each atom in
the structure. Each line contains: the sequential number within the structure,
an atomic symbol or name, the force field atom type number of the atom, and
internal coordinates in the usual Z-matrix format. For each atom the internal
coordinates consist of a distance to some previously defined atom, and either
two bond angles or a bond angle and a dihedral angle to previous atoms. The
length, angle and dihedral definitions do not have to represent real bonded
interactions. Following the last atom definition are two optional blank line
separated sets of atom number pairs. The first list contains pairs of atoms
that are covalently bonded, but whose bond length was not used as part of the
atom definitions. These pairs are typically used to close ring structures. The
second list contains ``bonds'' that are to be broken, <I>i.e.</I>, pairs of
atoms that are not covalently bonded, but which were used to define a distance
in the atom definitions.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SAMPLE.KEY</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The keyword
parameter file always has the extension </FONT><B><TT><FONT FACE="courier, courier new">.key</FONT></TT></B><FONT FACE="New Century Schoolbook">
and is optionally present during TINKER calculations. It contains values for
any of a wide variety of switches and parameters that are used to change the
course of the computation from the default. The detailed contents of this file
is explained in a latter section of this User's Guide. If a molecular system
specific keyfile, in this case </FONT><B><TT><FONT FACE="courier, courier new">sample.key</FONT></TT></B><FONT FACE="New Century Schoolbook">,
is not present, the the TINKER program will look in the same directory for a
generic file named </FONT><B><TT><FONT FACE="courier, courier new">tinker.key</FONT></TT></B><FONT FACE="New Century Schoolbook">.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SAMPLE.DYN</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The </FONT><B><TT><FONT FACE="courier, courier new">.dyn</FONT></TT></B><FONT FACE="New Century Schoolbook">
file contains values needed to restart a molecular or stochastic dynamics
computation. It stores the current position, current velocity and current and
previous accelerations for each atom, as well as the size and shape of any
periodic box or crystal unit cell. This information can be used to start a new
dynamics run from the final state of a previous run. Upon startup, the
dynamics programs always check for the presence of a </FONT><B><TT><FONT FACE="courier, courier new">.dyn</FONT></TT></B><FONT FACE="New Century Schoolbook">
file and make use of it whenever possible. The </FONT><B><TT><FONT FACE="courier, courier new">.dyn</FONT></TT></B><FONT FACE="New Century Schoolbook">
file is updated concurrent with the saving of a new dynamics trajectory snapshot.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SAMPLE.END</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The </FONT><B><TT><FONT FACE="courier, courier new">.end</FONT></TT></B><FONT FACE="New Century Schoolbook">
file type provides a mechanism to gracefully stop a running TINKER
calculation. At appropriate checkpoints during a calculation, TINKER will test
for the presence of a </FONT><B><TT><FONT FACE="courier, courier new">sample.end</FONT></TT></B><FONT FACE="New Century Schoolbook">
file, and if found will terminate the calculation after updating the output.
The </FONT><B><TT><FONT FACE="courier, courier new">.end</FONT></TT></B><FONT FACE="New Century Schoolbook">
file can be created at any time during a computation, and will be detected
when the next checkpoint is reached. The file may be of zero size, and its
contents are unimportant. In the current version of TINKER, the </FONT><B><TT><FONT FACE="courier, courier new">.end</FONT></TT></B><FONT FACE="New Century Schoolbook">
mechanism is only available within dynamics-based programs.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SAMPLE.001,
SAMPLE.002, ....</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Several
types of computations produce files containing a three or more digit extension (</FONT><B><TT><FONT FACE="courier, courier new">.001</FONT></TT></B><FONT FACE="New Century Schoolbook">
as shown; or </FONT><B><TT><FONT FACE="courier, courier new">.002</FONT></TT></B><FONT FACE="New Century Schoolbook">,
</FONT><B><TT><FONT FACE="courier, courier new">.137</FONT></TT></B><FONT FACE="New Century Schoolbook">,
</FONT><B><TT><FONT FACE="courier, courier new">.5678</FONT></TT></B><FONT FACE="New Century Schoolbook">,
<I>etc</I>.). These are referred to as cycle files, and are used to store
various types of output structures. The cycle files from a given computation
are identical in internal structure to either the </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
or </FONT><B><TT><FONT FACE="courier, courier new">.int</FONT></TT></B><FONT FACE="New Century Schoolbook">
files described above. For example, the vibrational analysis program can save
the tenth normal mode in </FONT><B><TT><FONT FACE="courier, courier new">sample.010</FONT></TT></B><FONT FACE="New Century Schoolbook">.
A molecular dynamics-based program might save its tenth 0.1 picosecond frame
(or an energy minimizer its tenth partially minimized intermediate) in a file
of the same name.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SAMPLE.ARC</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A TINKER
archive file is simply a series of </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
Cartesian coordinate files appended together one after another. This file can
be used to condense the results from intermediate stages of an optimization,
frames from a molecular dynamics trajectory, or set of normal mode vibrations
into a single file for storage.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SAMPLE.PDB</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">This file
type contains coordinate information in the PDB format developed by the
Brookhaven Protein Data Bank for deposition of model structures based on
macromolecular X-ray diffraction and NMR data. Although TINKER itself does not
use </FONT><B><TT><FONT FACE="courier, courier new">.pdb</FONT></TT></B><FONT FACE="New Century Schoolbook">
files directly for input/output, auxiliary programs are provided with the
system for interconverting </FONT><B><TT><FONT FACE="courier, courier new">.pdb</FONT></TT></B><FONT FACE="New Century Schoolbook">
files with the </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
format described above.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SAMPLE.SEQ</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">This file
type contains the primary sequence of a biopolymer in the standard one-letter
code with 50 residues per line. The </FONT><B><TT><FONT FACE="courier, courier new">.seq</FONT></TT></B><FONT FACE="New Century Schoolbook">
file for a biopolymer is generated automatically when a PDB file is converted
to TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
format or when using the PROTEIN or NUCLEIC programs to build a structure from
sequence It is required for the reverse conversion of a TINKER file back to
PDB format..</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SAMPLE.FRAC</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The
fractional coordinates corresponding to the asymmetric unit of a crystal unit
cell are stored in the </FONT><B><TT><FONT FACE="courier, courier new">.frac</FONT></TT></B><FONT FACE="New Century Schoolbook">
file. The internal format of this file is identical to the </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
file; except that the coordinates are fractional instead of in Angstrom units.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SAMPLE.XMOL</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The ARCHIVE
program has the option of converting a series of </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
cycle files into an XMakemol XYZ file. These files can be displayed as a movie
using the XMakemol display program. Note that the </FONT><B><TT><FONT FACE="courier, courier new">.xmol</FONT></TT></B><FONT FACE="New Century Schoolbook">
file format does not contain TINKER atom type information, so it is not
possible to convert an </FONT><B><TT><FONT FACE="courier, courier new">.xmol</FONT></TT></B><FONT FACE="New Century Schoolbook">
file back into a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
file.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SAMPLE.MSI</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The ARCHIVE
program has the option of converting a series of </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
cycle files into a MSI InsightII coordinate archive file. These files can be
displayed as a movie using the InsightII display program. Note that the </FONT><B><TT><FONT FACE="courier, courier new">.msi</FONT></TT></B><FONT FACE="New Century Schoolbook">
file format does not contain TINKER atom type information, so it is not
possible to convert a </FONT><B><TT><FONT FACE="courier, courier new">.msi</FONT></TT></B><FONT FACE="New Century Schoolbook">
file back into a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.XYZ</FONT></TT></B><FONT FACE="New Century Schoolbook">
file.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">PARAMETER FILES</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The
potential energy parameter files distributed with the TINKER package all end
in the extension </FONT><B><TT><FONT FACE="courier, courier new">.prm</FONT></TT></B><FONT FACE="New Century Schoolbook">,
although this is not required by the programs themselves. Each of these files
contains a definition of the potential energy functional forms for that force
field as well as values for individual energy parameters. For example, the </FONT><B><TT><FONT FACE="courier, courier new">mm3pro.prm</FONT></TT></B><FONT FACE="New Century Schoolbook">
file contains the energy parameters and definitions needed for a
protein-specific version of the MM3 force field.</FONT></FONT></P>
<TABLE WIDTH=450 CELLPADDING=0>
<TR VALIGN=TOP>
<TD COLSPAN=1 WIDTH="12%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">4.</FONT></FONT></U></B></P>
</TD>
<TD COLSPAN=1 WIDTH="88%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">Potential Energy Programs</FONT></FONT></U></B></P>
</TD></TR>
</TABLE>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;This
section of the manual contains a brief description of each of the TINKER
potential energy programs. A detailed example showing how to run each program
is included in a later section. The programs listed below are all part of the
main, supported distribution. Additional source code for various unsupported
programs can be found in the /other directory of the TINKER distribution.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">ALCHEMY</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A simple
program to perform very basic free energy perturbation calculations. This
program is provided mostly for demonstration purposes.  For example, we use
ALCHEMY in a molecular modeling course laboratory exercise to perform such
classic mutations as chloride to bromide and ethane to methanol in water. The
present version uses the perturbation formula and windowing with an explicit
mapping of atoms involved in the mutation (``AMBER''-style), instead of
thermodynamic integration and independent freely propagating groups of mutated
atoms (``CHARMM''-style). Some of the code specific to this program is limited
to the AMBER and OPLS potential functional forms, but could be easily
generalized to handle other potentials. A more general and sophisticated
version is currently under development.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">ANALYZE</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Provides
information about a specific molecular structure. The program will ask for the
name of a structure file, which must be in the TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
file format, and the type of analysis desired. Options allow output of:  (1)
total potential energy of the system, (2) breakdown of the energy by potential
function type or over individual atoms, (3) computation of the total dipole
moment and its components, moments of inertia and radius of gyration, (4)
listing of the parameters used to compute selected interaction energies, (5)
energies associated with specified individual interactions.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">ANNEAL</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Performs a
molecular dynamics simulated annealing computation. The program starts from a
specified input molecular structure in TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
format. The trajectory is updated using either a modified Beeman or a velocity
Verlet integration method. The annealing protocol is implemented by allowing
smooth changes between starting and final values of the system temperature via
the Groningen method of coupling to an external bath. The scaling can be
linear or sigmoidal in nature. In addition, parameters such as cutoff distance
can be transformed along with the temperature. The user must input the desired
number of dynamics steps for both the equilibration and cooling phases, a time
interval for the dynamics steps, and an interval between coordinate/trajectory
saves. All saved coordinate sets along the trajectory are placed in
sequentially numbered cycle files.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">DYNAMIC</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Performs a
molecular dynamics (MD) or stochastic dynamics (SD) computation. Starts either
from a specified input molecular structure (an </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
file) or from a structure-velocity-acceleration set saved from a previous
dynamics trajectory (a restart from a </FONT><B><TT><FONT FACE="courier, courier new">.dyn</FONT></TT></B><FONT FACE="New Century Schoolbook">
file). MD trajectories are propagated using either a modified Beeman or a
velocity Verlet integration method. SD is implemented via our own derivation
of a velocity Verlet-based algorithm. In addition the program can perform full
crystal calculations, and can operate in constant energy mode or with
maintenance of a desired temperature and/or pressure using the Groningen
method of coupling to external baths. The user must input the desired number
of dynamics steps, a time interval for the dynamics steps, and an interval
between coordinate/trajectory saves. Coordinate sets along the trajectory can
be saved as sequentially numbered cycle files or directly to a TINKER archive </FONT><B><TT><FONT FACE="courier, courier new">.arc</FONT></TT></B><FONT FACE="New Century Schoolbook">
file. At the same time that a point along the trajectory is saved, the
complete information needed to restart the trajectory from that point is
updated and stored in the </FONT><B><TT><FONT FACE="courier, courier new">.dyn</FONT></TT></B><FONT FACE="New Century Schoolbook">
file.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">GDA</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program to
implement Straub's Gaussian Density Annealing algorithm over an effective
series of analytically smoothed potential energy surfaces. This method can be
viewed as an extended stochastic version of the diffusion equation method of
Scheraga, <I>et al</I>., and also has many similar features to the TINKER
Potential Smoothing and Search (PSS) series of programs. The current version
of GDA is similar to but does not exactly reproduce Straub's published method
and is limited to argon clusters and other simple systems involving only van
der Waals interactions; further modification and development of this code is
currently underway in the Ponder research group. As with other programs
involving potential smoothing, GDA currently requires use of the </FONT><B><TT><FONT FACE="courier, courier new">smooth.prm</FONT></TT></B><FONT FACE="New Century Schoolbook">
force field parameters.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">MINIMIZE</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The MINIMIZE
program performs a limited memory L-BFGS minimization of an input structure
over Cartesian coordinates using a modified version of the algorithm of Jorge
Nocedal. The method requires only the potential energy and gradient at each
step along the minimization pathway. It requires storage space proportional to
the number of atoms in the structure. The MINIMIZE procedure is recommended
for preliminary minimization of trial structures to an <I>rms</I> gradient of
1.0 to 0.1 kcal/mole/&Aring;. It has a relatively fast cycle time and is
tolerant of poor initial structures, but converges in a slow, linear fashion
near the minimum. The user supplies the name of the TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
coordinates file and a target <I>rms</I> gradient value at which the
minimization will terminate. Output consists of minimization statistics
written to the screen or redirected to an output file, and the new coordinates
written to updated </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
files or to cycle files.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">MINIROT</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The MINIROT
program uses the same limited memory L-BFGS method as MINIMIZE, but performs
the computation in terms of dihedral angles instead of Cartesian coordinates.
Output is saved in an updated </FONT><B><TT><FONT FACE="courier, courier new">.int</FONT></TT></B><FONT FACE="New Century Schoolbook">
file or in cycle files.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">MINRIGID</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The MINRIGID
program is similar to MINIMIZE except that it operates on rigid bodies
starting from a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
coordinate file and the rigid body group definitions found in the
corresponding </FONT><B><TT><FONT FACE="courier, courier new">.key</FONT></TT></B><FONT FACE="New Century Schoolbook">
file. Output is saved in an updated </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
file or in cycle files.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">NEWTON</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A truncated
Newton minimization method which requires potential energy, gradient and
Hessian information. This procedure has significant advantages over standard
Newton methods, and is able to minimize very large structures completely.
Several options are provided with respect to minimization method and
preconditioning of the Newton equations. The default options are recommended
unless the user is familiar with the math involved. This program operates in
Cartesian coordinate space and is fairly tolerant of poor input structures.
Typical algorithm iteration times are longer than with nonlinear conjugate
gradient or variable metric methods, but many fewer iterations are required
for complete minimization. NEWTON is usually the best choice for minimizations
to the 0.01 to 0.000001 kcal/mole/&Aring; level of <I>rms</I> gradient
convergence. Tests for directions of negative curvature can be removed,
allowing NEWTON to be used for optimization to conformational transition state
structures (this only works if the starting point is very close to the
transition state). Input consists of a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
coordinates file; output is an updated set of minimized coordinates and
minimization statistics.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">NEWTROT</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The NEWTROT
program is similar to NEWTON except that it requires a </FONT><B><TT><FONT FACE="courier, courier new">.int</FONT></TT></B><FONT FACE="New Century Schoolbook">
file as input and then operates in terms of dihedral angles as the
minimization variables. Since the dihedral space Hessian matrix of an
arbitrary structure is often indefinite, this method will often not perform as
well as the other, simpler dihedral angle based minimizers.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">OPTIMIZE</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The OPTIMIZE
program performs a optimally conditioned variable metric minimization of an
input structure over Cartesian coordinates using an algorithm due to William
Davidon. The method does not perform line searches, but requires computation
of energies and gradients as well as storage for an estimate of the inverse
Hessian matrix. The program operates on Cartesian coordinates from a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
file. OPTIMIZE will typically converge somewhat faster and more completely
than MINIMIZE. However, the need to store and manipulate a full inverse
Hessian estimate limits its use to structures containing less than a few
hundred atoms on workstation class machines. As with the other minimizers,
OPTIMIZE needs input coordinates and an <I>rms</I> gradient cutoff criterion.
The output coordinates are saved in updated </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
files or as cycle files.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">OPTIROT</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The OPTIROT
program is similar to OPTIMIZE except that it operates on dihedral angles
starting from a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.int</FONT></TT></B><FONT FACE="New Century Schoolbook">
internal coordinate file. This program is usually the preferred method for
most dihedral angle optimization problems since Truncated Newton methods
appear, in our hands, to lose some of their efficacy in moving from Cartesian
to torsional coordinates.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">OPTRIGID</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The OPTRIGID
program is similar to OPTIMIZE except that it operates on rigid bodies
starting from a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
coordinate file and the rigid body atom group definitions found in the
corresponding </FONT><B><TT><FONT FACE="courier, courier new">.key</FONT></TT></B><FONT FACE="New Century Schoolbook">
file. Output is saved in an updated </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
file or in cycle files.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">PATH</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program
that implements a variant of Elber's Lagrangian multiplier-based reaction path
following algorithm. The program takes as input a pair of structural minima as
TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
files, and then generates a user specified number of points along a path
through conformational space connecting the input structures. The intermediate
structures are output as TINKER cycle files, and the higher energy
intermediates can be used as input to a Newton-based optimization to locate
conformational transition states.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">PSS</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Implements
our version of a potential smoothing and search algorithm for the global
optimization of molecular conformation. An initial structure in </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
format is first minimized in Cartesian coordinates on a series of increasingly
smoothed potential energy surfaces. Then the smoothing procedure is reversed
with minimization on each successive surface starting from the coordinates of
the minimum on the previous surface. A local search procedure is used during
the backtracking to explore for alternative minima better than the one found
during the current minimization. The final result is usually a very low energy
conformation or, in favorable cases, the global energy minimum conformation.
The minimum energy coordinate sets found on each surface during both the
forward smoothing and backtracking procedures are placed in sequentially
numbered cycle files.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">PSSRIGID</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">This program
implements the potential smoothing and search method as described above for
the PSS program, but performs the computation in terms of keyfile-defined
rigid body atom groups instead of Cartesian coordinates. Output is saved in
numbered cycle files with the </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
file format.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">PSSROT</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">This program
implements the potential smoothing and search method as described above for
the PSS program, but performs the computation in terms of a set of
user-specified dihedral angles instead of Cartesian coordinates. Output is
saved in numbered cycle files with the </FONT><B><TT><FONT FACE="courier, courier new">.int</FONT></TT></B><FONT FACE="New Century Schoolbook">
file format.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SADDLE</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program
for the location of a conformational transition state between two potential
energy minima. SADDLE uses a conglomeration of ideas from the Bell-Crighton
quadratic path and the Halgren-Lipscomb synchronous transit methods. The basic
idea is to perform a nonlinear conjugate gradient optimization in a subspace
orthogonal to a suitably defined reaction coordinate. The program requires as
input the coordinates (TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
files) of the two minima and an <I>rms</I> gradient convergence criterion for
the optimization. The current estimate of the transition state structure is
written to the file TSTATE.XYZ. Crude transition state structures generated by
SADDLE can sometimes be refined using the NEWTON program. Optionally, a scan
of the interconversion pathway can be made at each major iteration.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SCAN</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program
for general conformational search of an entire potential energy surface via a
basin hopping method. The program takes as input a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
coordinates file which is then minimized to find the first local minimum for a
search list. A series of activations along various normal modes from this
initial minimum are used as seed points for additional minimizations. Whenever
a previously unknown local minimum is located it is added to the search list.
When all minima on the search list have been subjected to the normal mode
activation without locating additional new minima, the program terminates. The
individual local minima are written to cycle files as they are discovered.
While the SCAN program can be used on standard undeformed potential energy
surfaces, we have found it to be most useful for quickly ``scanning'' a
smoothed energy surface to enumerate the major basins of attraction spaning
the entire surface.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SNIFFER</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program
that implements the Sniffer global optimization algorithm of Butler and
Slaminka, a discrete version of Griewank's global search trajectory method.
The program takes an input TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
coordinates file and shakes it vigorously via a modified dynamics trajectory
before, hopefully, settling into a low lying minimum. Some trial and error is
often required as the current implementation is sensitive to various
parameters and tolerances that govern the computation. At present, these
parameters are not user accessible, and must be altered in the source code.
However, this method can do a good job of quickly optimizing conformation
within a limited range of convergence.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">TESTGRAD</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The TESTGRAD
program computes and compares the analytical and numerical first derivatives (<I>i.e.</I>,
the gradient vector) of the potential energy for a Cartesian coordinate input
structure. The output can be used to test or debug the current potential or
any added user defined energy terms.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">TESTHESS</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The TESTHESS
program computes and compares the analytical and numerical second derivatives (<I>i.e.</I>,
the Hessian matrix) of the potential energy for a Cartesian coordinate input
structure. The output can be used to test or debug the current potential or
any added user defined energy terms.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">TESTLIGHT</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program to
compare the efficiency of different nonbonded neighbor methods for the current
molecular system. The program times the computation of energy and gradient for
the van der Waals and charge-charge electrostatic potential terms using a
simple double loop over all interactions and using the Method of Lights
algorithm to select neighbors. The results can be used to decide whether the
Method of Lights has any CPU time advantage for the current structure. Both
methods should give exactly the same answer in all cases, since the identical
individual interactions are computed by both methods. The default double loop
method is faster when cutoffs are not used, or when the cutoff sphere contains
about half or more of the total system of unit cell. In cases where the cutoff
sphere is much smaller than the system size, the Method of Lights can be much
faster since it avoids unnecessary calculation of distances beyond the cutoff range.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">TESTROT</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The TESTROT
program computes and compares the analytical and numerical first derivatives (<I>i.e.</I>,
the gradient vector) of the potential energy with respect to dihedral angles.
Input is a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.int</FONT></TT></B><FONT FACE="New Century Schoolbook">
internal coordinate file. The output can be used to test or debug the current
potential functions or any added user defined energy terms.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">TIMER</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A simple
program to provide timing statistics for energy function calls within the
TINKER package. TIMER requires an input </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
file and outputs the CPU time (wall clock time on some machine types) needed
to perform a specified number of energy, gradient and Hessian evaluations.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">TIMEROT</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">This program
is similar to TIMER, only it operates over dihedral angles via input of a
TINKER </FONT><B><TT><FONT FACE="courier, courier new">.int</FONT></TT></B><FONT FACE="New Century Schoolbook">
internal coordinate file. In the current version, the torsional Hessian is
computed numerically from the analytical torsional gradient.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">VIBRATE</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program to
perform vibrational analysis by computing and diagonalizing the full Hessian
matrix (<I>i.e.</I>, the second partial derivatives) for an input structure (a
TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
file). Eigenvalues and eigenvectors of the mass weighted Hessian (<I>i.e.</I>,
the vibrational frequencies and normal modes) are also calculated. Structures
corresponding to individual normal mode motions can be saved in cycle files.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">VIBROT</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The program
VIBROT forms the torsional Hessian matrix via numerical differentiation of the
analytical torsional gradient. The Hessian is then diagonalized and the
eigenvalues are output. The present version does not compute the kinetic
energy matrix elements needed to convert the Hessian into the torsional normal
modes; this will be added in a later version. The required input is a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.int</FONT></TT></B><FONT FACE="New Century Schoolbook">
internal coordinate file.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">XTALFIT</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The XTALFIT
program is of use in the automated fitting of potential parameters to crystal
structure and thermodynamic data. XTALFIT takes as input several crystal
structures (TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
files with unit cell parameters in corresponding keyfiles) as well as
information on lattice energies and dipole moments of monomers. The current
version uses a nonlinear least squares optimization to fit van der Waals and
electrostatic parameters to the input data. Bounds can be placed on the values
of the optimization parameters.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">XTALMIN</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program to
perform full crystal minimizations. The program takes as input the structure
coordinates and unit cell lattice parameters. It then alternates cycles of
Newton-style optimization of the structure and conjugate gradient optimization
of the crystal lattice parameters. This alternating minimization is slower
than more direct optimization of all parameters at once, but is somewhat more
robust in our hands. The symmetry of the original crystal is not enforced, so
interconversion of crystal forms may be observed in some cases.</FONT></FONT></P>
<TABLE WIDTH=450 CELLPADDING=0>
<TR VALIGN=TOP>
<TD COLSPAN=1 WIDTH="12%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">5.</FONT></FONT></U></B></P>
</TD>
<TD COLSPAN=1 WIDTH="88%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">Structure
Manipulation Programs</FONT></FONT></U></B></P>
</TD></TR>
</TABLE>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;This
section of the manual contains a brief description of each of the TINKER
structure manipulation, geometric calculation and auxiliary programs. A
detailed example showing how to run each program is included in a later
section. The programs listed below are all part of the main, supported
distribution. Additional source code for various unsupported programs can be
found in the /other directory of the TINKER distribution.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">ARCHIVE</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program
for concatenating TINKER cycle files into a single archive file; useful for
storing the intermediate results of minimizations, dynamics trajectories, and
so on. The archive file can be written in TINKER format, or in formats usable
with MSI's InsightII (their CAR file with <B>.msi</B> extension) or with
XMakemol (their file format with </FONT><B><TT><FONT FACE="courier, courier new">.xmol</FONT></TT></B><FONT FACE="New Century Schoolbook">
extension). Only active atoms are written into the InsightII and XMakemol
output files, allowing display of partial structures. The program can also
extract individual cycle files from a TINKER archive.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">CORRELATE</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program to
compute time correlation functions from collections of TINKER cycle files. Its
use requires a user supplied function <B>property</B> that computes the value
of the property for which a time correlation is desired for two input
structures. A sample routine is supplied that computes either a velocity
autocorrelation function or an <I>rms</I> structural superposition as a
function of time. The main body of the program organizes the overall
computation in an efficient manner and outputs the final time correlation function.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">CRYSTAL</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program
for the manipulation of crystal structures including interconversion of
fractional and Cartesian coordinates, generation of the unit cell from an
asymmetric unit, and building of a crystalline block of specified size via
replication of a single unit cell. The present version can handle about 25 of
the most common space groups, others can easily be added as needed by
modification of the routine <B>symmetry</B>.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">DIFFUSE</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program to
compute the self-diffusion constant for a homogeneous liquid via the Einstein
equation. A previously saved dynamics trajectory is read in and ``unfolded''
to reverse translation of molecules due to use of periodic boundary
conditions. The average motion over all molecules is then used to compute the
self-diffusion constant. While the current program assumes a homogeneous
system, it should be easy to modify the code to handle diffusion of individual
molecules or other desired effects.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">DISTGEOM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program to
perform distance geometry calculations using variations on the classic metric
matrix method. A user specified number of structures consistent with keyfile
input distance and dihedral restraints is generated. Bond length and angle
restraints are derived from the input structure. Trial distances between the
triangle smoothed lower and upper bounds can be chosen via any of several
metrization methods, including a very effective partial random pairwise
scheme. The correct radius of gyration of the structure is automatically
maintained by choosing trial distances from Gaussian distributions of
appropriate mean and width. The initial embedded structures can be further
refined against a geometric restraint-only potential using either a sequential
minimization protocol or simulated annealing.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">DOCUMENT</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The DOCUMENT
program is provided as a minimal listing and documentation tool. It operates
on the TINKER source code, either individual files or the complete source
listing produced by the command script </FONT><B><TT><FONT FACE="courier, courier new">listing.make</FONT></TT></B><FONT FACE="New Century Schoolbook">,
to generate lists of routines, common blocks or valid keywords. In addition,
the program has the ability to output a formatted parameter listing from the
standard TINKER parameter files.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">INTEDIT</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program to
allow interactive inspection and alteration of the internal coordinate
definitions and values of a TINKER structure. If the structure is altered, the
user has the option to write out a new internal coordinates file upon exit.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">INTXYZ</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program to
convert a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.int</FONT></TT></B><FONT FACE="New Century Schoolbook">
internal coordinates formatted file into a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
Cartesian coordinates formatted file.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">NUCLEIC</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program
for automated building of nucleic acid structures. Upon interactive input of a
nucleotide sequence with optional phosphate backbone angles, the program
builds internal and Cartesian coordinates. Standard bond lengths and angles
are used. Both DNA and RNA sequences are supported as are A-, B- and Z-form
structures. Double helixes of complementary sequence can be automatically
constructed via a rigid docking of individual strands.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">PDBXYZ</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program
for converting a Brookhaven Protein Data Bank file (a PDB file) into a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
Cartesian coordinate file. If the PDB file contains only protein/peptide amino
acid residues, then standard protein connectivity is assumed, and transferred
to the <B>.xyz</B> file. For non-protein portions of the PDB file, atom
connectivity is determined by the program based on interatomic distances. The
program also has the ability to add or remove hydrogen atoms from a protein as
required by the force field specified during the computation.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">POLARIZE</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program
for computing molecular polarizability from an atom-based distributed model of
polarizability. A damped interaction model due to Thole is optionally via
keyfile settings. A TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
file is required as input. The output consists of the overall polarizability
tensor in the global coordinates and its eigenvalues.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">PROTEIN</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program
for automated building of peptide and protein structures. Upon interactive
input of an amino acid sequence with optional phi/psi/omega/chi angles, D/L
chirality, etc., the program builds internal and Cartesian coordinates.
Standard bond lengths and angles are assumed for the peptide. The program will
optionally convert the structure to a cyclic peptide, or add either or both N-
and C-terminal capping groups. Atom type numbers are automatically assigned
for the specified force field. The final coordinates and a sequence file are
produced as the output.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">RADIAL</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program to
compute the pair radial distribution function between two atom types. The user
supplies the two atom names for which the distribution function is to be
computed, and the width of the distance bins for data analysis. A previously
saved dynamics trajectory is read as input. The raw radial distribution and a
spline smoothed version are then output from zero to a distance equal to half
the minimum periodic box dimension. The atom names are matched to the atom
name column of the TINKER <B>.xyz</B> file, independent of atom type.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SPACEFILL</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program to
compute the volume and surface areas of molecules. Using a modified version of
Connolly's original analytical description of the molecular surface, the
program determines either the van der Waals, accessible or molecular
(contact/reentrant) volume and surface area. Both surface area and volume are
broken down into their geometric components, and surface area is decomposed
into the convex contribution for each individual atom. The probe radius is
input as a user option, and atomic radii can be set via the keyword file. If
TINKER archive files are used as input, the program will compute the volume
and surface area of each structure in the input file.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SPECTRUM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program to
compute a power spectrum from velocity autocorrelation data. As input, this
program requires a velocity autocorrelation function as produced by the
CORRELATE program. This data, along with a user input time step, are Fourier
transformed to generate the spectral intensities over a wavelength range. The
result is a power spectrum, and the positions of the bands are those predicted
for an infrared or Raman spectrum. However, the data is not weighted by
molecular dipole moment derivatives as would be required to produce correct IR intensities.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SUPERPOSE</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program to
superimpose two molecular structures in 3-dimensions. A variety of options for
input of the atom sets to be used during the superposition are presented
interactively to the user. The superposition can be mass-weighted if desired,
and the coordinates of the second structure superimposed on the first
structure are optionally output. If TINKER archive files are used as input,
the program will compute all pairwise superpositions between structures in the
input files.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SYBYLXYZ</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program
for converting a TRIPOS Sybyl MOL2 file into a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
Cartesian coordinate file. The current version of the program does not attempt
to convert the Sybyl atoms types into the active TINKER force field types, <I>i.e.</I>,
all atoms types are simply set to zero.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">TVIEW</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">This is a
molecule viewing program derived from the well-know Rasmol program of Roger
Sayle. TVIEW is modified to remove most of the protein-specific options and to
directly read the TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
file format. The original RasMol program has been altered to allow selection
and specification by atoms instead of residues. We hope to provide additional
functionality in future versions of TVIEW, especially the ability to animate
the viewing of sequences of coordinate snapshots from a minimization or
dynamic trajectory.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">XYZEDIT</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program
that performs and of a variety of manipulations on an input TINKER .xyz
Cartesian coordinates formatted file. The present version of the program has
the following interactively selectable options: (1) Offset the Numbers of the
Current Atoms, (2) Deletion of Individual Specified Atoms, (3) Deletion of
Specified Types of Atoms, (4) Deletion of Atoms outside Cutoff Range, (5)
Insertion of Individual Specified Atoms, (6) Replace Old Atom Type with a New
Type, (7) Assign Connectivities based on Distance, (8) Convert Units from
Bohrs to Angstroms, (9) Invert thru Origin to give Mirror Image, (10)
Translate Center of Mass to the Origin, (11) Translate a Specified Atom to the
Origin, (12) Translate and Rotate to Inertial Frame, (13) Move to Specified
Rigid Body Coordinates, (14) Create and Fill a Periodic Boundary Box, (15)
Soak Current Molecule in Box of Solvent, (16) Append another XYZ file to
Current One. In most cases, multiply options can be applied sequentially to an
input file. At the end of the editing process, a new version of the original </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
file is written as output.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">XYZINT</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program
for converting a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
Cartesian coordinate formatted file into a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.int</FONT></TT></B><FONT FACE="New Century Schoolbook">
internal coordinates formatted file.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">XYZPDB</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program
for converting a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
Cartesian coordinate file into a Brookhaven Protein Data Bank file (a PDB file).</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">XYZSYBYL</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A program to
convert a TINKER </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
Cartesian coordinates file into a TRIPOS Sybyl MOL2 file. The conversion
generates only the MOLECULE, ATOM, BOND and SUBSTRUCTURE record type in the
MOL2 file. Generic Sybyl atom types are used in most cases; while these atom
types may need to be altered in some cases, Sybyl is usually able to correctly
display the resulting MOL2 file.</FONT></FONT></P>
<TABLE WIDTH=450 CELLPADDING=0>
<TR VALIGN=TOP>
<TD COLSPAN=1 WIDTH="12%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">6.</FONT></FONT></U></B></P>
</TD>
<TD COLSPAN=1 WIDTH="88%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">Force Field
Parameter Sets</FONT></FONT></U></B></P>
</TD></TR>
</TABLE>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The TINKER
package is distributed with several force field parameter sets, implementing a
selection of widely used literature force fields as well as the TINKER force
field currently under construction in the Ponder lab. We try to exactly
reproduce the intent of the original authors of our distributed, third-party
force fields. In all cases the parameter sets have been validated against
literature reports, results provided by the original developers, or
calculations made with the authentic programs. With the few exceptions noted
below, TINKER calculations can be treated as authentic results from the
genuine force fields. A brief description of each parameter set, including
some still in preparation and not distributed with the current version, is
provided below with lead literature references for the force field:</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">AMBER.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">AMBER-94/96
parameters for proteins and nucleic acids. Note that with their ``Cornell''
force field, the Kollman group has devised separate, fully independent partial
charge values for each of the N- and C-terminal amino acid residues. At
present, the terminal residue charges for TINKER's version maintain the
correct formal charge, but redistributed somewhat at the alpha carbon atoms
from the Kollman group values. The total magnitude of the redistribution is
less than 0.01 electrons in most cases. The file provided with TINKER
reproduces the original parm94 set; torsional parameter changes for parm96 are
noted in that section of the file. The newer, polarizable parm99 parameter set
is not distributed with TINKER at the present time.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. D.
Cornell, P. Cieplak, C. I. Bayly, I. R. Gould, K. M. Merz, Jr., D. M.
Ferguson, D. C. Spellmeyer, T. Fox, J. W. Caldwell and P. A. Kollman, A Second
Generation Force Field for the Simulation of Proteins, Nucleic Acids, and
Organic Molecules, <B><I>J. Am. Chem. Soc.</I></B>, <I>117</I>, 5179-5197
(1995)  [PARM94]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">P. Kollman,
R. Dixon, W. Cornell, T. Fox, C. Chipot and A. Pohorille, The Development/
Application of a 'Minimalist' Organic/Biochemical Molecular Mechanic Force
Field using a Combination of <I>ab Initio</I> Calculations and Experimental
Data, in <B>Computer Simulation of Biomolecular Systems</B>, W. F. van
Gunsteren, P. K. Weiner, A. J. Wilkinson, eds., Volume 3, 83-96 (1997)  [PARM96]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">G. Moyna, H.
J. Williams, R. J. Nachman and A. I. Scott, Conformation in Solution and
Dynamics of a Structurally Constrained Linear Insect Kinin Pentapeptide
Analogue, <B><I>Biopolymers</I></B>, <I>49</I>, 403-413 (1999)  [AIB charges]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. S. Ross
and C. C. Hardin, Ion-Induced Stabilization of the G-DNA Quadruplex: Free
Energy Perturbation Studies, <B><I>J. Am. Chem. Soc.</I></B>, <I>116</I>,
4363-4366 (1994)   [alkali metal ions]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. Aqvist,
Ion-Water Interaction Potentials Derived from Free Energy Perturbation
Simulations, <B><I>J. Phys. Chem.</I></B>, <I>94</I>, 8021-8024, 1990 
[alkaline earth Ions, radii adapted for AMBER combining rule]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Current
force field parameter values and suggested procedures for development of
parameters for additional molecules are available from the AMBER web site at
UCSF, http://www.amber.ucsf.edu/amber/amber.html/</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">CHARMM.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">CHARMM27
parameters for proteins. Most of the nucleic acid, lipid and small model
compound parameters are not yet implemented. We plan to provide the CHARMM27
nucleic acid parameters in due course as a separate parameter file.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">N. Foloppe
and A. D. MacKerell, Jr., All-Atom Empirical Force Field for Nucleic Acids: 1)
Parameter Optimization Based on Small Molecule and Condensed Phase
Macromolecular Target Data, <B><I>J. Comput. Chem.</I></B>, <I>21</I>, 86-104
(2000)  [CHARMM27]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">N. Banavali
and A. D. MacKerell, Jr., All-Atom Empirical Force Field for Nucleic Acids: 2)
Application to Molecular Dynamics Simulations of DNA and RNA in Solution, <B><I>J.
Comput. Chem.</I></B>, <I>21</I>, 105-120 (2000)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A. D.
MacKerrell, Jr., <I>et al.</I>, All-Atom Empirical Potential for Molecular
Modeling and Dynamics Studies of Proteins, <B><I>J. Phys. Chem. B</I></B>, <I>102</I>,
3586-3616 (1998)  [CHARMM22]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A. D.
MacKerell, Jr., J. Wiorkeiwicz-Kuczera and M. Karplus, An All-Atom Empirical
Energy Function for the Simulation of Nucleic Acids, <B><I>J. Am. Chem. Soc.</I></B>,
<I>117</I>, 11946-11975 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. E.
Feller, D. Yin, R. W. Pastor and A. D. MacKerell, Jr., Molecular Dynamics
Simulation of Unsaturated Lipids at Low Hydration: Parametrization and
Comparison with Diffraction Studies, <B><I>Biophysical Journal</I></B>, <I>73</I>,
2269-2279 (1997)  [alkenes]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">R. H. Stote
and M. Karplus, Zinc Binding in Proteins and Solution - A Simple but Accurate
Nonbonded Representation, <B><I>Proteins</I></B>, <I>23</I>, 12-31 (1995) 
[zinc ion]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Current and
legacy parameter values are available from the CHARMM force field web site on
Alex MacKerell's  Research Interests page at the University of Maryland School
of Pharmacy, https://rxsecure.umaryland.edu/research/amackere/research.html/</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">DUDEK.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Protein-only
parameters for the early 1990's TINKER force field with multipole values of
Dudek and Ponder. The current file contains only the multipole values from the
1995 paper by Dudek and Ponder. This set is now superceeded by the more recent
TINKER force field developed by Pengyu Ren (see WATER.PRM, below).</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. J. Dudek
and J. W. Ponder, Accurate Electrostatic Modelling of the Intramolecular
Energy of Proteins, <B><I>J. Comput. Chem.</I></B>, <I>16</I>, 791-816 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">EMR.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Reduced EMR
model adapted for flexible sidechains. Only a few amino acid residue types
have been implemented.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">R. V. Pappu,
W. J. Schneller and D. L. Weaver, Electrostatic Multipole Representation of a
Polypeptide Chain: An Algorithm for Simulation of Polypeptide Properties, <B><I>J.
Comput. Chem.</I></B>, <I>17</I>, 1033-1055 (1996)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">ENCAD.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ENCAD
parameters for proteins and nucleic acids.  (<I>in preparation</I>)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. Levitt,
M. Hirshberg, R. Sharon and V. Daggett, Potential Energy Function and
Parameters for Simulations of the Molecular Dynamics of Protein and Nucleic
Acids in Solution, <B><I>Comp. Phys. Commun.</I></B>, <I>91</I>, 215-231 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. Levitt,
M. Hirshberg, R. Sharon, K. E. Laidig and V. Daggett, Calibration and Testing
of a Water Model for Simulation of the Molecular Dynamics of Protein and
Nucleic Acids in Solution, <B><I>J. Phys. Chem. B</I></B>, <I>101</I>,
5051-5061 (1997)  [F3C water]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">HOCH.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Simple
NMR-NOE force field of Hoch and Stern.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. C. Hoch
and A. S. Stern, A Method for Determining Overall Protein Fold from NMR
Distance Restraints, <B><I>J. Biomol. NMR</I></B>, <I>2</I>, 535-543 (1992)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">MERCK.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Preliminary
MMFF vdw parameters using buffered 14/7 function.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. A.
Halgren, Representation of van der Waals (vdW) Interactions in Molecular
Mechanics Force Fields: Potential Form, Combination Rules, and vdW Parameters, <B><I>J.
Am. Chem. Soc.</I></B>, <I>114</I>, 7827-7843 (1992)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">MM2.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Full
MM2(1991) parameters including </FONT><FONT FACE="symbol">p</FONT><FONT FACE="New Century Schoolbook">-systems.
The anomeric and electronegativity correction terms included in some later
versions of MM2 are not implemented.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">N. L.
Allinger, Conformational Analysis. 130. MM2. A Hydrocarbon Force Field
Utilizing V1 and V2 Torsional Terms, <B><I>J. Am. Chem. Soc.</I></B>, <I>99</I>,
8127-8134 (1977)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. T.
Sprague, J. C. Tai, Y. Yuh and N. L. Allinger, The MMP2 Calculational Method, <B><I>J.
Comput. Chem.</I></B>, <I>8</I>, 581-603 (1987)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. C. Tai
and N. L. Allinger, Molecular Mechanics Calculations on Conjugated
Nitrogen-Containing Heterocycles,<B><I> J. Am. Chem. Soc.</I></B>, <I>110</I>,
2050-2055 (1988)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. C. Tai,
J.-H. Lii and N. L. Allinger, A Molecular Mechanics (MM2) Study of Furan,
Thiophene, and Related Compounds, <B><I>J. Comput. Chem.</I></B>, <I>10</I>,
635-647 (1989)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">N. L.
Allinger, R. A. Kok and M. R. Imam, Hydrogen Bonding in MM2, <B><I>J. Comput. Chem.</I></B>,
<I>9</I>, 591-595 (1988)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">L.
Norskov-Lauritsen and N. L. Allinger, A Molecular Mechanics Treatment of the
Anomeric Effect, <B><I>J. Comput. Chem.</I></B>, <I>5</I>, 326-335 (1984)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">All
parameters distributed with TINKER are from the ``MM2 (1991) Parameter Set'',
as provided by N. L. Allinger, University of Georgia</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">MM3.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Full
MM3(2000) parameters including pi-systems. The directional hydrogen bonding
term and electronegativity bond length corrections are implemented, but the
anomeric and Bohlmann correction terms are not implemented.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">N. L.
Allinger, Y. H. Yuh and J.-H. Lii, Molecular Mechanics. The MM3 Force Field
for Hydrocarbons. 1, <B><I>J. Am. Chem. Soc.</I></B>, <I>111</I>, 8551-8566 (1989)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J.-H. Lii
and N. L. Allinger, Molecular Mechanics. The MM3 Force Field for Hydrocarbons.
2. Vibrational Frequencies and Thermodynamics, <B><I>J. Am. Chem. Soc.</I></B>,
<I>111</I>, 8566-8575 (1989)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J.-H. Lii
and N. L. Allinger, Molecular Mechanics. The MM3 Force Field for Hydrocarbons.
3. The van der Waals' Potentials and Crystal Data for Aliphatic and Aromatic
Hydrocarbons, <B><I>J. Am. Chem. Soc.</I></B>, 111, 8576-8582 (1989)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">N. L.
Allinger, H. J. Geise, W. Pyckhout, L. A. Paquette and J. C. Gallucci,
Structures of Norbornane and Dodecahedrane by Molecular Mechanics Calculations
(MM3), X-ray Crystallography, and Electron Diffraction, <B><I>J. Am. Chem. Soc.</I></B>,
<I>111</I>, 1106-1114 (1989)  [stretch-torsion cross term]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">N. L.
Allinger, F. Li and L. Yan, Molecular Mechanics. The MM3 Force Field for
Alkenes, <B><I>J. Comput. Chem.</I></B>, <I>11</I>, 848-867 (1990)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">N. L.
Allinger, F. Li, L. Yan and J. C. Tai, Molecular Mechanics (MM3) Calculations
on Conjugated Hydrocarbons, <B><I>J. Comput. Chem.</I></B>, <I>11</I>, 868-895 (1990)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J.-H. Lii
and N. L. Allinger, Directional Hydrogen Bonding in the MM3 Force Field. I, <B><I>J.
Phys. Org. Chem.</I></B>, <I>7</I>, 591-609 (1994)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J.-H. Lii
and N. L. Allinger, Directional Hydrogen Bonding in the MM3 Force Field. II, <B><I>J.
Comput. Chem.</I></B>, <I>19</I>, 1001-1016 (1998)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">All
parameters distributed with TINKER are from the ``MM3 (2000) Parameter Set'',
as provided by N. L. Allinger, University of Georgia, August 2000</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">MM3PRO.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Protein-only
version of the MM3 parameters.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J.-H. Lii
and N. L. Allinger, The MM3 Force Field for Amides, Polypeptides and Proteins, <B><I>J.
Comput. Chem.</I></B>, <I>12</I>, 186-199 (1991)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">MMFFPRO.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Protein-only
version of the MMFF94 parameters.  (<I>in preparation</I>)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. A.
Halgren, Merck Molecular Force Field. I. Basis, Form, Scope, Parameterization,
and Performance of MMFF94, <B><I>J. Comput. Chem.</I></B>, <I>17</I>, 490-519, 1996</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">OPLS.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Complete
OPLS-UA with united-atom parameters for proteins and many classes of organic
molecules. Explicit hydrogens on polar atoms and aromatic carbons.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. L.
Jorgensen and J. Tirado-Rives, The OPLS Potential Functions for Proteins.
Energy Minimizations for Crystals of Cyclic Peptides and Crambin, <B><I>J. Am.
Chem. Soc.</I></B>, <I>110</I>, 1657-1666 (1988)  [peptide and proteins]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. L.
Jorgensen and D. L. Severance, Aromatic-Aromatic Interactions: Free Energy
Profiles for the Benzene Dimer in Water, Chloroform, and Liquid Benzene, <B><I>J.
Am. Chem. Soc.</I></B>, <I>112</I>, 4768-4774 (1990)  [aromatic hydrogens]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. J.
Weiner, P. A. Kollman, D. A. Case, U. C. Singh, C. Ghio, G. Alagona, S.
Profeta, Jr. and P. Weiner, A New Force Field for Molecular Mechanical
Simulation of Nucleic Acids and Proteins, <B><I>J. Am. Chem. Soc.</I></B>, <I>106</I>,
765-784 (1984)  [united-atom ``AMBER/OPLS'' local geometry]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. J.
Weiner, P. A. Kollman, D. T. Nguyen and D. A. Case, An All Atom Force Field
for Simulations of Proteins and Nucleic Acids, <B><I>J. Comput. Chem.</I></B>, <I>7</I>,
230-252 (1986)  [all-atom &quot;AMBER/OPLS&quot; local geometry]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">L. X. Dang
and B. M. Pettitt, Simple Intramolecular Model Potentials for Water, <B><I>J.
Phys. Chem.</I></B>, <I>91</I>, 3349-3354 (1987)  [flexible TIP3P and SPC water]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. L.
Jorgensen, J. D. Madura and C. J. Swenson, Optimized Intermolecular Potential
Functions for Liquid Hydrocarbons, <B><I>J. Am. Chem. Soc.</I></B>, <I>106</I>,
6638-6646 (1984)  [hydrocarbons]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. L.
Jorgensen, E. R. Laird, T. B. Nguyen and J. Tirado-Rives, Monte Carlo
Simulations of Pure Liquid Substituted Benzenes with OPLS Potential Functions, <B><I>J.
Comput. Chem.</I></B>, <I>14</I>, 206-215 (1993)  [substituted benzenes]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">E. M. Duffy,
P. J. Kowalczyk and W. L. Jorgensen, Do Denaturants Interact with Aromatic
Hydrocarbons in Water?, <B><I>J. Am. Chem. Soc.</I></B>, <I>115</I>, 9271-9275
(1993)  [benzene, naphthalene, urea, guanidinium, tetramethyl ammonium]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. L.
Jorgensen and C. J. Swenson, Optimized Intermolecular Potential Functions for
Amides and Peptides. Structure and Properties of Liquid Amides, <B><I>J. Am.
Chem. Soc.</I></B>, <I>106</I>, 765-784 (1984)  [amides]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. L.
Jorgensen, J. M. Briggs and M. L. Contreras, Relative Partition Coefficients
for Organic Solutes form Fluid Simulations, <B><I>J. Phys. Chem.</I></B>, <I>94</I>,
1683-1686 (1990)  [chloroform, pyridine, pyrazine, pyrimidine]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. M.
Briggs, T. B. Nguyen and W. L. Jorgensen, Monte Carlo Simulations of Liquid
Acetic Acid and Methyl Acetate with the OPLS Potential Functions, <B><I>J.
Phys. Chem.</I></B>, <I>95</I>, 3315-3322 (1991)  [acetic acid, methyl acetate]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">H. Liu, F.
Muller-Plathe and W. F. van Gunsteren, A Force Field for Liquid Dimethyl
Sulfoxide and Physical Properties of Liquid Dimethyl Sulfoxide Calculated
Using Molecular Dynamics Simulation, <B><I>J. Am. Chem. Soc.</I></B>, <I>117</I>,
4363-4366 (1995)  [dimethyl sulfoxide]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. Gao, X.
Xia and T. F. George, Importance of Bimolecular Interactions in Developing
Empirical Potential Functions for Liquid Ammonia, <B><I>J. Phys. Chem.</I></B>,
<I>97</I>, 9241-9246 (1993)  [ammonia]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. Aqvist,
Ion-Water Interaction Potentials Derived from Free Energy Perturbation
Simulations, <B><I>J. Phys. Chem.</I></B>, <I>94</I>, 8021-8024 (1990)  [metal ions]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. S. Ross
and C. C. Hardin, Ion-Induced Stabilization of the G-DNA Quadruplex: Free
Energy Perturbation Studies, <B><I>J. Am. Chem. Soc.</I></B>, <I>116</I>,
4363-4366 (1994)  [alkali metal ions]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J.
Chandrasekhar, D. C. Spellmeyer and W. L. Jorgensen, Energy Component Analysis
for Dilute Aqueous Solutions of Li+, Na+, F-, and Cl- Ions, <B><I>J. Am. Chem. Soc.</I></B>,
<I>106</I>, 903-910 (1984)  [halide ions]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Most
parameters distributed with TINKER are from ``OPLS and OPLS-AA Parameters for
Organic Molecules, Ions, and Nucleic Acids'' as provided by W. L. Jorgensen,
Yale University, October 1997</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">OPLSAA.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">OPLS-AA with
all-atom parameters for proteins and many general classes of organic molecules.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. L.
Jorgensen, D. S. Maxwell and J. Tirado-Rives, Development and Testing of the
OPLS All-Atom Force Field on Conformational Energetics and Properties of
Organic Liquids, <B><I>J. Am. Chem. Soc.</I></B>, <I>117</I>, 11225-11236 (1996)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. L.
Jorgensen and N. A. McDonald, Development of an All-Atom Force Field for
Heterocycles. Properties of Liquid Pyridine and Diazenes, <B><I>THEOCHEM-J.
Mol. Struct.</I></B>, <I>424</I>, 145-155 (1998)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">N. A.
McDonald and W. L. Jorgensen, Development of an All-Atom Force Field for
Heterocycles. Properties of Liquid Pyrrole, Furan, Diazoles, and Oxazoles, <B><I>J.
Phys. Chem. B</I></B>, <I>102</I>, 8049-8059 (1998)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">All
parameters distributed with TINKER are from ``OPLS and OPLS-AA Parameters for
Organic Molecules, Ions, and Nucleic Acids'' as provided by W. L. Jorgensen,
Yale University, October 1997</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SMOOTH.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Version of
OPLS-UA for use with potential smoothing. Largely adapted largely from
standard OPLS-UA parameters with modifications to the vdw and improper torsion terms.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">R. V. Pappu,
R. K. Hart and J. W. Ponder, Analysis and Application of Potential Energy
Smoothing and Search Methods for Global Optimization, <B><I>J. Phys, Chem. B</I></B>,
<I>102</I>, 9725-9742 (1998)  [smoothing modifications]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SMOOTHAA.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Version of
OPLS-AA for use with potential smoothing. Largely adapted largely from
standard OPLS-AA parameters with modifications to the vdw and improper torsion terms.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">R. V. Pappu,
R. K. Hart and J. W. Ponder, Analysis and Application of Potential Energy
Smoothing and Search Methods for Global Optimization, <B><I>J. Phys, Chem. B</I></B>,
<I>102</I>, 9725-9742 (1998)  [smoothing modifications]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">TINKER.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Preliminary
parameters for the TINKER polarizable multipole force field. As the release of
TINKER 3.9 we have completed parametrization for over 50 small molecule
systems, aimed toward the first version of a TINKER force field for proteins
and peptides. For further information, or if you are interested in testing a
beta parameter set, please contact the TINKER developers.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">WATER.PRM</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The current
TINKER water parameters for a polarizable multipole electrostatics model. This
model is equal or better to the best available water models for many bulk and
cluster properties.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Y. Kong and
J. W. Ponder, Calculation of the Reaction Field Due to Off-Center Point
Multipoles, <B><I>J. Chem. Phys.</I></B>, <I>107</I>, 481-492 (1997)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The
parameters distributed with TINKER are modified values based on the work of
Pengyu Ren starting from the Ph.D. thesis of Yong Kong, ``Multipole
Electrostatic Methods for Protein Modeling with Reaction Field Treatment'',
Biochemistry &amp; Molecular Biophysics, Washington University, St. Louis,
August, 1997</FONT></FONT></P>
<TABLE WIDTH=450 CELLPADDING=0>
<TR VALIGN=TOP>
<TD COLSPAN=1 WIDTH="12%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">7.</FONT></FONT></U></B></P>
</TD>
<TD COLSPAN=1 WIDTH="88%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">Use of the Keyword
Control File</FONT></FONT></U></B></P>
</TD></TR>
</TABLE>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;This
section contains a description of the keyword parameters which may be used to
define or alter the course of a TINKER calculation. The keyword control file
is optional in the sense that all of the TINKER programs will run in the
absence of a keyfile and will simply use default values or query the user for
needed information. However, the keywords allow use of a wide variety of
algorithmic and procedural options, many of which are unavailable interactively.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Keywords
are read from the keyword control file. All programs look first for a keyfile
with the same base name as the input molecular system and ending in the
extension </FONT><B><TT><FONT FACE="courier, courier new">.key</FONT></TT></B><FONT FACE="New Century Schoolbook">.
If this file does not exist, then TINKER tries to use a generic keyfile with
the name </FONT><B><TT><FONT FACE="courier, courier new">tinker.key</FONT></TT></B><FONT FACE="New Century Schoolbook">
and located in the same directory as the input system. If neither a system
specific nor a generic keyfile is present, TINKER will continue by using
default values for keyword options and asking interactive questions as necessary.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;TINKER
searches the keyfile during the course of a calculation for relevant keywords
that may be present. All keywords must appear as the first word on the line.
Any blank space to the left of the keyword is ignored, and all contents of the
keyfiles are case insensitive. Some keywords take modifiers; <I>i.e.</I>,
TINKER looks further on the same line for additional information, such as the
value of some parameter related to the keyword. Modifier information is read
in free format, but must be completely contained on the same line as the
original keyword. Any lines contained in the keyfile which do not qualify as
valid keyword lines are treated as comments and are simply ignored.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Several
keywords take a list of integer values (atom numbers, for example) as
modifiers. For these keywords the integers can simply be listed explicitly and
separated by spaces, commas or tabs. If a range of numbers is desired, it can
be specified by listing the negative of the first number of the range,
followed by a separator and the last number of the range. For example, the
keyword line </FONT><B><TT><FONT FACE="courier, courier new">ACTIVE 4 -9 17 23</FONT></TT></B><FONT FACE="New Century Schoolbook">
could be used to add atoms 4, 9 through 17, and 23 to the set of active atoms
during a TINKER calculation.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Listed
below are the valid TINKER keywords sorted into groups by general function.
The section ends with an alphabetical listing of the individual keywords along
with brief descriptions of their action and possible modifiers, and examples
of usage.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT FACE="New Century Schoolbook">Keywords Grouped by Functionality</FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">OUTPUT
CONTROL KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ARCHIVE&#9;DEBUG&#9;DIGITS</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ECHO&#9;EXIT-PAUSE&#9;NOVERSION&#9;OVERWRITE&#9;PRINTOUT&#9;SAVE-CYCLE</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">SAVE-INDUCED&#9;SAVE-VELOCITY&#9;VERBOSE&#9;WRITEOUT&#9;</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">FORCE
FIELD SELECTION KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">FORCEFIELD&#9;PARAMETERS</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">POTENTIAL
FUNCTION SELECTION KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ANGANGTERM&#9;ANGLETERM&#9;BONDTERM</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">CHARGETERM&#9;CHGDPLTERM&#9;DIPOLETERM</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">EXTRATERM&#9;IMPROPTERM&#9;IMPTORSTERM&#9;METALTERM&#9;MPOLETERM&#9;OPBENDTERM&#9;OPDISTTERM&#9;POLARIZETERM&#9;RESTRAINTERM&#9;RXNFIELDTERM&#9;SOLVATETERM&#9;STRBNDTERM&#9;STRTORTERM&#9;TORSIONTERM&#9;TORTORTERM&#9;UREYTERM&#9;VDWTERM</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;</FONT></FONT></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">POTENTIAL
FUNCTION PARAMETER KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ANGANG&#9;ANGLE&#9;ANGLE3</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ANGLE4&#9;ANGLE5&#9;ANGLEF</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ATOM&#9;BIOTYPE&#9;BOND</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">BOND3&#9;BOND4&#9;BOND5</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">CHARGE&#9;DIPOLE&#9;DIPOLE3</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">DIPOLE4&#9;DIPOLE5&#9;ELECTNEG</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">HBOND&#9;IMPROPER&#9;IMPTORS&#9;METAL&#9;MULTIPOLE&#9;OPBEND</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">OPDIST&#9;PIATOM&#9;PIBOND</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">POLARIZE&#9;SOLVATE&#9;STRBND</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">STRTORS&#9;TORSION&#9;TORSION4&#9;TORSION5&#9;UREYBRAD&#9;VDW</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">VDW14&#9;VDWPR</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">ENERGY
UNIT CONVERSION KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ANGLEUNIT&#9;ANGANGUNIT&#9;BONDUNIT</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">IMPROPUNIT&#9;IMPTORUNIT&#9;OPBENDUNIT&#9;OPDISTUNIT&#9;STRBNDUNIT&#9;STRTORUNIT&#9;TORSIONUNIT&#9;UREYUNIT</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">LOCAL
GEOMETRY FUNCTIONAL FORM KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ANGLE-CUBIC&#9;ANGLE-QUARTIC&#9;ANGLE-PENTIC</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ANGLE-SEXTIC&#9;BOND-CUBIC&#9;BOND-QUARTIC&#9;BONDTYPE&#9;MM2-STRBND&#9;PISYSTEM</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">UREY-CUBIC&#9;UREY-QUARTIC</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">VAN
DER WAALS FUNCTIONAL FORM KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A-EXPTERM&#9;B-EXPTERM&#9;C-EXPTERM</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">DELTA-HALGREN&#9;EPSILONRULE&#9;GAMMA-HALGREN&#9;GAUSSTYPE&#9;RADIUSRULE&#9;RADIUSSIZE&#9;RADIUSTYPE&#9;VDW-12-SCALE&#9;VDW-13-SCALE</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">VDW-14-SCALE&#9;VDW-15-SCALE&#9;VDWTYPE</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">ELECTROSTATICS
FUNCTIONAL FORM KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">CHG-12-SCALE&#9;CHG-13-SCALE&#9;CHG-14-SCALE</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">CHG-15-SCALE&#9;DIELECTRIC&#9;DIRECT-11-SCALE</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">DIRECT-12-SCALE&#9;DIRECT-13-SCALE&#9;DIRECT-14-SCALE</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">MUTUAL-11-SCALE&#9;MUTUAL-12-SCALE&#9;MUTUAL-13-SCALE</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">MUTUAL-14-SCALE&#9;POLAR-11-SCALE&#9;POLAR-12-SCALE</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">POLAR-13-SCALE&#9;POLAR-14-SCALE&#9;POLAR-DAMP&#9;POLAR-EPS&#9;POLAR-OLD&#9;POLAR-SOR</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">POLARIZATION&#9;REACTIONFIELD</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">NONBONDED
CUTOFF KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">CHG-CUTOFF&#9;CHG-TAPER&#9;CUTOFF</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">DPL-CUTOFF&#9;DPL-TAPER&#9;HESS-CUTOFF&#9;LIGHTS&#9;MPOLE-CUTOFF&#9;MPOLE-TAPER</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">NEIGHBOR-GROUPS&#9;NEUTRAL-GROUPS&#9;POLYMER-CUTOFF</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">TAPER&#9;TRUNCATE&#9;VDW-CUTOFF</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">VDW-TAPER</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">EWALD
SUMMATION KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">EWALD&#9;EWALD-ALPHA&#9;EWALD-BOUNDARY&#9;EWALD-CUTOFF&#9;EWALD-FRACTION&#9;PME-GRID</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">PME-ORDER</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">CRYSTAL
LATTICE &amp; PERIODIC BOUNDARY KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A-AXIS&#9;B-AXIS&#9;C-AXIS</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ALPHA&#9;BETA&#9;GAMMA</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">OCTAHEDRON&#9;SPACEGROUP</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">OPTIMIZATION
KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ANGMAX&#9;CAPPA&#9;FCTMIN</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">HGUESS&#9;INTMAX&#9;LBFGS-VECTORS&#9;MAXITER&#9;NEWHESS&#9;NEXTITER&#9;STEPMAX&#9;STPMIN</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">DYNAMICS
KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">COLLISION&#9;COMPRESS&#9;FRICTION&#9;FRICTION-SCALING&#9;INTEGRATE&#9;TAU-PRESSURE&#9;TAU-TEMPERATURE&#9;THERMOSTAT</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">TRANSITION
STATE KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">DIVERGE&#9;GAMMAMIN&#9;REDUCE</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">SADDLEPOINT</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">DISTANCE
GEOMETRY KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">TRIAL-DISTANCE&#9;TRIAL-DISTRIBUTION</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">RANDOM
NUMBER KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">RANDOMSEED</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">FREE
ENERGY PERTURBATION KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">LAMBDA&#9;MUTATE</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">PARTIAL
STRUCTURE KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ACTIVE&#9;GROUP&#9;GROUP-INTER</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">GROUP-INTRA&#9;GROUP-MOLECULE&#9;GROUP-SELECT&#9;INACTIVE</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">CONSTRAINT
&amp; RESTRAINT KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">BASIN&#9;RATTLE&#9;RATTLE-BOND&#9;RESTRAIN-DIHEDRAL&#9;RESTRAIN-DISTANCE&#9;RESTRAIN-POSITION</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">SPHERE&#9;WALL</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">POTENTIAL
SMOOTHING KEYWORDS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">DEFORM&#9;DIFFUSE-CHARGE&#9;DIFFUSE-TORSION</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">DIFFUSE-VDW</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT FACE="New Century Schoolbook">Description of
Individual Keywords</FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">The
following is an alphabetical list of the TINKER keywords along with a brief
description of the action of each keyword and required or optional parameters
that can be used to extend or modify each keyword. The form of possible
modifiers, if any, is shown in brackets following each keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>A-AXIS [real]</B>
    Sets the value of the a-axis length for a crystal unit cell, or,
equivalently,  the X-axis length for a periodic box. The length value in
Angstroms is listed after the keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>A-EXPTERM
[real]     </B>Sets the value of the ``A'' premultiplier term in the
Buckingham van der Waals function, <I>i.e.</I>, the value of A in the formula E</FONT></FONT><FONT FACE="New Century Schoolbook"><SUB><FONT SIZE=1>vdw</FONT></SUB><FONT SIZE=2>
= </FONT></FONT><FONT FACE="symbol">e</FONT><FONT FACE="New Century Schoolbook"><FONT SIZE=2>
{ A exp[-B(R</FONT><SUB><FONT SIZE=1>o</FONT></SUB><FONT SIZE=2>/R)] - C (R</FONT><SUB><FONT SIZE=1>o</FONT></SUB><FONT SIZE=2>/R)</FONT><SUP><FONT SIZE=1>6</FONT></SUP><FONT SIZE=2>
}.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ACTIVE
[integer list]</B>     Sets the list of active atoms during a TINKER
computation. Individual potential energy terms are computed when at least one
atom involved in the term is active. For Cartesian space calculations, active
atoms are those allowed to move. For torsional space calculations, rotations
are allowed when all atoms on one side of the rotated bond are active.
Multiple ACTIVE lines can be present in the keyfile and are treated
cumulatively.  On each line the keyword can be followed by one or more atom
numbers or atom ranges. The presence of any ACTIVE keyword overrides any
INACTIVE keywords in the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ALPHA [real]</B>
    Sets the value of the </FONT><FONT FACE="symbol">a</FONT><FONT FACE="New Century Schoolbook">
angle of a crystal unit cell, <I>i.e.</I>, the angle between the b-axis and
c-axis of a unit cell, or, equivalently, the angle between the Y-axis and
Z-axis of a periodic box. The default value in the absence of the ALPHA
keyword is 90 degrees.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ANGANG [1
integer &amp; 3 reals]</B>     This keyword provides the values for a single
angle-angle cross term potential parameter.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ANGANGTERM
[NONE/ONLY]</B>     This keyword controls use of the angle-angle cross term
potential energy. In the absence of a modifying option, this keyword turns on
use of the potential. The NONE option turns off use of this potential energy
term. The ONLY option turns off all potential energy terms except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT FACE="New Century Schoolbook"><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ANGANGUNIT
[real]</B>     Sets the scale factor needed to convert the energy value
computed by the angle-angle cross term potential into units of kcal/mole. The
correct value is force field dependent and typically provided in the header of
the master force field parameter file. The default of (</FONT><FONT FACE="symbol">p</FONT>/180)</FONT><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>
= 0.0003046 is used, if the ANGANGUNIT keyword is not given in the force field
parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ANGLE [3
integers &amp; 4 reals]</B>     This keyword provides the values for a single
bond angle bending parameter. The integer modifiers give the atom class
numbers for the three kinds of atoms involved in the angle which is to be
defined. The real number modifiers give the force constant value for the angle
and up to three ideal bond angles in degrees. In most cases only one ideal
bond angle is given, and that value is used for all occurrences of the
specified bond angle. If all three ideal angles are given, the values apply
when the central atom of the angle is attached to 0, 1 or 2 additional
hydrogen atoms, respectively. This ``hydrogen environment'' option is provided
to implement the corresponding feature of Allinger's MM force fields. The
default units for the force constant are kcal/mole/radian</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>,
but this can be controlled via the ANGLEUNIT keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ANGLE-CUBIC
[real]</B>     Sets the value of the cubic term in the Taylor series expansion
form of the bond angle bending potential energy. The real number modifier
gives the value of the coefficient as a multiple of the quadratic coefficient.
This term multiplied by the angle bending energy unit conversion factor, the
force constant, and the cube of the deviation of the bond angle from its ideal
value gives the cubic contribution to the angle bending energy. The default
value in the absence of the ANGLE-CUBIC keyword is zero; <I>i.e.</I>, the
cubic angle bending term is omitted.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ANGLE-PENTIC
[real]</B>     Sets the value of the fifth power term in the Taylor series
expansion form of the bond angle bending potential energy. The real number
modifier gives the value of the coefficient as a multiple of the quadratic
coefficient. This term multiplied by the angle bending energy unit conversion
factor, the force constant, and the fifth power of the deviation of the bond
angle from its ideal value gives the pentic contribution to the angle bending
energy. The default value in the absence of the ANGLE-PENTIC keyword is zero; <I>i.e.</I>,
the pentic angle bending term is omitted.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ANGLE-QUARTIC
[real]</B>     Sets the value of the quartic term in the Taylor series
expansion form of the bond angle bending potential energy. The real number
modifier gives the value of the coefficient as a multiple of the quadratic
coefficient. This term multiplied by the angle bending energy unit conversion
factor, the force constant, and the forth power of the deviation of the bond
angle from its ideal value gives the quartic contribution to the angle bending
energy. The default value in the absence of the ANGLE-QUARTIC keyword is zero; <I>i.e.</I>,
the quartic angle bending term is omitted.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ANGLE-SEXTIC
[real]</B>     Sets the value of the sixth power term in the Taylor series
expansion form of the bond angle bending potential energy. The real number
modifier gives the value of the coefficient as a multiple of the quadratic
coefficient. This term multiplied by the angle bending energy unit conversion
factor, the force constant, and the sixth power of the deviation of the bond
angle from its ideal value gives the sextic contribution to the angle bending
energy. The default value in the absence of the ANGLE-SEXTIC keyword is zero; <I>i.e.</I>,
the sextic angle bending term is omitted.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ANGLE3 [3
integers &amp; 4 reals]</B>     This keyword provides the values for a single
bond angle bending parameter specific to atoms in 3-membered rings. The
integer modifiers give the atom class numbers for the three kinds of atoms
involved in the angle which is to be defined. The real number modifiers give
the force constant value for the angle and up to three ideal bond angles in
degrees. If all three ideal angles are given, the values apply when the
central atom of the angle is attached to 0, 1 or 2 additional hydrogen atoms,
respectively. The default units for the force constant are kcal/mole/radian</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>,
but this can be controlled via the ANGLEUNIT keyword. If any ANGLE3 keywords
are present, either in the master force field parameter file or the keyfile,
then TINKER requires that special ANGLE3 parameters be given for all angles in
3-membered rings. In the absence of any ANGLE3 keywords, standard ANGLE
parameters will be used for bonds in 3-membered rings.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ANGLE4 [3
integers &amp; 4 reals]</B>     This keyword provides the values for a single
bond angle bending parameter specific to atoms in 4-membered rings. The
integer modifiers give the atom class numbers for the three kinds of atoms
involved in the angle which is to be defined. The real number modifiers give
the force constant value for the angle and up to three ideal bond angles in
degrees. If all three ideal angles are given, the values apply when the
central atom of the angle is attached to 0, 1 or 2 additional hydrogen atoms,
respectively. The default units for the force constant are kcal/mole/radian</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>,
but this can be controlled via the ANGLEUNIT keyword. If any ANGLE4 keywords
are present, either in the master force field parameter file or the keyfile,
then TINKER requires that special ANGLE4 parameters be given for all angles in
4-membered rings. In the absence of any ANGLE4 keywords, standard ANGLE
parameters will be used for bonds in 4-membered rings.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ANGLE5 [3
integers &amp; 4 reals]</B>     This keyword provides the values for a single
bond angle bending parameter specific to atoms in 5-membered rings. The
integer modifiers give the atom class numbers for the three kinds of atoms
involved in the angle which is to be defined. The real number modifiers give
the force constant value for the angle and up to three ideal bond angles in
degrees. If all three ideal angles are given, the values apply when the
central atom of the angle is attached to 0, 1 or 2 additional hydrogen atoms,
respectively. The default units for the force constant are kcal/mole/radian</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>,
but this can be controlled via the ANGLEUNIT keyword. If any ANGLE5 keywords
are present, either in the master force field parameter file or the keyfile,
then TINKER requires that special ANGLE5 parameters be given for all angles in
5-membered rings. In the absence of any ANGLE5 keywords, standard ANGLE
parameters will be used for bonds in 5-membered rings.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ANGLEF [3
integers &amp; 3 reals]</B>     This keyword provides the values for a single
bond angle bending parameter for a SHAPES-style Fourier potential function.
The integer modifiers give the atom class numbers for the three kinds of atoms
involved in the angle which is to be defined. The real number modifiers give
the force constant value for the angle, the angle shift in degrees, and the
periodicity value. Note that the force constant should be given as the
``harmonic'' value and not the native Fourier value. The default units for the
force constant are kcal/mole/radian</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>,
but this can be controlled via the ANGLEUNIT keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ANGLETERM [NONE/ONLY]</B>
    This keyword controls use of the bond angle bending potential energy term.
In the absence of a modifying option, this keyword turns on use of the
potential. The NONE option turns off use of this potential energy term. The
ONLY option turns off all potential energy terms except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT FACE="New Century Schoolbook"><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ANGLEUNIT
[real]</B>     Sets the scale factor needed to convert the energy value
computed by the bond angle bending potential into units of kcal/mole. The
correct value is force field dependent and typically provided in the header of
the master force field parameter file. The default value of (</FONT><FONT FACE="symbol">p</FONT>/180)</FONT><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>
= 0.0003046 is used, if the ANGLEUNIT keyword is not given in the force field
parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ANGMAX [real]</B>
    Set the maximum permissible angle between the current optimization search
direction and the negative of the gradient direction. If this maximum angle
value is exceeded, the optimization routine will note an error condition and
may restart from the steepest descent direction. The default value in the
absence of the ANGMAX keyword is usually 88 degrees for conjugate gradient
methods and 180 degrees (<I>i.e.</I>, disabled) for variable metric optimizations.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ARCHIVE</B>
    Causes TINKER molecular dynamics-based programs to write trajectories
directly to a single plain-text archive file with the </FONT><TT><FONT FACE="courier, courier new"><B>.arc</B>
</FONT></TT><FONT FACE="New Century Schoolbook">format. If an archive file
already exists at the start of the calculation, then the newly generated
trajectory is appended to the end of the existing file. The default in the
absence of this keyword is to write the trajectory snapshots to consecutively
numbered cycle files.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ATOM [2
integers, name, quoted string, integer, real &amp; integer]</B>     This
keyword provides the values needed to define a single force field atom type.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>B-AXIS [real]</B>
    Sets the value of the b-axis length for a crystal unit cell, or,
equivalently,  the Y-axis length for a periodic box. The length value in
Angstroms is listed after the keyword. If the keyword is absent, the b-axis
length is set equal to the a-axis length.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>B-EXPTERM
[real]     </B>Sets the value of the ``B'' exponential factor in the
Buckingham van der Waals function, <I>i.e.</I>, the value of B in the formula E</FONT></FONT><FONT FACE="New Century Schoolbook"><SUB><FONT SIZE=1>vdw</FONT></SUB><FONT SIZE=2>
= </FONT></FONT><FONT FACE="symbol">e</FONT><FONT FACE="New Century Schoolbook"><FONT SIZE=2>
{ A exp[-B(R</FONT><SUB><FONT SIZE=1>o</FONT></SUB><FONT SIZE=2>/R)] - C (R</FONT><SUB><FONT SIZE=1>o</FONT></SUB><FONT SIZE=2>/R)</FONT><SUP><FONT SIZE=1>6</FONT></SUP><FONT SIZE=2>
}.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>BASIN [2 reals]</B>
    Presence of this keyword turns on a ``basin'' restraint potential function
that serves to drive the system toward a compact structure. The actual
function is a Gaussian of the form E</FONT></FONT><SUB><FONT SIZE=1><FONT FACE="New Century Schoolbook">basin</FONT></FONT></SUB><FONT FACE="New Century Schoolbook"><FONT SIZE=2><FONT FACE="New Century Schoolbook">
= </FONT><FONT FACE="symbol">S</FONT> A exp[-B R</FONT><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>],
summed over all pairs of atoms where R is the distance between atoms. The A
and B values are the depth and width parameters given as modifiers to the
BASIN keyword. This potential is currently used to control the degree of
expansion during potential energy smooth procedures through the use of
shallow, broad basins.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>BETA [real]</B>
    Sets the value of the </FONT><FONT FACE="symbol">b</FONT><FONT FACE="New Century Schoolbook">
angle of a crystal unit cell, <I>i.e.</I>, the angle between the a-axis and
c-axis of a unit cell, or, equivalently, the angle between the X-axis and
Z-axis of a periodic box. The default value in the absence of the BETA keyword
is to set the </FONT><FONT FACE="symbol">b</FONT><FONT FACE="New Century Schoolbook">
angle equal to the </FONT><FONT FACE="symbol">a</FONT><FONT FACE="New Century Schoolbook">
angle as given by the keyword ALPHA.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>BIOTYPE
[integer, name, quoted string &amp; integer]</B>     This keyword provides the
values to define the correspondence between a single biopolymer atom type and
its force field atom type.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>BOND [2
integers &amp; 2 reals]</B>     This keyword provides the values for a single
bond stretching parameter. The integer modifiers give the atom class numbers
for the two kinds of atoms involved in the bond which is to be defined. The
real number modifiers give the force constant value for the bond and the ideal
bond length in &Aring;. The default units for the force constant are kcal/mole/&Aring;</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>,
but this can be controlled via the BONDUNIT keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>BOND-CUBIC
[real]</B>     Sets the value of the cubic term in the Taylor series expansion
form of the bond stretching potential energy. The real number modifier gives
the value of the coefficient as a multiple of the quadratic coefficient. This
term multiplied by the bond stretching energy unit conversion factor, the
force constant, and the cube of the deviation of the bond length from its
ideal value gives the cubic contribution to the bond stretching energy. The
default value in the absence of the BOND-CUBIC keyword is zero; <I>i.e.</I>,
the cubic bond stretching term is omitted.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>BOND-QUARTIC
[real]</B>     Sets the value of the quartic term in the Taylor series
expansion form of the bond stretching potential energy. The real number
modifier gives the value of the coefficient as a multiple of the quadratic
coefficient. This term multiplied by the bond stretching energy unit
conversion factor, the force constant, and the forth power of the deviation of
the bond length from its ideal value gives the quartic contribution to the
bond stretching energy. The default value in the absence of the BOND-QUARTIC
keyword is zero; <I>i.e.</I>, the quartic bond stretching term is omitted.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>BOND3 [2
integers &amp; 2 reals]</B>     This keyword provides the values for a single
bond stretching parameter specific to atoms in 3-membered rings. The integer
modifiers give the atom class numbers for the two kinds of atoms involved in
the bond which is to be defined. The real number modifiers give the force
constant value for the bond and the ideal bond length in &Aring;. The default
units for the force constant are kcal/mole/&Aring;</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>,
but this can be controlled via the BONDUNIT keyword. If any BOND3 keywords are
present, either in the master force field parameter file or the keyfile, then
TINKER requires that special BOND3 parameters be given for all bonds in
3-membered rings. In the absence of any BOND3 keywords, standard BOND
parameters will be used for bonds in 3-membered rings.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>BOND4 [2
integers &amp; 2 reals]</B>     This keyword provides the values for a single
bond stretching parameter specific to atoms in 4-membered rings. The integer
modifiers give the atom class numbers for the two kinds of atoms involved in
the bond which is to be defined. The real number modifiers give the force
constant value for the bond and the ideal bond length in &Aring;. The default
units for the force constant are kcal/mole/&Aring;</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>,
but this can be controlled via the BONDUNIT keyword. If any BOND4 keywords are
present, either in the master force field parameter file or the keyfile, then
TINKER requires that special BOND4 parameters be given for all bonds in
4-membered rings. In the absence of any BOND4 keywords, standard BOND
parameters will be used for bonds in 4-membered rings</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>BOND5 [2
integers &amp; 2 reals]</B>     This keyword provides the values for a single
bond stretching parameter specific to atoms in 5-membered rings. The integer
modifiers give the atom class numbers for the two kinds of atoms involved in
the bond which is to be defined. The real number modifiers give the force
constant value for the bond and the ideal bond length in &Aring;. The default
units for the force constant are kcal/mole/&Aring;</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>,
but this can be controlled via the BONDUNIT keyword. If any BOND5 keywords are
present, either in the master force field parameter file or the keyfile, then
TINKER requires that special BOND5 parameters be given for all bonds in
5-membered rings. In the absence of any BOND5 keywords, standard BOND
parameters will be used for bonds in 5-membered rings</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>BONDTERM [NONE/ONLY]</B>
    This keyword controls use of the bond stretching potential energy term. In
the absence of a modifying option, this keyword turns on use of the potential.
The NONE option turns off use of this potential energy term. The ONLY option
turns off all potential energy terms except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>BONDTYPE [TAYLOR/MORSE/GAUSSIAN]</B>
    Chooses the functional form of the bond stretching potential. The TAYLOR
option selects a Taylor series expansion containing terms from harmonic
through quartic. The MORSE option selects a Morse potential fit to the ideal
bond length and stretching force constant parameter values. The GAUSSIAN
option uses an inverted Gaussian with amplitude equal to the Morse bond
dissociation energy and width set to reproduce the vibrational frequency of a
harmonic potential. The default is to use the TAYLOR potential.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>BONDUNIT [real]</B>
    Sets the scale factor needed to convert the energy value computed by the
bond stretching potential into units of kcal/mole. The correct value is force
field dependent and typically provided in the header of the master force field
parameter file. The default value of 1.0 is used, if the BONDUNIT keyword is
not given in the force field parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>C-AXIS [real]</B>
    Sets the value of the C-axis length for a crystal unit cell, or,
equivalently,  the Z-axis length for a periodic box. The length value in
Angstroms is listed after the keyword. If the keyword is absent, the C-axis
length is set equal to the A-axis length.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>C-EXPTERM
[real]     </B>Sets the value of the ``C'' dispersion multiplier in the
Buckingham van der Waals function, <I>i.e.</I>, the value of C in the formula E</FONT></FONT><FONT FACE="New Century Schoolbook"><SUB><FONT SIZE=1>vdw</FONT></SUB><FONT SIZE=2>
= </FONT></FONT><FONT FACE="symbol">e</FONT><FONT FACE="New Century Schoolbook"><FONT SIZE=2>
{ A exp[-B(R</FONT><SUB><FONT SIZE=1>o</FONT></SUB><FONT SIZE=2>/R)] - C (R</FONT><SUB><FONT SIZE=1>o</FONT></SUB><FONT SIZE=2>/R)</FONT><SUP><FONT SIZE=1>6</FONT></SUP><FONT SIZE=2>
}.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>CAPPA [real]</B>
    This keyword is used to set the normal termination criterion for the line
search phase of TINKER optimization routines. The line search exits
successfully if the ratio of the current gradient projection on the line to
the projection at the start of the line search falls below the value of CAPPA.
A default value of 0.1 is used in the absence of the CAPPA keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>CHARGE [1
integer &amp; 1 real]</B>     This keyword provides a value for a single
atomic partial charge electrostatic parameter. The integer modifier, if
positive, gives the atom type number for which the charge parameter is to be
defined. Note that vdw parameters are given for atom types, not atom classes.
If the integer modifier is negative, then the parameter value to follow
applies only to the individual atom whose atom number is the negative of the
modifier. The real number modifier gives the values of the atomic partial
charge in electrons.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>CHARGETERM
[NONE/ONLY]</B>     This keyword controls use of the charge-charge potential
energy term between pairs of atomic partial charges. In the absence of a
modifying option, this keyword turns on use of the potential. The NONE option
turns off use of this potential energy term. The ONLY option turns off all
potential energy terms except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>CHG-12-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to charge-charge electrostatic interactions between 1-2 connected atoms,
<I>i.e.</I>, atoms that are directly bonded. The default value of 0.0 is used,
if the CHG-12-SCALE keyword is not given in either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>CHG-13-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to charge-charge electrostatic interactions between 1-3 connected atoms,
<I>i.e.</I>, atoms separated by two covalent bonds. The default value of 0.0
is used, if the CHG-13-SCALE keyword is not given in either the parameter file
or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>CHG-14-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to charge-charge electrostatic interactions between 1-4 connected atoms,
<I>i.e.</I>, atoms separated by three covalent bonds. The default value of 1.0
is used, if the CHG-14-SCALE keyword is not given in either the parameter file
or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>CHG-15-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to charge-charge electrostatic interactions between 1-5 connected atoms,
<I>i.e.</I>, atoms separated by four covalent bonds. The default value of 1.0
is used, if the CHG-15-SCALE keyword is not given in either the parameter file
or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>CHG-CUTOFF
[real]</B>     Sets the cutoff distance value in Angstroms for charge-charge
electrostatic potential energy interactions. The energy for any pair of sites
beyond the cutoff distance will be set to zero. Other keywords can be used to
select a smoothing scheme near the cutoff distance. The default cutoff
distance in the absence of the CHG-CUTOFF keyword is infinite for nonperiodic
systems and 9.0 for periodic systems.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>CHG-TAPER [real]</B>
    This keyword allows modification of the cutoff window for charge-charge
electrostatic potential energy interactions. It is similar in form and action
to the TAPER keyword, except that its value applies only to the charge-charge
potential. The default value in the absence of the CHG-TAPER keyword is to
begin the cutoff window at 0.65 of the corresponding cutoff distance.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>CHGDPLTERM
[NONE/ONLY]</B>     This keyword controls use of the charge-dipole potential
energy term between atomic partial charges and bond dipoles. In the absence of
a modifying option, this keyword turns on use of the potential. The NONE
option turns off use of this potential energy term. The ONLY option turns off
all potential energy terms except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>COLLISION [real]</B>
    Sets the value of the random collision frequency used in the Andersen
stochastic collision dynamics thermostat. The supplied value has units of fs</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>-1</FONT></SUP><FONT SIZE=2>
atom</FONT><SUP><FONT SIZE=1>-1</FONT></SUP><FONT SIZE=2> and is multiplied
internal to TINKER by the time step in fs and N</FONT><SUP><FONT SIZE=1>-2/3</FONT></SUP><FONT SIZE=2>
where N is the number of atoms. The default value used in the absence of the
COLLISION keyword is 0.1 which is appropriate for many systems but may need
adjustment to achieve adequate temperature control without perturbing the dynamics.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>COMPRESS [real]</B>
    Sets the value of the bulk solvent isothermal compressibility in Atm</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>-1</FONT></SUP><FONT SIZE=2>
for use during pressure computation and scaling in molecular dynamics
computations. The default value used in the absence of the COMPRESS keyword is
0.000046, appropriate for water. This parameter serves as a scale factor for
the Groningen-style pressure bath coupling time, and its exact value should
not be of critical importance.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>CUTOFF [real]</B>
    Sets the cutoff distance value for all nonbonded potential energy
interactions. The energy for any of the nonbonded potentials of a pair of
sites beyond the cutoff distance will be set to zero. Other keywords can be
used to select a smoothing scheme near the cutoff distance, or to apply
different cutoff distances to various nonbonded energy terms.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DEBUG</B>
    Turns on printing of detailed information and intermediate values
throughout the progress of a TINKER computation; not recommended for use with
large structures or full potential energy functions since a summary of every
individual interaction will usually be output.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DEFORM [real]</B>
    Sets the amount of diffusion equation-style smoothing that will be applied
to the potential energy surface when using the SMOOTH force field. The real
number option is equivalent to the ``time'' value in the original Piela, <I>et al.</I>
formalism; the larger the value, the greater the smoothing. The default value
is zero, meaning that no smoothing will be applied.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DELTA-HALGREN
[real]</B>     Sets the value of the </FONT><FONT FACE="symbol">d</FONT><FONT FACE="New Century Schoolbook">
parameter in Halgren's buffered 14-7 vdw potential energy functional form. In
the absence of the DELTA-HALGREN keyword, a default value of 0.07 is used.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DIELECTRIC
[real]</B>     Sets the value of the bulk dielectric constant used to damp all
electrostatic interaction energies for any of the TINKER electrostatic
potential functions. The default value is force field dependent, but is
usually equal to 1.0 (for Allinger's MM force fields the default is 1.5).</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"> </FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DIFFUSE-CHARGE
[real]</B>     This keyword is used during potential function smoothing
procedures to specify the effective diffusion coefficient to be applied to the
smoothed form of the Coulomb's Law charge-charge potential function. In the
absence of the DIFFUSE-CHARGE keyword, a default value of 3.5 is used.</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"> </FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DIFFUSE-TORSION
[real]</B>     This keyword is used during potential function smoothing
procedures to specify the effective diffusion coefficient to be applied to the
smoothed form of the torsion angle potential function. In the absence of the
DIFFUSE-TORSION keyword, a default value of 0.0225 is used.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DIFFUSE-VDW
[real]</B>     This keyword is used during potential function smoothing
procedures to specify the effective diffusion coefficient to be applied to the
smoothed Gaussian approximation to the Lennard-Jones van der Waals potential
function. In the absence of the DIFFUSE-VDW keyword, a default value of 1.0 is used.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DIGITS [integer]</B>
    This keyword controls the number of digits of precision  output by TINKER
in reporting potential energies and atomic coordinates. The allowed values for
the integer modifier are 4, 6 and 8. Input values less than 4 will be set to
4, and those greater than 8 will be set to 8. Final energy values reported by
most TINKER programs will contain the specified number of digits to the right
of the decimal point. The number of decimal places to be output for atomic
coordinates is generally two larger than the value of DIGITS. In the absence
of the DIGITS keyword a default value of 4 is used, and  energies will be
reported to 4 decimal places with coordinates to 6 decimal places.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DIPOLE [2
integers &amp; 2 reals]</B>     This keyword provides the values for a single
bond dipole electrostatic parameter. The integer modifiers give the atom type
numbers for the two kinds of atoms involved in the bond dipole which is to be
defined. The real number modifiers give the value of the bond dipole in Debyes
and the position of the dipole site along the bond. If the bond dipole value
is positive, then the first of the two atom types is the positive end of the
dipole. For a negative bond dipole value, the first atom type listed is
negative. The position along the bond is an optional modifier that gives the
postion of the dipole site as a fraction between the first atom type
(position=0) and the second atom type (position=1). The default for the dipole
position in the absence of a specified value is 0.5, placing the dipole at the
midpoint of the bond.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DIPOLE3
[2 integers &amp; 2 reals]</B>     This keyword provides the values for a
single bond dipole electrostatic parameter specific to atoms in 3-membered
rings. The integer modifiers give the atom type numbers for the two kinds of
atoms involved in the bond dipole which is to be defined. The real number
modifiers give the value of the bond dipole in Debyes and the position of the
dipole site along the bond. The default for the dipole position in the absence
of a specified value is 0.5, placing the dipole at the midpoint of the bond.
If any DIPOLE3 keywords are present, either in the master force field
parameter file or the keyfile, then TINKER requires that special DIPOLE3
parameters be given for all bond dipoles in 3-membered rings. In the absence
of any DIPOLE3 keywords, standard DIPOLE parameters will be used for bonds in
3-membered rings.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DIPOLE4
[2 integers &amp; 2 reals]</B>     This keyword provides the values for a
single bond dipole electrostatic parameter specific to atoms in 4-membered
rings. The integer modifiers give the atom type numbers for the two kinds of
atoms involved in the bond dipole which is to be defined. The real number
modifiers give the value of the bond dipole in Debyes and the position of the
dipole site along the bond. The default for the dipole position in the absence
of a specified value is 0.5, placing the dipole at the midpoint of the bond.
If any DIPOLE4 keywords are present, either in the master force field
parameter file or the keyfile, then TINKER requires that special DIPOLE4
parameters be given for all bond dipoles in 4-membered rings. In the absence
of any DIPOLE4 keywords, standard DIPOLE parameters will be used for bonds in
4-membered rings.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DIPOLE5
[2 integers &amp; 2 reals]</B>     This keyword provides the values for a
single bond dipole electrostatic parameter specific to atoms in 5-membered
rings. The integer modifiers give the atom type numbers for the two kinds of
atoms involved in the bond dipole which is to be defined. The real number
modifiers give the value of the bond dipole in Debyes and the position of the
dipole site along the bond. The default for the dipole position in the absence
of a specified value is 0.5, placing the dipole at the midpoint of the bond.
If any DIPOLE5 keywords are present, either in the master force field
parameter file or the keyfile, then TINKER requires that special DIPOLE5
parameters be given for all bond dipoles in 5-membered rings. In the absence
of any DIPOLE5 keywords, standard DIPOLE parameters will be used for bonds in
5-membered rings.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DIPOLETERM
[NONE/ONLY]</B>     This keyword controls use of the dipole-dipole potential
energy term between pairs of bond dipoles. In the absence of a modifying
option, this keyword turns on use of the potential. The NONE option turns off
use of this potential energy term. The ONLY option turns off all potential
energy terms except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DIRECT-11-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to the permanent (direct) field due to atoms within a polarization
group during an induced dipole calculation, <I>i.e.</I>, atoms that are in the
same polarization group as the atom being polarized. The default value of 0.0
is used, if the DIRECT-11-SCALE keyword is not given in either the parameter
file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DIRECT-12-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to the permanent (direct) field due to atoms in 1-2 polarization
groups during an induced dipole calculation, <I>i.e.</I>, atoms that are in
polarization groups directly connected to the group containing the atom being polarized.
The default value of 0.0 is used, if the DIRECT-12-SCALE keyword is not given
in either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DIRECT-13-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to the permanent (direct) field due to atoms in 1-3 polarization
groups during an induced dipole calculation, <I>i.e.</I>, atoms that are in
polarization groups separated by one group from the group containing the atom
being polarized. The default value of 0.0 is used, if the DIRECT-13-SCALE
keyword is not given in either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DIRECT-14-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to the permanent (direct) field due to atoms in 1-4 polarization
groups during an induced dipole calculation, <I>i.e.</I>, atoms that are in
polarization groups separated by two groups from the group containing the atom
being polarized. The default value of 1.0 is used, if the DIRECT-14-SCALE
keyword is not given in either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DIVERGE [real]</B>
    This keyword is used by the SADDLE program to set the maximum allowed
value of the ratio of the gradient length along the path to the total gradient
norm at the end of a cycle of minimization perpendicular to the path. If the
value provided by the DIVERGE keyword is exceeded, then another cycle of
maximization along the path is required. A default value of 0.005 is used in
the absence of the DIVERGE keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DPL-CUTOFF
[real]</B>     Sets the cutoff distance value in Angstroms for bond
dipole-bond dipole electrostatic potential energy interactions. The energy for
any pair of bond dipole sites beyond the cutoff distance will be set to zero.
Other keywords can be used to select a smoothing scheme near the cutoff
distance. The default cutoff distance in the absence of the DPL-CUTOFF keyword
is essentially infinite for nonperiodic systems and 10.0 for periodic systems.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>DPL-TAPER [real]</B>
    This keyword allows modification of the cutoff windows for bond
dipole-bond dipole electrostatic potential energy interactions. It is similar
in form and action to the TAPER keyword, except that its value applies only to
the vdw potential. The default value in the absence of the DPL-TAPER keyword
is to begin the cutoff window at 0.75 of the dipole cutoff distance.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ECHO
[text string]</B>     The presence of this keyword causes whatever text
follows it on the line to be copied directly to the output file. This keyword
is also active in parameter files. It has no default value; if no text follows
the ECHO keyword, a blank line is placed in the output file.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ELECTNEG
[3 integers &amp; 1 real]</B>     This keyword provides the values for a
single electronegativity bond length correction parameter. The first two
integer modifiers give the atom class numbers of the atoms involved in the
bond to be corrected. The third integer modifier is the atom class of an
electronegative atom. In the case of a primary correction, an atom of this
third class must be directly bonded to an atom of the second atom class. For a
secondary correction, the third class is one atom removed from an atom of the
second class. The real number modifier is the value in &Aring; by which the
original ideal bond length is to be corrected.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>EPSILONRULE
[GEOMETRIC/ARITHMETIC/HARMONIC/HHG]</B>     This keyword selects the combining
rule used to derive the </FONT><FONT FACE="symbol">e</FONT><FONT FACE="New Century Schoolbook">
value for van der Waals interactions. The default in the absence of the
EPSILONRULE keyword is to use the GEOMETRIC mean of the individual </FONT><FONT FACE="symbol">e</FONT><FONT FACE="New Century Schoolbook">
values of the two atoms involved in the van der Waals interaction.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>EWALD</B>
    This keyword turns on the use of Ewald summation during computation of
electrostatic interactions in periodic systems. In the current version of
TINKER, regular Ewald is used for polarizable atomic multipoles, and smooth
particle mesh Ewald (PME) is used for charge-charge interactions. Ewald
summation is not available for interactions involving bond-centered dipoles.
By default, in the absence of the EWALD keyword, distance-based cutoffs are
used for electrostatic interactions.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>EWALD-ALPHA
[real]</B>     Sets the value of the Ewald coefficient which controls the
width of the Gaussian screening charges during particle mesh Ewald summation.
In the absence of the EWALD-ALPHA keyword, a value is chosen which causes
interactions outside the real-space cutoff to be below a fixed tolerance. For
most standard applications of Ewald summation, the program default should be used.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>EWALD-BOUNDARY</B>
    This keyword invokes the use of ``vacuum'' boundary conditions during
Ewald summation, corresponding to the media surrounding the system having a
dielectric value of 1. The default in the absence of the EWALD-BOUNDARY
keyword is to use ``tinfoil'' boundary conditions where the surrounding media
is assumed to have an infinite dielectric value.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>EWALD-CUTOFF
[real]</B>     Sets the value in Angstroms of the real-space distance cutoff
for use during Ewald summation. By default, in the absence of the EWALD-CUTOFF
keyword, a value of 9.0 is used.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>EWALD-FRACTION
[real]</B>    Sets the fraction between 0 and 1 of reciprocal space included
in the reciprocal sum when using regular Ewald summation. The keyword has no
effect on PME calculations. A default value of 0.5 is used in the absence of
the EWALD-FRACTION keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>EXIT-PAUSE</B>
    This keyword causes TINKER programs to pause and wait for a carriage
return at the end of executation prior to returning control to the operating
system. This is useful to keep the execution window open following termination
on machines running Microsoft Windows or Apple MacOS. The default in the
absence of the EXIT-PAUSE keyword, is to return control to the operating
system immediately at program termination.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>EXTRATERM [NONE/ONLY]</B>
    This keyword controls use of the user defined extra potential energy term.
In the absence of a modifying option, this keyword turns on use of the
potential. The NONE option turns off use of this potential energy term. The
ONLY option turns off all potential energy terms except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>FCTMIN [real]</B>
    This keyword sets a convergence criterion for successful completion of a
TINKER optimization. If the value of the optimization objective function,
typically the potential energy, falls below the value set by FCTMIN, then the
optimization is deemed to have converged. The default value in the absence of
the FCTMIN keyword is -1000000, effectively removing this criterion as a
possible agent for termination.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>FORCEFIELD
[name]</B>     This keyword provides a name for the force field to be used in
the current calculation. Its value is usually set in the master force field
parameter file for the calculation (see the PARAMETERS keyword) instead of in
the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>FRICTION [real]</B>
    Sets the value of the frictional coefficient in ps</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>-1</FONT></SUP><FONT SIZE=2>
for use with stochastic dynamics. The default value used in the absence of the
FRICTION keyword is 91.0, which is generally appropriate for water.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>GAMMA [real]</B>
    Sets the value of the </FONT><FONT FACE="symbol">g</FONT><FONT FACE="New Century Schoolbook">
angle of a crystal unit cell, <I>i.e.</I>, the angle between the a-axis and
b-axis of a unit cell, or, equivalently, the angle between the X-axis and
Z-axis of a periodic box. The default value in the absence of the GAMMA
keyword is to set the </FONT><FONT FACE="symbol">g</FONT><FONT FACE="New Century Schoolbook">
angle equal to the </FONT><FONT FACE="symbol">a</FONT><FONT FACE="New Century Schoolbook">
angle as given by the keyword ALPHA.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>GAMMA-HALGREN
[real]</B>     Sets the value of the </FONT><FONT FACE="symbol">g</FONT><FONT FACE="New Century Schoolbook">
parameter in Halgren's buffered 14-7 vdw potential energy functional form. In
the absence of the DELTA-HALGREN keyword, a default value of 0.12 is used.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>GAMMAMIN [real]</B>
    Sets the convergence target value for </FONT><FONT FACE="symbol">g</FONT><FONT FACE="New Century Schoolbook">
during searches for maxima along the quadratic synchronous transit used by the
SADDLE program. The value of </FONT><FONT FACE="symbol">g</FONT><FONT FACE="New Century Schoolbook">
is the square of the ratio of the gradient projection along the path to the
total gradient. A default value of 0.00001 is used in the absence of the
GAMMAMIN keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"> </FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>GAUSSTYPE [LJ-2/LJ-4/MM2-2/MM3-2/IN-PLACE]</B>
    This keyword specifies the underlying vdw form that a Gaussian vdw
approximation will attempt to fit.number of terms to be used in a Gaussian
approximation of the Lennard-Jones van der Waals potential. The text modifier
gives the name of the functional form to be used. Thus LJ-2 as a modifier will
result in a 2-Gaussian fit to a Lennard-Jones vdw potential. The GAUSSTYPE
keyword only takes effect when VDWTYPE is set to GAUSSIAN. This keyword has no
default value.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>GROUP
[integer, integer list]</B>     This keyword defines an atom group as a
substructure within the full input molecular structure. The value of the first
integer is the group number which must be in the range from 1 to the maximum
number of allowed groups. The remaining intergers give the atom or atoms
contained in this group as one or more atom numbers or ranges. Multiple
keyword lines can be used to specify additional atoms in the same group. Note
that an atom can only be in one group, the last group to which it is assigned
is the one used.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>GROUP-INTER</B>
    This keyword assigns a value of 1.0 to all inter-group interactions and a
value of 0.0 to all intra-group interactions. For example, combination with
the GROUP-MOLECULE keyword provides for rigid-body calculations.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>GROUP-INTRA</B>
    This keyword assigns a value of 1.0 to all intra-group interactions and a
value of 0.0 to all inter-group interactions.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>GROUP-MOLECULE</B>
    This keyword sets each individual molecule in the system to be a separate
atom group, but does not assign weights to group-group interactions.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>GROUP-SELECT
[2 integers, real]</B>     This keyword gives the weight in the final
potential energy of a specified set of intra- or intergroup interactions. The
integer modifiers give the group numbers of the groups involved. If the two
numbers are the same, then an intragroup set of interactions is specified. The
real modifier gives the weight by which all energetic interactions in this set
will be multiplied before incorporation into the final potential energy. If
omitted as a keyword modifier, the weight will be set to 1.0 by default. If
any SELECT-GROUP keywords are present, then any set of interactions not
specified in a SELECT-GROUP keyword is given a zero weight. The default when
no SELECT-GROUP keywords are specified is to use all intergroup interactions
with a weight of 1.0 and to set all intragroup interactions to zero.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>HBOND [2
integers &amp; 2 reals]</B>     This keyword provides the values for the
MM3-style directional hydrogen bonding parameters for a single pair of atoms.
The integer modifiers give the pair of atom class numbers for which hydrogen
bonding parameters are to be defined. The two real number modifiers give the
values of the minimum energy contact distance in &Aring; and the well depth at
the minimum distance in kcal/mole.     </FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>HESS-CUTOFF
[real]</B>     This keyword defines a lower limit for significant Hessian
matrix elements. During computation of the Hessian matrix of partial second
derivatives, any matrix elements with absolute value below HESS-CUTOFF will be
set to zero and omitted from the sparse matrix Hessian storage scheme used by
TINKER. For most calculations, the default in the absence of this keyword is
zero, i.e., all elements will be stored. For most Truncated Newton
optimizations the Hessian cutoff will be chosen dynamically by the optimizer.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>HGUESS [real]</B>
    Sets an initial guess for the average value of the diagonal elements of
the scaled inverse Hessian matrix used by the optimally conditioned variable
metric optimization routine. A default value of 0.4 is used in the absence of
the HGUESS keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>IMPROPER
[4 integers &amp; 2 reals]</B>     This keyword provides the values for a
single CHARMM-style improper dihedral angle parameter.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>IMPROPTERM
[NONE/ONLY]</B>     This keyword controls use of the CHARMM-style improper
dihedral angle potential energy term. In the absence of a modifying option,
this keyword turns on use of the potential. The NONE option turns off use of
this potential energy term. The ONLY option turns off all potential energy
terms except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>IMPROPUNIT
[real]</B>     Sets the scale factor needed to convert the energy value
computed by the CHARMM-style improper dihedral angle potential into units of
kcal/mole. The correct value is force field dependent and typically provided
in the header of the master force field parameter file. The default value of
1.0 is used, if the IMPROPUNIT keyword is not given in the force field
parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>IMPTORS
[4 integers &amp; up to 3 real/real/integer triples]</B>     This keyword
provides the values for a single AMBER-style improper torsional angle
parameter. The first four integer modifiers give the atom class numbers for
the atoms involved in the improper torsional angle to be defined. By
convention, the third atom class of the four is the trigonal atom on which the
improper torsion is centered. The torsional angle computed is literally that
defined by the four atom classes in the order specified by the keyword. Each
of the remaining triples of real/real/integer modifiers give the
half-amplitude, phase offset in degrees and periodicity of a particular
improper torsional term, respectively. Periodicities through 3-fold are
allowed for improper torsional parameters.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>IMPTORSTERM
[NONE/ONLY]</B>     This keyword controls use of the AMBER-style improper
torsional angle potential energy term. In the absence of a modifying option,
this keyword turns on use of the potential. The NONE option turns off use of
this potential energy term. The ONLY option turns off all potential energy
terms except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>IMPTORSUNIT
[real]</B>     Sets the scale factor needed to convert the energy value
computed by the AMBER-style improper torsional angle potential into units of
kcal/mole. The correct value is force field dependent and typically provided
in the header of the master force field parameter file. The default value of
1.0 is used, if the IMPTORSUNIT keyword is not given in the force field
parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>INACTIVE
[integer list]</B>     Sets the list of inactive atoms during a TINKER
computation. Individual potential energy terms are not computed when all atoms
involved in the term are inactive. For Cartesian space calculations, inactive
atoms are not allowed to move. For torsional space calculations, rotations are
not allowed when there are inactive atoms on both sides of the rotated bond.
Multiple INACTIVE lines can be present in the keyfile, and on each line the
keyword can be followed by one or more atom numbers or ranges. If any INACTIVE
keys are found, all atoms are set to active except those listed on the
INACTIVE lines. The ACTIVE keyword overrides all INACTIVE keywords found in
the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>INTEGRATE [VERLET/BEEMAN/STOCHASTIC/RIGIDBODY]</B>
    Chooses the integration method for propagation of dynamics trajectories.
The keyword is followed on the same line by the name of the option. Standard
Newtonian MD can be run using either VERLET for the Velocity Verlet method, or
BEEMAN for the velocity form of Bernie Brook's ``Better Beeman'' method. A
Velocity Verlet-based stochastic dynamics trajectory is selected by the
STOCHASTIC modifier. A rigid-body dynamics method is selected by the RIGIDBODY
modifier. The default integration scheme is MD using the BEEMAN method.<I> </I><B><I>Note
the RIGIDBODY option is still under development and should be treated as
experimental code in the current version of TINKER.</I></B></FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>INTMAX [integer]</B>
    Sets the maximum number of interpolation cycles that will be allowed
during the line search phase of an optimization. All gradient-based TINKER
optimization routines use a common line search routine involving quadratic
extrapolation and cubic interpolation. If the value of INTMAX is reached, an
error status is set for the line search and the search is repeated with a much
smaller initial step size. The default value in the absence of this keyword is
optimization routine dependent, but is usually in the range 5 to 10.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>LAMBDA [real]</B>
    This keyword sets the value of the </FONT><FONT FACE="symbol">l</FONT><FONT FACE="New Century Schoolbook">
path parameter for free energy perturbation calculations. The real number
modifier specifies the position along the mutation path and must be a number
in the range from 0 (initial state) to 1 (final state). The actual atoms
involved in the mutation are given separately in individual MUTATE keyword lines.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>LBFGS-VECTORS
[integer]</B>     Sets the number of correction vectors used by the
limited-memory L-BFGS optimization routine. The current maximum allowable
value, and the default in the absence of the LBFGS-VECTORS keyword is 15.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>LIGHTS</B>
    This keyword turns on Method of Lights neighbor generation for the
charge-charge potential and any of the van der Waals potentials. This method
will yield identical energetic results to the standard double loop method.
Method of Lights will be faster when the volume of a sphere with radius equal
to the nonbond cutoff distance is significantly less than half the volume of
the total system (<I>i.e.</I>, the full molecular system, the crystal unit
cell or the periodic box).</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MAXITER [integer]</B>
    Sets the maximum number of minimization iterations that will be allowed
for any TINKER program that uses any of the nonlinear optimization routines.
The default value in the absence of this keyword is program dependent, but is
always set to a very large number.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>METAL</B>
    This keyword provides the values for a single transition metal ligand
field parameter. <B><I>Note this keyword is present in the code, but not
active in the current version of TINKER.</I></B></FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>METALTERM [NONE/ONLY]</B>
    This keyword controls use of the transition metal ligand field potential
energy term. In the absence of a modifying option, this keyword turns on use
of the potential. The NONE option turns off use of this potential energy term.
The ONLY option turns off all potential energy terms except for this one. <B><I>Note
this keyword is present in the code, but not active in the current version of TINKER.</I></B></FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MM2-STRBND</B>
    This keyword switches the behavior of the stretch-bend potential function
to match the formulation used by the MM2 force field. In MM2, stretching of
bonds to attached hydrogen atoms is not including in computing the
stretch-bend cross term energy. The default behavior in the absence of this
keyword is to include stretching of attached hydrogen atoms as in the MM3
force field.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MPOLE-12-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to permanent atomic multipole electrostatic interactions between 1-2
connected atoms, <I>i.e.</I>, atoms that are directly bonded. The default
value of 0.0 is used, if the MPOLE-12-SCALE keyword is not given in either the
parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MPOLE-13-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to permanent atomic multipole  electrostatic interactions between 1-3
connected atoms, <I>i.e.</I>, atoms separated by two covalent bonds. The
default value of 0.0 is used, if the MPOLE-13-SCALE keyword is not given in
either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MPOLE-14-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to permanent atomic multipole  electrostatic interactions between 1-4
connected atoms, <I>i.e.</I>, atoms separated by three covalent bonds. The
default value of 1.0 is used, if the MPOLE-14-SCALE keyword is not given in
either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MPOLE-15-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to permanent atomic multipole  electrostatic interactions between 1-5
connected atoms, <I>i.e.</I>, atoms separated by four covalent bonds. The
default value of 1.0 is used, if the MPOLE-15-SCALE keyword is not given in
either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MPOLE-CUTOFF
[real]</B>     Sets the cutoff distance value in Angstroms for atomic
multipole potential energy interactions. The energy for any pair of sites
beyond the cutoff distance will be set to zero. Other keywords can be used to
select a smoothing scheme near the cutoff distance. The default cutoff
distance in the absence of the MPOLE-CUTOFF keyword is infinite for
nonperiodic systems and 9.0 for periodic systems.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MPOLE-TAPER
[real]</B>     This keyword allows modification of the cutoff window for
atomic multipole potential energy interactions. It is similar in form and
action to the TAPER keyword, except that its value applies only to the atomic
multipole potential. The default value in the absence of the MPOLE-TAPER
keyword is to begin the cutoff window at 0.65 of the corresponding cutoff distance.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MPOLETERM [NONE/ONLY]</B>
    This keyword controls use of the atomic multipole electrostatics potential
energy term. In the absence of a modifying option, this keyword turns on use
of the potential. The NONE option turns off use of this potential energy term.
The ONLY option turns off all potential energy terms except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MULTIPOLE
[5 lines with: 3 integers &amp; 1 real; 3 reals; 1 real; 2 reals; 3 reals]</B>
    This keyword provides the values for a set of atomic multipole parameters
at a single site. A complete keyword entry consists of three consequtive
lines, the first line containing the MULTIPOLE keyword and the two following
lines. The first line contains three integers which define the atom type on
which the multipoles are centered, and the Z-axis and X-axis defining atom
types for this center. The real number on the first line gives the monopole
(atomic charge) in electrons. The second line contains three real numbers
which give the X-, Y- and Z-components of the atomic dipole in
electron-&Aring;. The final three lines, consisting of one, two and three real
numbers give the upper triangle of the traceless atomic quadrupole tensor in electron-&Aring;</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MUTATE [3 integers]</B>
    This keyword is used to specify atoms to be mutated during free energy
perturbation calculations. The first integer modifier gives the atom number of
an atom in the current system. The final two modifier values give the atom
types corresponding the the </FONT><FONT FACE="symbol">l</FONT><FONT FACE="New Century Schoolbook">=0
and </FONT><FONT FACE="symbol">l</FONT><FONT FACE="New Century Schoolbook">=1
states of the specified atom.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MUTUAL-11-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to the induced (mutual) field due to atoms within a polarization group
during an induced dipole calculation, <I>i.e.</I>, atoms that are in the same
polarization group as the atom being polarized. The default value of 1.0 is
used, if the MUTUAL-11-SCALE keyword is not given in either the parameter file
or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MUTUAL-12-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to the induced (mutual) field due to atoms in 1-2 polarization groups
during an induced dipole calculation, <I>i.e.</I>, atoms that are in
polarization groups directly connected to the group containing the atom being polarized.
The default value of 1.0 is used, if the MUTUAL-12-SCALE keyword is not given
in either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MUTUAL-13-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to the induced (mutual) field due to atoms in 1-3 polarization groups
during an induced dipole calculation, <I>i.e.</I>, atoms that are in
polarization groups separated by one group from the group containing the atom
being polarized. The default value of 1.0 is used, if the MUTUAL-13-SCALE
keyword is not given in either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MUTUAL-14-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to the induced (mutual) field due to atoms in 1-4 polarization groups
during an induced dipole calculation, <I>i.e.</I>, atoms that are in
polarization groups separated by two groups from the group containing the atom
being polarized. The default value of 1.0 is used, if the MUTUAL-14-SCALE
keyword is not given in either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>NEIGHBOR-GROUPS</B>
    This keyword causes the attached atom to be used in determining the
charge-charge neighbor distance for all monovalent atoms in the molecular
system. Its use causes all monovalent atoms to be treated the same as their
attached atoms for purposes of including or scaling 1-2, 1-3 and 1-4
interactions. This option works only for the simple charge-charge
electrostatic potential; it does not affect bond dipole or atomic multipole
potentials. The NEIGHBOR-GROUPS scheme is similar to that used by some common
force fields such as ENCAD.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>NEUTRAL-GROUPS</B>
    The keyword causes the attached atom to be used in determining the
charge-charge interaction cutoff distance for all monovalent atoms in the
molecular system. Its use reduces cutoff discontinuities by avoiding splitting
many of the largest charge separations found in typical molecules. Note that
this keyword does not rigorously implement the usual concept of a ``neutral
group'' as used in the literature with AMBER/OPLS and other force fields. This
option works only for the simple charge-charge electrostatic potential; it
does not affect bond dipole or atomic multipole potentials.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>NEWHESS [integer]</B>
    Sets the number of algorithmic iterations between recomputation of the
Hessian matrix. At present this keyword applies exclusively to optimizations
using the Truncated Newton method. The default value in the absence of this
keyword is 1, <I>i.e.,</I> the Hessian is computed on every iteration.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>NEXTITER [integer]</B>
    Sets the iteration number to be used for the first iteration of the
current computation. At present this keyword applies to optimization
procedures where its use can effect convergence criteria, timing of restarts,
and so forth. The default in the absence of this keyword is to take the
initial iteration as iteration 1.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>NOVERSION</B>
    Turns off the use of version numbers appended to the end of filenames as
the method for generating filenames for updated copies of an existing file.
The presence of this keyword results in direct use of input file names without
a search for the highest available version, and requires the entry of specific
output file names in many additional cases. By default, in the absence of this
keyword, TINKER generates and attaches version numbers in a manner similar to
the Digital OpenVMS operating system. For example, subsequent new versions of
the file </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
would be written first to the file</FONT><B><TT><FONT FACE="courier, courier new">
molecule.xyz_2</FONT></TT></B><FONT FACE="New Century Schoolbook">, then to </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz_3</FONT></TT></B><FONT FACE="New Century Schoolbook">,
<I>etc</I>.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>OCTAHEDRON</B>
    Specifies that the periodic ``box'' is a truncated octahedron with maximal
distance across the truncated octahedron as given by the A-AXIS keyword. All
other unit cell and periodic box size-defining keywords are ignored if the
OCTAHEDRON keyword is present.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>OPBEND [2
integers &amp; 1 real]</B>     This keyword provides the values for a single
Allinger MM-style out-of-plane angle bending potential parameter. The first
integer modifier is the atom class of the central trigonal atom and the second
integer is the atom class of the out-of-plane atom. The real number modifier
gives the force constant value for the out-of-plane angle. The default units
for the force constant are kcal/mole/radian</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>,
but this can be controlled via the OPBENDUNIT keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>OPBENDTERM
[NONE/ONLY]</B>     This keyword controls use of the Allinger MM-style
out-of-plane bending potential energy term. In the absence of a modifying
option, this keyword turns on use of the potential. The NONE option turns off
use of this potential energy term. The ONLY option turns off all potential
energy terms except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT FACE="New Century Schoolbook"><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>OPBENDUNIT
[real]</B>     Sets the scale factor needed to convert the energy value
computed by the Allinger MM-style out-of-plane bending potential into units of
kcal/mole. The correct value is force field dependent and typically provided
in the header of the master force field parameter file. The default of (</FONT><FONT FACE="symbol">p</FONT>/180)</FONT><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>
= 0.0003046 is used, if the OPBENDUNIT keyword is not given in the force field
parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>OPDIST [4
integers &amp; 1 real]</B>     This keyword provides the values for a single
out-of-plane distance potential parameter. The first integer modifier is the
atom class of the central trigonal atom and the three following integer
modifiers are the atom classes of the three attached atoms. The real number
modifier is the force constant for the harmonic function of the out-of-plane
distance of the central atom. The default units for the force constant are kcal/mole/&Aring;</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>,
but this can be controlled via the OPDISTUNIT keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>OPDISTTERM
[NONE/ONLY]</B>     This keyword controls use of the out-of-plane distance
potential energy term. In the absence of a modifying option, this keyword
turns on use of the potential. The NONE option turns off use of this potential
energy term. The ONLY option turns off all potential energy terms except for
this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>OPDISTUNIT
[real]</B>     Sets the scale factor needed to convert the energy value
computed by the out-of-plane distance potential into units of kcal/mole. The
correct value is force field dependent and typically provided in the header of
the master force field parameter file. The default value of 1.0 is used, if
the OPDISTUNIT keyword is not given in the force field parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>OVERWRITE</B>
    Causes TINKER programs, such as minimizations, that output intermediate
coordinate sets to create a single disk file for the intermediate results
which is successively overwritten with the new intermediate coordinates as
they become available. This keyword is essentially the opposite of the
SAVECYCLE keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>PARAMETERS
[file name]</B>     Provides the name of the force field parameter file to be
used for the current TINKER calculation. The standard file name extension for
parameter files, </FONT><B><TT><FONT FACE="courier, courier new">.prm</FONT></TT></B><FONT FACE="New Century Schoolbook">,
is an optional part of the file name modifier. The default in the absence of
the PARAMETERS keyword is to look for a parameter file with the same base name
as the molecular system and ending in the </FONT><B><TT><FONT FACE="courier, courier new">.prm</FONT></TT></B><FONT FACE="New Century Schoolbook">
extension. If a valid parameter file is not found, the user will asked to
provide a file name interactively.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>PIATOM [1
integer &amp; 3 reals]</B>     This keyword provides the values for the
pisystem MO potential parameters for a single atom class belonging to a pisystem.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>PIBOND [2
integers &amp; 2 reals]</B>     This keyword provides the values for the
pisystem MO potential parameters for a single type of pisystem bond.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>PISYSTEM
[integer list]</B>     This keyword sets the atoms within a molecule that are
part of a conjugated </FONT><FONT FACE="symbol">p</FONT><FONT FACE="New Century Schoolbook">-system.
The keyword is followed on the same line by a list of atom numbers and/or atom
ranges that constitute the </FONT><FONT FACE="symbol">p</FONT><FONT FACE="New Century Schoolbook">-system.
The Allinger MM force fields use this information to set up an MO calculation
used to scale bond and torsion parameters involving </FONT><FONT FACE="symbol">p</FONT><FONT FACE="New Century Schoolbook">-system
atoms.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>PME-GRID
[3 integers]</B>     This keyword sets the dimensions of the charge grid used
during particle mesh Ewald summation. The three modifiers give the size along
the X-, Y- and Z-axes, respectively. If either the Y- or Z-axis dimensions are
omitted, then they are set equal to the X-axis dimension. The default in the
absence of the PME-GRID keyword is to set the grid size along each axis to the
smallest power of 2, 3 and/or 5 which is at least as large as 1.5 times the
axis length in Angstoms. Note that the FFT used by PME is not restricted to,
but is most efficient for, grid sizes which are powers of 2, 3 and/or 5.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>PME-ORDER [integer]</B>
    This keyword sets the order of the B-spline interpolation used during
particle mesh Ewald summation. A default value of 8 is used in the absence of
the PME-ORDER keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>POLAR-11-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to polarization interactions within a polarization group, <I>i.e.</I>,
pairs of atoms that are in the same polarization group. The default value of
0.0 is used, if the POLAR-11-SCALE keyword is not given in either the
parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>POLAR-12-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to polarization interactions between 1-2 polarization groups, <I>i.e.</I>,
pairs of atoms that are in directly connected polarization groups. The default
value of 0.0 is used, if the POLAR-12-SCALE keyword is not given in either the
parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>POLAR-13-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to polarization interactions between 1-3 polarization groups, <I>i.e.</I>,
pairs of atoms that are in polarization groups separated by one other group.
The default value of 0.0 is used, if the POLAR-13-SCALE keyword is not given
in either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>POLAR-14-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to polarization interactions between 1-4 polarization groups, <I>i.e.</I>,
pairs of atoms that are in polarization groups separated by two other groups.
The default value of 1.0 is used, if the POLAR-14-SCALE keyword is not given
in either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>POLAR-DAMP
[2 reals]</B>     Controls the strength of the damping function applied to
induced dipoles and dipole polarization interaction energies. The first
modifier sets the radius in Angstoms of a hypothetical atom with unit
polarizability, while the second modifier sets the scale factor for the
exponent of the  damping function. The default values for the radius and the
scale factor are 1.662 and 1.0, respectively. Damping is eliminated entirely
by using this keyword to set the radius value to zero. </FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>POLAR-EPS [real]</B>
    This keyword sets the convergence criterion applied during computation of
self-consistent induced dipoles. The calculation is deemed to have converged
when the <I>rms</I> change (in Debyes) of the induced dipoles at all
polarizable sites is less than the value specified with this keyword. The
default value in the absence of the keyword is 10</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>-6</FONT></SUP><FONT SIZE=2>
Debyes.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>POLAR-OLD</B>
    This keyword selects the polarization damping scheme used in TINKER 3.8
and earlier. Beginning with the 3.9 release, TINKER implements a short range
polarization damping method due to Thole. This option is included primarily to
allow continued use of the early TINKER polarizable water model based on the
originally implemented flat multiplicative damping.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>POLAR-SOR [real]</B>
    Sets a successive overrelaxation (SOR) factor for use in computation of
induced atomic dipoles. Optimal values for this keyword will speed the induced
dipole calculation, and poor values can result in convergence failure. The
default value in the absence of the POLAR-SOR keyword is 0.7 which often a
reasonable value when short-range intramolecular polarization is present. For
models lacking intramolecular polarization, keyword values closer to 1.0 may
be optimal.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>POLARIZATION
[DIRECT/MUTUAL]</B>     Selects between the use of direct and mutual dipole
polarization for force fields that incorporate the polarization term. The
DIRECT modifier avoids an iterative calculation by using only the permanent
electric field in computation of induced dipoles. The MUTUAL option, which is
the default in the absence of the POLARIZATION keyword, iterates the induced
dipoles to self-consistency.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>POLARIZE
[1 integer, 1 real &amp; up to 4 integers]</B>     This keyword provides the
values for a single atomic dipole polarizability parameter. The integer
modifier, if positive, gives the atom type number for which a polarizability
parameter is to be defined. If the first integer modifier is negative, then
the parameter value to follow applies only to the individual atom whose atom
number is the negative of the modifier. The real number modifier gives the
value of the dipole polarizability in &Aring;</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>3</FONT></SUP><FONT SIZE=2>.
The final integer modifiers list the atom type numbers of atoms directly
bonded to the current atom and which will be considered to be part of the
current atom's polarization group.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>POLARIZETERM
[NONE/ONLY]</B>     This keyword controls use of the atomic dipole
polarization potential energy term. In the absence of a modifying option, this
keyword turns on use of the potential. The NONE option turns off use of this
potential energy term. The ONLY option turns off all potential energy terms
except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>POLYMER-CUTOFF
[real]</B>     Sets the value of an additional cutoff parameter needed for
infinite polymer systems. This value must be set to less than half the minimal
periodic box dimension and should be greater than the largest possible
interatomic distance that can be subject to scaling or exclusion as a local
electrostatic or van der Waals interaction. The default in the absence of the
POLYMER-CUTOFF keyword is 5.5 Angstroms.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>PRINTOUT [integer]</B>
    A general parameter for iterative procedures such as minimizations that
sets the number of iterations between writes of status information to the
standard output. The default value in the absence of the keyword is 1, <I>i.e.</I>,
the calculation status is given every iteration.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>RADIUSRULE
[ARITHMETIC/GEOMETRIC/CUBIC-MEAN]</B>     Sets the functional form of the
radius combining rule for heteroatomic van der Waals potential energy
interactions. The default in the absence of the RADIUSRULE keyword is to use
the arithmetic mean combining rule to get radii for heteroatomic interactions.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>RADIUSSIZE
[RADIUS/DIAMETER]</B>     Determines whether the atom size values given in van
der Waals parameters read from VDW keyword statements are interpreted as
atomic radius or diameter values. The default in the absence of the RADIUSSIZE
keyword is to assume that vdw size parameters are given as radius values.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>RADIUSTYPE
[R-MIN/SIGMA]</B>     Determines whether atom size values given in van der
Waals parameters read from VDW keyword statements are interpreted as potential
minimum (R</FONT></FONT><SUB><FONT SIZE=1><FONT FACE="New Century Schoolbook">min</FONT></FONT></SUB><FONT FACE="New Century Schoolbook"><FONT SIZE=2><FONT FACE="New Century Schoolbook">)
or LJ-style sigma (</FONT><FONT FACE="symbol">s</FONT>) values. The default in
the absence of the RADIUSTYPE keyword is to assume that vdw size parameters
are given as R</FONT><SUB><FONT SIZE=1>min</FONT></SUB><FONT SIZE=2> values.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>RANDOMSEED
[integer]</B>     Followed by an integer value, this keyword sets the initial
seed value for the random number generator used by TINKER. Setting RANDOMSEED
to the same value as an earlier run will allow exact reproduction of the
earlier calculation. (Note that this will not hold across different machine
types.) RANDOMSEED should be set to a positive integer less than about 2
billion. In the absence of the RANDOMSEED keyword the seed is chosen
``randomly'' based upon the number of seconds that have elapsed in the current decade.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>RATTLE [BONDS/ANGLES/DIATOMIC/TRIATOMIC/WATER]</B>
    Invokes the rattle algorithm, a velocity version of shake, on portions of
a molecular system during a molecular dynamic calculation. The RATTLE keyword
can be followed by any of the modifiers shown, in which case all occurrences
of the modifier species are constrained at ideal values taken from the bond
and angle parameters of the force field in use. In the absence of any
modifier, RATTLE constrains all bonds to hydrogen atoms at ideal bond lengths.</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">  </FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>RATTLE-BOND
[2 integers]</B>     This keyword allows the use of rattle (see above) on a
the bond between the two atoms whose numbers are specified on the keyword
line. If the two atoms are involved in a covalent bond, then their distance is
constrained to the ideal bond length from the force field. For nonbonded
atoms, the rattle constraint fixes their distance at the distance in the input
coordinate file.</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">  </FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>REACTIONFIELD
[2 reals &amp; 1 integer]</B>     This keyword provides parameters needed for
the reaction field potential energy calculation. The two real modifiers give
the radius of the dielectric cavity and the ratio of the bulk dielectric
outside the cavity to that inside the cavity. The integer modifier gives the
number of terms in the reaction field summation to be used. In the absence of
the REACTIONFIELD keyword, the default values are a cavity of radius 1000000
&Aring;, a dielectric ratio of 80 and use of only the first term of the
reaction field summation.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>REDUCE [real]</B>
    Specifies the fraction between zero and one by which the path between
starting and final conformational state will be shortened at each major cycle
of the transition state location algorithm implemented by the SADDLE program.
This causes the path endpoints to move up and out of the terminal structures
toward the transition state region. In favorable cases, a nonzero value of the
REDUCE modifier can speed convergence to the transition state. The default
value in the absence of the REDUCE keyword is zero.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>RESTRAIN-ANGLE
[3 integers &amp; 3 reals]</B>     This keyword implements a flat-welled
harmonic potential that can be used to restrain the angle between three atoms
to lie within a specified angle range. The initial integer modifiers contains
the atom numbers of the three atoms whose angle is to be restrained. The first
two real number modifiers give the lower and upper bounds in degrees on the
allowed angle values. If the angle lies between the lower and upper bounds,
the restraint potential is zero. Outside the bounds, a harmonic potential with
force constant in kcal/degree</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>
given by the final real modifier is applied. If the force constant is omitted,
a default value of 10.0 is used. If all the real modifiers are omitted, then
the atoms are restrained to an angle of zero with the default force constant.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>RESTRAIN-DISTANCE
[2 integers &amp; 3 reals]</B>     This keyword implements a flat-welled
harmonic potential that can be used to restrain two atoms to lie within a
specified distance range. The initial integer modifiers contains the atom
numbers of the two atoms to be restrained. The first two real number modifiers
give the lower and upper bounds in Angstroms on the allowed distance values.
If the interatomic distance lies between the lower and upper bounds, the
restraint potential is zero. Outside the bounds, a harmonic potential with
force constant in kcal/&Aring;</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>
given by the final real modifier is applied. If the force constant is omitted,
a default value of 100.0 is used. If all the real modifiers are omitted, then
the atoms are restrained to an interatomic distance of zero with the default
force constant.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>RESTRAIN-POSITION
[1 integer &amp; 4 reals]</B>     This keyword provides the ability to
restrain an individual atom to a specified coordinate position. The initial
integer modifier contains the atom number of the atom to be restrained. The
first three real number modifiers give the X-, Y- and Z-coordinates to which
the atom is tethered. The final real modifier sets the force constant in kcal/&Aring;</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>
for the harmonic restraint potential. If the force constant is omitted, a
default value of 100.0 is used. If all the real modifiers are omitted, then
the atom is restrained to the origin with the default force constant.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>RESTRAIN-TORSION
[4 integers &amp; 3 reals]</B>     This keyword implements a flat-welled
harmonic potential that can be used to restrain the torsional angle between
four atoms to lie within a specified angle range. The initial integer
modifiers contains the atom numbers of the four atoms whose torsional angle,
computed in the atom order listed, is to be restrained. The first two real
number modifiers give the lower and upper bounds in degrees on the allowed
torsional angle values. The angle values given can wrap around across -180 and
+180 degrees. Outside the allowed angle range, a harmonic potential with force
constant in kcal/degree</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>
given by the final real modifier is applied. If the force constant is omitted,
a default value of 1.0 is used. If all the real modifiers are omitted, then
the atoms are restrained to a torsional angle of zero with the default force constant.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>RESTRAINTERM
[NONE/ONLY]</B>     This keyword controls use of the restraint potential
energy terms. In the absence of a modifying option, this keyword turns on use
of these potentials. The NONE option turns off use of these potential energy
terms. The ONLY option turns off all potential energy terms except for these terms.</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"> </FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>RXNFIELDTERM
[NONE/ONLY]</B>     This keyword controls use of the reaction field continuum
solvation potential energy term. In the absence of a modifying option, this
keyword turns on use of the potential. The NONE option turns off use of this
potential energy term. The ONLY option turns off all potential energy terms
except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>SADDLEPOINT</B>
    The presence of this keyword allows Newton-style second derivative-based
optimization routine used by NEWTON, NEWTROT and other programs to converge to
saddlepoints as well as minima on the potential surface. By default, in the
absence of the SADDLEPOINT keyword, checks are applied that prevent
convergence to stationary points having directions of negative curvature.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>SAVE-CYCLE</B>
    This keyword causes TINKER programs, such as minimizations, that output
intermediate coordinate sets to save each successive set to the next
consecutively numbered cycle file. The SAVE-CYCLE keyword is the opposite of
the OVERWRITE keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>SAVE-INDUCED</B>
    This keyword causes TINKER molecular dynamics calculations that involve
polarizable atomic multipoles to save the values of the induced dipole
components on each polarizable atom to a separate cycle file. These files are
written whenever the atomic coordinate snapshots are written during the
dynamics run. Each induced dipole file name contains as a suffix the cycle
number followed by the letter </FONT><B><TT><FONT FACE="courier, courier new">u</FONT></TT></B><FONT FACE="New Century Schoolbook">.
</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>SAVE-VELOCITY</B>
    This keyword causes TINKER molecular dynamics calculations to save the
values of the velocity components on each atom to a separate cycle file. These
files are written whenever the atomic coordinate snapshots are written during
the dynamics run. Each velocity file name contains as a suffix the cycle
number followed by the letter </FONT><B><TT><FONT FACE="courier, courier new">v</FONT></TT></B><FONT FACE="New Century Schoolbook">.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>SOLVATE [ASP/SASA/ONION/STILL/HCT/ACE/GBSA]</B>
    Use of this keyword during energy calculations with any of the standard
force fields turns on a continuum solvation free energy term. Several
algorithms are available based on the modifier used: ASP= Eisenberg-McLachlan
ASP method using the Wesson-Eisenberg vacuum-to-water parameters; SASA= the
Ooi-Scheraga SASA method; ONION= the original 1990 Still ``Onion-shell'' GB/SA
method; STILL= the 1997 analytical GB/SA method from Still's group; HCT= the
pairwise descreening method of Hawkins, Cramer and Truhlar; ACE= the
analytical continuum solvation method from the Karplus group; GBSA= equivalent
to the STILL modifier. At present, GB/SA-style methods are only valid for
force fields that use simple partial charge electrostatics. <B><I>The ACE
method is not recommended for use in the current version of TINKER; the
algorithm is fully implemented in the source code, but is not yet completely parametrized.</I></B></FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>SOLVATETERM
[NONE/ONLY]</B>     This keyword controls use of the macroscopic solvation
potential energy term. In the absence of a modifying option, this keyword
turns on use of the potential. The NONE option turns off use of this potential
energy term. The ONLY option turns off all potential energy terms except for
this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>SPACEGROUP
[name]</B>     This keyword selects the space group to be used in manipulation
of crystal unit cells and asymmetric units. The name option must be chosen
from one of the following currently implemented space groups: P1, P1(-), P21,
Cc, P21/a, P21/n, P21/c, C2/c, P212121, Pna21, Pn21a, Cmc21, Pccn, Pbcn, Pbca,
P41, I41/a, P4(-)21c, P4(-)m2, R3c, P6(3)/mcm, Fm3(-)m, Im3(-)m.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>SPHERE [4
reals, or 1 integer &amp; 1 real]</B>     This keyword provides an alternative
to the ACTIVE and INACTIVE keywords for specification of subsets of active
atoms. If four real number modifiers are provided, the first three are taken
as X-, Y- and Z-coordinates and the fourth is the radius of a sphere centered
at these coordinates. In this case, all atoms within the sphere at the <I>start</I>
of the calculation are active <I>throughout</I> the calculation, while all
other atoms are inactive. Similarly if one integer and real number are given,
an ``active'' sphere with radius set by the real is centered on the system
atom with atom number given by the integer modifier. Multiple SPHERE keyword
lines can be present in a single keyfile, and the list of active atoms
specified by the spheres is cumulative.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>STEPMAX [real]</B>
    This keyword and its modifying value set the maximum size of an individual
step during the line search phase of conjugate gradient or truncated Newon
optimizations. The step size is computed as the norm of the vector of changes
in parameters being optimized. The default value depends on the particular
TINKER program, but is usually in the range from 1.0 to 5.0 when not specified
via the STEPMAX keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>STEPMIN [real]</B>
    This keyword and its modifying value set the minimum size of an individual
step during the line search phase of conjugate gradient or truncated Newon
optimizations. The step size is computed as the norm of the vector of changes
in parameters being optimized. The default value is usually set to about 10</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>-16</FONT></SUP><FONT SIZE=2>
when not specified via the STEPMIN keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>STRBND [1
integer &amp; 3 reals]</B>     This keyword provides the values for a single
stretch-bend cross term potential parameter. The integer modifier gives the
atom class number for the central atom of the bond angle involved in
stretch-bend interactions. The real number modifiers give the force constant
values to be used when the central atom of the angle is attached to 0, 1 or 2
additional hydrogen atoms, respectively. The default units for the
stretch-bend force constant are kcal/mole/&Aring;-degree, but this can be
controlled via the STRBNDUNIT keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>STRBNDTERM
[NONE/ONLY]</B>     This keyword controls use of the bond stretching-angle
bending cross term potential energy. In the absence of a modifying option,
this keyword turns on use of the potential. The NONE option turns off use of
this potential energy term. The ONLY option turns off all potential energy
terms except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>STRBNDUNIT
[real]</B>     Sets the scale factor needed to convert the energy value
computed by the bond stretching-angle bending cross term potential into units
of kcal/mole. The correct value is force field dependent and typically
provided in the header of the master force field parameter file. The default
value of 1.0 is used, if the STRBNDUNIT keyword is not given in the force
field parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>STRTORS
[2 integers &amp; 1 real]</B>     This keyword provides the values for a
single stretch-torsion cross term potential parameter. The two integer
modifiers give the atom class numbers for the atoms involved in the central
bond of the torsional angles to be parameterized. The real modifier gives the
value of the stretch-torsion force constant for all torsional angles with the
defined central bond atom classes. The default units for the stretch-torsion
force constant can be controlled via the STRTORUNIT keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>STRTORTERM
[NONE/ONLY]</B>     This keyword controls use of the bond stretching-torsional
angle cross term potential energy. In the absence of a modifying option, this
keyword turns on use of the potential. The NONE option turns off use of this
potential energy term. The ONLY option turns off all potential energy terms
except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>STRTORUNIT
[real]</B>     Sets the scale factor needed to convert the energy value
computed by the bond stretching-torsional angle cross term potential into
units of kcal/mole. The correct value is force field dependent and typically
provided in the header of the master force field parameter file. The default
value of 1.0 is used, if the STRTORUNIT keyword is not given in the force
field parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>TAPER [real]</B>
    This keyword allows modification of the cutoff windows for nonbonded
potential energy interactions. The nonbonded terms are smoothly reduced from
their standard value at the beginning of the cutoff window to zero at the far
end of the window. The far end of the window is specified via the CUTOFF
keyword or its potential function specific variants. The modifier value
supplied with the TAPER keyword sets the beginning of the cutoff window. The
modifier can be given either as an absolute distance value in Angstroms, or as
a fraction between zero and one of the CUTOFF distance. The default value in
the absence of the TAPER keyword ranges from 0.65 to 0.9 of the CUTOFF
distance depending on the type of potential function. The windows are
implemented via polynomial-based switching functions, in some cases combined
with energy shifting.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>TAU-PRESSURE
[real]</B>     Sets the coupling time in picoseconds for the Groningen-style
pressure bath coupling used to control the system pressure during molecular
dynamics calculations. A default value of 2.0 is used for TAU-PRESSURE in the
absence of the keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>TAU-TEMPERATURE
[real]</B>     Sets the coupling time in picoseconds for the Groningen-style
temperature bath coupling used to control the system temperature during
molecular dynamics calculations. A default value of 0.1 is used for
TAU-TEMPERATURE in the absence of the keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>THERMOSTAT
[BERENDSEN/ANDERSEN]</B>     This keyword selects a thermostat algorithm for
use during molecular dynamics. Two modifiers are available, a Berendsen bath
coupling method, and an Andersen stochastic collision method. The default in
the absence of the THERMOSTAT keyword is to use the BERENDSEN algorithm.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>TORSION
[4 integers &amp; up to 6 real/real/integer triples]</B>     This keyword
provides the values for a single torsional angle parameter. The first four
integer modifiers give the atom class numbers for the atoms involved in the
torsional angle to be defined. Each of the remaining triples of
real/real/integer modifiers give the half-amplitude, phase offset in degrees
and periodicity of a particular torsional function term, respectively.
Periodicities through 6-fold are allowed for torsional parameters.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>TORSION4
[4 integers &amp; up to 6 real/real/integer triples]</B>     This keyword
provides the values for a single torsional angle parameter specific to atoms
in 4-membered rings. The first four integer modifiers give the atom class
numbers for the atoms involved in the torsional angle to be defined. The
remaining triples of real number and integer modifiers operate as described
above for the TORSION keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>TORSION5
[4 integers &amp; up to 6 real/real/integer triples]</B>     This keyword
provides the values for a single torsional angle parameter specific to atoms
in 5-membered rings. The first four integer modifiers give the atom class
numbers for the atoms involved in the torsional angle to be defined. The
remaining triples of real number and integer modifiers operate as described
above for the TORSION keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>TORSIONTERM
[NONE/ONLY]</B>     This keyword controls use of the torsional angle potential
energy term. In the absence of a modifying option, this keyword turns on use
of the potential. The NONE option turns off use of this potential energy term.
The ONLY option turns off all potential energy terms except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>TORSIONUNIT
[real]</B>     Sets the scale factor needed to convert the energy value
computed by the torsional angle potential into units of kcal/mole. The correct
value is force field dependent and typically provided in the header of the
master force field parameter file. The default value of 1.0 is used, if the
TORSIONUNIT keyword is not given in the force field parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>TORTORTERM
[NONE/ONLY]</B>     This keyword controls use of the torsion-torsion potential
energy term. In the absence of a modifying option, this keyword turns on use
of the potential. The NONE option turns off use of this potential energy term.
The ONLY option turns off all potential energy terms except for this one. This
energy term is not implemented in the current version of TINKER.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>TRIAL-DISTANCE
             [CLASSIC/RANDOM/TRICOR/HAVEL integer/PAIRWISE integer]</B>    
Sets the method for selection of a trial distance matrix during distance
geometry computations. The keyword takes a modifier that selects the method to
be used. The HAVEL and PAIRWISE modifiers also require an additional integer
value that specifies the number of atoms used in metrization and the
percentage of metrization, respectively. The default in the absence of this
keyword is to use the PAIRWISE method with 100 percent metrization. Further
information on the various methods is given with the description of the TINKER
distance geometry program.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>TRIAL-DISTRIBUTION
[real]</B>     Sets the initial value for the mean of the Gaussian
distribution used to select trial distances between the lower and upper bounds
during distance geometry computations. The value given must be between 0 and 1
which represent the lower and upper bounds respectively. This keyword is
rarely needed since TINKER will usually be able to choose a reasonable value
by default.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>TRUNCATE</B>
    Causes all distance-based nonbond energy cutoffs to be sharply truncated
to an energy of zero at distances greater than the value set by the cutoff
keyword(s) without use of any shifting, switching or smoothing schemes. At all
distances within the cutoff sphere, the full interaction energy is computed.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>UREY-CUBIC
[real]</B>     Sets the value of the cubic term in the Taylor series expansion
form of the Urey-Bradley potential energy. The real number modifier gives the
value of the coefficient as a multiple of the quadratic coefficient. The
default value in the absence of the UREY-CUBIC keyword is zero; <I>i.e.</I>,
the cubic Urey-Bradley term is omitted.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>UREY-QUARTIC
[real]</B>     Sets the value of the quartic term in the Taylor series
expansion form of the Urey-Bradley potential energy. The real number modifier
gives the value of the coefficient as a multiple of the quadratic coefficient.
The default value in the absence of the UREY-QUARTIC keyword is zero; <I>i.e.</I>,
the quartic Urey-Bradley term is omitted.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>UREYBRAD
[3 integers &amp; 2 reals]</B>     This keyword provides the values for a
single Urey-Bradley cross term potential parameter. The integer modifiers give
the atom class numbers for the three kinds of atoms involved in the angle for
which a Urey-Bradley term is to be defined. The real number modifiers give the
force constant value for the term and the target value for the 1-3 distance in
&Aring;. The default units for the force constant are kcal/mole/&Aring;</FONT></FONT><FONT FACE="New Century Schoolbook"><SUP><FONT SIZE=1>2</FONT></SUP><FONT SIZE=2>,
but this can be controlled via the UREYUNIT keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>UREYTERM [NONE/ONLY]</B>
    This keyword controls use of the Urey-Bradley potential energy term. In
the absence of a modifying option, this keyword turns on use of the potential.
The NONE option turns off use of this potential energy term. The ONLY option
turns off all potential energy terms except for this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>UREYUNIT [real]</B>
    Sets the scale factor needed to convert the energy value computed by the
Urey-Bradley potential into units of kcal/mole. The correct value is force
field dependent and typically provided in the header of the master force field
parameter file. The default value of 1.0 is used, if the UREYUNIT keyword is
not given in the force field parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>VDW [1
integer &amp; 3 reals]</B>     This keyword provides values for a single van
der Waals parameter. The integer modifier, if positive, gives the atom class
number for which vdw parameters are to be defined. Note that vdw parameters
are given for atom classes, not atom types. The three real number modifiers
give the values of the atom size in &Aring;, homoatomic well depth in
kcal/mole, and an optional reduction factor for univalent atoms.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>VDW-12-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to van der Waals potential interactions between 1-2 connected atoms, <I>i.e.</I>,
atoms that are directly bonded. The default value of 0.0 is used, if the
VDW-12-SCALE keyword is not given in either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>VDW-13-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to van der Waals potential interactions between 1-3 connected atoms, <I>i.e.</I>,
atoms separated by two covalent bonds. The default value of 0.0 is used, if
the VDW-13-SCALE keyword is not given in either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>VDW-14-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to van der Waals potential interactions between 1-4 connected atoms, <I>i.e.</I>,
atoms separated by three covalent bonds. The default value of 1.0 is used, if
the VDW-14-SCALE keyword is not given in either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>VDW-15-SCALE
[real]</B>     This keyword provides a multiplicative scale factor that is
applied to van der Waals potential interactions between 1-5 connected atoms, <I>i.e.</I>,
atoms separated by four covalent bonds. The default value of 1.0 is used, if
the VDW-15-SCALE keyword is not given in either the parameter file or the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>VDW-CUTOFF
[real]</B>     Sets the cutoff distance value in Angstroms for van der Waals
potential energy interactions. The energy for any pair of van der Waals sites
beyond the cutoff distance will be set to zero. Other keywords can be used to
select a smoothing scheme near the cutoff distance. The default cutoff
distance in the absence of the VDW-CUTOFF keyword is infinite for nonperiodic
systems and 9.0 for periodic systems.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>VDW-TAPER [real]</B>
    This keyword allows modification of the cutoff windows for van der Waals
potential energy interactions. It is similar in form and action to the TAPER
keyword, except that its value applies only to the vdw potential. The default
value in the absence of the VDW-TAPER keyword is to begin the cutoff window at
0.9 of the vdw cutoff distance.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>VDW14 [1
integer &amp; 2 reals]</B>     This keyword provides values for a single van
der Waals parameter for use in 1-4 nonbonded interactions. The integer
modifier, if positive, gives the atom class number for which vdw parameters
are to be defined. Note that vdw parameters are given for atom classes, not
atom types. The two real number modifiers give the values of the atom size in
&Aring; and the homoatomic well depth in kcal/mole. Reduction factors, if
used, are carried over from the VDW keyword for the same atom class.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>VDWPR [2
integers &amp; 2 reals]</B>     This keyword provides the values for the vdw
parameters for a single special heteroatomic pair of atoms. The integer
modifiers give the pair of atom class numbers for which special vdw parameters
are to be defined. The two real number modifiers give the values of the
minimum energy contact distance in &Aring; and the well depth at the minimum
distance in kcal/mole.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>VDWTERM [NONE/ONLY]</B>
    This keyword controls use of the van der Waals repulsion-dispersion
potential energy term. In the absence of a modifying option, this keyword
turns on use of the potential. The NONE option turns off use of this potential
energy term. The ONLY option turns off all potential energy terms except for
this one.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>VDWTYPE
[LENNARD-JONES/BUCKINGHAM/BUFFERED-14-7/MM3-HBOND /GAUSSIAN]</B>     Sets the
functional form for the van der Waals potential energy term. The text modifier
gives the name of the functional form to be used. The GAUSSIAN modifier value
implements a two or four Gaussian fit to the corresponding Lennard-Jones
function for use with potential energy smoothing schemes. The default in the
absence of the VDWTYPE keyword is to use the standard two parameter
Lennard-Jones function.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>VERBOSE</B>
    Turns on printing of secondary and informational output during a variety
of TINKER computations; a subset of the more extensive output provided by the
DEBUG keyword.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>WALL [real]</B>
    Sets the radius of a spherical boundary used to maintain droplet boundary
conditions. The real modifier specifies the desired approximate radius of the
droplet. In practice, an artificial van der Waals wall is constructed at a
fixed buffer distance of 2.5 &Aring; outside the specified radius. The effect
is that atoms which attempt to move outside the region defined by the droplet
radius will be forced toward the center.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>WRITEOUT [integer]</B>
    A general parameter for iterative procedures such as minimizations that
sets the number of iterations between writes of intermediate results (such as
the current coordinates) to disk file(s). The default value in the absence of
the keyword is 1, <I>i.e.</I>, the intermediate results are written to file on
every iteration. Whether successive results are saved to new files or replace
previously written intermediate results is controlled by the OVERWRITE and
SAVECYCLE keywords.</FONT></FONT></P>
<TABLE WIDTH=450 CELLPADDING=0>
<TR VALIGN=TOP>
<TD COLSPAN=1 WIDTH="12%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">8.</FONT></FONT></U></B></P>
</TD>
<TD COLSPAN=1 WIDTH="88%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">Notes on Special
Features &amp; Methods</FONT></FONT></U></B></P>
</TD></TR>
</TABLE>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;This
section contains several short notes with further information about TINKER
methodology, algorithms and special features. The discussion is not intended
to be exhaustive, but rather to explain features and capabilities so that
users can make more complete use of the package.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">FILE
VERSION NUMBERS</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;All of
the input and output file types routinely used by the TINKER package are
capable of existing as multiple versions of a base file name. For example, if
the program XYZINT is run on the input file </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">,
the output internal coordinates file will be written to </FONT><B><TT><FONT FACE="courier, courier new">molecule.int</FONT></TT></B><FONT FACE="New Century Schoolbook">.
If a file named </FONT><B><TT><FONT FACE="courier, courier new">molecule.int</FONT></TT></B><FONT FACE="New Century Schoolbook">
is already present prior to running XYZINT, then the output will be written
instead to the next available version, in this case to </FONT><B><TT><FONT FACE="courier, courier new">molecule.int_2</FONT></TT></B><FONT FACE="New Century Schoolbook">.
In fact the output is generally written to the lowest available, previously
unused version number (</FONT><B><TT><FONT FACE="courier, courier new">molecule.int_3</FONT></TT></B><FONT FACE="New Century Schoolbook">,
</FONT><B><TT><FONT FACE="courier, courier new">molecule.int_4</FONT></TT></B><FONT FACE="New Century Schoolbook">,
<I>etc</I>., as high as needed). Input file names are handled similarly. If
simply </FONT><B><TT><FONT FACE="courier, courier new">molecule</FONT></TT></B><FONT FACE="New Century Schoolbook">
or </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
is entered as the input file name upon running XYZINT, then the highest
version of </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
will be used as the actual input file. If an explicit version number is
entered as part of the input file name, then the specified version will be
used as the input file.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The
version number scheme will be recognized by many older users as a holdover
from the VMS origins of the first version of the TINKER software. It has been
maintained to make it easier to chain together multiple calculations that may
create several new versions of a given file, and to make it more difficult to
accidently overwrite a needed result. The version scheme applies to most uses
of many common TINKER file types such as </FONT><B><TT><FONT FACE="courier, courier new">.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">,
</FONT><B><TT><FONT FACE="courier, courier new">.int</FONT></TT></B><FONT FACE="New Century Schoolbook">,
</FONT><B><TT><FONT FACE="courier, courier new">.key</FONT></TT></B><FONT FACE="New Century Schoolbook">,
</FONT><B><TT><FONT FACE="courier, courier new">.arc</FONT></TT></B><FONT FACE="New Century Schoolbook">.
It is not used when an overwritten file ``update'' is obviously the correct
action, for example, the </FONT><B><TT><FONT FACE="courier, courier new">.dyn</FONT></TT></B><FONT FACE="New Century Schoolbook">
molecular dynamics restart files. For those users who prefer a more Unix-like
operation, and do not desire use of file versions, this feature can be turned
off by adding the NOVERSION keyword to the applicable TINKER keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The
version scheme as implemented in TINKER does have two known quirks. First, it
becomes impossible to directly use the original unversioned copy of a file if
higher version numbers are present. For example, if the files </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
and </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz_2</FONT></TT></B><FONT FACE="New Century Schoolbook">
both exist, then </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
cannot be accessed as input by XYZINT. If </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
is entered in response to the input file name question, </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz_2</FONT></TT></B><FONT FACE="New Century Schoolbook">
(or the highest present version number) will be used as input. The only
workaround is to copy or rename </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
to something else, say </FONT><B><TT><FONT FACE="courier, courier new">molecule.new</FONT></TT></B><FONT FACE="New Century Schoolbook">,
and use that name for the input file. Secondly, missing version numbers always
end the search for the highest available version number; <I>i.e.</I>, version
numbers are assumed to be consecutive and without gaps. For example, if </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">,
</FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz_2</FONT></TT></B><FONT FACE="New Century Schoolbook">
and </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz_4</FONT></TT></B><FONT FACE="New Century Schoolbook">
are present, but not </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz_3</FONT></TT></B><FONT FACE="New Century Schoolbook">,
then </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz_2</FONT></TT></B><FONT FACE="New Century Schoolbook">
will be used as input to XYZINT if </FONT><B><TT><FONT FACE="courier, courier new">molecule</FONT></TT></B><FONT FACE="New Century Schoolbook">
is given as the input file name. Similarly, output files will fill in gaps in
an already existing set of file versions.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">COMMAND
LINE OPTIONS</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Many
operating systems or compiler supplied-libraries make available something like
the standard Unix iargc and getarg routines for capturing command line
arguments. On these machines most of the TINKER programs support a selection
of command line arguments and options. The name of the keyfile to be used for
a calculation is read from the argument following a </FONT><B><TT><FONT FACE="courier, courier new">-k</FONT></TT></B><FONT FACE="New Century Schoolbook">
(equivalent to either</FONT><B><FONT FACE="New Century Schoolbook"> </FONT><TT><FONT FACE="courier, courier new">-key</FONT></TT></B><FONT FACE="New Century Schoolbook">
or </FONT><B><TT><FONT FACE="courier, courier new">-keyfile</FONT></TT></B><FONT FACE="New Century Schoolbook">,
case insensitive) command line argument. Note that the </FONT><B><TT><FONT FACE="courier, courier new">-k</FONT></TT></B><FONT FACE="New Century Schoolbook">
options can appear anywhere on the command line following the executable name.
All other command line arguments, excepting the name of the executable program
itself, are treated as input arguments. These input arguments are read from
left to right and interpreted in order as the answers to questions that would
be asked by an interactive invocation of the same TINKER program. For example,
the following command line:</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><TT><FONT SIZE=2><FONT FACE="courier, courier new">newton
molecule -k test a a 0.01</FONT></FONT></TT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">will invoke
the NEWTON program on the structure file </FONT><B><TT><FONT FACE="courier, courier new">molecule.xyz</FONT></TT></B><FONT FACE="New Century Schoolbook">
using the keyfile </FONT><B><TT><FONT FACE="courier, courier new">test.key</FONT></TT></B><FONT FACE="New Century Schoolbook">,
automatic mode </FONT><B><TT><FONT FACE="courier, courier new">[a]</FONT></TT></B><FONT FACE="New Century Schoolbook">
for both the method and preconditioning, and </FONT><B><TT><FONT FACE="courier, courier new">0.01</FONT></TT></B><FONT FACE="New Century Schoolbook">
for the RMS gradient per atom termination criterion in kcal/mole/&Aring;.
Provided that the force field parameter set, <I>etc</I>. is provided in </FONT><B><TT><FONT FACE="courier, courier new">test.key</FONT></TT></B><FONT FACE="New Century Schoolbook">,
the above compuation will procede directly from the command line invocation
without further interactive input.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">USE ON
MICROSOFT WINDOWS SYSTEMS</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;TINKER
executables for Microsoft PC systems should be run from the DOS Prompt window
available under the various versions of Windows. The TINKER executable
directory should be added to your path via the autoexec.bat file or similar.
If using Win2000, set the number of scrollable lines in the DOS Prompt window
to a very large number, so that you will be able to inspect screen output
after it flies by. With Win95/98, these DOS Prompt windows are only able to
scroll a small number of lines (<I>amazing!</I>), so TINKER programs which
generate large amounts of screen output should be run such that output will be
redirected to a file. This can be accomplished by running the TINKER program
in batch mode or by using the Unix-like output redirection build into DOS. For
example, the command:</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><TT><FONT SIZE=2><FONT FACE="courier, courier new">dynamic
&lt; molecule.inp &gt; molecule.log</FONT></FONT></TT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">will run the
TINKER dynamic program taking input from the file </FONT><B><TT><FONT FACE="courier, courier new">molecule.inp</FONT></TT></B><FONT FACE="New Century Schoolbook">
and sending output to </FONT><B><TT><FONT FACE="courier, courier new">molecule.log</FONT></TT></B><FONT FACE="New Century Schoolbook">.
Also note that command line options as described above are available with the
distributed TINKER executables.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Another
alternative, particularly attractive to those already familiar with Linux or
Unix systems, is to download the Cygwin package currently available under GPL
license from the site </FONT><B><TT><FONT FACE="courier, courier new">http://source.redhat.com/cygwin/</FONT></TT></B><FONT FACE="New Century Schoolbook">.
The cygwin tools provide many of the GNU tools, including a bash shell window
from which TINKER programs can be run.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;If the
distributed TINKER executables are directly from Windows by double clicking on
the program icon, then the program will run in its own window. However, upon
completion of the program the window will close and screen output will be
lost. Any output files written by the program will, of course, still be
available. The Windows behavior can be changed by adding the EXIT-PAUSE
keyword to the keyfile.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">USE ON
APPLE MACINTOSH SYSTEMS</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The
TINKER executables can be run under MacOS by double clicking on a program
icon. The program will run in its own window to which all ``screen'' output
will be directed. Upon program termination the window will remain active
pending a final return entered by the user which will close the window. Prior
to the final return, the contents of the screen window can be saved to a file
via the clipboard for permanent storage. Note that Macintosh uses a colon
instead of a forward- or back-slash as the directory separator, so keyfiles
transfered from other machines will need to be altered accordingly.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">ATOM
TYPES VS. ATOM CLASSES</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Manipulation
of atom types and the proliferation of parameters as atoms are further
subdivided into new types is the bane of force field calculation. For example,
if each topologically distinct atom arising from the 20 natural amino acids is
given a different atom type, then about 300 separate type are required (this
ignores the different N- and C-terminal forms of the residues, diastereotopic
hydrogens, <I>etc</I>.). However, all these types lead to literally thousands
of different force field parameters. In fact, there are many thousands of
distinct torsional parameters alone. It is impossible at present to fully
optimize each of these parameters; and even if we could, a great many of the
parameters would be nearly identical. Two somewhat complimentary solutions are
available to handle the proliferation of parameters. The first is to specify
the molecular fragments to which a given parameter can be applied in terms of
a chemical structure language, SMILES strings for example. Some commercial
systems, such as the TRIPOS Sybyl software, make use of such a scheme to parse
structures and assign force field parameters.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;A second
general approach is to use hierarchical cascades of parameter groups. TINKER
uses a simple version of this scheme. Each TINKER force field atom has both an
atom <I>type</I> number and an atom <I>class</I> number. The types are subsets
of the atom classes, <I>i.e.</I>, several different atom types can belong to
the same atom class. Force field parameters that are somewhat less sensitive
to local environment, such as local geometry terms, are then provided and
assigned based on atom class. Other energy parameters, such as electrostatic
parameters, that are very environment dependent are assigned over the atom
types. This greatly reduces the number of independent multiple-atom parameters
like the four-atom torsional parameters.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">CALCULATIONS
ON PARTIAL STRUCTURES</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Two
methods are available for performing energetic calculations on portions or
substructures within a full molecular system. TINKER allows division of the
entire system into <I>active</I> and <I>inactive</I> parts which can be
defined via keywords. In subsequent calculations, such as minimization or
dynamics, only the active portions of the system are allowed to move. The
force field engine responds to the active/inactive division by computing all
energetic interactions involving at least one active atom; <I>i.e.</I>, any
interaction whose energy can change with the motion of one or more active
atoms is computed.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The
second method for partial structure computation involves dividing the original
system into a set of atom <I>groups</I>. As before, the groups can be
specified via appropriate keywords. The current TINKER implementation allows
specification of up to a maximum number of groups as given in the </FONT><B><TT><FONT FACE="courier, courier new">sizes.i</FONT></TT></B><FONT FACE="New Century Schoolbook">
dimensioning file. The groups must be disjoint in that no atom can belong to
more than one group. Further keywords allow the user to specify which intra-
and intergroup sets of energetic interactions will contribute to the total
force field energy. Weights for each set of interactions in the total energy
can also be input. A specific energetic interaction is assigned to a
particular intra- or intergroup set if all the atoms involved in the
interaction belong to the group (intra-) or pair of groups (inter-).
Interactions involving atoms from more than two groups are not computed.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Note
that the groups method and active/inactive method use different assignment
procedures for individual interactions. The active/inactive scheme is intended
for situations where only a portion of a system is allowed to move, but the
total energy needs to reflect the presence of the remaining inactive portion
of the structure. The groups method is intended for use in rigid body
calculations, and is needed for certain kinds of free energy perturbation calculations.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">METAL
COMPLEXES AND HYPERVALENT SPECIES</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The
distribution version of TINKER comes dimensioned for a maximum atomic
coordination number of four as needed for standard organic compounds. In order
to use TINKER for calculations on species containing higher coordination
numbers, simply change the value of the parameter </FONT><B><TT><FONT FACE="courier, courier new">maxval</FONT></TT></B><FONT FACE="New Century Schoolbook">
in the master dimensioning file </FONT><B><TT><FONT FACE="courier, courier new">sizes.i</FONT></TT></B><FONT FACE="New Century Schoolbook">
and rebuilt the package. Note that this parameter value should not be set
larger than necessary since large values can slow the execution of portions of
some TINKER programs.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Many
molecular mechanics approaches to inorganic and metal structures use an angle
bending term which is softer than the usual harmonic bending potential. TINKER
implements a Fourier bending term similar to that used by the Landis group's
SHAPES force field. The parameters for specific Fourier angle terms are
supplied via the ANGLEF parameter and keyword format. Note that a Fourier term
will only be used for a particular angle if a corresponding harmonic angle
term is not present in the parameter file.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;We are
now collaborating with Anders Carlsson's group in St. Louis to add his
transition metal ligand field term to TINKER. Support for this additional
potential functional form is already in the TINKER source code, and we plan to
release the energy routines after further testing and parameterization.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">NEIGHBOR
METHODS FOR NONBONDED TERMS</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;In
addition to standard double loop methods, the Method of Lights is available to
speed neighbor searching. This method based on taking intersections of sorted
atom lists can be much faster for problems where the cutoff distance is
significantly smaller than half the maximal cell dimension. The current
version of TINKER does not implement the ``neighbor list'' schemes common to
many other simulation packages.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">PERIODIC
BOUNDARY CONDITIONS</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Both
spherical cutoff images or replicates of a cell are supported by all TINKER
programs that implement periodic boundary conditions. Whenever the cutoff
distance is too large for the minimum image to be the only relevant neighbor (<I>i.e.</I>,
half the minimum box dimension for orthogonal cells), TINKER will
automatically switch from the image formalism to use of replicated cells.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">DISTANCE
CUTOFFS FOR ENERGY FUNCTIONS</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Polynomial
energy switching over a window is used for terms whose energy is small near
the cutoff distance. For monopole electrostatic interactions, which are quite
large in typical cutoff ranges, a two polynomial multiplicative-additive
shifted energy switch unique to TINKER is applied. The TINKER method is
similar in spirit to the force switching methods of Steinbach and Brooks, <I>J.
Comput. Chem.</I>, <B><I>15</I></B>, 667-683 (1994). While the particle mesh
Ewald method is preferred when periodic boundary conditions are present,
TINKER's shifted energy switch with reasonable switching windows is quite
satisfactory for most routine modeling problems. The shifted energy switch
minimizes the perturbation of the energy and the gradient at the cutoff to
acceptable levels. Problems should arise only if the property you wish to
monitor is known to require explicit inclusion of long range components (<I>i.e.</I>,
calculation of the dielectric constant, <I>etc</I>.).</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">EWALD
SUMMATION METHODS</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;TINKER
contains a versions of the Ewald summation technique for inclusion of long
range electrostatic interactions via periodic boundaries. The particle mesh
Ewald (PME) method is available for simple charge-charge potentials, while
regular Ewald is provided for polarizable atomic multipole interactions. The
accuracy and speed of the regular and PME calculations is dependent on several
interrelated parameters. For both methods, the Ewald coefficient and
real-space cutoff distance must be set to reasonable and complementary values.
Additional control variables for regular Ewald are the fractional coverage and
number of vectors used in reciprocal space. For PME the additional control
values are the B-spline order and charge grid dimensions. Complete control
over all of these parameters is available via the TINKER keyfile mechanism. By
default TINKER will select a set of parameters which provide a reasonable
compromise between accuracy and speed, but these should be checked and
modified as necessary for each individual system.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">CONTINUUM
SOLVATION MODELS</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Several
alternative continuum solvation algorithms are contained within TINKER. All of
these are accessed via the SOLVATE keyword and its modifiers. Two simple
surface area methods are implemented: the ASP method of Eisenberg and
McLachlan, and the SASA method from Scheraga's group. These methods are
applicable to any of the standard TINKER force fields. Various schemes based
on the generalized Born formalism are also available: the original 1990
numerical ``Onion-shell'' GB/SA method from Still's group, the 1997 analytical
GB/SA method also due to Still, a pairwise descreening algorithm originally
proposed by Hawkins, Cramer and Truhlar, and the analytical continuum
solvation (ACE) method of Schaefer and Karplus. At present, the generalized
Born methods should only be used with force fields having simple partial
charge electrostatic interactions.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Some
further comments are in order regarding the GB/SA-style solvation models. The
``Onion-shell'' model is provided mostly for comparison purposes. It uses an
exact, analytical surface area calculation for the cavity term and the
numerical scheme described in the original paper for the polarization term.
This method is very slow, especially for large systems, and does not contain
the contribution of the Born radii chain rule term to the first derivatives.
We recommend its use only for single-point energy calculations. The other
GB/SA methods (``analytical'' Still, H-C-T pairwise descreening, and ACE) use
an approximate cavity term based on Born radii, and do contain fully correct
derivatives including the Born radii chain rule contribution. These methods
all scale in CPU time with the square of the size of the system, and can be
used with minimization, molecular dynamics and large molecules.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Finally,
we note that the ACE solvation model should not be used with the current
version of TINKER. The algorithm is fully implemented in the source code, but
parameterization is not complete. As of late 2000, parameter values are only
available in the literature for use of ACE with the older CHARMM19 force
field. We plan to develop values for use with more modern all-atom force
fields, and these will be incorporated into TINKER sometime in the future.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">POLARIZABLE
MULTIPOLE ELECTROSTATICS</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Atomic
multipole electrostatics through the quadrupole moment is supported by the
current version of TINKER, as is either mutual or direct dipole polarization.
Ewald summation is available for inclusion of long range interactions.
Calculations are implemented via a mixture of the CCP5 algorithms of W. Smith
and the Applequist-Dykstra Cartesian polytensor method. At present analytical
energy and Cartesian gradient code is provided.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The
TINKER package allows intramolecular polarization to be treated via a version
of the interaction damping scheme of Thole. To implement the Thole scheme, it
is necessary to set all the </FONT><B><TT><FONT FACE="courier, courier new">mutual-1x-scale</FONT></TT></B><FONT FACE="New Century Schoolbook">
keywords to a value of one. The other polarization scaling keyword series, </FONT><B><TT><FONT FACE="courier, courier new">direct-1x-scale</FONT></TT></B><FONT FACE="New Century Schoolbook">
and </FONT><B><TT><FONT FACE="courier, courier new">polar-1x-scale</FONT></TT></B><FONT FACE="New Century Schoolbook">,
can be set independently to enable a wide variety of polarization models. In
order to use an Applequist-style model without polarization damping, simply
set the </FONT><B><TT><FONT FACE="courier, courier new">polar-damp</FONT></TT></B><FONT FACE="New Century Schoolbook">
keyword to zero.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">POTENTIAL
ENERGY SMOOTHING</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Versions
of our Potential Smoothing and Search (PSS) methodology have been implemented
within TINKER. This methods belong to the same general family as Scheraga's
Diffusion Equation Method, Straub's Gaussian Density Annealing, Shalloway's
Packet Annealing and Verschelde's Effective Diffused Potential, but our
algorithms reflect our own ongoing research in this area. In many ways the
TINKER potential smoothing methods are the deterministic analog of stochastic
simulated annealing. The PSS algorithms are very powerful, but are relatively
new and are still undergoing modification, testing and calibration within our
research group. This version of TINKER also includes a basin-hopping
conformational scanning algorithm in the program SCAN which is particularly
effective on smoothed potential surfaces.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">DISTANCE
GEOMETRY METRIZATION</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;A much
improved and very fast random pairwise metrization scheme is available which
allows good sampling during trial distance matrix generation without the usual
structural anomalies and CPU constraints of other metrization procedures. An
outline of the methodology and its application to NMR NOE-based structure
refinement is described in the paper by Hodsdon, <I>et al</I>. in <I>J. Mol. Biol.</I>,
<B><I>264</I></B>, 585-602 (1996). We have obtained good results with
something like the keyword phrase </FONT><B><TT><FONT FACE="courier, courier new">trial-distribution
pairwise 5</FONT></TT></B><FONT FACE="New Century Schoolbook">, which performs
5% partial random pairwise metrization. For structures over several hundred
atoms, a value less than 5 for the percentage of metrization should be fine.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<TABLE WIDTH=450 CELLPADDING=0>
<TR VALIGN=TOP>
<TD COLSPAN=1 WIDTH="12%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">9.</FONT></FONT></U></B></P>
</TD>
<TD COLSPAN=1 WIDTH="88%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">Descriptions of
TINKER Routines</FONT></FONT></U></B></P>
</TD></TR>
</TABLE>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The
distribution version of the TINKER package contains over 600 separate
programs, subroutines and functions. This section contains a brief description
of the purpose of most of these code units. Further information can be found
in the comments located at the top of each source code file.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ACTIVE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;active&quot;
sets the list of atoms that are used during each potential energy function calculation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ADDBASE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;addbase&quot;
builds the Cartesian coordinates for a single nucleic acid base; coordinates
are read from the Protein Data Bank file or found from internal coordinates,
then atom types are assigned and connectivity data generated</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ADDBOND Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;addbond&quot;
adds entries to the attached atoms list in order to generate a direct
connection between two atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ADDSIDE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;addside&quot;
builds the Cartesian coordinates for a single amino acid side chain;
coordinates are read from the Protein Data Bank file or found from internal
coordinates, then atom types are assigned and connectivity data generated</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ADJACENT Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;adjacent&quot;
finds an atom connected to atom &quot;i1&quot; other than atom &quot;i2&quot;;
if no such atom exists, then the closest atom in space is returned</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ALCHEMY Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;alchemy&quot;
computes the free energy difference corresponding to a small perturbation by
Boltzmann weighting the potential energy difference over a number of sample
states; current version (incorrectly) considers the charge energy to be
intermolecular in finding the perturbation energies</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ANALYSIS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;analysis&quot;
calls the series of routines needed to calculate the potential energy and
perform energy partitioning analysis in terms of type of interaction or atom number</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ANALYZ4 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;analyz4&quot;
prints the energy to 4 decimal places and number of interactions for each
component of the potential energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ANALYZ6 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;analyz6&quot;
prints the energy to 6 decimal places and number of interactions for each
component of the potential energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ANALYZ8 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;analyz8&quot;
prints the energy to 8 decimal places and number of interactions for each
component of the potential energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ANALYZE Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;analyze&quot;
computes and displays the total potential; options are provided to partition
the energy by atom or by potential function type; parameters used in computing
interactions can also be displayed by atom; output of large energy
interactions and of electrostatic and inertial properties is available</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ANGLES Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;angles&quot;
finds the total number of bond angles and stores the atom numbers of the atoms
defining each angle; for each angle to a tricoordinate central atom, the third
bonded atom is stored for use in out-of-plane bending</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ANNEAL Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;anneal&quot;
performs a simulated annealing protocol by means of variable temperature
molecular dynamics using either linear, exponential or sigmoidal cooling schedules</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ANORM Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;anorm&quot;
finds the norm (length) of a vector; used as a service routine by the Connolly
surface area and volume computation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ARCHIVE Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;archive&quot;
is a utility program for coordinate files which concatenates multiple
coordinate sets into a single archive file, or extracts individual coordinate
sets from an archive</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ASET Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ATOMYZE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;atomyze&quot;
prints the potential energy components broken down by atom and to a choice of precision</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ATTACH Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;attach&quot;
generates lists of 1-3, 1-4 and 1-5 connectivities starting from the
previously determined list of attached atoms (ie, 1-2 connectivity)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BASEFILE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;basefile&quot;
extracts from an input filename the portion consisting of any directory name
and the base filename</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BEEMAN Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;beeman&quot;
performs a single molecular dynamics time step by means of a Beeman multistep
recursion formula; the actual coefficients are Brooks' &quot;Better
Beeman&quot; values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BETACF Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;betacf&quot;
computes a rapidly convergent continued fraction needed by routine
&quot;betai&quot; to evaluate the cumulative Beta distribution</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BETAI Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;betai&quot;
evaluates the cumulative Beta distribution function as the probability that a
random variable from a distribution with Beta parameters &quot;a&quot; and
&quot;b&quot; will be less than &quot;x&quot;</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BIGBLOCK Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;bigblock&quot;
replicates the coordinates of a single unit cell to give a larger block of
repeated units</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BMAX Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BNDERR Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;bnderr&quot;
is the distance bound error function and derivatives; this version implements
the original and Havel's normalized lower bound penalty, the normalized
version is preferred when lower bounds are small (as with NMR NOE restraints),
the original penalty is needed if large lower bounds are present</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BONDS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;bonds&quot;
finds the total number of covalent bonds and stores the atom numbers of the
atoms defining each bond</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BORN Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;born&quot;
computes the Born radius of each atom for use with the various GB/SA solvation models</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BORN1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;born1&quot;
computes derivatives of the Born radii with respect to atomic coordinates and
increments total energy derivatives and virial components for potentials
involving Born radii</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BOUNDS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;bounds&quot;
finds the center of mass of each molecule, translates any stray molecules back
into the periodic box, and saves the offset of each atom relative to the
molecular center of mass</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BSET Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BSPLINE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;bspline&quot;
calculates the coefficients for an n-th order B-spline approximation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BSPLINE1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;bspline1&quot;
calculates the coefficients and derivative coefficients for an n-th order
B-spline approximation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BSSTEP Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CALENDAR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;calendar&quot;
returns the current time as a set of integer values representing the year,
month, day, hour, minute and second</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CELLATOM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;cellatom&quot;
completes the addition of a symmetry related atom to a unit cell by updating
the atom type and attachment arrays</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CENTER Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;center&quot;
moves the weighted centroid of each coordinate set to the origin during least
squares superposition</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CERROR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;cerror&quot;
is the error handling routine for the Connolly surface area and volume computation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CFFTB Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;cfftb&quot;
computes the backward complex discrete Fourier transform, the Fourier synthesis</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CFFTB1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CFFTF Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;cfftf&quot;
computes the forward complex discrete Fourier transform, the Fourier analysis</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CFFTF1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CFFTI Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;cffti&quot;
initializes the array &quot;wsave&quot; which is used in both forward and
backward transforms; the prime factorization of &quot;n&quot; together with a
tabulation of the trigonometric functions are computed and stored in &quot;wsave&quot;</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CFFTI1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CHIRER Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;chirer&quot;
computes the chirality error and its derivatives with respect to atomic
Cartesian coordinates as a sum the squares of deviations of chiral volumes
from target values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CHKSIZE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;chksize&quot;
computes a measure of overall global structural expansion or compaction from
the number of excess upper or lower bounds matrix violations</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CHKTREE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;chktree&quot;
tests a minimum energy structure to see if it belongs to the correct
progenitor in the existing map</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CHKXYZ Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;chkxyz&quot;
finds any pairs of atoms with identical Cartesian coordinates, and prints a
warning message</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CHOLESKY Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;cholesky&quot;
uses a modified Cholesky method to solve the linear system Ax = b, returning
&quot;x&quot; in &quot;b&quot;; &quot;A&quot; is assumed to be a real
symmetric positive definite matrix with diagonal and upper triangle stored by
rows in &quot;A&quot;; thus the actual size of the passed portion of
&quot;A&quot; is nvar*(nvar+1)/2</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CIRPLN Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CJKM Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CLIMBER Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CLIMBRGD Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CLIMBROT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CLIMBTOR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CLIMBXYZ Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CLOCK Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;clock&quot;
determines elapsed CPU time in seconds since the start of the job</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CLUSTER Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;cluster&quot;
gets the partitioning of the system into groups and stores a list of the group
to which each atom belongs</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">COLUMN Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;column&quot;
takes the off-diagonal Hessian elements stored as sparse rows and sets up
indices to allow column access</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">COMMAND Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;command&quot;
uses the standard Unix-like iargc/getarg routines to get the number and values
of arguments specified on the command line at program runtime</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">COMPRESS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;compress&quot;
transfers only the non-buried tori from the temporary tori arrays to the final
tori arrays</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CONNECT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;connect&quot;
sets up the attached atom arrays starting from a set of internal coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CONNOLLY Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;connolly&quot;
uses the algorithms from the AMS/VAM programs of Michael Connolly to compute
the analytical molecular surface area and volume of a collection of spherical
atoms; thus it implements Fred Richards' molecular surface definition as a set
of analytically defined spherical and toroidal polygons</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CONTACT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;contact&quot;
constructs the contact surface, cycles and convex faces</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CONTROL Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;control&quot;
gets initial values for parameters that determine the output style and
information level provided by TINKER</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">COORDS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;coords&quot;
converts the three principal eigenvalues/vectors from the metric matrix into
atomic coordinates, and calls a routine to compute the rms deviation from the bounds</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CORRELATE Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;correlate&quot;
computes the time correlation function of some user-supplied property from
individual snapshot frames taken from a molecular dynamics or other trajectory</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CRYSTAL Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;crystal&quot;
is a utility program which converts between fractional and Cartesian
coordinates, and can generate full unit cells from asymmetric units</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CUTOFFS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;cutoffs&quot;
initializes and stores spherical energy cutoff distance windows, Hessian
element and Ewald sum cutoffs, and the pairwise neighbor generation method</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">D1D2 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DELETE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;delete&quot;
removes a specified atom from the Cartesian coordinates list and shifts the
remaining atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DEPTH Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DFTMOD Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;dftmod&quot;
computes the modulus of the discrete Fourier transform of &quot;bsarray&quot;,
storing it into &quot;bsmod&quot;</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DIAGQ Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;diagq&quot;
is a matrix diagonalization routine which is derived from the classical given,
housec, and eigen algorithms with several modifications to increase the
efficiency and accuracy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DIFFEQ Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;diffeq&quot;
performs the numerical integration of an ordinary differential equation using
an adaptive stepsize method to solve the corresponding coupled first-order
equations of the general form dyi/dx = f(x,y1,...,yn) for yi = y1,...,yn</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DIFFUSE Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;diffuse&quot;
finds the self-diffusion constant for a homogeneous liquid via the Einstein
relation from a set of stored molecular dynamics frames; molecular centers of
mass are unfolded and mean squared displacements are computed versus time separation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DIST2 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;dist2&quot;
finds the distance squared between two points; used as a service routine by
the Connolly surface area and volume computation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DISTGEOM Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;distgeom&quot;
uses a metric matrix distance geometry procedure to generate structures with
interpoint distances that lie within specified bounds, with chiral centers
that maintain chirality, and with torsional angles restrained to desired
values; the user also has the ability to interactively inspect and alter the
triangle smoothed bounds matrix prior to embedding</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DMDUMP Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;dmdump&quot;
puts the distance matrix of the final structure into the upper half of a
matrix, the distance of each atom to the centroid on the diagonal, and the
individual terms of the bounds errors into the lower half of the matrix</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DOCUMENT Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;document&quot;
generates a formatted description of all the code modules or common blocks, a
listing of all valid keywords, a list of include file dependencies as needed
by a Unix-style Makefile, or a formatted force field parameter set summary</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DOT Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;dot&quot;
finds the dot product of two vectors</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DSTMAT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;dstmat&quot;
selects a distance matrix containing values between the previously smoothed
upper and lower bounds; the distance values are chosen from uniform
distributions, in a triangle correlated fashion, or using random partial metrization</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DYNAMIC Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;dynamic&quot;
computes a molecular dynamics trajectory in any of several statistical
mechanical ensembles with optional periodic boundaries and optional coupling
to temperature and pressure baths alternatively a stochastic dynamics
trajectory can be generated</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EANGANG Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eangang&quot;
calculates the angle-angle potential energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EANGANG1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eangang1&quot;
calculates the angle-angle potential energy and first derivatives with respect
to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EANGANG2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eangang2&quot;
calculates the angle-angle potential energy second derivatives with respect to
Cartesian coordinates using finite difference methods</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EANGANG2A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eangang2a&quot;
calculates the angle-angle first derivatives for a single interaction with
respect to Cartesian coordinates; used in computation of finite difference
second derivatives</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EANGANG3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eangang3&quot;
calculates the angle-angle potential energy; also partitions the energy among
the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EANGLE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eangle&quot;
calculates the angle bending potential energy; projected in-plane angles at
trigonal centers or Fourier angle bending terms are optionally used</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EANGLE1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eangle1&quot;
calculates the angle bending potential energy and the first derivatives with
respect to Cartesian coordinates; projected in-plane angles at trigonal
centers or Fourier angle bending terms are optionally used</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EANGLE2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eangle2&quot;
calculates second derivatives of the angle bending energy for a single atom
using a mixture of analytical and finite difference methods; projected
in-plane angles at trigonal centers or Fourier angle bending terms are
optionally used</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EANGLE2A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eangle2a&quot;
calculates bond angle bending potential energy second derivatives with respect
to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EANGLE2B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eangle2b&quot;
computes projected in-plane bending first derivatives for a single angle with
respect to Cartesian coordinates; used in computation of finite difference
second derivatives</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EANGLE3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eangle3&quot;
calculates the angle bending potential energy, also partitions the energy
among the atoms; projected in-plane angles at trigonal centers or Fourier
angle bending terms are optionally used</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EBOND Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ebond&quot;
calculates the bond stretching energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EBOND1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ebond1&quot;
calculates the bond stretching energy and first derivatives with respect to
Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EBOND2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ebond2&quot;
calculates second derivatives of the bond stretching energy for a single atom
at a time</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EBOND3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ebond3&quot;
calculates the bond stretching energy; also partitions the energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EBUCK Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ebuck&quot;
calculates the Buckingham exp-6 van der Waals energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EBUCK0A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ebuck0a&quot;
calculates the Buckingham exp-6 van der Waals energy using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EBUCK0B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ebuck0b&quot;
calculates the Buckingham exp-6 van der Waals energy using the method of
lights to locate neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EBUCK1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ebuck1&quot;
calculates the Buckingham exp-6 van der Waals energy and its first derivatives
with respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EBUCK1A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ebuck1a&quot;
calculates the Buckingham exp-6 van der Waals energy and its first derivatives
with respect to Cartesian coordinates using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EBUCK1B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ebuck1b&quot;
calculates the Buckingham exp-6 van der Waals energy and its first derivatives
with respect to Cartesian coordinates using the method of lights to locate
neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EBUCK2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ebuck2&quot;
calculates the Buckingham exp-6 van der Waals second derivatives for a single
atom at a time</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EBUCK3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ebuck3&quot;
calculates the Buckingham exp-6 van der Waals energy and partitions the energy
among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EBUCK3A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ebuck3a&quot;
calculates the Buckingham exp-6 van der Waals energy and partitions the energy
among the atoms using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EBUCK3B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ebuck3b&quot;
calculates the Buckingham exp-6 van der Waals energy and also partitions the
energy among the atoms using the method of lights to locate neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge&quot;
calculates the charge-charge interaction energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE0A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge0a&quot;
calculates the charge-charge interaction energy using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE0B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge0b&quot;
calculates the charge-charge interaction energy using the method of lights to
locate neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE0C Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge0c&quot;
calculates the charge-charge interaction energy for use with potential
smoothing methods</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE0D Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge0d&quot;
calculates the charge-charge interaction energy using a particle mesh Ewald summation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge1&quot;
calculates the charge-charge interaction energy and first derivatives with
respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE1A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge1a&quot;
calculates the charge-charge interaction energy and first derivatives with
respect to Cartesian coordinates using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE1B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge1b&quot;
calculates the charge-charge interaction energy and first derivatives with
respect to Cartesian coordinates using the method of lights to locate
neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE1C Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge1c&quot;
calculates the charge-charge interaction energy and first derivatives with
respect to Cartesian coordinates for use with potential smoothing methods</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE1D Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge1d&quot;
calculates the charge-charge interaction energy and first derivatives with
respect to Cartesian coordinates using a particle mesh Ewald summation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge2&quot;
calculates second derivatives of the charge-charge interaction energy for a
single atom</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE2A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge2a&quot;
calculates second derivatives of the charge-charge interaction energy for a
single atom using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE2B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge2b&quot;
calculates second derivatives of the charge-charge interaction energy for a
single atom for use with potential smoothing methods</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE2C Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge2c&quot;
calculates second derivatives of the charge-charge interaction energy for a
single atom using a particle mesh Ewald summation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge3&quot;
calculates the charge-charge interaction energy and partitions the energy
among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE3A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge3a&quot;
calculates the charge-charge interaction energy and partitions the energy
among the atoms using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE3B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge3b&quot;
calculates the charge-charge interaction energy and partitions the energy
among the atoms using the method of lights to locate neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE3C Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge3c&quot;
calculates the charge-charge interaction energy and partitions the energy
among the atoms for use with potential smoothing methods</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHARGE3D Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echarge3d&quot;
calculates the charge-charge interaction energy and partitions the energy
among the atoms using a particle mesh Ewald summation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHGDPL Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echgdpl&quot;
calculates the charge-dipole interaction energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHGDPL1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echgdpl1&quot;
calculates the charge-dipole interaction energy and first derivatives with
respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHGDPL2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echgdpl2&quot;
calculates second derivatives of the charge-dipole interaction energy for a
single atom</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ECHGDPL3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;echgdpl3&quot;
calculates the charge-dipole interaction energy; also partitions the energy
among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EDIPOLE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;edipole&quot;
calculates the dipole-dipole interaction energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EDIPOLE1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;edipole1&quot;
calculates the dipole-dipole interaction energy and first derivatives with
respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EDIPOLE2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;edipole2&quot;
calculates second derivatives of the dipole-dipole interaction energy for a
single atom</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EDIPOLE3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;edipole3&quot;
calculates the dipole-dipole interaction energy; also partitions the energy
among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGAUSS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egauss&quot;
calculates the Gaussian expansion van der Waals interaction energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGAUSS1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egauss1&quot;
calculates the Gaussian expansion van der Waals interaction energy and its
first derivatives with respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGAUSS2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egauss2&quot;
calculates the Gaussian expansion van der Waals second derivatives for a
single atom at a time</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGAUSS3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egauss3&quot;
calculates the Gaussian expansion van der Waals interaction energy and
partitions the energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGBSA0A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egbsa0a&quot;
calculates the generalized Born polarization energy for the GB/SA and ACE
solvation models</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGBSA0B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egbsa0b&quot;
calculates the generalized Born polarization energy for the GB/SA and ACE
solvation models for use with potential smoothing methods via analogy to the
smoothing of Coulomb's law</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGBSA1A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egbsa1a&quot;
calculates the generalized Born energy and first derivatives with respect to
Cartesian coordinates for the GB/SA solvation models</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGBSA1B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egbsa1b&quot;
calculates the generalized Born energy and first derivatives with respect to
Cartesian coordinates for the GB/SA solvation models</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGBSA2A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egbsa2a&quot;
calculates second derivatives of the generalized Born energy term for the
GB/SA solvation models</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGBSA2B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egbsa2b&quot;
calculates second derivatives of the generalized Born energy term for the
GB/SA solvation models via analogy to the smoothing of Coulomb's law</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGBSA3A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egbsa3a&quot;
calculates the generalized Born energy term for the GB/SA and ACE solvation
models; also partitions the energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGBSA3B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egbsa3b&quot;
calculates the generalized Born polarization energy for the GB/SA and ACE
solvation models for use with potential smoothing methods via analogy to the
smoothing of Coulomb's law; also partitions the energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGEOM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egeom&quot;
calculates the energy due to restraints on positions, distances, angles and
torsions as well as Gaussian basin and spherical droplet restraints</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGEOM1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egeom1&quot;
calculates the energy and first derivatives with respect to Cartesian
coordinates due to restraints on positions, distances, angles and torsions as
well as Gaussian basin and spherical droplet restraints</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGEOM2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egeom2&quot;
calculates second derivatives of restraints on positions, distances, angles
and torsions as well as Gaussian basin and spherical droplet restraints</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EGEOM3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;egeom3&quot;
calculates the energy due to restraints on positions, distances, angles and
torsions as well as Gaussian basin and droplet restraints; also partitions
energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EHAL Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ehal&quot;
calculates the buffered 14-7 van der Waals energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EHAL0A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ehal0a&quot;
calculates the buffered 14-7 van der Waals energy using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EHAL0B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ehal0a&quot;
calculates the buffered 14-7 van der Waals energy using the method of lights
to locate neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EHAL1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ehal1&quot;
calculates the buffered 14-7 van der Waals energy and its first derivatives
with respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EHAL1A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ehal1a&quot;
calculates the buffered 14-7 van der Waals energy and its first derivatives
with respect to Cartesian coordinates using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EHAL1B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ehal1b&quot;
calculates the buffered 14-7 van der Waals energy and its first derivatives
with respect to Cartesian coordinates using the method of lights to locate
neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EHAL2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ehal2&quot;
calculates the buffered 14-7 van der Waals second derivatives for a single
atom at a time</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EHAL3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ehal3&quot;
calculates the buffered 14-7 van der Waals energy and partitions the energy
among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EHAL3A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ehal3a&quot;
calculates the buffered 14-7 van der Waals energy and partitions the energy
among the atoms using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EHAL3B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ehal3b&quot;
calculates the buffered 14-7 van der Waals energy and also partitions the
energy among the atoms using the method of lights to locate neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EIGEN Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eigen&quot;
uses the power method to compute the largest eigenvalues and eigenvectors of
the metric matrix, &quot;valid&quot; is set true if the first three
eigenvalues are positive</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EIGENRGD Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EIGENROT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EIGENROT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EIGENTOR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EIGENXYZ Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EIMPROP Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eimprop&quot;
calculates the improper dihedral potential energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EIMPROP1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eimprop1&quot;
calculates improper dihedral energy and its first derivatives with respect to
Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EIMPROP2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eimprop2&quot;
calculates second derivatives of the improper dihedral angle energy for a
single atom</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EIMPROP3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eimprop3&quot;
calculates the improper dihedral potential energy; also partitions the energy
terms among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EIMPTOR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eimptor&quot;
calculates the improper torsion potential energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EIMPTOR1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eimptor1&quot;
calculates improper torsion energy and its first derivatives with respect to
Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EIMPTOR2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eimptor2&quot;
calculates second derivatives of the improper torsion energy for a single atom</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EIMPTOR3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eimptor3&quot;
calculates the improper torsion potential energy; also partitions the energy
terms among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ELJ Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;elj&quot;
calculates the Lennard-Jones 6-12 van der Waals energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ELJ0A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;elj0a&quot;
calculates the Lennard-Jones 6-12 van der Waals energy using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ELJ0B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;elj0b&quot;
calculates the Lennard-Jones 6-12 van der Waals energy using the method of
lights to locate neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ELJ1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;elj1&quot;
calculates the Lennard-Jones 6-12 van der Waals energy and its first
derivatives with respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ELJ1A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;elj1a&quot;
calculates the Lennard-Jones 6-12 van der Waals energy and its first
derivatives with respect to Cartesian coordinates using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ELJ1B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;elj1b&quot;
calculates the Lennard-Jones 6-12 van der Waals energy and its first
derivatives with respect to Cartesian coordinates using the method of lights
to locate neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ELJ2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;elj2&quot;
calculates the Lennard-Jones 6-12 van der Waals second derivatives for a
single atom at a time</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ELJ3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;elj3&quot;
calculates the Lennard-Jones 6-12 van der Waals energy and also partitions the
energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ELJ3A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;elj3a&quot;
calculates the Lennard-Jones 6-12 van der Waals energy and also partitions the
energy among the atoms using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ELJ3B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;elj3b&quot;
calculates the Lennard-Jones 6-12 van der Waals energy and also partitions the
energy among the atoms using the method of lights to locate neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMBED Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;embed&quot;
is a distance geometry routine patterned after the ideas of Gordon Crippen,
Irwin Kuntz and Tim Havel; it takes as input a set of upper and lower bounds
on the interpoint distances, chirality restraints and torsional restraints,
and attempts to generate a set of coordinates that satisfy the input bounds
and restraints</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMETAL Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;emetal&quot;
calculates the transition metal ligand field energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMETAL1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;emetal1&quot;
calculates the transition metal ligand field energy and its first derivatives
with respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMETAL2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;emetal2&quot;
calculates the transition metal ligand field second derivatives for a single
atom at a time</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMETAL3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;emetal3&quot;
calculates the transition metal ligand field energy and also partitions the
energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMM3HB Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;emm3hb&quot;
calculates the MM3 exp-6 van der Waals and directional charge transfer
hydrogen bonding energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMM3HB0A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;emm3hb0a&quot;
calculates the MM3 exp-6 van der Waals and directional charge transfer
hydrogen bonding energy using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMM3HB0B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;emm3hb0b&quot;
calculates the MM3 exp-6 van der Waals and directional charge transfer
hydrogen bonding energy using the method of lights to locate neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMM3HB1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;emm3hb1&quot;
calculates the MM3 exp-6 van der Waals and directional charge transfer
hydrogen bonding energy with respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMM3HB1A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;emm3hb1a&quot;
calculates the MM3 exp-6 van der Waals and directional charge transfer
hydrogen bonding energy with respect to Cartesian coordinates using a pairwise
double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMM3HB1B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;emm3hb1b&quot;
calculates the MM3 exp-6 van der Waals and directional charge transfer
hydrogen bonding energy with respect to Cartesian coordinates using the method
of lights to locate neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMM3HB2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;emm3hb2&quot;
calculates the MM3 exp-6 van der Waals and directional charge transfer
hydrogen bonding second derivatives for a single atom at a time</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMM3HB3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;emm3hb3&quot;
calculates the MM3 exp-6 van der Waals and directional charge transfer
hydrogen bonding energy, and partitions the energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMM3HB3A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;emm3hb3&quot;
calculates the MM3 exp-6 van der Waals and directional charge transfer
hydrogen bonding energy, and partitions the energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMM3HB3B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;emm3hb3b&quot;
calculates the MM3 exp-6 van der Waals and directional charge transfer
hydrogen bonding energy using the method of lights to locate neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMPOLE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;empole&quot;
calculates the electrostatic energy due to atomic multipole interactions and
dipole polarizability</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMPOLE0A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;empole0a&quot;
calculates the electrostatic energy due to atomic multipole interactions and
dipole polarizability using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMPOLE0B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;empole0b&quot;
calculates the electrostatic energy due to atomic multipole interactions and
dipole polarizability using a regular Ewald summation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMPOLE1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;empole1&quot;
calculates the multipole and dipole polarization energy and derivatives with
respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMPOLE1A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;empole1a&quot;
calculates the multipole and dipole polarization energy and derivatives with
respect to Cartesian coordinates using a pairwise double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMPOLE1B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;empole1b&quot;
calculates the multipole and dipole polarization energy and derivatives with
respect to Cartesian coordinates using a regular Ewald summation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMPOLE2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;empole2&quot;
calculates second derivatives of the multipole and dipole polarization energy
for a single atom at a time</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMPOLE2A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;empole2a&quot;
computes multipole and dipole polarization first derivatives for a single atom
with respect to Cartesian coordinates; used to get finite difference second derivatives</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMPOLE3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;empole3&quot;
calculates the electrostatic energy due to atomic multipole interactions and
dipole polarizability, and partitions the energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMPOLE3A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;empole3a&quot;
calculates the electrostatic energy due to atomic multipole interactions and
dipole polarizability, and partitions the energy among the atoms using a
double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EMPOLE3B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;empole3b&quot;
calculates the electrostatic energy due to atomic multipole interactions and
dipole polarizability, and partitions the energy among the atoms using a
regular Ewald summation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ENERGY Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;energy&quot;
calls the subroutines to calculate the potential energy terms and sums up to
form the total energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ENRGYZE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;energyze&quot;
is an auxiliary routine for the analyze program that performs the energy
analysis and prints the total and intermolecular energies</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EOPBEND Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eopbend&quot;
computes the out-of-plane bend potential energy at trigonal centers via a
Wilson-Decius-Cross angle bend</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EOPBEND1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eopbend1&quot;
computes the out-of-plane bend potential energy and first derivatives at
trigonal centers via a Wilson-Decius-Cross angle bend</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EOPBEND2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eopbend2&quot;
calculates second derivatives of the out-of-plane bend energy via a
Wilson-Decius-Cross angle bend for a single atom using finite difference methods</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EOPBEND2A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eopbend2a&quot;
calculates out-of-plane bending first derivatives at a trigonal center via a
Wilson-Decius-Cross angle bend; used in computation of finite difference
second derivatives</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EOPBEND3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eopbend3&quot;
computes the out-of-plane bend potential energy at trigonal centers via a
Wilson-Decius-Cross angle bend; also partitions the energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EOPDIST Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eopdist&quot;
computes the out-of-plane distance potential energy at trigonal centers via
the central atom height</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EOPDIST1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eopdist1&quot;
computes the out-of-plane distance potential energy and first derivatives at
trigonal centers via the central atom height</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EOPDIST2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eopdist2&quot;
calculates second derivatives of the out-of-plane distance energy for a single
atom via the central atom height</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EOPDIST3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eopdist3&quot;
computes the out-of-plane distance potential energy at trigonal centers via
the central atom height; also partitions the energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EPME Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;epme&quot;
computes the reciprocal space energy for a particle mesh Ewald summation over
partial charges</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EPME1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;epme1&quot;
computes the reciprocal space energy and first derivatives for a particle mesh
Ewald summation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EPME3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;epme3&quot;
computes the reciprocal space energy for a particle mesh Ewald summation over
partial charges and prints information about the energy over the charge grid points</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EPUCLC Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EREAL Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ereal&quot;
evaluates the real space portion of the regular Ewald summation energy due to
atomic multipole interactions and dipole polarizability</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EREAL1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ereal1&quot;
evaluates the real space portion of the regular Ewald summation energy and
gradient due to atomic multipole interactions and dipole polarizability</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EREAL3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ereal3&quot;
evaluates the real space portion of the regular Ewald summation energy due to
atomic multipole interactions and dipole polarizability and partitions the
energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ERECIP Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;erecip&quot;
evaluates the reciprocal space portion of the regular Ewald summation energy
due to atomic multipole interactions and dipole polarizability</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ERECIP1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;erecip1&quot;
evaluates the reciprocal space portion of the regular Ewald summation energy
and gradient due to atomic multipole interactions and dipole polarizability</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ERECIP3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;erecip3&quot;
evaluates the reciprocal space portion of the regular Ewald summation energy
due to atomic multipole interactions and dipole polarizability, and prints
information about the energy over the reciprocal lattice vectors</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ERF Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;erf&quot;
computes a numerical approximation to the value of the error function via a
Chebyshev approximation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ERFC Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;erfc&quot;
computes a numerical approximation to the value of the complementary error
function via a Chebyshev approximation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ERFCORE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;erfcore&quot;
evaluates erf(x) or erfc(x) for a real argument x; when called with mode set
to 0 it returns erf, a mode of 1 returns erfc; uses rational functions that
approximate erf(x) and erfc(x) to at least 18 significant decimal digits</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ERFIK Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ERFINV Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;erfinv&quot;
evaluates the inverse of the error function erf for a real argument in the
range (-1,1) using a rational function approximation followed by cycles of
Newton-Raphson correction</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ERXNFLD Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;erxnfld&quot;
calculates the macroscopic reaction field energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ERXNFLD1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;erxnfld1&quot;
calculates the macroscopic reaction field energy and derivatives with respect
to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ERXNFLD2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;erxnfld2&quot;
calculates second derivatives of the macroscopic reaction field energy for a
single atom at a time</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ERXNFLD3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;erxnfld3&quot;
calculates the macroscopic reaction field energy, and also partitions the
energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ESOLV Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;esolv&quot;
calculates the continuum solvation energy via either the Eisenberg-McLachlan
ASP model, Ooi-Scheraga SASA model, various GB/SA methods or the ACE model</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ESOLV1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;esolv1&quot;
calculates the continuum solvation energy and first derivatives with respect
to Cartesian coordinates using either the Eisenberg-McLachlan ASP,
Ooi-Scheraga SASA or various GB/SA solvation models</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ESOLV2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;esolv2&quot;
calculates second derivatives of the continuum solvation energy using either
the Eisenberg-McLachlan ASP, Ooi-Scheraga SASA or various GB/SA solvation models</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ESOLV3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;esolv3&quot;
calculates the continuum solvation energy using either the Eisenberg-McLachlan
ASP model, Ooi-Scheraga SASA model, various GB/SA methods or the ACE model;
also partitions the energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ESTRBND Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;estrbnd&quot;
calculates the stretch-bend potential energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ESTRBND1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;estrbnd1&quot;
calculates the stretch-bend potential energy and first derivatives with
respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ESTRBND2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;estrbnd2&quot;
calculates the stretch-bend potential energy second derivatives with respect
to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ESTRBND3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;estrbnd3&quot;
calculates the stretch-bend potential energy; also partitions the energy among
the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ESTRTOR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;estrtor&quot;
calculates the stretch-torsion potential energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ESTRTOR1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;estrtor1&quot;
calculates the stretch-torsion energy and first derivatives with respect to
Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ESTRTOR2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;estrtor2&quot;
calculates the stretch-torsion potential energy second derivatives with
respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ESTRTOR3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;estrtor3&quot;
calculates the stretch-torsion potential energy; also partitions the energy
terms among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ETORS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;etors&quot;
calculates the torsional potential energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ETORS1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;etors1&quot;
calculates torsional potential energy and first derivatives with respect to
Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ETORS2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;etors2&quot;
calculates second derivatives of the torsional energy for a single atom</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ETORS3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;etors3&quot;
calculates the torsional potential energy; also partitions the energy terms
among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EUREY Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eurey&quot;
calculates the Urey-Bradley 1-3 interaction energy</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EUREY1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eurey1&quot;
calculates the Urey-Bradley interaction energy and its first derivatives with
respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EUREY2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eurey2&quot;
calculates second derivatives of the Urey-Bradley interaction energy for a
single atom at a time</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EUREY3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;eurey3&quot;
calculates the Urey-Bradley energy; also partitions the energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EWALDCOF Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ewaldcof&quot;
finds a value of the Ewald coefficient such that all terms beyond the
specified cutoff distance will have an value less than a specified tolerance</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EXPLORE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;explore&quot;
uses simulated annealing on an initial crude embedded distance geoemtry
structure to refine versus the bound, chirality, planarity and torsional error functions</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EXTRA Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;extra&quot;
calculates any additional user defined potential energy contribution</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EXTRA1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;extra1&quot;
calculates any additional user defined potential energy contribution and its
first derivatives</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EXTRA2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;extra2&quot;
calculates second derivatives of any additional user defined potential energy
contribution for a single atom at a time</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EXTRA3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;extra3&quot;
calculates any additional user defined potential contribution and also
partitions the energy among the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">FATAL Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;fatal&quot;
terminates execution due to a user request, a severe error or some other
nonstandard condition</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">FFTBACK Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">FFTFRONT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">FFTSETUP Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">FIELD Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;field&quot;
sets the force field potential energy functions from a parameter file and
modifications specified in a keyfile</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">FINAL Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;final&quot;
performs any final program actions, prints a status message, and then pauses
if necessary to avoid closing the execution window</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">FINDATM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;findatm&quot;
locates a specific PDB atom name type within a range of atoms from the PDB
file, returns zero if the name type was not found</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">FIXPDB Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;fixpdb&quot;
corrects problems with PDB files by converting residue and atom names to the
forms used by TINKER</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">FRACDIST Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;fracdist&quot;
computes a normalized distribution of the pairwise fractional distances
between the smoothed upper and lower bounds</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">FREEUNIT Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;freeunit&quot;
finds an unopened Fortran I/O unit and returns its numerical value from 1 to
99; the units already assigned to &quot;input&quot; and &quot;iout&quot;
(usually 5 and 6) are skipped since they have special meaning as the default
I/O units</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GAMMLN Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;gammln&quot;
uses a series expansion due to Lanczos to compute the natural logarithm of the
Gamma function at &quot;x&quot; in [0,1]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GDA Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;gda&quot;
implements Gaussian Density Annealing (GDA) algorithm for global optimization
via simulated annealing</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GDA1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GDA2 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GDA3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GDASTAT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GENDOT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;gendot&quot;
finds the coordinates of a specified number of surface points for a sphere
with the input radius and coordinate center</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GEODESIC Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;geodesic&quot;
smooths the upper and lower distance bounds via the triangle inequality using
a sparse matrix version of a shortest path algorithm</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GEOMETRY Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;geometry&quot;
finds the value of the interatomic distance, angle or dihedral angle defined
by two to four input atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETBASE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getbase&quot;
finds the base heavy atoms for a single nucleotide residue and copies the
names and coordinates to the Protein Data Bank file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETIME Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getime&quot;
gets elapsed CPU time in seconds for an interval</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETINT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getint&quot;
asks for an internal coordinate file name, then reads the internal coordinates
and computes Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETKEY Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getkey&quot;
finds a valid keyfile and stores its contents as line images for subsequent
keyword parameter searching</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETMOL2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getmol2&quot;
asks for a Sybyl MOL2 molecule file name, then reads the coordinates from the file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETNUCH Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getnuch&quot;
finds the nucleotide hydrogen atoms for a single residue and copies the names
and coordinates to the Protein Data Bank file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETNUMB Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getnumb&quot;
searchs an input string from left to right for an integer and puts the numeric
value in &quot;number&quot;; returns zero with &quot;next&quot; unchanged if
no integer value is found</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETPDB Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getpdb&quot;
asks for a Protein Data Bank file name, then reads in the coordinates file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETPRB Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getprb&quot;
tests for a possible probe position at the interface between three neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETPRM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getprm&quot;
finds the potential energy parameter file and then opens and reads the parameters</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETPROH Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getproh&quot;
finds the hydrogen atoms for a single amino acid residue and copies the names
and coordinates to the Protein Data Bank file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETREF Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getref&quot;
copies structure information from the reference area into the standard
variables for the current system structure</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETSEQ Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getseq&quot;
asks the user for the amino acid sequence and torsional angle values needed to
define a peptide</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETSEQN Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getseqn&quot;
asks the user for the nucleotide sequence and torsional angle values needed to
define a nucleic acid</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETSIDE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getside&quot;
finds the side chain heavy atoms for a single amino acid residue and copies
the names and coordinates to the Protein Data Bank file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETSTRING Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getstring&quot;
searchs for a quoted text string within an input character string; the region
between the first and second quotes is returned as the &quot;text&quot;; if
the actual text is too long, only the first part is returned</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETTEXT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;gettext&quot;
searchs an input string for the first string of non-blank characters; the
region from a non-blank character to the first blank space is returned as
&quot;text&quot;; if the actual text is too long, only the first part is returned</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETTOR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;gettor&quot;
tests for a possible torus position at the interface between two atoms, and
finds the torus radius, center and axis</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETWORD Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getword&quot;
searchs an input string for the first alphabetic character (A-Z or a-z); the
region from this first character to the first blank space or comma is returned
as a &quot;word&quot;; if the actual word is too long, only the first part is returned</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GETXYZ Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;getxyz&quot;
asks for a Cartesian coordinate file name, then reads in the coordinates file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GRADIENT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;gradient&quot;
calls subroutines to calculate the potential energy and first derivatives with
respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GRADRGD Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;gradrgd&quot;
calls subroutines to calculate the potential energy and first derivatives with
respect to rigid body coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GRADROT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;gradrot&quot;
calls subroutines to calculate the potential energy and its torsional first derivatives</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GRAFIC Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;grafic&quot;
outputs the upper &amp; lower triangles and diagonal of a square matrix in a
schematic form for visual inspection</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GROUPS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;groups&quot;
tests a set of atoms to see if all are members of a single atom group or a
pair of atom groups; if so, then the correct intra- or intergroup weight is assigned</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GRPLINE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;grpline&quot;
tests each atom group for linearity of the sites contained in the group</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GYRATE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;gyrate&quot;
computes the radius of gyration of a molecular system from its atomic coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HANGLE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;hangle&quot;
constructs hybrid angle bending parameters given an initial state, final state
and &quot;lambda&quot; value</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HATOM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;hatom&quot;
assigns a new atom type to each hybrid site</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HBOND Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;hbond&quot;
constructs hybrid bond stretch parameters given an initial state, final state
and &quot;lambda&quot; value</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HCHARGE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;hcharge&quot;
constructs hybrid charge interaction parameters given an initial state, final
state and &quot;lambda&quot; value</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HDIPOLE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;hdipole&quot;
constructs hybrid dipole interaction parameters given an initial state, final
state and &quot;lambda&quot; value</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HESSIAN Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;hessian&quot;
calls subroutines to calculate the Hessian elements for each atom in turn with
respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HESSRGD Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;hessrgd&quot;
computes the numerical Hessian elements with respect to rigid body
coordinates; either the full matrix or just the diagonal can be calculated;
the full matrix needs 6*ngroup+1 gradient evaluations while the diagonal
requires just two gradient calls</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HESSROT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;hessrot&quot;
computes the numerical Hessian elements with respect to torsional angles;
either the full matrix or just the diagonal can be calculated; the full matrix
needs nomega+1 gradient evaluations while the diagonal requires just two
gradient calls</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HIMPTOR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;himptor&quot;
constructs hybrid improper torsional parameters given an initial state, final
state and &quot;lambda&quot; value</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HSTRBND Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;hstrbnd&quot;
constructs hybrid stretch-bend parameters given an initial state, final state
and &quot;lambda&quot; value</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HSTRTOR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;hstrtor&quot;
constructs hybrid stretch-torsion parameters given an initial state, final
state and &quot;lambda&quot; value</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HTORS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;htors&quot;
constructs hybrid torsional parameters for a given initial state, final state
and &quot;lambda&quot; value</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HVDW Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;hvdw&quot;
constructs hybrid van der Waals  parameters given an initial state, final
state and &quot;lambda&quot; value</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HYBRID Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;hybrid&quot;
constructs the hybrid hamiltonian for a specified initial state, final state
and mutation parameter &quot;lambda&quot;</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">IJK_PT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">IMAGE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;image&quot;
takes the components of pairwise distance between two points in the same or
neighboring periodic boxes and converts to the components of the minimum image distance</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">IMPOSE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;impose&quot;
performs the least squares best superposition of two atomic coordinate sets
via a quaternion method; upon return, the first coordinate set is unchanged
while the second set is translated and rotated to give best fit; the final
root mean square fit is returned in &quot;rmsvalue&quot;</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INDUCE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;induce&quot;
computes the induced dipole moment at each polarizable site due to direct or
mutual polarization</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INDUCE0A Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;induce0a&quot;
computes the induced dipole moment at each polarizable site using a pairwise
double loop</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INDUCE0B Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;induce0b&quot;
computes the induced dipole moment at each polarizable site using a regular
Ewald summation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INEDGE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;inedge&quot;
inserts a concave edge into the linked list for its temporary torus</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INERTIA Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;inertia&quot;
computes the principal moments of inertia for the system, and optionally
translates the center of mass to the origin and rotates the principal axes
onto the global axes</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INITERR Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;initerr&quot;
is the initial error function and derivatives for a distance geometry
embedding; it includes components from the local geometry and torsional
restraint errors</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INITIAL Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;initial&quot;
sets up original values for some parameters and variables that might not
otherwise get initialized</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INITPRM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;initprm&quot;
completely initializes a force field by setting all parameters to zero and
using defaults for control values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INITRES Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;initres&quot;
sets names for biopolymer residue types used in PDB file conversion and
automated generation of structures</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INITROT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;initrot&quot;
asks for torsional angles which are to be rotated in subsequent computation,
it will automatically locate all rotatable single bonds if desired; assumes
that an appropriate internal coordinates file has already been read in</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INSERT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;insert&quot;
adds the specified atom to the Cartesian coordinates list and shifts the
remaining atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INTEDIT Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;intedit&quot;
allows the user to extract information from or alter the values within an
internal coordinates file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INTXYZ Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;intxyz&quot;
takes as input an internal coordinates file, converts to and then writes out
Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INVBETA Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;invbeta&quot;
computes the inverse Beta distribution function via a combination of Newton
iteration and bisection search</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INVERT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;invert&quot;
inverts a matrix using the Gauss-Jordan method</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">IPEDGE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ipedge&quot;
inserts convex edge into linked list for atom</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">JACOBI Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;jacobi&quot;
performs a matrix diagonalization of a real symmetric matrix by the method of
Jacobi rotations</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KANGANG Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kangang&quot;
assigns the parameters for angle-angle cross term interactions and processes
new or changed parameter values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KANGLE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kangle&quot;
assigns the force constants and ideal angles for the bond angles; also
processes new or changed parameters</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KATOM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;katom&quot;
assigns an atom type definitions to each atom in the structure and processes
any new or changed values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KBOND Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kbond&quot;
assigns a force constant and ideal bond length to each bond in the structure
and processes any new or changed parameter values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KCHARGE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kcharge&quot;
assigns partial charges to the atoms within the structure and processes any
new or changed values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KCHIRAL Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kchiral&quot;
determines the target value for each chirality and planarity restraint as the
signed volume of the parallelpiped spanned by vectors from a common atom to
each of three other atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KDIPOLE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kdipole&quot;
assigns bond dipoles to the bonds within the structure and processes any new
or changed values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KENEG Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;keneg&quot;
applies primary and secondary electronegativity bond length corrections to
applicable bond parameters</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KEWALD Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kewald&quot;
assigns both regular Ewald summation and particle mesh Ewald parameters for a
periodic box</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KGEOM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kgeom&quot;
asisgns parameters for geometric restraint terms to be included in the
potential energy calculation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KIMPROP Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kimprop&quot;
assigns potential parameters to each improper dihedral in the structure and
processes any changed values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KIMPTOR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kimptor&quot;
assigns torsional parameters to each improper torsion in the structure and
processes any changed values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KMETAL Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kmetal&quot;
assigns ligand field parameters to transition metal atoms and processes any
new or changed parameter values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KMPOLE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kmpole&quot;
assigns atomic multipole moments to the atoms of the structure and processes
any new or changed values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KOPBEND Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kopbend&quot;
assigns the force constants for out-of-plane bending at trigonal centers via
Wilson-Decius-Cross angle bends; also processes any new or changed parameter values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KOPDIST Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kopdist&quot;
assigns the force constants for out-of-plane distance at trigonal centers via
the central atom height; also processes any new or changed parameter values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KORBIT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;korbit&quot;
assigns pi-orbital parameters to conjugated systems and processes any new or
changed parameters</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KPOLAR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kpolar&quot;
assigns atomic dipole polarizabilities to the atoms within the structure and
processes any new or changed values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KSOLV Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ksolv&quot;
assigns continuum solvation energy parameters for the Eisenberg-McLachlan ASP,
Ooi-Scheraga SASA or various GB/SA solvation models</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KSTRBND Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kstrbnd&quot;
assigns the parameters for the stretch-bend interactions and processes new or
changed parameter values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KSTRTOR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kstrtor&quot;
assigns stretch-torsion parameters to torsions needing them, and processes any
new or changed values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KTORS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ktors&quot;
assigns torsional parameters to each torsion in the structure and processes
any new or changed values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KUREY Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kurey&quot;
assigns the force constants and ideal distances for the Urey-Bradley 1-3
interactions; also processes any new or changed parameter values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KVDW Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;kvdw&quot;
assigns the parameters to be used in computing the van der Waals interactions
and processes any new or changed values for these parameters</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">LATTICE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;lattice&quot;
stores the periodic box dimensions and sets angle values to be used in
computing fractional coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">LBFGS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;lbfgs&quot;
is a limited memory BFGS quasi-newton nonlinear optimization routine</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">LIGASE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ligase&quot;
translates a nucleic acid structure in Protein Data Bank format to a Cartesian
coordinate file and sequence file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">LIGHTS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;lights&quot;
computes the set of nearest neighbor interactions using the method of lights algorithm</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">LINBODY Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;linbody&quot;
finds the angular velocity of a linear rigid body</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">LMSTEP Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;lmstep&quot;
computes the Levenberg-Marquardt step during a nonlinear least squares
calculation; this version is based upon ideas from the Minpack routine LMPAR
together with with the internal doubling strategy of Dennis and Schnabel</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">LOCALMIN Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;localmin&quot;
is used during normal mode local search to perform a Cartesian coordinate
energy minimization</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">LOCALRGD Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;localrgd&quot;
is used during the PSS local search procedure to perform a rigid body energy minimization</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">LOCALROT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;localrot&quot;
is used during the PSS local search procedure to perform a torsional space
energy minimization</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">LOCALXYZ Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;localxyz&quot;
is used during the potential smoothing and search procedure to perform a local
optimization at the current smoothing level</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">LOCERR Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;locerr&quot;
is the local geometry error function and derivatives including the 1-2, 1-3
and 1-4 distance bound restraints</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">LOWCASE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;lowcase&quot;
converts a text string to all lower case letters</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MAJORIZE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;majorize&quot;
refines the projected coordinates by attempting to minimize the least square
residual between the trial distance matrix and the distances computed from the coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MAKEINT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;makeint&quot;
converts Cartesian to internal coordinates where selection of internal
coordinates is controlled by &quot;mode&quot;</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MAKEPDB Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;makexyz&quot;
converts a set of Cartesian coordinates to Protein Data Bank format with
special handling for systems consisting of polypeptide chains, ligands and
water molecules</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MAKEREF Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;makeref&quot;
copies the information contained in the &quot;xyz&quot; file of the current
structure into corresponding reference areas</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MAKEXYZ Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;makexyz&quot;
generates a complete set of Cartesian coordinates for a full structure from
the internal coordinate values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MAPCHECK Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;mapcheck&quot;
checks the current minimum energy structure for possible addition to the
master list of local minima</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MAXWELL Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;maxwell&quot;
returns a speed in Angstroms/picosecond randomly selected from a 3-D
Maxwell-Boltzmann distribution for the specified particle mass and system temperature</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MDINIT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;mdinit&quot;
initializes the velocities and accelerations for a molecular dynamics
trajectory, including restarts</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MDREST Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;mdrest&quot;
finds and removes any translational or rotational kinetic energy of the
overall system center of mass</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MDSAVE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;mdsave&quot;
writes molecular dynamics trajectory snapshots and auxiliary files with
velocity and induced dipole information; also checks for user requested
termination of a simulation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MDSTAT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;mdstat&quot;
is called at each molecular dynamics time step to form statistics on various
average values and fluctuations, and to periodically save the state of the trajectory</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MEASFN Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MEASFP Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MEASFS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MEASPM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;measpm&quot;
computes the volume of a single prism section of the full interior polyhedron</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MECHANIC Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;mechanic&quot;
sets up needed parameters for the potential energy calculation and reads in
many of the user selectable options</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MERGE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;merge&quot;
combines the reference and current structures into a single new
&quot;current&quot; structure containing the reference atoms followed by the
atoms of the current structure</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">METRIC Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;metric&quot;
takes as input the trial distance matrix and computes the metric matrix of all
possible dot products between the atomic vectors and the center of mass using
the law of cosines and the following formula for the distances to the center
of mass:</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MIDERR Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;miderr&quot;
is the secondary error function and derivatives for a distance geometry
embedding; it includes components from the distance bounds, local geometry,
chirality and torsional restraint errors</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MINIMIZ1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;minimiz1&quot;
is a service routine that computes the energy and gradient for a low storage
BFGS optimization in Cartesian coordinate space</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MINIMIZE Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;minimize&quot;
performs energy minimization in Cartesian coordinate space using a low storage
BFGS nonlinear optimization</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MINIROT Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;minirot&quot;
performs an energy minimization in torsional angle space using a low storage
BFGS nonlinear optimization</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MINIROT1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;minirot1&quot;
is a service routine that computes the energy and gradient for a low storage
BFGS nonlinear optimization in torsional angle space</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MINPATH Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;minpath&quot;
is a routine for finding the triangle smoothed upper and lower bounds of each
atom to a specified root atom using a sparse variant of the Bellman-Ford
shortest path algorithm</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MINRIGID Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;minrigid&quot;
performs an energy minimization of rigid body atom groups using a low storage
BFGS nonlinear optimization</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MINRIGID1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;minrigid1&quot;
is a service routine that computes the energy and gradient for a low storage
BFGS nonlinear optimization of rigid bodies</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MMID Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;mmid&quot;
implements a modified midpoint method to advance the integration of a set of
first order differential equations</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MODECART Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MODEROT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MODESRCH Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MODETORS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MODULI Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;moduli&quot;
sets the moduli of the inverse discrete Fourier transform of the B-splines;
bsmod[1-3] hold these values, nfft[1-3] are the grid dimensions, bsorder is
the order of B-spline approximation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MOLECULE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;molecule&quot;
counts the molecules, assigns each atom to its molecule and computes the mass
of each molecule</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MOLUIND Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;moluind&quot;
computes the molecular induced dipole components in the presence of an
external electric field</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MOMENTS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;moments&quot;
computes the total electric charge, dipole and quadrupole moments for the
entire system as a sum over the partial charges, bond dipoles and atomic
multipole moments</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MUTATE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;mutate&quot;
constructs the hybrid hamiltonian for a specified initial state, final state
and mutation parameter &quot;lambda&quot;</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NEIGHBOR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;neighbor&quot;
finds all of the neighbors of each atom</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NEWATM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;newatm&quot;
creates and defines an atom needed for the Cartesian coordinates file, but
which may not present in the original Protein Data Bank file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NEWCRD Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;newcrd&quot;
computes updated atomic coordinates for a rigid body given the previous
coordinates, the rotation matrix and shift in the center of mass corresponding
to the motion</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NEWTON Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;newton&quot;
performs an energy minimization in Cartesian coordinate space using a
truncated Newton method</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NEWTON1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;newton1&quot;
is a service routine that computes the energy and gradient for truncated
Newton optimization in Cartesian coordinate space</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NEWTON2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;newton2&quot;
is a service routine that computes the sparse matrix Hessian elements for
truncated Newton optimization in Cartesian coordinate space</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NEWTROT Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;newtrot&quot;
performs an energy minimization in torsional angle space using a truncated
Newton conjugate gradient method</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NEWTROT1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;newtrot1&quot;
is a service routine that computes the energy and gradient for truncated
Newton conjugate gradient optimization in torsional angle space</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NEWTROT2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;newtrot2&quot;
is a service routine that computes the sparse matrix Hessian elements for
truncated Newton optimization in torsional angle space</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NEXTARG Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;nextarg&quot;
finds the next unused command line argument and returns it in the input
character string</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NEXTTEXT Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;nexttext&quot;
finds and returns the location of the first non-blank character within an
input text string; zero is returned if no such character is found</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NORMAL Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;normal&quot;
generates a random number from a normal Gaussian distribution with a mean of
zero and a variance of one</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NUCBASE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;nucbase&quot;
builds the side chain for a single nucleotide base in terms of internal coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NUCCHAIN Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;nucchain&quot;
builds up the internal coordinates for a nucleic acid sequence from the sugar
type, backbone and glycosidic torsional values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NUCLEIC Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;nucleic&quot;
builds the internal and Cartesian coordinates of a polynucleotide from nucleic
acid sequence and torsional angle values for the nucleic acid backbone and
side chains</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NUMBER Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;number&quot;
converts a text numeral into an integer value; the input string must contain
only numeric characters</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NUMERAL Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;numeral&quot;
converts an input integer number into the corresponding right- or
left-justified text numeral</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NUMGRAD Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;numgrad&quot;
computes the gradient of the objective function &quot;fvalue&quot; with
respect to Cartesian coordinates of the atoms via a two-sided numerical differentiation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">OCVM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ocvm&quot;
is an optimally conditioned variable metric nonlinear optimization routine
without line searches</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">OLDATM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;oldatm&quot;
get the Cartesian coordinates for an atom from the Protein Data Bank file,
then assigns the atom type and atomic connectivities</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">OPENEND Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;openend&quot;
opens a file on a Fortran unit such that the position is set to the bottom for
appending to the end of the file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">OPTIMIZ1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;optimiz1&quot;
is a service routine that computes the energy and gradient for optimally
conditioned variable metric optimization in Cartesian coordinate space</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">OPTIMIZE Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;optimize&quot;
performs energy minimization in Cartesian coordinate space using an optimally
conditioned variable metric method</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">OPTIROT Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;optirot&quot;
performs an energy minimization in torsional angle space using an optimally
conditioned variable metric method</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">OPTIROT1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;optirot1&quot;
is a service routine that computes the energy and gradient for optimally
conditioned variable metric optimization in torsional angle space</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">OPTRIGID Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;optrigid&quot;
performs an energy minimization of rigid body atom groups using an optimally
conditioned variable metric method</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">OPTRIGID1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;optrigid1&quot;
is a service routine that computes the energy and gradient for optimally
conditioned variable metric optimization of rigid bodies</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ORBITAL Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;orbital&quot;
finds and organizes lists of atoms in a pisystem, bonds connecting pisystem
atoms and torsions whose two central atoms are both pisystem atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ORIENT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;orient&quot;
computes a set of reference Cartesian coordinates in standard orientation for
each rigid body atom group</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ORTHOG Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;orthog&quot;
performs an orthogonalization of an input matrix via the modified Gram-Schmidt algorithm</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">OVERLAP Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;overlap&quot;
computes the overlap for two parallel p-orbitals given the atomic numbers and
distance of separation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PARAMYZE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;paramyze&quot;
prints the force field parameters used in the computation of each of the
potential energy terms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PASSB Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PASSB2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PASSB3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PASSB4 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PASSB5 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PASSF Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PASSF2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PASSF3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PASSF4 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PASSF5 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PATH Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;path&quot;
locates a series of structures equally spaced along a conformational pathway
connecting the input reactant and product structures; a series of constrained
optimizations orthogonal to the path is done via Lagrangian multipliers</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PATH1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PATHPNT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pathpnt&quot;
finds a structure on the synchronous transit path with the specified path
value &quot;t&quot;</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PATHSCAN Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pathscan&quot;
makes a scan of a synchronous transit pathway by computing structures and
energies for specific path values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PATHVAL Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pathval&quot;
computes the synchronous transit path value for the specified structure</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PDBATM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pdbatm&quot;
adds an atom to the Protein Data Bank file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PDBXYZ Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pdbxyz&quot;
takes as input a Protein Data Bank file and then converts to and writes out a
Cartesian coordinates file and, for polypeptides, a sequence file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PIALTER Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pialter&quot;
first modifies bond lengths and force constants according to the standard bond
slope parameters and the bond order values stored in &quot;pnpl&quot;; also
alters some 2-fold torsional parameters based on the bond-order * beta matrix</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PIMOVE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pimove&quot;
rotates the vector between atoms &quot;list(1)&quot; and &quot;list(2)&quot;
so that atom 1 is at the origin and atom 2 along the x-axis; the atoms
defining the respective planes are also moved and their bond lengths normalized</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PIPLANE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;piplane&quot;
selects the three atoms which specify the plane perpendicular to each
p-orbital; the current version will fail in certain situations, including
ketenes, allenes, and isolated or adjacent triple bonds</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PISCF Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;piscf&quot;
performs an scf molecular orbital calculation for the pisystem using a
modified Pariser-Parr-Pople method</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PITILT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pitilt&quot;
calculates for each pibond the ratio of the actual p-orbital overlap integral
to the ideal overlap if the same orbitals were perfectly parallel</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PLACE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;place&quot;
finds the probe sites by putting the probe sphere tangent to each triple of
neighboring atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">POLARGRP Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;polargrp&quot;
generates members of the polarization group of each atom and separate lists of
the 1-2, 1-3 and 1-4 group connectivities</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">POLARIZE Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;polarize&quot;
computes the molecular polarizability by applying an external field along each
axis followed by diagonalization of the resulting polarizability tensor</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">POLYMER Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;polymer&quot;
tests for the presence of an infinite polymer extending across periodic boundaries</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">POLYP Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">POTNRG Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">POTOFF Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;potoff&quot;
clears the forcefield definition by turning off the use of each of the
potential energy functions</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">POWER Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;power&quot;
uses the power method with deflation to compute the few largest eigenvalues
and eigenvectors of a symmetric matrix</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PRECISE Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;precise&quot;
finds a machine precision value as selected by the input argument: (1) the
smallest positive floating point value, (2) the smallest relative floating
point spacing, (3) the largest relative floating point spacing</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PRECOND Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;precond&quot;
solves a simplified version of the Newton equations Ms = r, and uses the
result to precondition linear conjugate gradient iterations on the full Newton
equations in &quot;tnsolve&quot;</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PRESSURE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pressure&quot;
uses the internal virial to find the pressure in a periodic box and maintains
a constant desired pressure by scaling the coordinates via coupling to an
external constant pressure bath</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PRMKEY Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;field&quot;
parses a text string to extract keywords related to force field potential
energy functional forms and constants</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PROCHAIN Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;prochain&quot;
builds up the internal coordinates for an amino acid sequence from the phi,
psi, omega and chi values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PROJCT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PROMO Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;promo&quot;
writes a short message containing information about the TINKER version number
and the copyright notice</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PROPERTY Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;property&quot;
takes two input snapshot frames and computes the value of the property for
which the correlation function is being accumulated</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PROPYZE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;propyze&quot;
finds and prints the total charge, dipole moment components, radius of
gyration and moments of inertia</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PROSIDE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;proside&quot;
builds the side chain for a single amino acid residue in terms of internal coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PROTEIN Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;protein&quot;
builds the internal and Cartesian coordinates of a polypeptide from amino acid
sequence and torsional angle values for the peptide backbone and side chains</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PRTDYN Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;prtdyn&quot;
writes out the information needed to restart a molecular dynamics trajectory
to an external disk file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PRTERR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;prterr&quot;
writes out a set of coordinates to a disk file prior to aborting on a serious error</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PRTINT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;prtint&quot;
writes out a set of Z-matrix internal coordinates to an external disk file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PRTMOL2 Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;prtmol2&quot;
writes out a set of coordinates in Sybyl MOL2 format to an external disk file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PRTMSI Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;prtmsi&quot;
writes out a set of Cartesian coordinates for all active atoms in the MSI
Insight II archive format</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PRTPDB Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;prtpdb&quot;
writes out a set of Protein Data Bank coordinates to an external disk file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PRTPRM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;prtprm&quot;
writes out a formatted listing of the default set of potential energy
parameters for a force field</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PRTSEQ Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;prtseq&quot;
writes out a biopolymer sequence to an external disk file with 15 residues per
line and distinct chains separated by blank lines</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PRTXMOL Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;prtxmol&quot;
writes out a set of Cartesian coordinates for all active atoms in a simple,
generic XYZ format originally due to the XMOL program</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PRTXYZ Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;prtxyz&quot;
writes out a set of Cartesian coordinates to an external disk file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PSS Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pss&quot;
implements the potential smoothing plus search method for global optimization
in Cartesian coordinate space with local searches performed in Cartesian or
torsional space</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PSS1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pss1&quot;
is a service routine that computes the energy and gradient during PSS global
optimization in Cartesian coordinate space</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PSS2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pss2&quot;
is a service routine that computes the sparse matrix Hessian elements during
PSS global optimization in Cartesian coordinate space</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PSSRGD1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pssrgd1&quot;
is a service routine that computes the energy and gradient during PSS global
optimization over rigid bodies</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PSSRIGID Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pssrigid&quot;
implements the potential smoothing plus search method for global optimization
for a set of rigid bodies</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PSSROT Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pssrot&quot;
implements the potential smoothing plus search method for global optimization
in torsional space</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PSSROT1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;pssrot1&quot;
is a service routine that computes the energy and gradient during PSS global
optimization in torsional space</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PSSWRITE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PTINCY Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PZEXTR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">QRFACT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;qrfact&quot;
performs Householder transformations with column pivoting (optional) to
compute a QR factorization of the m by n matrix a; the routine determines an
orthogonal matrix q, a permutation matrix p, and an upper trapezoidal matrix r
with diagonal elements of nonincreasing magnitude, such that a*p = q*r; the
Householder transformation for column k, k = 1,2,...,min(m,n), is of the form</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">QRSOLVE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;qrsolve&quot;
solves a*x=b and d*x=0 in the least squares sense; normally used in
combination with routine &quot;qrfact&quot; to solve least squares problems</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">QUATFIT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;quatfit&quot;
uses a quaternion-based method to achieve the best fit superposition of two
sets of coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RADIAL Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;radial&quot;
finds the radial distribution function for a specified pair of atom types via
analysis of a set of stored coordinate frames from a liquid simulation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RANDOM Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;random&quot;
generates a random number on [0,1] via a long period generator due to L'Ecuyer
with Bays-Durham shuffle</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RANVEC Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ranvec&quot;
generates a unit vector in 3-dimensional space with uniformly distributed
random orientation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RATTLE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;rattle&quot;
implements the first portion of the rattle algorithm by correcting atomic
positions and half-step velocities to maintain constrained interatomic distances</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RATTLE2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;rattle2&quot;
implements the second portion of the rattle algorithm by correcting the
full-step velocities in order to maintain constrained interatomic distances</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">READBLK Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;readblk&quot;
reads in a set of snapshot frames and transfers the values to internal arrays
for use in the computation of time correlation functions</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">READDYN Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;readdyn&quot;
get the positions, velocities and accelerations for a molecular dynamics
restart from an external disk file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">READINT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;readint&quot;
gets a set of Z-matrix internal coordinates from an external file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">READMOL2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;readmol2&quot;
gets a set of Sybyl MOL2 coordinates from an external disk file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">READPDB Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;readpdb&quot;
gets a set of Protein Data Bank coordinates from an external disk file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">READPRM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;readprm&quot;
processes the potential energy parameter file in order to define the default
force field parameters</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">READSEQ Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;readseq&quot;
gets a biopolymer sequence containing one or more separate chains from an
external file; all lines containing sequence must begin with the starting
sequence number, the actual sequence is read from subsequent nonblank characters</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">READXYZ Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;readxyz&quot;
gets a set of Cartesian coordinates from an external disk file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">REFINE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;refine&quot;
performs minimization of the atomic coordinates of an initial crude embedded
distance geometry structure versus the bound, chirality, planarity and
torsional error functions</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">REGBODY Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;regbody&quot;
finds the angular velocity of a regular, nonlinear rigid body</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">REPLICA Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;replica&quot;
decides between images and replicates for generation of periodic boundary
conditions, and sets the cell replicate list if the replicates method is to be used</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RFINDEX Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RGDSRCH Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RGDSTEP Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;rgdstep&quot;
performs a single molecular dynamics time step for a rigid-body calculation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RIBOSOME Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ribosome&quot;
translates a polypeptide structure in Protein Data Bank format to a Cartesian
coordinate file and sequence file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RIGIDXYZ Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;rigidxyz&quot;
computes Cartesian coordinates for a rigid body group via rotation and
translation of reference coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RINGS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;rings&quot;
searches the structure for small rings and stores their component atoms; code
to remove the reducible rings consisting of smaller rings is commented in this
version since reducible rings are needed for parameter assignment</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RMSERROR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;rmserror&quot;
computes the maximum absolute deviation and the rms deviation from the
distance bounds, and the number and rms value of the distance restraint violations</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RMSFIT Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;rmsfit&quot;
computes the rms fit of two coordinate sets</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ROTANG Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ROTCHECK Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;rotcheck&quot;
tests a specified candidate rotatable bond for the disallowed case where
inactive atoms are found on both sides of the candidate bond</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ROTEULER Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;roteuler&quot;
computes a set of Euler angle values consistent with an input rotation matrix</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ROTLIST Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;rotlist&quot;
generates the minimum list of all the atoms lying to one side of a pair of
directly bonded atoms; optionally finds the minimal list by choosing the side
with fewer atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ROTMAT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;rotmat&quot;
find the rotation matrix that converts from the local coordinate system at
each multipole site to the global system</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ROTPOLE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;rotpole&quot;
computes the atomic multipole values in the global coordinate frame by
applying a rotation matrix to a set of locally defined multipoles</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ROTRGD Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;rotrgd&quot;
finds the rotation matrix for a rigid body due to a single step of dynamics</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SADDLE Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;saddle&quot;
finds a transition state between two conformational minima using a combination
of ideas from the synchronous transit (Halgren-Lipscomb) and quadratic path
(Bell-Crighton) methods</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SADDLE1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SADDLES Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;saddles&quot;
constructs circles, convex edges and saddle faces</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SCAN Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;scan&quot;
attempts to find all the local minima on a potential energy surface via an
iterative series of local searches</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SCAN1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;scan1&quot;
is a service routine that computes the energy and gradient during exploration
of a potential energy surface via iterative local search</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SCAN2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;scan2&quot;
is a service routine that computes the sparse matrix Hessian elements during
exploration of a potential energy surface via iterative local search</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SDAREA Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sdarea&quot;
optionally scales the atomic friction coefficient of each atom based on its
accessible surface area</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SDSTEP Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sdstep&quot;
performs a single stochastic dynamics time step via a velocity Verlet
integration algorithm</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SDTERM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sdterm&quot;
gets frictional and random force terms needed to update positions and
velocities via stochastic dynamics</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SEARCH Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;search&quot;
is a line search minimizer based upon parabolic extrapolation and cubic
interpolation using both function and gradient values; if forced to search in
an uphill direction, return is after the initial step</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SETIME Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;setime&quot;
initializes the elapsed interval CPU timer</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SHAKEUP Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;shakeup&quot;
initializes any holonomic constraints for use with the rattle algorithm during
molecular dynamics</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SIGMOID Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sigmoid&quot;
implements a normalized sigmoidal function on the interval [0,1]; the curves
connect (0,0) to (1,1) and have a cooperativity controlled by beta, they
approach a straight line as beta -&gt; 0 and get more nonlinear as beta increases</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SLATER Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;slater&quot;
is a general routine for computing the overlap integrals between two
Slater-type orbitals</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SMOOTH Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;smooth&quot;
sets extent of potential surface deformation for use with potential smoothing
plus search, the diffusion equation method or Gaussian density annealing</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SNIFFER Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sniffer&quot;
performs a global energy minimization using a discrete version of Griewank's
global search trajectory</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SNIFFER1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sniffer1&quot;
is a service routine that computes the energy and gradient for the Sniffer
global optimization method</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SOAK Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;soak&quot;
takes a currently defined solute system and places it into a solvent box, with
removal of any solvent molecules that overlap the solute</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SORT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sort&quot;
takes an input list of integers and sorts it into ascending order using the
Heapsort algorithm</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SORT2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sort2&quot;
takes an input list of reals and sorts it into ascending order using the
Heapsort algorithm; it also returns a key into the original ordering</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SORT3 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sort3&quot;
takes an input list of integers and sorts it into ascending order using the
Heapsort algorithm; it also returns a key into the original ordering</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SORT4 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sort4&quot;
takes an input list of integers and sorts it into ascending absolute value
using the Heapsort algorithm</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SORT5 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sort5&quot;
takes an input list of integers and sorts it into ascending order based on
each value modulo &quot;m&quot;</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SORT6 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sort6&quot;
takes an input list of character strings and sorts it into alphabetical order
using the Heapsort algorithm</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SORT7 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sort7&quot;
takes an input list of character strings and sorts it into alphabetical order
using the Heapsort algorithm; it also returns a key into the original ordering</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SORT8 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sort8&quot;
takes an input list of integers and sorts it into ascending order using the
Heapsort algorithm, duplicate values are removed from the final sorted list</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SPACEFILL Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;spacefill&quot;
computes the surface area and volume of a structure; the van der Waals,
accessible-excluded, and contact-reentrant definitions are available</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SPECTRUM Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;spectrum&quot;
computes a power spectrum over a wavelength range from the velocity
autocorrelation as a function of time</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SQUARE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;square&quot;
is a nonlinear least squares routine derived from the IMSL routine BCLSF and
More's Minpack routine LMDER; the Jacobian is estimated by finite differences
and bounds can be specified for the variables to be refined</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SUFFIX Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;suffix&quot;
checks a filename for the presence of an extension, and appends an extension
if none is found</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SUPERPOSE Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;superpose&quot;
takes pairs of structures and superimposes them in the optimal least squares
sense; it will attempt to match all atom pairs or only those specified by the user</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SURFACE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;surface&quot;
performs an analytical computation of the weighted solvent accessible surface
area of each atom and the first derivatives of the area with respect to
Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SURFATOM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;surfatom&quot;
performs an analytical computation of the surface area of a specified atom; a
simplified version of &quot;surface&quot;</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SWITCH Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;switch&quot;
sets the coeffcients used by the fifth and seventh order polynomial switching
functions for spherical cutoffs</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SYBYLXYZ Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sybylxyz&quot;
takes as input a Sybyl MOL2 coordinates file, converts to and then writes out
Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SYMMETRY Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;symmetry&quot;
applies symmetry operators to the fractional coordinates of the asymmetric
unit in order to generate the symmetry related atoms of the full unit cell</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TANGENT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;tangent&quot;
finds the projected gradient on the synchronous transit path for a point along
the transit pathway</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TEMPER Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;temper&quot;
maintains a constant desired temperature via either Berendsen's velocity
scaling coupled to an external temperature bath or Andersen's stochastic
collision method</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TESTGRAD Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;testgrad&quot;
computes and compares the analytical and numerical gradient vectors of the
potential energy function with respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TESTHESS Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;testhess&quot;
computes and compares the analytical and numerical Hessian matrices of the
potential energy function with respect to Cartesian coordinates</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TESTLIGHT Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;testlight&quot;
performs a set of timing tests to compare the evaluation of potential energy
and energy/gradient using the method of lights with a double loop over all
atom pairs</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TESTROT Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;testrot&quot;
computes and compares the analytical and numerical gradient vectors of the
potential energy function with respect to rotatable torsional angles</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TIMER Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;timer&quot;
measures the CPU time required for file reading and parameter assignment,
potential energy computation, energy and gradient computation, and Hessian
matrix evaluation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TIMEROT Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;timerot&quot;
measures the CPU time required for file reading and parameter assignment,
potential energy computation, energy and gradient over torsions, and torsional
angle Hessian matrix evaluation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TNCG Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;tncg&quot;
implements a truncated Newton optimization algorithm in which a preconditioned
linear conjugate gradient method is used to approximately solve Newton's
equations; special features include use of an explicit sparse Hessian or
finite-difference gradient-Hessian products within the PCG iteration; the
exact Newton search directions can be used optionally; by default the
algorithm checks for negative curvature to prevent convergence to a stationary
point having negative eigenvalues; if a saddle point is desired this test can
be removed by disabling &quot;negtest&quot;</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TNSOLVE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;tnsolve&quot;
uses a linear conjugate gradient method to find an approximate solution to the
set of linear equations represented in matrix form by Hp = -g (Newton's equations)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TORPHASE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;torphase&quot;
sets the n-fold amplitude and phase values for each torsion via sorting of the
input parameters</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TORQUE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;torque&quot;
takes the torque values on sites defined by local coordinate frames and
distributes thme to convert to forces on the original sites and sites
specifying the local frames</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TORQUE1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;torque1&quot;
takes the torque value on a site defined by a local coordinate frame and
distributes it to convert to forces on the original site and sites specifying
the local frame</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TORSER Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;torser&quot;
computes the torsional error function and its first derivatives with respect
to the atomic Cartesian coordinates based on the deviation of specified
torsional angles from desired values, the contained bond angles are also
restrained to avoid a numerical instability</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TORSIONS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;torsions&quot;
finds the total number of dihedral angles and the numbers of the four atoms
defining each dihedral angle</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TORUS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;torus&quot;
sets a list of all of the temporary torus positions by testing for a torus
between each atom and its neighbors</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TOTERR Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;toterr&quot;
is the error function and derivatives for a distance geometry embedding; it
includes components from the distance bounds, hard sphere contacts, local
geometry, chirality and torsional restraint errors</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TRANSIT Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;transit&quot;
evaluates the synchronous transit function and gradient; linear and quadratic
transit paths are available</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TRIANGLE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;triangle&quot;
smooths the upper and lower distance bounds via the triangle inequality using
a full-matrix variant of the Floyd-Warshall shortest path algorithm; this
routine is usually much slower than the sparse matrix shortest path methods in
&quot;geodesic&quot; and &quot;trifix&quot;, and should be used only for
comparison with answers generated by those routines</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TRIFIX Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;trifix&quot;
rebuilds both the upper and lower distance bound matrices following tightening
of one or both of the bounds between a specified pair of atoms, &quot;p&quot;
and &quot;q&quot;, using a modification of Murchland's shortest path update algorithm</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TRIMTEXT Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;trimtext&quot;
finds and returns the location of the last non-blank character before the
first null character in an input text string; the function returns zero if no
such character is found</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TRIPLE Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;triple&quot;
finds the triple product of three vectors; used as a service routine by the
Connolly surface area and volume computation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TRUST Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;trust&quot;
updates the model trust region for a nonlinear least squares calculation; this
version is based on the ideas found in NL2SOL and in Dennis and Schnabel's book</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">UDIRECT1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;udirect1&quot;
computes the reciprocal space contribution of the permanent atomic multipole
moments to the electrostatic field for use in finding the direct induced
dipole moments via a regular Ewald summation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">UDIRECT2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;udirect2&quot;
computes the real space contribution of the permanent atomic multipole moments
to the electrostatic field for use in finding the direct induced dipole
moments via a regular Ewald summation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">UFIELD Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;ufield&quot;
finds the field at each polarizable site due to the induced dipoles at the
other sites using Thole's method to damp the field at close range</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">UMUTUAL1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;umutual1&quot;
computes the reciprocal space contribution of the induced atomic dipole
moments to the electrostatic field for use in iterative calculation of induced
dipole moments via a regular Ewald summation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">UMUTUAL2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;umutual2&quot;
computes the real space contribution of the induced atomic dipole moments to
the electrostatic field for use in iterative calculation of induced dipole
moments via a regular Ewald summation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">UNITCELL Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;unitcell&quot;
gets the periodic boundary box size and related values from an external
keyword file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">UPCASE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;upcase&quot;
converts a text string to all upper case letters</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">VAM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;vam&quot;
takes the analytical molecular surface defined as a collection of spherical
and toroidal polygons and uses it to compute the volume and surface area</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">VCROSS Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;vcross&quot;
finds the cross product of two vectors</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">VDWERR Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;vdwerr&quot;
is the hard sphere van der Waals bound error function and derivatives that
penalizes close nonbonded contacts, pairwise neighbors are generated via the
method of lights</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">VECANG Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;vecang&quot;
finds the angle between two vectors handed with respect to a coordinate axis;
returns an angle in the range [0,2*pi]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">VERLET Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;verlet&quot;
performs a single molecular dynamics time step by means of the velocity Verlet
multistep recursion formula</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">VERSION Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;version&quot;
checks the name of a file about to be opened; if if &quot;old&quot; status is
passed, the name of the highest current version is returned; if
&quot;new&quot; status is passed the filename of the next available unused
version is generated</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">VIBRATE Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;vibrate&quot;
performs a vibrational normal mode analysis; the Hessian matrix of second
derivatives is determined and then diagonalized both directly and after mass
weighting; output consists of the eigenvalues of the force constant matrix as
well as the vibrational frequencies and displacements</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">VIBROT Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">VNORM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;vnorm&quot;
normalizes a vector to unit length; used as a service routine by the Connolly
surface area and volume computation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">VOLUME Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;volume&quot;
calculates the excluded volume via the Connolly analytical volume and surface
area algorithm</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">VOLUME1 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;volume1&quot;
calculates first derivatives of the total excluded volume with respect to the
Cartesian coordinates of each atom</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">VOLUME2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;volume2&quot;
calculates second derivatives of the total excluded volume with respect to the
Cartesian coordinates of the atoms</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">WATSON Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;watson&quot;
uses a rigid-body optimization to approximately align the paired strands of a
nucleic acid double helix</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">WATSON1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;watson1&quot;
is a service routine that computes the energy and gradient for optimally
conditioned variable metric optimization of rigid bodies</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">WRITEOUT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;writeout&quot;
is used by each of the optimization routines to save imtermediate atomic
coordinates to a disk file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XTALERR Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XTALFIT Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;xtalfit&quot;
computes an optimized set of potential energy parameters for user specified
van der Waals and electrostatic interactions by fitting to crystal structure,
lattice energy and monomer dipole moment data</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XTALLAT1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;xtalmol1&quot;
is a service routine that computes the energy and numerical gradient with
respect to the six lattice lengths and angles for a crystal energy minimization</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XTALMIN Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;xtalmin&quot;
performs a full crystal energy minimization by alternating cycles of truncated
Newton optimization over atomic coordinates with variable metric optimization
over the six lattice dimensions and angles</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XTALMOL1 Function</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;xtalmol1&quot;
is a service routine that computes the energy and gradient with respect to the
atomic Cartesian coordinates for a crystal energy minimization</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XTALMOL2 Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;xtalmol2&quot;
is a service routine that computes the sparse matrix Hessian elements with
respect to the atomic Cartesian coordinates for a crystal energy minimization</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XTALMOVE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XTALPRM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;xtalprm&quot;
stores or retrieves a crystal structure; used to make a previously stored
structure the currently active structure, or to store a structure for later
use; only the intermolecular energy terms are provided for</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XTALWRT Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XYZATM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;xyzatm&quot;
computes the Cartesian coordinates of a single atom from its defining internal
coordinate values</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XYZEDIT Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;xyzedit&quot;
provides for modification and manipulation of the contents of a Cartesian
coordinates file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XYZINT Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;xyzint&quot;
takes as input a Cartesian coordinates file, then converts to and writes out
an internal coordinates file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XYZPDB Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;xyzpdb&quot;
takes as input a Cartesian coordinates file, then converts to and writes out a
Protein Data Bank file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XYZRIGID Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;xyzrigid&quot;
computes the center of mass and Euler angle rigid body coordinates for each
atom group in the system</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XYZSYBYL Program</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;xyzsybyl&quot;
takes as input a Cartesian coordinates file, converts to and then writes out a
Sybyl MOL2 file</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ZATOM Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;zatom&quot;
adds an atom to the end of the current Z-matrix and then increments the atom
counter; atom type, defining atoms and internal coordinates are passed as arguments</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ZHELP Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;zhelp&quot;
prints the general information and instructions for the Z-matrix editing program</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ZVALUE Subroutine</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;zvalue&quot;
gets user supplied values for selected coordinates as needed by the internal
coordinate editing program</FONT></FONT></P>
<TABLE WIDTH=450 CELLPADDING=0>
<TR VALIGN=TOP>
<TD COLSPAN=1 WIDTH="12%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">10.</FONT></FONT></U></B></P>
</TD>
<TD COLSPAN=1 WIDTH="88%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">Contents of Common
Block Variables</FONT></FONT></U></B></P>
</TD></TR>
</TABLE>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The
Fortran common blocks found in the TINKER package are listed below along with
a brief description of the contents of each variable in each common block.
Each individual common block is present as a separate &quot;.i&quot; file in
the /source subdirectory. A source code listing containing each of the source
code modules and each of the common blocks can be produced by running the
&quot;listing.make&quot; script found in the distribution.</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ACTION&#9;total number of
each energy term computed</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">neb&#9;number of bond stretch
energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nea&#9;number of angle bend
energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">neba&#9;number of stretch-bend
energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">neub&#9;number of Urey-Bradley
energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">neaa&#9;number of angle-angle
energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">neopb&#9;number of
out-of-plane bend energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">neopd&#9;number of
out-of-plane distance energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">neid&#9;number of improper
dihedral energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">neit&#9;number of improper
torsion energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">net&#9;number of torsional
energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nebt&#9;number of
stretch-torsion energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nett&#9;number of
torsion-torsion energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nev&#9;number of van der Waals
energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nec&#9;number of charge-charge
energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">necd&#9;number of
charge-dipole energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ned&#9;number of dipole-dipole
energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nem&#9;number of multipole
energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nep&#9;number of polarization
energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">new&#9;number of Ewald
summation energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ner&#9;number of reaction
field energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nes&#9;number of solvation
energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nelf&#9;number of metal ligand
field energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">neg&#9;number of geometric
restraint energy terms computed</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nex&#9;number of extra energy
terms computed</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ALIGN&#9;information for
superposition of structures</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">wfit&#9;weights assigned to
atom pairs during superposition</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nfit&#9;number of atoms to use
in superimposing two structures</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ifit&#9;atom numbers of pairs
of atoms to be superimposed</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ANALYZ&#9;energy components
partitioned over atoms</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aeb&#9;bond stretch energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aea&#9;angle bend energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aeba&#9;stretch-bend energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aeub&#9;Urey-Bradley energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aeaa&#9;angle-angle energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aeopb&#9;out-of-plane bend
energy partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aeopd&#9;out-of-plane distance
energy partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aeid&#9;improper dihedral
energy partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aeit&#9;improper torsion
energy partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aet&#9;torsional energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aebt&#9;stretch-torsion energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aett&#9;torsion-torsion energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aev&#9;van der Waals energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aec&#9;charge-charge energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aecd&#9;charge-dipole energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aed&#9;dipole-dipole energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aem&#9;multipole energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aep&#9;polarization energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aer&#9;reaction field energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aes&#9;solvation energy
partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aelf&#9;metal ligand field
energy partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aeg&#9;geometric restraint
energy partitioned over atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aex&#9;extra energy term
partitioned over atoms</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ANGANG&#9;angle-angle terms
in current structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kaa&#9;force constant for
angle-angle cross terms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nangang&#9;total number of
angle-angle interactions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iaa&#9;angle numbers used in
each angle-angle term</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ANGLE&#9;bond angles within
the current structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ak&#9;harmonic angle force
constant (kcal/mole/rad**2)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">anat&#9;ideal bond angle or
phase shift angle (degrees)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">afld&#9;periodicity for
Fourier bond angle term</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nangle&#9;total number of bond
angles in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iang&#9;numbers of the atoms
in each bond angle</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">angtyp&#9;potential energy
function type for each bond angle</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ANGPOT&#9;specifics of bond
angle functional forms</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">cang&#9;cubic coefficient in
angle bending potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">qang&#9;quartic coefficient in
angle bending potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pang&#9;quintic coefficient in
angle bending potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">sang&#9;sextic coefficient in
angle bending potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">angunit&#9;convert angle force
constant to kcal/mole/deg**2</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">stbnunit&#9;convert str-bend
constant to kcal/mole/deg-Ang**2</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aaunit&#9;convert angle-angle
constant to kcal/mole/deg**2</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">opbunit&#9;convert
out-of-plane bend force to kcal/mole/deg**2</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">opdunit&#9;convert
out-of-plane distance to kcal/mole/Ang**2</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">mm2stbn&#9;logical flag
governing use of MM2-style stretch-bend</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ARGUE&#9;command line
arguments at program startup</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxarg&#9;maximum number of
command line arguments</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">narg&#9;number of command line
arguments to the program</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">listarg&#9;flag to mark
available command line arguments</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">arg&#9;strings containing the
command line arguments</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ATMLST&#9;local geometry
terms involving each atom</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bndlist&#9;list of the bond
numbers involving each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">anglist&#9;list of the angle
numbers centered on each atom</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ATMTYP&#9;atomic properties
for each current atom</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">mass&#9;atomic weight for each
atom in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tag&#9;integer atom labels
from input coordinates file</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">class&#9;atom class number for
each atom in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">atomic&#9;atomic number for
each atom in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">valence&#9;valence number for
each atom in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">name&#9;atom name for each
atom in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">story&#9;descriptive type for
each atom in system</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ATOMS&#9;number, position
and type of current atoms</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">x&#9;current x-coordinate for
each atom in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">y&#9;current y-coordinate for
each atom in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">z&#9;current z-coordinate for
each atom in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">n&#9;total number of atoms in
the current system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">type&#9;atom type number for
each atom in the system</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BATH&#9;temperature and
pressure control parameters</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kelvin&#9;target value for the
system temperature (K)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">atmsph&#9;target value for the
system pressure (atm)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tautemp&#9;time constant in
psec for temperature bath coupling</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">taupres&#9;time constant in
psec for pressure bath coupling</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">compress&#9;isothermal
compressibility of medium (atm-1)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">collide&#9;collision frequency
for Andersen thermostat</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">isothermal&#9;logical flag
geverning use of temperature bath</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">isobaric&#9;logical flag
governing use of pressure bath</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">thermostat&#9;type of
thermostat, either Berendsen or Andersen</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BNDPOT&#9;specifics of bond
stretch functional forms</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">cbnd&#9;cubic coefficient in
bond stretch potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">qbnd&#9;quartic coefficient in
bond stretch potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bndunit&#9;convert bond force
constant to kcal/mole/Ang**2</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bndtyp&#9;type of bond stretch
potential energy function</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BOND&#9;covalent bonds in
the current structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bk&#9;bond stretch force
constants (kcal/mole/Ang**2)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bl&#9;ideal bond length values
in Angstroms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nbond&#9;total number of bond
stretches in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ibnd&#9;numbers of the atoms
in each bond stretch</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BORDER&#9;bond orders for a
conjugated pisystem</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pbpl&#9;pi-bond orders for
bonds in &quot;planar&quot; pisystem</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pnpl&#9;pi-bond orders for
bonds in &quot;nonplanar&quot; pisystem</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BOUND&#9;control of
periodic boundary conditions</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">polycut&#9;cutoff distance for
infinite polymer nonbonds</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">polycut2&#9;square of infinite
polymer nonbond cutoff</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_bounds&#9;flag to use
periodic boundary conditions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_image&#9;flag to use
images for periodic system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_replica&#9;flag to use
replicates for periodic system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_polymer&#9;flag to mark
presence of infinite polymer</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">BOXES&#9;parameters for
periodic boundary conditions</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xbox&#9;length in Angs of
a-axis of periodic box</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ybox&#9;length in Angs of
b-axis of periodic box</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zbox&#9;length in Angs of
c-axis of periodic box</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">alpha&#9;angle in degrees
between b- and c-axes of box</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">beta&#9;angle in degrees
between a- and c-axes of box</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">gamma&#9;angle in degrees
between a- and b-axes of box</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xbox2&#9;half of the a-axis
length of periodic box</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ybox2&#9;half of the b-axis
length of periodic box</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zbox2&#9;half of the c-axis
length of periodic box</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">box34&#9;three-fourths axis
length of truncated octahedron</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">recip&#9;reciprocal lattice
vectors as matrix columns</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">volbox&#9;volume in Ang**3 of
the periodic box</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">beta_sin&#9;sine of the beta
periodic box angle</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">beta_cos&#9;cosine of the beta
periodic box angle</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">gamma_sin&#9;sine of the gamma
periodic box angle</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">gamma_cos&#9;cosine of the
gamma periodic box angle</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">beta_term&#9;term used in
generating triclinic box</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">gamma_term&#9;term used in
generating triclinic box</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">orthogonal&#9;flag to mark
periodic box as orthogonal</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">monoclinic&#9;flag to mark
periodic box as monoclinic</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">triclinic&#9;flag to mark
periodic box as triclinic</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">octahedron&#9;flag to mark box
as truncated octahedron</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">spacegrp&#9;space group symbol
for the unitcell type</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CELL&#9;periodic boundaries
using replicated cells</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xcell&#9;length of the a-axis
of the complete replicated cell</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ycell&#9;length of the b-axis
of the complete replicated cell</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zcell&#9;length of the c-axis
of the complete replicated cell</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xcell2&#9;half the length of
the a-axis of the replicated cell</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ycell2&#9;half the length of
the b-axis of the replicated cell</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zcell2&#9;half the length of
the c-axis of the replicated cell</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ncell&#9;total number of cell
replicates for periodic boundaries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">icell&#9;offset along axes for
each replicate periodic cell</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CENTRE&#9;atom coordinates
relative to center of mass</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xcm&#9;offset of each atom
from center of mass x-coordinate</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ycm&#9;offset of each atom
from center of mass y-coordinate</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zcm&#9;offset of each atom
from center of mass z-coordinate</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CHARGE&#9;partial charges
for the current structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pchg&#9;magnitude of the
partial charges (e-)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nion&#9;total number of
partial charges in system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iion&#9;number of the atom
site for each partial charge</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">jion&#9;neighbor generation
site for each partial charge</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kion&#9;cutoff switching site
for each partial charge</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">chglist&#9;partial charge site
for each atom (0=no charge)</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CHGPOT&#9;specifics of
charge-charge functional form</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dielec&#9;dielectric constant
for electrostatic interactions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">c2scale&#9;factor by which 1-2
charge interactions are scaled</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">c3scale&#9;factor by which 1-3
charge interactions are scaled</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">c4scale&#9;factor by which 1-4
charge interactions are scaled</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">c5scale&#9;factor by which 1-5
charge interactions are scaled</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">neutnbr&#9;logical flag
governing use of neutral group neighbors</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">neutcut&#9;logical flag
governing use of neutral group cutoffs</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CHRONO&#9;timing statistics
for the current program</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">cputim&#9;elapsed cpu time in
seconds since start of program</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">COUPLE&#9;near-neighbor
atom connectivity lists</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxn13&#9;maximum number of
atoms 1-3 connected to an atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxn14&#9;maximum number of
atoms 1-4 connected to an atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxn15&#9;maximum number of
atoms 1-5 connected to an atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">n12&#9;number of atoms
directly bonded to each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">i12&#9;atom numbers of atoms
1-2 connected to each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">n13&#9;number of atoms in a
1-3 relation to each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">i13&#9;atom numbers of atoms
1-3 connected to each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">n14&#9;number of atoms in a
1-4 relation to each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">i14&#9;atom numbers of atoms
1-4 connected to each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">n15&#9;number of atoms in a
1-5 relation to each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">i15&#9;atom numbers of atoms
1-5 connected to each atom</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">CUTOFF&#9;cutoff distances
for energy interactions</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">vdwcut&#9;cutoff distance for
van der Waals interactions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">chgcut&#9;cutoff distance for
charge-charge interactions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dplcut&#9;cutoff distance for
dipole-dipole interactions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">mpolecut&#9;cutoff distance
for atomic multipole interactions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">vdwtaper&#9;distance at which
van der Waals switching begins</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">chgtaper&#9;distance at which
charge-charge switching begins</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dpltaper&#9;distance at which
dipole-dipole switching begins</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">mpoletaper&#9;distance at
which atomic multipole switching begins</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ewaldcut&#9;cutoff distance
for direct space Ewald summation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_ewald&#9;logical flag
governing use of Ewald summation term</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_lights&#9;logical flag to
use method of lights neighbors</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DERIV&#9;Cartesian
coordinate derivative components</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">deb&#9;bond stretch Cartesian
coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dea&#9;angle bend Cartesian
coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">deba&#9;stretch-bend Cartesian
coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">deub&#9;Urey-Bradley Cartesian
coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">deaa&#9;angle-angle Cartesian
coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">deopb&#9;out-of-plane bend
Cartesian coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">deopd&#9;out-of-plane distance
Cartesian coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">deid&#9;improper dihedral
Cartesian coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">deit&#9;improper torsion
Cartesian coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">det&#9;torsional Cartesian
coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">debt&#9;stretch-torsion
Cartesian coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dett&#9;torsion-torsion
Cartesian coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dev&#9;van der Waals Cartesian
coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dec&#9;charge-charge Cartesian
coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">decd&#9;charge-dipole
Cartesian coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ded&#9;dipole-dipole Cartesian
coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dem&#9;multipole Cartesian
coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dep&#9;polarization Cartesian
coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">der&#9;reaction field
Cartesian coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">des&#9;solvation Cartesian
coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">delf&#9;metal ligand field
Cartesian coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">deg&#9;geometric restraint
Cartesian coordinate derivatives</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dex&#9;extra energy term
Cartesian coordinate derivatives</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DIPOLE&#9;atom &amp; bond
dipoles for current structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bdpl&#9;magnitude of each of
the dipoles (Debyes)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">sdpl&#9;position of each
dipole between defining atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ndipole&#9;total number of
dipoles in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">idpl&#9;numbers of atoms that
define each dipole</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DISGEO&#9;distance geometry
bounds and parameters</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bnd&#9;distance geometry upper
and lower bounds matrix</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">vdwrad&#9;hard sphere radii
for distance geometry atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">vchir&#9;signed volume values
for chirality constraints</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">compact&#9;index of local
distance compaction on embedding</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pathmax&#9;maximum value of
upper bound after smoothing</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">vdwmax&#9;maximum value of
hard sphere sum for an atom pair</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nchir&#9;total number of
chirality constraints</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ichir&#9;numbers of atoms in
each chirality constraint</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_invert&#9;flag to use
enantiomer closest to input structure</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_anneal&#9;flag to use
simulated annealing refinement</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">DOMEGA&#9;derivative
components over dihedrals</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">teb&#9;bond stretch
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tea&#9;angle bend derivatives
over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">teba&#9;stretch-bend
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">teub&#9;Urey-Bradley
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">teaa&#9;angle-angle
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">teopb&#9;out-of-plane bend
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">teopd&#9;out-of-plane distance
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">teid&#9;improper dihedral
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">teit&#9;improper torsion
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tet&#9;torsional derivatives
over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tebt&#9;stretch-torsion
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tett&#9;torsion-torsion
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tev&#9;van der Waals
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tec&#9;charge-charge
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tecd&#9;charge-dipole
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ted&#9;dipole-dipole
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tem&#9;atomic multipole
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tep&#9;polarization
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ter&#9;reaction field
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tes&#9;solvation derivatives
over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">telf&#9;metal ligand field
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">teg&#9;geometric restraint
derivatives over torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tex&#9;extra energy term
derivatives over torsions</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ENERGI&#9;individual
potential energy components</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">eb&#9;bond stretch potential
energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ea&#9;angle bend potential
energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">eba&#9;stretch-bend potential
energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">eub&#9;Urey-Bradley potential
energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">eaa&#9;angle-angle potential
energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">eopb&#9;out-of-plane bend
potential energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">eopd&#9;out-of-plane distance
potential energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">eid&#9;improper dihedral
potential energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">eit&#9;improper torsion
potential energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">et&#9;torsional potential
energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ebt&#9;stretch-torsion
potential energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ett&#9;torsion-torsion
potential energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ev&#9;van der Waals potential
energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ec&#9;charge-charge potential
energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ecd&#9;charge-dipole potential
energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ed&#9;dipole-dipole potential
energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">em&#9;atomic multipole
potential energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ep&#9;polarization potential
energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">er&#9;reaction field potential
energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">es&#9;solvation potential
energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">elf&#9;metal ligand field
potential energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">eg&#9;geometric restraint
potential energy of the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ex&#9;extra term potential
energy of the system</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EWALD&#9;parameters for
regular or PM Ewald summation</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aewald&#9;Ewald convergence
coefficient value (Ang-1)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">frecip&#9;fractional cutoff
value for reciprocal sphere</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tinfoil&#9;flag governing use
of tinfoil boundary conditions</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">EWREG&#9;exponential
factors for regular Ewald sum</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxvec&#9;maximum number of
k-vectors per reciprocal axis</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ejc&#9;exponental factors for
cosine along the j-axis</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ejs&#9;exponental factors for
sine along the j-axis</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ekc&#9;exponental factors for
cosine along the k-axis</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">eks&#9;exponental factors for
sine along the k-axis</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">elc&#9;exponental factors for
cosine along the l-axis</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">els&#9;exponental factors for
sine along the l-axis</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">FACES&#9;variables for
Connolly area and volume</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnbr&#9;maximum number of
neighboring atom pairs</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxtt&#9;maximum number of
temporary tori</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxt&#9;maximum number of
total tori</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxp&#9;maximum number of
probe positions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxv&#9;maximum number of vertices</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxen&#9;maximum number of
concave edges</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxfn&#9;maximum number of
concave faces</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxc&#9;maximum number of circles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxep&#9;maximum number of
convex edges</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxfs&#9;maximum number of
saddle faces</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxcy&#9;maximum number of cycles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">mxcyep&#9;maximum number of
cycle convex edges</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxfp&#9;maximum number of
convex faces</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">mxfpcy&#9;maximum number of
convex face cycles</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">FIELDS&#9;molecular
mechanics force field description</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">biotyp&#9;force field atom
type of each biopolymer type</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">forcefield&#9;string used to
describe the current forcefield</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">FILES&#9;name and number of
current structure files</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nprior&#9;number of previously
existing cycle files</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ldir&#9;length in characters
of the directory name</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">leng&#9;length in characters
of the base filename</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">filename&#9;base filename used
by default for all files</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">outfile&#9;output filename
used for intermediate results</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">FRACS&#9;atom distances to
molecular center of mass</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xfrac&#9;fractional coordinate
along a-axis of center of mass</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">yfrac&#9;fractional coordinate
along b-axis of center of mass</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zfrac&#9;fractional coordinate
along c-axis of center of mass</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">GROUP&#9;partitioning of
system into atom groups</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">grpmass&#9;total mass of all
the atoms in each group</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">wgrp&#9;weight for each set of
group-group interactions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">grpnum&#9;original group
number for each nonempty group</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ngrp&#9;total number of atom
groups in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kgrp&#9;contiguous list of the
atoms in each group</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">igrp&#9;first and last atom of
each group in the list</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">grplist&#9;number of the group
to which each atom belongs</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_group&#9;flag to use
partitioning of system into groups</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_intra&#9;flag to include
only intragroup interactions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_inter&#9;flag to include
only intergroup interactions</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HESCUT&#9;cutoff value for
Hessian matrix elements</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">hesscut&#9;magnitude of
smallest allowed Hessian element</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">HESSN&#9;Cartesian Hessian
elements for a single atom</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">hessx&#9;Hessian elements for
x-component of current atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">hessy&#9;Hessian elements for
y-component of current atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">hessz&#9;Hessian elements for
z-component of current atom</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">IMPROP&#9;improper
dihedrals in the current structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kprop&#9;force constant values
for improper dihedral angles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">vprop&#9;ideal improper
dihedral angle value in degrees</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">niprop&#9;total number of
improper dihedral angles in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iiprop&#9;numbers of the atoms
in each improper dihedral angle</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">IMPTOR&#9;improper torsions
in the current structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">itors1&#9;1-fold amplitude and
phase for each improper torsion</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">itors2&#9;2-fold amplitude and
phase for each improper torsion</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">itors3&#9;3-fold amplitude and
phase for each improper torsion</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nitors&#9;total number of
improper torsional angles in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iitors&#9;numbers of the atoms
in each improper torsional angle</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INFORM&#9;control values
for I/O and program flow</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">digits&#9;decimal places
output for energy and coordinates</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iprint&#9;steps between status
printing (0=no printing)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iwrite&#9;steps between
coordinate dumps (0=no dumps)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">verbose&#9;logical flag to
turn on extra information</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">debug&#9;logical flag to turn
on full debug printing</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">holdup&#9;logical flag to wait
for carriage return on exit</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">abort&#9;logical flag to stop
execution at next chance</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">INTER&#9;sum of
intermolecular energy components</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">einter&#9;total intermolecular
potential energy</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">IOUNIT&#9;Fortran
input/output (I/O) unit numbers</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iout&#9;Fortran I/O unit for
major output (default=6)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">input&#9;Fortran I/O unit for
major input (default=5)</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KANANG&#9;forcefield
parameters for angle-angle terms</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">anan&#9;angle-angle cross term
parameters for each atom class</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KANGS&#9;forcefield
parameters for bond angle bending</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxna&#9;maximum number of
harmonic angle bend parameter entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxna5&#9;maximum number of
5-membered ring angle bend entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxna4&#9;maximum number of
4-membered ring angle bend entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxna3&#9;maximum number of
3-membered ring angle bend entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnaf&#9;maximum number of
Fourier angle bend parameter entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">acon&#9;force constant
parameters for harmonic angle bends</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">acon5&#9;force constant
parameters for 5-ring angle bends</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">acon4&#9;force constant
parameters for 4-ring angle bends</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">acon3&#9;force constant
parameters for 3-ring angle bends</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aconf&#9;force constant
parameters for Fourier angle bends</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ang&#9;bond angle parameters
for harmonic angle bends</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ang5&#9;bond angle parameters
for 5-ring angle bends</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ang4&#9;bond angle parameters
for 4-ring angle bends</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ang3&#9;bond angle parameters
for 3-ring angle bends</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">angf&#9;phase shift angle and
periodicity for Fourier bends</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ka&#9;string of atom classes
for harmonic angle bends</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ka5&#9;string of atom classes
for 5-ring angle bends</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ka4&#9;string of atom classes
for 4-ring angle bends</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ka3&#9;string of atom classes
for 3-ring angle bends</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kaf&#9;string of atom classes
for Fourier angle bends</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KATOMS&#9;forcefield
parameters for the atom types</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">weight&#9;average atomic mass
of each atom type</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">atmcls&#9;atom class number
for each of the atom types</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">atmnum&#9;atomic number for
each of the atom types</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ligand&#9;number of atoms to
be attached to each atom type</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">symbol&#9;modified atomic
symbol for each atom type</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">describe&#9;string identifing
each of the atom types</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KBONDS&#9;forcefield
parameters for bond stretching</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnb&#9;maximum number of
bond stretch parameter entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnb5&#9;maximum number of
5-membered ring bond stretch entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnb4&#9;maximum number of
4-membered ring bond stretch entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnb3&#9;maximum number of
3-membered ring bond stretch entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnel&#9;maximum number of
electronegativity bond corrections</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bcon&#9;force constant
parameters for harmonic bond stretch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bcon5&#9;force constant
parameters for 5-ring bond stretch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bcon4&#9;force constant
parameters for 4-ring bond stretch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bcon3&#9;force constant
parameters for 3-ring bond stretch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">blen&#9;bond length parameters
for harmonic bond stretch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">blen5&#9;bond length
parameters for 5-ring bond stretch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">blen4&#9;bond length
parameters for 4-ring bond stretch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">blen3&#9;bond length
parameters for 3-ring bond stretch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dlen&#9;electronegativity bond
length correction parameters</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kb&#9;string of atom classes
for harmonic bond stretch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kb5&#9;string of atom classes
for 5-ring bond stretch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kb4&#9;string of atom classes
for 4-ring bond stretch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kb3&#9;string of atom classes
for 3-ring bond stretch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kel&#9;string of atom classes
for electronegativity corrections</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KCHRGE&#9;forcefield
parameters for partial charges</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">chg&#9;partial charge
parameters for each atom type</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KDIPOL&#9;forcefield
parameters for bond dipoles</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnd&#9;maximum number of
bond dipole parameter entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnd5&#9;maximum number of
5-membered ring dipole entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnd4&#9;maximum number of
4-membered ring dipole entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnd3&#9;maximum number of
3-membered ring dipole entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dpl&#9;dipole moment
parameters for bond dipoles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dpl5&#9;dipole moment
parameters for 5-ring dipoles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dpl4&#9;dipole moment
parameters for 4-ring dipoles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dpl3&#9;dipole moment
parameters for 3-ring dipoles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pos&#9;dipole position
parameters for bond dipoles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pos5&#9;dipole position
parameters for 5-ring dipoles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pos4&#9;dipole position
parameters for 4-ring dipoles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pos3&#9;dipole position
parameters for 3-ring dipoles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kd&#9;string of atom classes
for bond dipoles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kd5&#9;string of atom classes
for 5-ring dipoles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kd4&#9;string of atom classes
for 4-ring dipoles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kd3&#9;string of atom classes
for 3-ring dipoles</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KEYS&#9;contents of current
keyword parameter file</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nkey&#9;number of nonblank
lines in the keyword file</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">keyline&#9;contents of each
individual keyword file line</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KGEOMS&#9;parameters for
the geometrical restraints</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">depth&#9;depth of shallow
Gaussian basin restraint</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">width&#9;exponential width
coefficient of Gaussian basin</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">rwall&#9;radius of spherical
droplet boundary restraint</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xpfix&#9;x-coordinate target
for each restrained position</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ypfix&#9;y-coordinate target
for each restrained position</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zpfix&#9;z-coordinate target
for each restrained position</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pfix&#9;flat-well range and
force constant for each position</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dfix&#9;target range and force
constant for each distance</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">afix&#9;target range and force
constant for each angle</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tfix&#9;target range and force
constant for each torsion</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">npfix&#9;number of position
restraints to be applied</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ipfix&#9;atom number involved
in each position restraint</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ndfix&#9;number of distance
restraints to be applied</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">idfix&#9;atom numbers defining
each distance restraint</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nafix&#9;number of angle
restraints to be applied</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iafix&#9;atom numbers defining
each angle restraint</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ntfix&#9;number of torsional
restraints to be applied</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">itfix&#9;atom numbers defining
each torsional restraint</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_basin&#9;logical flag
governing use of Gaussian basin</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_wall&#9;logical flag
governing use of droplet boundary</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KHBOND&#9;forcefield
parameters for H-bonding terms</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnhb&#9;maximum number of
hydrogen bonding pair entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">radhb&#9;radius parameter for
hydrogen bonding pairs</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">epshb&#9;well depth parameter
for hydrogen bonding pairs</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">khb&#9;string of atom types
for hydrogen bonding pairs</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KIPROP&#9;forcefield
parameters for improper dihedral</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxndi&#9;maximum number of
improper dihedral parameter entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dcon&#9;force constant
parameters for improper dihedrals</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tdi&#9;ideal dihedral angle
values for improper dihedrals</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kdi&#9;string of atom classes
for improper dihedral angles</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KITORS&#9;forcefield
parameters for improper torsions</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnti&#9;maximum number of
improper torsion parameter entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ti1&#9;torsional parameters
for improper 1-fold rotation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ti2&#9;torsional parameters
for improper 2-fold rotation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ti3&#9;torsional parameters
for improper 3-fold rotation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kti&#9;string of atom classes
for improper torsional parameters</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KMULTI&#9;forcefield
parameters for atomic multipoles</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnmp&#9;maximum number of
atomic multipole parameter entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">multip&#9;atomic monopole,
dipole and quadrupole values</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">mpaxis&#9;type of local axis
definition for atomic multipoles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kmp&#9;string of atom types
for atomic multipoles</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KOPBND&#9;forcefield
parameters for out-of-plane bend</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnopb&#9;maximum number of
out-of-plane bending entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">copb&#9;force constant
parameters for out-of-plane bending</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kaopb&#9;string of atom
classes for out-of-plane bending</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KOPDST&#9;forcefield
parameters for out-plane distance</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnopb&#9;maximum number of
out-of-plane distance entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">copb&#9;force constant
parameters for out-of-plane distance</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kaopb&#9;string of atom
classes for out-of-plane distance</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KORBS&#9;forcefield
parameters for pisystem orbitals</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnpi&#9;maximum number of
pisystem bond parameter entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">electron&#9;number of
pi-electrons for each atom class</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ionize&#9;ionization potential
for each atom class</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">repulse&#9;repulsion integral
value for each atom class</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">sslope&#9;slope for bond
stretch vs. pi-bond order</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tslope&#9;slope for 2-fold
torsion vs. pi-bond order</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kpi&#9;string of atom classes
for pisystem bonds</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KPOLR&#9;forcefield
parameters for polarizability</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">polr&#9;dipole polarizability
parameters for each atom type</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pgrp&#9;connected types in
polarization group of each atom type</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KSTBND&#9;forcefield
parameters for stretch-bending</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">stbn&#9;stretch-bending
parameters for each atom class</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KSTTOR&#9;forcefield
parameters for stretch-torsions</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnbt&#9;maximum number of
stretch-torsion parameter entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">btcon&#9;force constant
parameters for stretch-torsion</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kbt&#9;string of atom classes
for bonds in stretch-torsion</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KTORSN&#9;forcefield
parameters for torsional angles</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnt&#9;maximum number of
torsional angle parameter entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnt5&#9;maximum number of
5-membered ring torsion entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnt4&#9;maximum number of
4-membered ring torsion entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t1&#9;torsional parameters for
standard 1-fold rotation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t2&#9;torsional parameters for
standard 2-fold rotation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t3&#9;torsional parameters for
standard 3-fold rotation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t4&#9;torsional parameters for
standard 4-fold rotation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t5&#9;torsional parameters for
standard 5-fold rotation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t6&#9;torsional parameters for
standard 6-fold rotation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t15&#9;torsional parameters
for 1-fold rotation in 5-ring</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t25&#9;torsional parameters
for 2-fold rotation in 5-ring</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t35&#9;torsional parameters
for 3-fold rotation in 5-ring</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t45&#9;torsional parameters
for 4-fold rotation in 5-ring</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t55&#9;torsional parameters
for 5-fold rotation in 5-ring</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t65&#9;torsional parameters
for 6-fold rotation in 5-ring</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t14&#9;torsional parameters
for 1-fold rotation in 4-ring</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t24&#9;torsional parameters
for 2-fold rotation in 4-ring</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t34&#9;torsional parameters
for 3-fold rotation in 4-ring</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t44&#9;torsional parameters
for 4-fold rotation in 4-ring</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t54&#9;torsional parameters
for 5-fold rotation in 4-ring</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t64&#9;torsional parameters
for 6-fold rotation in 4-ring</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kt&#9;string of atom classes
for torsional angles</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kt5&#9;string of atom classes
for 5-ring torsions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kt4&#9;string of atom classes
for 4-ring torsions</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KURYBR&#9;forcefield
parameters for Urey-Bradley terms</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnu&#9;maximum number of
Urey-Bradley parameter entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ucon&#9;force constant
parameters for Urey-Bradley terms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dst13&#9;ideal 1-3 distance
parameters for Urey-Bradley terms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ku&#9;string of atom classes
for Urey-Bradley terms</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KVDWPR&#9;forcefield
parameters for special vdw terms</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnvp&#9;maximum number of
special van der Waals pair entries</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">radpr&#9;radius parameter for
special van der Waals pairs</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">epspr&#9;well depth parameter
for special van der Waals pairs</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kvpr&#9;string of atom classes
for special van der Waals pairs</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">KVDWS&#9;forcefield
parameters for van der Waals terms</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">rad&#9;van der Waals radius
parameter for each atom class</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">eps&#9;van der Waals well
depth parameter for each atom class</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">rad4&#9;van der Waals radius
parameter in 1-4 interactions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">eps4&#9;van der Waals well
depth parameter in 1-4 interactions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">reduct&#9;van der Waals
reduction factor for each atom class</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">LIGHT&#9;indices for method
of lights pair neighbors</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nlight&#9;total number of
sites for method of lights calculation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kbx&#9;low index of neighbors
of each site in the x-sorted list</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kby&#9;low index of neighbors
of each site in the y-sorted list</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kbz&#9;low index of neighbors
of each site in the z-sorted list</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kex&#9;high index of neighbors
of each site in the x-sorted list</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">key&#9;high index of neighbors
of each site in the y-sorted list</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kez&#9;high index of neighbors
of each site in the z-sorted list</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">locx&#9;pointer from x-sorted
list into original interaction list</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">locy&#9;pointer from y-sorted
list into original interaction list</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">locz&#9;pointer from z-sorted
list into original interaction list</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">rgx&#9;pointer from original
interaction list into x-sorted list</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">rgy&#9;pointer from original
interaction list into y-sorted list</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">rgz&#9;pointer from original
interaction list into z-sorted list</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">LINMIN&#9;parameters for
line search minimization</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">stpmin&#9;minimum step length
in current line search direction</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">stpmax&#9;maximum step length
in current line search direction</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">cappa&#9;stringency of line
search (0=tight &lt; cappa &lt; 1=loose)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">angmax&#9;maximum angle
between search direction and -gradient</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">intmax&#9;maximum number of
inner iterations during line search</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MATH&#9;mathematical and
geometrical constants</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">radian&#9;conversion factor
from radians to degrees</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pi&#9;numerical value of the
geometric constant</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">sqrtpi&#9;numerical value of
the square root of Pi</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">logten&#9;numerical value of
the natural log of ten</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">twosix&#9;numerical value of
the sixth root of two</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MDSTUF&#9;control of
molecular dynamics trajectory</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nfree&#9;total number of
degrees of freedom for a system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">velsave&#9;flag to save atomic
velocity components to a file</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">uindsave&#9;flag to save
induced atomic dipoles to a file</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">integrate&#9;type of molecular
dynamics integration algorithm</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MINIMA&#9;general
parameters for minimizations</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">fctmin&#9;value below which
function is deemed optimized</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">hguess&#9;initial value for
the H-matrix diagonal elements</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxiter&#9;maximum number of
iterations during optimization</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nextiter&#9;iteration number
to use for the first iteration</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MOLCUL&#9;individual
molecules within current system</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">molmass&#9;molecular weight
for each molecule in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">totmass&#9;total weight of all
the molecules in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nmol&#9;total number of
separate molecules in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kmol&#9;contiguous list of the
atoms in each molecule</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">imol&#9;first and last atom of
each molecule in the list</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">molcule&#9;number of the
molecule to which each atom belongs</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MOLDYN&#9;velocity and
acceleration on MD trajectory</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">v&#9;current velocity of each
atom along the x,y,z-axes</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">a&#9;current acceleration of
each atom along x,y,z-axes</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">aold&#9;previous acceleration
of each atom along x,y,z-axes</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MOMENT&#9;components of
electric multipole moments</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">netchg&#9;net electric charge
for the total system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">netdpl&#9;dipole moment
magnitude for the total system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">netqdp&#9;diagonal quadrupole
(Qxx, Qyy, Qzz) for system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xdpl&#9;dipole vector
x-component in the global frame</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ydpl&#9;dipole vector
y-component in the global frame</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zdpl&#9;dipole vector
z-component in the global frame</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xxqdp&#9;quadrupole tensor
xx-component in global frame</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xyqdp&#9;quadrupole tensor
xy-component in global frame</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xzqdp&#9;quadrupole tensor
xz-component in global frame</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">yxqdp&#9;quadrupole tensor
yx-component in global frame</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">yyqdp&#9;quadrupole tensor
yy-component in global frame</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">yzqdp&#9;quadrupole tensor
yz-component in global frame</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zxqdp&#9;quadrupole tensor
zx-component in global frame</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zyqdp&#9;quadrupole tensor
zy-component in global frame</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zzqdp&#9;quadrupole tensor
zz-component in global frame</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">MPLPOT&#9;specifics of atomic
multipole functions</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">m2scale&#9;factor by which 1-2
multipole interactions are scaled</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">m3scale&#9;factor by which 1-3
multipole interactions are scaled</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">m4scale&#9;factor by which 1-4
multipole interactions are scaled</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">m5scale&#9;factor by which 1-5
multipole interactions are scaled</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MPOLE&#9;multipole
components for current structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxpole&#9;max components (monopole=1,dipole=4,quadrupole=13)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pole&#9;multipole values for
each site in the local frame</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">rpole&#9;multipoles rotated to
the global coordinate system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">npole&#9;total number of
multipole sites in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ipole&#9;number of the atom
for each multipole site</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">polsiz&#9;number of mutipole
components at each multipole site</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zaxis&#9;number of the z-axis
defining atom for each site</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xaxis&#9;number of the x-axis
defining atom for each site</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">polaxe&#9;local axis type for
each multipole site</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">MUTANT&#9;hybrid atoms for
free energy perturbation</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">lambda&#9;weighting of initial
state in hybrid Hamiltonian</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nhybrid&#9;number of atoms
mutated from initial to final state</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ihybrid&#9;atomic sites
differing in initial and final state</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">type0&#9;atom type of each
atom in the initial state system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">class0&#9;atom class of each
atom in the initial state system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">type1&#9;atom type of each
atom in the final state system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">class1&#9;atom class of each
atom in the final state system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">alter&#9;true if an atom is to
be mutated, false otherwise</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">NUCLEO&#9;parameters for
nucleic acid structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bkbone&#9;phosphate backbone
angles for each nucleotide</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">glyco&#9;glycosidic torsional
angle for each nucleotide</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pucker&#9;sugar pucker, either
2=2'-endo or 3=3'-endo</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dblhlx&#9;flag to mark system
as nucleic acid double helix</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">deoxy&#9;flag to mark
deoxyribose or ribose sugar units</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">hlxform&#9;helix form (A, B or
Z) of polynucleotide strands</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">OMEGA&#9;dihedrals for
torsional space computations</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dihed&#9;current value in
radians of each dihedral angle</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nomega&#9;number of dihedral
angles allowed to rotate</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iomega&#9;numbers of two atoms
defining rotation axis</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zline&#9;line number in
Z-matrix of each dihedral angle</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">OPBEND&#9;out-of-plane
bends in the current structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kopb&#9;force constant values
for out-of-plane bending</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nopbend&#9;total number of
out-of-plane bends in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iopb&#9;bond angle numbers
used in out-of-plane bending</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">OPDIST&#9;out-of-plane
distances in current structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kopd&#9;force constant values
for out-of-plane distance</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nopdist&#9;total number of
out-of-plane distances in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iopb&#9;numbers of the atoms
in each out-of-plane distance</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ORBITS&#9;orbital energies
for conjugated pisystem</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">q&#9;number of pi-electrons
contributed by each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">w&#9;ionization potential of
each pisystem atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">em&#9;repulsion integral for
each pisystem atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nfill&#9;number of filled
pisystem molecular orbitals</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">OUTPUT&#9;control of
coordinate output file format</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">archive&#9;logical flag to
save structures in an archive</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">noversion&#9;logical flag
governing use of filename versions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">overwrite&#9;logical flag to
overwrite intermediate files inplace</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">cyclesave&#9;logical flag to
mark use of numbered cycle files</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">coordtype&#9;selects
Cartesian, internal, rigid body or none</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PATHS&#9;parameters for
Elber reaction path method</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">p0&#9;reactant Cartesian
coordinates as variables</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">p1&#9;product Cartesian
coordinates as variables</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pmid&#9;midpoint between the
reactant and product</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pvect&#9;vector connecting the
reactant and product</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pstep&#9;step per cycle along
reactant-product vector</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pzet&#9;current projection on
reactant-product vector</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pnorm&#9;length of the
reactant-product vector</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">acoeff&#9;transformation
matrix 'A' from Elber paper</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">gc&#9;gradients of the path constraints</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PDB&#9;definition of a
Protein Data Bank structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xpdb&#9;x-coordinate of each
atom stored in PDB format</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ypdb&#9;y-coordinate of each
atom stored in PDB format</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zpdb&#9;z-coordinate of each
atom stored in PDB format</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">npdb&#9;number of atoms stored
in Protein Data Bank format</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">resnum&#9;number of the
residue to which each atom belongs</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">npdb12&#9;number of atoms
directly bonded to each CONECT atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ipdb12&#9;atom numbers of
atoms connected to each CONECT atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pdblist&#9;list of the Protein
Data Bank atom number of each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pdbtyp&#9;Protein Data Bank
record type assigned to each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">atmnam&#9;Protein Data Bank
atom name assigned to each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">resnam&#9;Protein Data Bank
residue name assigned to each atom</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PHIPSI&#9;phi-psi-omega-chi
angles for a protein</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">phi&#9;value of the phi angle
for each amino acid residue</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">psi&#9;value of the psi angle
for each amino acid residue</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">omega&#9;value of the omega
angle for each amino acid residue</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">chi&#9;values of the chi
angles for each amino acid residue</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">chiral&#9;chirality of each
amino acid residue (1=L, -1=D)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">disulf&#9;residue joined to
each residue via a disulfide link</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PIORBS&#9;conjugated system
in the current structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">norbit&#9;total number of
pisystem orbitals in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iorbit&#9;numbers of the atoms
containing pisystem orbitals</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">reorbit&#9;number of
evaluations between orbital updates</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">piperp&#9;atoms defining a
normal plane to each orbital</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">npibond&#9;total number of
bonds affected by the pisystem</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pibond&#9;bond and piatom
numbers for each pisystem bond</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">npitors&#9;total number of
torsions affected by the pisystem</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pitors&#9;torsion and pibond
numbers for each pisystem torsion</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">listpi&#9;atom list indicating
whether each atom has an orbital</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PISTUF&#9;bonds and
torsions in the current pisystem</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bkpi&#9;bond stretch force
constants for pi-bond order of 1.0</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">blpi&#9;ideal bond length
values for a pi-bond order of 1.0</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kslope&#9;rate of force
constant decrease with bond order decrease</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">lslope&#9;rate of bond length
increase with a bond order decrease</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">torspi&#9;2-fold torsional
energy barrier for pi-bond order of 1.0</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PME&#9;parameters for
particle mesh Ewald summation</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxfft&#9;maximum number of
points along each FFT direction</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxorder&#9;maximum order of
the B-spline approximation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxtable&#9;maximum size of
the FFT table array</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxgrid&#9;maximum dimension
of the PME charge grid array</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bsmod1&#9;B-spline moduli
along the a-axis direction</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bsmod2&#9;B-spline moduli
along the b-axis direction</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bsmod3&#9;B-spline moduli
along the c-axis direction</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">table&#9;intermediate array
used by the FFT calculation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nfft1&#9;number of grid points
along the a-axis direction</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nfft2&#9;number of grid points
along the b-axis direction</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nfft3&#9;number of grid points
along the c-axis direction</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bsorder&#9;order of the PME
B-spline approximation</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">POLAR&#9;polarizabilities
and induced dipole moments</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">polarity&#9;dipole
polarizability for each multipole site (Ang**3)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pdamp&#9;value of
polarizability damping factor for each site</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">uind&#9;induced dipole
components at each multipole site</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">uinp&#9;induced dipoles in
field used for energy interactions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">npolar&#9;total number of
polarizable sites in the system</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">POLGRP&#9;polarizable site
group connectivity lists</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxp11&#9;maximum number of
atoms in a polarization group</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxp12&#9;maximum number of
atoms in groups 1-2 to an atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxp13&#9;maximum number of
atoms in groups 1-3 to an atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxp14&#9;maximum number of
atoms in groups 1-4 to an atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">np11&#9;number of atoms in
polarization group of each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ip11&#9;atom numbers of atoms
in same group as each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">np12&#9;number of atoms in
groups 1-2 to each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ip12&#9;atom numbers of atoms
in groups 1-2 to each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">np13&#9;number of atoms in
groups 1-3 to each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ip13&#9;atom numbers of atoms
in groups 1-3 to each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">np14&#9;number of atoms in
groups 1-4 to each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ip14&#9;atom numbers of atoms
in groups 1-4 to each atom</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">POLPOT&#9;specifics of
polarization functional form</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">poleps&#9;induced dipole
convergence criterion (rms Debyes/atom)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">polsor&#9;induced dipole SOR
convergence acceleration factor</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pgamma&#9;prefactor in
exponential polarization damping term</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">p1scale&#9;field intra-group
scale factor for energy evaluations</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">p2scale&#9;field 1-2 group
scale factor for energy evaluations</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">p3scale&#9;field 1-3 group
scale factor for energy evaluations</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">p4scale&#9;field 1-4 group
scale factor for energy evaluations</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">d1scale&#9;field intra-group
scale factor for direct induced</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">d2scale&#9;field 1-2 group
scale factor for direct induced</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">d3scale&#9;field 1-3 group
scale factor for direct induced</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">d4scale&#9;field 1-4 group
scale factor for direct induced</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">u1scale&#9;field intra-group
scale factor for mutual induced</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">u2scale&#9;field 1-2 group
scale factor for mutual induced</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">u3scale&#9;field 1-3 group
scale factor for mutual induced</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">u4scale&#9;field 1-4 group
scale factor for mutual induced</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">polold&#9;logical flag for
scale factor instead of Thole damping</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">poltyp&#9;type of polarization
potential (direct or mutual)</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">POTENT&#9;usage of each
potential energy component</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_bond&#9;logical flag
governing use of bond stretch potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_angle&#9;logical flag
governing use of angle bend potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_strbnd&#9;logical flag
governing use of stretch-bend potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_urey&#9;logical flag
governing use of Urey-Bradley potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_angang&#9;logical flag
governing use of angle-angle cross term</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_opbend&#9;logical flag
governing use of out-of-plane bend term</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_opdist&#9;logical flag
governing use of out-of-plane distance</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_improp&#9;logical flag
governing use of improper dihedral term</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_imptor&#9;logical flag
governing use of improper torsion term</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_tors&#9;logical flag
governing use of torsional potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_strtor&#9;logical flag
governing use of stretch-torsion term</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_tortor&#9;logical flag
governing use of torsion-torsion term</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_vdw&#9;logical flag
governing use of vdw der Waals potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_charge&#9;logical flag
governing use of charge-charge potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_chgdpl&#9;logical flag
governing use of charge-dipole potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_dipole&#9;logical flag
governing use of dipole-dipole potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_mpole&#9;logical flag
governing use of multipole potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_polar&#9;logical flag
governing use of polarization term</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_rxnfld&#9;logical flag
governing use of reaction field term</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_solv&#9;logical flag
governing use of surface area solvation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_gbsa&#9;logical flag
governing use of GB/SA solvation term</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_metal&#9;logical flag
governing use of ligand field term</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_geom&#9;logical flag
governing use of geometric restraints</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_extra&#9;logical flag
governing use of extra potential term</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_orbit&#9;logical flag
governing use of pisystem computation</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">PRECIS&#9;values of machine
precision tolerances</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tiny&#9;the smallest positive
floating point value</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">small&#9;the smallest relative
floating point spacing</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">huge&#9;the largest relative
floating point spacing</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">REFER&#9;storage of
reference atomic coordinate set</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xref&#9;reference x-coordinate
for each atom in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">yref&#9;reference y-coordinate
for each atom in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zref&#9;reference z-coordinate
for each atom in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nref&#9;total number of atoms
in the reference system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">reftyp&#9;atom type for each
atom in the reference system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">n12ref&#9;number of atoms
bonded to each reference atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">i12ref&#9;atom numbers of
atoms 1-2 connected to each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">refleng&#9;length in
characters of the reference filename</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">refltitle&#9;length in
characters of the reference title string</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">refnam&#9;atom name for each
atom in the reference system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">reffile&#9;base filename for
the reference structure</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">reftitle&#9;title used to
describe the reference structure</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RESDUE&#9;standard
biopolymer residue abbreviations</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">amino&#9;three-letter
abbreviations for amino acids types</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nuclz&#9;three-letter
abbreviations for nucleic acids types</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">amino1&#9;one-letter
abbreviations for amino acids types</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nuclz1&#9;one-letter
abbreviations for nucleic acids types</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RGDDYN&#9;velocities and
momenta for rigid-body MD</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">vcm&#9;current translational
velocity of each rigid-body</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">wcm&#9;current angular
velocity of each rigid-body</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">lm&#9;current angular momenta
of each rigid-body</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">linear&#9;logical flag to mark
group as linear or nonlinear</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RIGID&#9;rigid body
coordinates for atom groups</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xrb&#9;rigid body reference
x-coordinate for each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">yrb&#9;rigid body reference
y-coordinate for each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zrb&#9;rigid body reference
z-coordinate for each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">rbc&#9;current rigid body
coordinates for each atom group</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RING&#9;number and location
of small ring structures</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nring3&#9;total number of
3-membered rings in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iring3&#9;numbers of the atoms
involved in each 3-ring</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nring4&#9;total number of
4-membered rings in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iring4&#9;numbers of the atoms
involved in each 4-ring</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nring5&#9;total number of
5-membered rings in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iring5&#9;numbers of the atoms
involved in each 5-ring</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nring6&#9;total number of
6-membered rings in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iring6&#9;numbers of the atoms
involved in each 6-ring</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ROTATE&#9;molecule
partitions for rotation of a bond</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nrot&#9;total number of atoms
moving when bond rotates</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">rot&#9;atom numbers of atoms
moving when bond rotates</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_short&#9;logical flag
governing use of shortest atom list</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RXNFLD&#9;reaction field
matrix elements and indices</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">b1</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">b2</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ijk</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">RXNPOT&#9;specifics of
reaction field functional form</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">rfsize&#9;radius of reaction
field sphere centered at origin</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">rfbulkd&#9;bulk dielectric
constant of reaction field continuum</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">rfterms&#9;number of terms to
use in reaction field summation</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SCALES&#9;parameter scale
factors for optimization</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">scale&#9;multiplicative factor
for each optimization parameter</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">set_scale&#9;logical flag to
show if scale factors have been set</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SEQUEN&#9;sequence
information for a biopolymer</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nseq&#9;total number of
residues in biopolymer sequences</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nchain&#9;number of separate
biopolymer sequence chains</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ichain&#9;first and last
residue in each biopolymer chain</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">seqtyp&#9;residue type for
each residue in the sequence</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">seq&#9;three-letter code for
each residue in the sequence</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">chnnam&#9;one-letter
identifier for each sequence chain</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SHAKE&#9;definition of
Shake/Rattle constraints</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">krat&#9;ideal distance value
for rattle constraint</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nrat&#9;number of rattle
constraints to be applied</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">irat&#9;atom numbers of atoms
in a rattle constraint</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ratimage&#9;flag to use
minimum image for rattle constraint</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_rattle&#9;logical flag to
set use of rattle contraints</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SHUNT&#9;polynomial
switching function coefficients</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">off&#9;distance at which the
potential energy goes to zero</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">off2&#9;square of distance at
which the potential goes to zero</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">cut&#9;distance at which
switching of the potential begins</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">cut2&#9;square of distance at
which the switching begins</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">c0&#9;zeroth order coefficient
of multiplicative switch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">c1&#9;first order coefficient
of multiplicative switch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">c2&#9;second order coefficient
of multiplicative switch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">c3&#9;third order coefficient
of multiplicative switch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">c4&#9;fourth order coefficient
of multiplicative switch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">c5&#9;fifth order coefficient
of multiplicative switch</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">f0&#9;zeroth order coefficient
of additive switch function</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">f1&#9;first order coefficient
of additive switch function</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">f2&#9;second order coefficient
of additive switch function</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">f3&#9;third order coefficient
of additive switch function</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">f4&#9;fourth order coefficient
of additive switch function</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">f5&#9;fifth order coefficient
of additive switch function</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">f6&#9;sixth order coefficient
of additive switch function</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">f7&#9;seventh order
coefficient of additive switch function</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SIZES&#9;parameter values
to set array dimensions</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&quot;sizes.i&quot; sets
values for critical array dimensions used throughout the software; these
parameters will fix the size of the largest systems that can be handled;
values too large</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">for the computer's memory
and/or swap space to accomodate will result in poor performance or outright failure</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">parameter:&#9;maximum allowed
number of:</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxatm&#9;atoms in the
molecular system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxval&#9;atoms directly
bonded to an atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxgrp&#9;user-defined groups
of atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxtyp&#9;force field atom
type definitions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxclass&#9;force field atom
class definitions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxkey&#9;lines in the keyword file</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxrot&#9;bonds for torsional rotation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxvar&#9;optimization
variables (vector storage)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxopt&#9;optimization
variables (matrix storage)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxhess&#9;off-diagonal
Hessian elements</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxlight&#9;sites for method
of lights neighbors</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxvib&#9;vibrational frequencies</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxgeo&#9;distance geometry points</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxcell&#9;unit cells in
replicated crystal</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxring&#9;3-, 4-, or
5-membered rings</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxfix&#9;geometric restraints</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxbio&#9;biopolymer atom definitions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxres&#9;residues in the macromolecule</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxamino&#9;amino acid residue types</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxnuc&#9;nucleic acid residue types</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxbnd&#9;covalent bonds in
molecular system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxang&#9;bond angles in
molecular system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxtors&#9;torsional angles in
molecular system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxpi&#9;atoms in conjugated pisystem</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxpib&#9;covalent bonds
involving pisystem</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxpit&#9;torsional angles
involving pisystem</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SOLUTE&#9;parameters for
continuum solvation models</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">rsolv&#9;atomic radius of each
atom for continuum solvation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">vsolv&#9;atomic volume of each
atom for continuum solvation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">asolv&#9;atomic solvation
parameters (kcal/mole/Ang**2)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">rborn&#9;Born radius of each
atom for GB/SA solvation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">drb&#9;solvation derivatives
with respect to Born radii</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">doffset&#9;dielectric offset
to continuum solvation atomic radii</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">p1&#9;single-atom scale factor
for analytical Still GB/SA</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">p2&#9;1-2 interaction scale
factor for analytical Still GB/SA</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">p3&#9;1-3 interaction scale
factor for analytical Still GB/SA</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">p4&#9;nonbonded scale factor
for analytical Still GB/SA</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">p5&#9;soft cutoff parameter
for analytical Still GB/SA</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">gpol&#9;polarization
self-energy values for each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">shct&#9;overlap scaling
factors for Hawkins-Cramer-Truhlar GB/SA</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">wace&#9;&quot;omega&quot;
values for atom class pairs for use with ACE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">s2ace&#9;&quot;sigma^2&quot;
values for atom class pairs for use with ACE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">uace&#9;&quot;mu&quot; values
for atom class pairs for use with ACE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">solvtyp&#9;solvation model
(ASP, SASA, ONION, STILL, HCT, ACE)</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">STODYN&#9;frictional
coefficients for SD trajectory</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">friction&#9;global frictional
coefficient for exposed particle</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">gamma&#9;atomic frictional
coefficients for each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_sdarea&#9;logical flag to
use surface area friction scaling</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">STRBND&#9;stretch-bends in
the current structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ksb&#9;force constant for
stretch-bend terms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nstrbnd&#9;total number of
stretch-bend interactions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">isb&#9;angle and bond numbers
used in stretch-bend</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">STRTOR&#9;stretch-torsions
in the current structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kst&#9;1-, 2- and 3-fold
stretch-torsion force constants</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nstrtor&#9;total number of
stretch-torsion interactions</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ist&#9;torsion and bond
numbers used in stretch-torsion</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">SYNTRN&#9;definition of
synchronous transit path</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">t&#9;value of the path
coordinate (0=reactant, 1=product)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">pm&#9;path coordinate for
extra point in quadratic transit</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xmin1&#9;reactant coordinates
as array of optimization variables</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xmin2&#9;product coordinates
as array of optimization variables</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">xm&#9;extra coordinate set for
quadratic synchronous transit</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TITLES&#9;title for the
current molecular system</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ltitle&#9;length in characters
of the nonblank title string</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">title&#9;title used to
describe the current structure</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TORPOT&#9;specifics of
torsional functional forms</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">idihunit&#9;convert improper
dihedral force to kcal/mole/deg**2</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">itorunit&#9;convert improper
torsion amplitudes to kcal/mole</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">torsunit&#9;convert torsional
parameter amplitudes to kcal/mole</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">storunit&#9;convert
stretch-torsion force to kcal/mole/Ang</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TORS&#9;torsional angles
within the current structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tors1&#9;1-fold amplitude and
phase for each torsional angle</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tors2&#9;2-fold amplitude and
phase for each torsional angle</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tors3&#9;3-fold amplitude and
phase for each torsional angle</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tors4&#9;4-fold amplitude and
phase for each torsional angle</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tors5&#9;5-fold amplitude and
phase for each torsional angle</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">tors6&#9;6-fold amplitude and
phase for each torsional angle</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ntors&#9;total number of
torsional angles in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">itors&#9;numbers of the atoms
in each torsional angle</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">TREE&#9;potential smoothing
&amp; search tree levels</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">maxpss&#9;maximum number of
potential smoothing levels</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">etree&#9;energy reference
value at the top of the tree</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ilevel&#9;smoothing
deformation value at each tree level</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nlevel&#9;number of levels of
potential smoothing used</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">UNITS&#9;physical constants
and unit conversions</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">avogadro&#9;Avogadro's number
(N) in particles/mole</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">boltzmann&#9;Boltzmann
constant (kB) in g*Ang**2/ps**2/K/mole</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">gasconst&#9;ideal gas constant
(R) in kcal/mole/K</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">lightspd&#9;speed of light in
vacuum (c) in cm/ps</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bohr&#9;conversion from Bohrs
to Angstroms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">joule&#9;conversion from
calories to joules</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">evolt&#9;conversion from
Hartree to electron-volts</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">hartree&#9;conversion from
Hartree to kcal/mole</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">electric&#9;conversion from
electron**2/Ang to kcal/mole</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">debye&#9;conversion from
electron-Ang to Debyes</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">prescon&#9;conversion from
kcal/mole/Ang**3 to Atm</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">convert&#9;conversion from
kcal to g*Ang**2/ps**2</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">UREY&#9;Urey-Bradley
interactions in the structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">uk&#9;Urey-Bradley force
constants (kcal/mole/Ang**2)</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ul&#9;ideal 1-3 distance
values in Angstroms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nurey&#9;total number of
Urey-Bradley terms in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iury&#9;numbers of the atoms
in each Urey-Bradley interaction</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">URYPOT&#9;specifics of
Urey-Bradley functional form</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">cury&#9;cubic coefficient in
Urey-Bradley potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">qury&#9;quartic coefficient in
Urey-Bradley potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ureyunit&#9;convert
Urey-Bradley constant to kcal/mole/Ang**2</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">USAGE&#9;atoms active
during energy computation</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nuse&#9;number of active atoms
used in energy calculation</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use&#9;true if an atom is
active, false if inactive</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">VDW&#9;van der Waals
parameters for current structure</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">radmin&#9;minimum energy
distance for each atom class pair</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">epsilon&#9;well depth
parameter for each atom class pair</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">radmin4&#9;minimum energy
distance for 1-4 interaction pairs</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">epsilon4&#9;well depth
parameter for 1-4 interaction pairs</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">radhbnd&#9;minimum energy
distance for hydrogen bonding pairs</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">epshbnd&#9;well depth
parameter for hydrogen bonding pairs</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">kred&#9;value of reduction
factor parameter for each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ired&#9;attached atom from
which reduction factor is applied</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nvdw&#9;total number van der
Waals active sites in the system</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ivdw&#9;number of the atom for
each van der Waals active site</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">VDWPOT&#9;specifics of van
der Waals functional form</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">abuck&#9;value of
&quot;A&quot; constant in Buckingham vdw potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">bbuck&#9;value of
&quot;B&quot; constant in Buckingham vdw potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">cbuck&#9;value of
&quot;C&quot; constant in Buckingham vdw potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ghal&#9;value of
&quot;gamma&quot; in buffered 14-7 vdw potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">dhal&#9;value of
&quot;delta&quot; in buffered 14-7 vdw potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">v2scale&#9;factor by which 1-2
vdw interactions are scaled</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">v3scale&#9;factor by which 1-3
vdw interactions are scaled</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">v4scale&#9;factor by which 1-4
vdw interactions are scaled</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">v5scale&#9;factor by which 1-5
vdw interactions are scaled</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">igauss&#9;coefficients of
Gaussian fit to vdw potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ngauss&#9;number of Gaussians
used in fit to vdw potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">vdwtyp&#9;type of van der
Waals potential energy function</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">radtyp&#9;type of parameter
(sigma or R-min) for atomic size</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">radsiz&#9;atomic size provided
as radius or diameter</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">radrule&#9;combining rule for
atomic size parameters</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">epsrule&#9;combining rule for
vdw well depth parameters</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">gausstyp&#9;type of Gaussian
fit to van der Waals potential</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">VIRIAL&#9;components of the
internal virial</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">virx&#9;x-component of the
total internal virial</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">viry&#9;y-component of the
total internal virial</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">virz&#9;z-component of the
total internal virial</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">WARP&#9;parameters for
potential surface smoothing</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">m2&#9;second moment of
Gaussian representing each atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">deform&#9;value of diffusional
smoothing deformation parameter</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">difft&#9;diffusion coefficient
for torsional potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">diffv&#9;diffusion coefficient
for van der Waals potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">diffc&#9;diffusion coefficient
for charge-charge potential</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_deform&#9;flag to use
diffusion smoothed potential terms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">use_gda&#9;flag to use
Straub's GDA instead of Scheraga's DEM</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">XTALS&#9;crystal structures
for parameter fitting</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">e0_lattice&#9;ideal lattice
energy for the current crystal</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">moment_0&#9;ideal dipole
moment for monomer from crystal</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nxtal&#9;number of crystal
structures to be stored</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nvary&#9;number of potential
parameters to optimize</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ivary&#9;index for the types
of potential parameters</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">vary&#9;atom numbers involved
in potential parameters</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iresid&#9;crystal structure to
which each residual refers</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">rsdtyp&#9;experimental
variable for each of the residuals</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">vartyp&#9;type of potential
parameter to be optimized</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ZCLOSE&#9;ring openings and
closures for Z-matrix</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">nadd&#9;number of added bonds
between Z-matrix atoms</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iadd&#9;numbers of the atom
pairs defining added bonds</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ndel&#9;number of bonds
between Z-matrix bonds to delete</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">idel&#9;numbers of the atom
pairs defining deleted bonds</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="Century Schoolbook">ZCOORD&#9;Z-matrix internal
coordinate definitions</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zbond&#9;bond length used to
define each Z-matrix atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">zang&#9;bond angle used to
define each Z-matrix atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">ztors&#9;angle or torsion used
to define Z-matrix atom</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">iz&#9;defining atom numbers
for each Z-matrix atom</FONT></FONT></P>
<TABLE WIDTH=450 CELLPADDING=0>
<TR VALIGN=TOP>
<TD COLSPAN=1 WIDTH="12%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">11.</FONT></FONT></U></B></P>
</TD>
<TD COLSPAN=1 WIDTH="88%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">Index of Function
&amp; Subroutine Calls</FONT></FONT></U></B></P>
</TD></TR>
</TABLE>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;This
section contains an alphabetical cross index listing of the routines called by
each TINKER program, subroutine and function. Routines not present in the left
hand column do not make calls to any other portion of the TINKER package.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT FACE="New Century Schoolbook"><U>Routine</U>&#9;<U>List
of Source Code Units called by this Routine</U></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ACTIVE</B>&#9;GETTEXT&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ADDBASE</B>&#9;ADDBOND&#9;FINDATM&#9;JACOBI&#9;NEWATM&#9;OLDATM</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;OVERLAP&#9;PIALTER&#9;PIMOVE&#9;PITILT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ADDSIDE</B>&#9;ADDBASE&#9;ADDBOND&#9;FATAL&#9;FINDATM&#9;FREEUNIT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;JACOBI&#9;NEWATM&#9;OLDATM&#9;OVERLAP&#9;PIALTER</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;PIMOVE&#9;PITILT&#9;PRTSEQ&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ALCHEMY</B>&#9;ENERGY&#9;FINAL&#9;FREEUNIT&#9;GETTEXT&#9;GETXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;HATOM&#9;HYBRID&#9;INITIAL&#9;MECHANIC&#9;NUMERAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;READXYZ&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ANALYSIS</B>&#9;BOUNDS&#9;EANGANG3&#9;EANGLE3&#9;EBOND3&#9;EBUCK3</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;ECHARGE3&#9;ECHGDPL3&#9;EDIPOLE3&#9;EGAUSS3&#9;EGEOM3</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;EHAL3&#9;EIMPROP3&#9;EIMPTOR3&#9;ELJ3&#9;EMETAL3</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;EMM3HB3&#9;EMPOLE3&#9;EOPBEND3&#9;EOPDIST3&#9;ERXNFLD3</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;ESOLV3&#9;ESTRBND3&#9;ESTRTOR3&#9;ETORS3&#9;EUREY3</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;EXTRA3&#9;PISCF&#9;REPLICA</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ANALYZE</B>&#9;ANALYZ4&#9;ANALYZ6&#9;ANALYZ8&#9;ATOMYZE&#9;ENRGYZE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;FINAL&#9;FREEUNIT&#9;GETXYZ&#9;INITIAL&#9;MECHANIC</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;NEXTARG&#9;PARAMYZE&#9;PROPYZE&#9;READXYZ&#9;SUFFIX</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;TRIMTEXT&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ANGLES</B>&#9;FATAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ANNEAL</B>&#9;BEEMAN&#9;FINAL&#9;GETTEXT&#9;GETXYZ&#9;INITIAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;MDINIT&#9;MDREST&#9;MECHANIC&#9;NEXTARG&#9;SDSTEP</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SHAKEUP&#9;SIGMOID&#9;UPCASE&#9;VERLET</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ARCHIVE</B>&#9;ACTIVE&#9;BASEFILE&#9;FINAL&#9;FREEUNIT&#9;GETTEXT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITIAL&#9;NEXTARG&#9;NUMERAL&#9;PRTMSI&#9;PRTXMOL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;PRTXYZ&#9;READXYZ&#9;SUFFIX&#9;TRIMTEXT&#9;UPCASE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ATTACH</B>&#9;SORT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>BASEFILE</B>&#9;CONTROL&#9;GETKEY&#9;TRIMTEXT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>BEEMAN</B>&#9;GRADIENT&#9;MDSAVE&#9;MDSTAT&#9;PRESSURE&#9;RATTLE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;RATTLE2&#9;TEMPER</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>BETAI</B>&#9;BETACF&#9;GAMMLN</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>BIGBLOCK</B>&#9;CELLATOM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>BONDS</B>&#9;FATAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>BORN</B>&#9;SURFATOM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>BSET</B>&#9;BMAX</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>BSSTEP</B>&#9;FATAL&#9;MMID&#9;PZEXTR</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CALENDAR</B>&#9;IDATE&#9;ITIME</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CERROR</B>&#9;FATAL&#9;TRIMTEXT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CFFTB</B>&#9;CFFTB1</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CFFTB1</B>&#9;PASSB&#9;PASSB2&#9;PASSB3&#9;PASSB4&#9;PASSB5</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CFFTF</B>&#9;CFFTF1</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CFFTF1</B>&#9;PASSF&#9;PASSF2&#9;PASSF3&#9;PASSF4&#9;PASSF5</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CFFTI</B>&#9;CFFTI1</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CHKTREE</B>&#9;LOCALXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CIRPLN</B>&#9;ANORM&#9;DOT&#9;VCROSS&#9;VNORM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CLIMBER</B>&#9;ENERGY&#9;GETREF&#9;LOCALMIN&#9;MAKEINT&#9;MAKEXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CLIMBRGD</B>&#9;ENERGY&#9;LOCALRGD&#9;RIGIDXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CLIMBROT</B>&#9;ENERGY&#9;LOCALROT&#9;MAKEXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CLIMBTOR</B>&#9;CHKTREE&#9;ENERGY&#9;GETREF&#9;LOCALXYZ&#9;MAKEINT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;MAKEXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CLIMBXYZ</B>&#9;CHKTREE&#9;ENERGY&#9;GETREF&#9;LOCALXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CLUSTER</B>&#9;CUTOFFS&#9;FATAL&#9;GETNUMB&#9;GETTEXT&#9;SORT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SORT3&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>COMMAND</B>&#9;GETARG&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>COMPRESS</B>&#9;CERROR&#9;GETTOR</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CONNECT</B>&#9;SORT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CONNOLLY</B>&#9;COMPRESS&#9;CONTACT&#9;NEIGHBOR&#9;PLACE&#9;SADDLES</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;TORUS&#9;VAM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CONTACT</B>&#9;ANORM&#9;CERROR&#9;PTINCY</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CONTROL</B>&#9;GETTEXT&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>COORDS</B>&#9;GYRATE&#9;RMSERROR</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CORRELATE</B>&#9;FINAL&#9;INITIAL&#9;NEXTARG&#9;PROPERTY&#9;READBLK</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;TRIMTEXT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CRYSTAL</B>&#9;BIGBLOCK&#9;BOUNDS&#9;FIELD&#9;FINAL&#9;FREEUNIT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;GETTEXT&#9;GETXYZ&#9;INITIAL&#9;KATOM&#9;LATTICE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;MOLECULE&#9;NEXTARG&#9;PRTXYZ&#9;SYMMETRY&#9;UNITCELL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>CUTOFFS</B>&#9;GETTEXT&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>DEPTH</B>&#9;DOT&#9;VCROSS&#9;VNORM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>DIAGQ</B>&#9;GETIME&#9;SETIME</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>DIFFEQ</B>&#9;BSSTEP&#9;DERIVS&#9;GDASTAT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>DIFFUSE</B>&#9;BASEFILE&#9;FATAL&#9;FIELD&#9;FINAL&#9;FREEUNIT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;GETWORD&#9;INITIAL&#9;KATOM&#9;MOLECULE&#9;NEXTARG</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;READXYZ&#9;SUFFIX&#9;UNITCELL&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>DISTGEOM</B>&#9;ACTIVE&#9;ANGLES&#9;ATTACH&#9;BONDS&#9;EMBED</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;FATAL&#9;FINAL&#9;FREEUNIT&#9;GEODESIC&#9;GETIME</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;GETTEXT&#9;GETXYZ&#9;GRAFIC&#9;IMPOSE&#9;INITIAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;KCHIRAL&#9;KGEOM&#9;MAKEREF&#9;NEXTARG&#9;NUMERAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;PRTXYZ&#9;SETIME&#9;TORSIONS&#9;TRIFIX&#9;UPCASE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>DMDUMP</B>&#9;GRAFIC</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>DOCUMENT</B>&#9;FINAL&#9;FREEUNIT&#9;GETPRM&#9;GETTEXT&#9;GETWORD</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITIAL&#9;LOWCASE&#9;NEXTARG&#9;NEXTTEXT&#9;PRTPRM</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SORT6&#9;SORT7&#9;SORT9&#9;SUFFIX&#9;TRIMTEXT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>DSTMAT</B>&#9;GETIME&#9;GETNUMB&#9;GETTEXT&#9;INVBETA&#9;LOWCASE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;RANDOM&#9;SETIME&#9;SORT2&#9;TRIFIX&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>DYNAMIC</B>&#9;BEEMAN&#9;FINAL&#9;GETXYZ&#9;INITIAL&#9;MDINIT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;MDREST&#9;MECHANIC&#9;NEXTARG&#9;RGDSTEP&#9;SDSTEP</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SHAKEUP&#9;VERLET</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EANGANG</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EANGANG1</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EANGANG2</B>&#9;EANGANG2A&#9;GROUP</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EANGANG2A</B>&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EANGANG3</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EANGLE</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EANGLE1</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EANGLE2</B>&#9;EANGLE2A&#9;EANGLE2B&#9;GROUPS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EANGLE2A</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EANGLE2B</B>&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EANGLE3</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EBOND</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EBOND1</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EBOND2</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EBOND3</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EBUCK</B>&#9;EBUCK0A&#9;EBUCK0B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EBUCK0A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EBUCK0B</B>&#9;GROUPS&#9;LIGHTS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EBUCK1</B>&#9;EBUCK1A&#9;EBUCK1B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EBUCK1A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EBUCK1B</B>&#9;GROUPS&#9;LIGHTS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EBUCK2</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EBUCK3</B>&#9;EBUCK3A&#9;EBUCK3B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EBUCK3A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EBUCK3B</B>&#9;GROUPS&#9;LIGHTS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE</B>&#9;ECHARGE0A&#9;ECHARGE0B&#9;ECHARGE0C&#9;ECHARGE0D</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE0A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE0B</B>&#9;GROUPS&#9;LIGHTS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE0C</B>&#9;ERF&#9;GROUPS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE0D</B>&#9;EPME&#9;ERFC&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE1</B>&#9;ECHARGE1A&#9;ECHARGE1B&#9;ECHARGE1C&#9;ECHARGE1D</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE1A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE1B</B>&#9;GROUPS&#9;LIGHTS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE1C</B>&#9;ERF&#9;GROUPS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE1D</B>&#9;EPME1&#9;ERFC&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE2</B>&#9;ECHARGE2A&#9;ECHARGE2B&#9;ECHARGE2C</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE2A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE2B</B>&#9;ERF&#9;GROUPS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE2C</B>&#9;ERFC&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE3</B>&#9;ECHARGE3A&#9;ECHARGE3B&#9;ECHARGE3C&#9;ECHARGE3D</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE3A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE3B</B>&#9;GROUPS&#9;LIGHTS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE3C</B>&#9;ERF&#9;GROUPS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHARGE3D</B>&#9;EPME3&#9;ERFC&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHGDPL</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHGDPL1</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHGDPL2</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ECHGDPL3</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EDIPOLE</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EDIPOLE1</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EDIPOLE2</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EDIPOLE3</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGAUSS</B>&#9;GROUPS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGAUSS1</B>&#9;GROUPS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGAUSS2</B>&#9;GROUPS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGAUSS3</B>&#9;GROUPS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGBSA0A</B>&#9;GROUPS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGBSA0B</B>&#9;ERF&#9;GROUPS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGBSA1A</B>&#9;GROUPS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGBSA1B</B>&#9;ERF&#9;GROUPS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGBSA2A</B>&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGBSA2B</B>&#9;ERF</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGBSA3A</B>&#9;GROUPS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGBSA3B</B>&#9;ERF&#9;GROUPS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGEOM</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGEOM1</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGEOM2</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EGEOM3</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EHAL</B>&#9;EHAL0A&#9;EHAL0B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EHAL0A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EHAL0B</B>&#9;GROUPS&#9;LIGHTS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EHAL1</B>&#9;EHAL1A&#9;EHAL1B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EHAL1A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EHAL1B</B>&#9;GROUPS&#9;LIGHTS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EHAL2</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EHAL3</B>&#9;EHAL3A&#9;EHAL3B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EHAL3A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EHAL3B</B>&#9;GROUPS&#9;LIGHTS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EIGEN</B>&#9;GETIME&#9;POWER&#9;SETIME</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EIGENRGD</B>&#9;DIAGQ&#9;HESSRGD</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EIGENROT</B>&#9;DIAGQ&#9;HESSROT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EIGENROT</B>&#9;DIAGQ&#9;HESSROT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EIGENTOR</B>&#9;DIAGQ&#9;HESSROT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EIGENXYZ</B>&#9;DIAGQ&#9;HESSIAN</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EIMPROP</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EIMPROP1</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EIMPROP2</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EIMPROP3</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EIMPTOR</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EIMPTOR1</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EIMPTOR2</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EIMPTOR3</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ELJ</B>&#9;ELJ0A&#9;ELJ0B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ELJ0A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ELJ0B</B>&#9;GROUPS&#9;LIGHTS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ELJ1</B>&#9;ELJ1A&#9;ELJ1B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ELJ1A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ELJ1B</B>&#9;GROUPS&#9;LIGHTS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ELJ2</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ELJ3</B>&#9;ELJ3A&#9;ELJ3B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ELJ3A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ELJ3B</B>&#9;GROUPS&#9;LIGHTS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMBED</B>&#9;BNDERR&#9;CHIRER&#9;CHKSIZE&#9;COORDS&#9;DMDUMP</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;DSTMAT&#9;EIGEN&#9;EXPLORE&#9;FRACDIST&#9;FREEUNIT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;GETIME&#9;GYRATE&#9;IMPOSE&#9;LOCERR&#9;MAJORIZE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;METRIC&#9;NUMERAL&#9;PRTXYZ&#9;REFINE&#9;RMSERROR</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SETIME&#9;TORSER&#9;VDWERR</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMETAL</B>&#9;FATAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMETAL1</B>&#9;FATAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMETAL3</B>&#9;EMETAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMM3HB</B>&#9;EMM3HB0A&#9;EMM3HB0B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMM3HB0A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMM3HB0B</B>&#9;GROUPS&#9;LIGHTS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMM3HB1</B>&#9;EMM3HB1A&#9;EMM3HB1B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMM3HB1A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMM3HB1B</B>&#9;GROUPS&#9;LIGHTS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMM3HB2</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMM3HB3</B>&#9;EMM3HB3A&#9;EMM3HB3B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMM3HB3A</B>&#9;GROUPS&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMM3HB3B</B>&#9;GROUPS&#9;LIGHTS&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMPOLE</B>&#9;EMPOLE0A&#9;EMPOLE0B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMPOLE0A</B>&#9;GROUPS&#9;IMAGE&#9;INDUCE&#9;ROTMAT&#9;ROTPOLE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMPOLE0B</B>&#9;EREAL&#9;ERECIP&#9;INDUCE&#9;ROTMAT&#9;ROTPOLE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMPOLE1</B>&#9;EMPOLE1A&#9;EMPOLE1B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMPOLE1A</B>&#9;GROUPS&#9;IMAGE&#9;INDUCE&#9;ROTMAT&#9;ROTPOLE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SWITCH&#9;TORQUE&#9;TORQUE1</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMPOLE1B</B>&#9;EREAL1&#9;ERECIP1&#9;INDUCE&#9;ROTMAT&#9;ROTPOLE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;TORQUE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMPOLE2</B>&#9;EMPOLE2A</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMPOLE2A</B>&#9;GROUPS&#9;IMAGE&#9;ROTMAT&#9;ROTPOLE&#9;SWITCH</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;TORQUE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMPOLE3</B>&#9;EMPOLE3A&#9;EMPOLE3B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMPOLE3A</B>&#9;GROUPS&#9;IMAGE&#9;INDUCE&#9;ROTMAT&#9;ROTPOLE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EMPOLE3B</B>&#9;EREAL3&#9;ERECIP3&#9;INDUCE&#9;ROTMAT&#9;ROTPOLE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ENERGY</B>&#9;BOUNDS&#9;EANGANG&#9;EANGLE&#9;EBOND&#9;EBUCK</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;ECHARGE&#9;ECHGDPL&#9;EDIPOLE&#9;EGAUSS&#9;EGEOM</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;EHAL&#9;EIMPROP&#9;EIMPTOR&#9;ELJ&#9;EMETAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;EMM3HB&#9;EMPOLE&#9;EOPBEND&#9;EOPDIST&#9;ERXNFLD</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;ESOLV&#9;ESTRBND&#9;ESTRTOR&#9;ETORS&#9;EUREY</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;EXTRA&#9;PISCF&#9;REPLICA</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ENRGYZE</B>&#9;ANALYSIS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EOPBEND</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EOPBEND1</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EOPBEND2</B>&#9;EOPBEND2A&#9;GROUPS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EOPBEND2A</B>&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EOPBEND3</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EOPDIST</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EOPDIST1</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EOPDIST2</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EOPDIST3</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EPME</B>&#9;BSPLINE&#9;FFTFRONT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EPME1</B>&#9;BSPLINE1&#9;FFTBACK&#9;FFTFRONT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EPME3</B>&#9;BSPLINE&#9;FFTFRONT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EPUCLC</B>&#9;ANORM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EREAL</B>&#9;ERFC&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EREAL1</B>&#9;ERFC&#9;IMAGE&#9;SWITCH&#9;TORQUE&#9;TORQUE1</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EREAL3</B>&#9;ERFC&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ERECIP1</B>&#9;TORQUE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ERF</B>&#9;ERFCORE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ERFC</B>&#9;ERFCORE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ERFIK</B>&#9;D1D2&#9;RFINDEX</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ERFINV</B>&#9;ERF&#9;FATAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ERXNFLD</B>&#9;ERFIK&#9;IJK_PT&#9;ROTMAT&#9;ROTPOLE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ERXNFLD3</B>&#9;ERFIK&#9;IJK_PT&#9;ROTMAT&#9;ROTPOLE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ESOLV</B>&#9;BORN&#9;EGBSA0A&#9;EGBSA0B&#9;SURFACE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ESOLV1</B>&#9;BORN&#9;BORN1&#9;EGBSA1A&#9;EGBSA1B&#9;SURFACE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ESOLV2</B>&#9;EGBSA2A&#9;EGBSA2B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ESOLV3</B>&#9;BORN&#9;EGBSA3A&#9;EGBSA3B&#9;SURFACE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ESTRBND</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ESTRBND1</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ESTRBND2</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ESTRBND3</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ESTRTOR</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ESTRTOR1</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ESTRTOR2</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ESTRTOR3</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ETORS</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ETORS1</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ETORS2</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ETORS3</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EUREY</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EUREY1</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EUREY2</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EUREY3</B>&#9;GROUPS&#9;IMAGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EWALDCOF</B>&#9;ERFC</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>EXPLORE</B>&#9;INITERR&#9;MIDERR&#9;SIGMOID&#9;TOTERR</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>FFTBACK</B>&#9;CFFTB</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>FFTFRONT</B>&#9;CFFTF</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>FFTSETUP</B>&#9;CFFTI</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>FIELD</B>&#9;GETPRM&#9;PRMKEY</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>FRACDIST</B>&#9;DIST2&#9;TRIMTEXT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>FREEUNIT</B>&#9;FATAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GDA</B>&#9;DIFFEQ&#9;FINAL&#9;FREEUNIT&#9;GDASTAT&#9;GETTEXT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;GETXYZ&#9;INITIAL&#9;MECHANIC&#9;NEXTARG&#9;NUMERAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;PRTXYZ&#9;RANDOM&#9;TNCG&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GDA1</B>&#9;GRADIENT&#9;HESSIAN</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GDA2</B>&#9;GRADIENT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GDA3</B>&#9;HESSIAN</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GDASTAT</B>&#9;ENERGY&#9;GYRATE&#9;WRITEOUT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GEODESIC</B>&#9;MINPATH&#9;SORT3</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETBASE</B>&#9;PDBATM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETIME</B>&#9;CLOCK</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETINT</B>&#9;BASEFILE&#9;CHKXYZ&#9;CONNECT&#9;FATAL&#9;FREEUNIT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;MAKEXYZ&#9;NEXTARG&#9;READINT&#9;SUFFIX&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETKEY</B>&#9;FATAL&#9;FREEUNIT&#9;GETTEXT&#9;SUFFIX&#9;TRIMTEXT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETMOL2</B>&#9;BASEFILE&#9;FREEUNIT&#9;NEXTARG&#9;READMOL2&#9;SUFFIX</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETNUCH</B>&#9;PDBATM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETNUMB</B>&#9;TRIMTEXT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETPDB</B>&#9;BASEFILE&#9;FREEUNIT&#9;NEXTARG&#9;READPDB&#9;SUFFIX</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETPRB</B>&#9;DIST2&#9;DOT&#9;GETTOR&#9;VCROSS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETPRM</B>&#9;FREEUNIT&#9;GETTEXT&#9;INITPRM&#9;NEXTARG&#9;READPRM</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SUFFIX&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETPROH</B>&#9;PDBATM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETSEQ</B>&#9;GETWORD&#9;TRIMTEXT&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETSEQN</B>&#9;GETTEXT&#9;GETWORD&#9;TRIMTEXT&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETSIDE</B>&#9;PDBATM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETTOR</B>&#9;DIST2</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GETXYZ</B>&#9;BASEFILE&#9;FATAL&#9;FREEUNIT&#9;NEXTARG&#9;READXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SUFFIX&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GRADIENT</B>&#9;BOUNDS&#9;EANGANG1&#9;EANGLE1&#9;EBOND1&#9;EBUCK1</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;ECHARGE1&#9;ECHGDPL1&#9;EDIPOLE1&#9;EGAUSS1&#9;EGEOM1</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;EHAL1&#9;EIMPROP1&#9;EIMPTOR1&#9;ELJ1&#9;EMETAL1</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;EMM3HB1&#9;EMPOLE1&#9;EOPBEND1&#9;EOPDIST1&#9;ERXNFLD1</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;ESOLV1&#9;ESTRBND1&#9;ESTRTOR1&#9;ETORS1&#9;EUREY1</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;EXTRA1&#9;PISCF&#9;REPLICA</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GRADRGD</B>&#9;GRADIENT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GRADROT</B>&#9;GRADIENT&#9;ROTLIST</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>HANGLE</B>&#9;NUMERAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>HBOND</B>&#9;NUMERAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>HDIPOLE</B>&#9;NUMERAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>HESSIAN</B>&#9;BORN&#9;BOUNDS&#9;EANGANG2&#9;EANGLE2&#9;EBOND2</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;EBUCK2&#9;ECHARGE2&#9;ECHGDPL2&#9;EDIPOLE2&#9;EGAUSS2</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;EGEOM2&#9;EHAL2&#9;EIMPROP2&#9;EIMPTOR2&#9;ELJ2</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;EMETAL2&#9;EMM3HB2&#9;EMPOLE2&#9;EOPBEND2&#9;EOPDIST2</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;ERXNFLD2&#9;ESOLV2&#9;ESTRBND2&#9;ESTRTOR2&#9;ETORS2</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;EUREY2&#9;EXTRA2&#9;FATAL&#9;INDUCE&#9;PISCF</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;REPLICA</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>HESSRGD</B>&#9;GRADRGD&#9;RIGIDXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>HESSROT</B>&#9;GRADROT&#9;MAKEXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>HIMPTOR</B>&#9;NUMERAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>HSTRTOR</B>&#9;NUMERAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>HTORS</B>&#9;NUMERAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>HYBRID</B>&#9;HANGLE&#9;HATOM&#9;HBOND&#9;HCHARGE&#9;HDIPOLE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;HIMPTOR&#9;HSTRBND&#9;HSTRTOR&#9;HTORS&#9;HVDW</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>IMPOSE</B>&#9;CENTER&#9;QUATFIT&#9;RMSFIT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>INDUCE</B>&#9;INDUCE0A&#9;INDUCE0B</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>INDUCE0A</B>&#9;FATAL&#9;GROUPS&#9;IMAGE&#9;PRTERR&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>INDUCE0B</B>&#9;FATAL&#9;PRTERR&#9;UDIRECT1&#9;UDIRECT2&#9;UMUTUAL1</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;UMUTUAL2</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>INEDGE</B>&#9;CERROR</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>INERTIA</B>&#9;JACOBI</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>INITERR</B>&#9;LOCERR&#9;TORSER</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>INITIAL</B>&#9;COMMAND&#9;INITRES&#9;PRECISE&#9;PROMO</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>INITROT</B>&#9;FATAL&#9;NEXTARG&#9;ROTCHECK&#9;ROTLIST</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>INTEDIT</B>&#9;FIELD&#9;FINAL&#9;FREEUNIT&#9;GEOMETRY&#9;GETINT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;GETWORD&#9;INITIAL&#9;MAKEXYZ&#9;NUMBER&#9;PRTINT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;TRIMTEXT&#9;UPCASE&#9;VERSION&#9;ZHELP&#9;ZVALUE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>INTXYZ</B>&#9;FINAL&#9;FREEUNIT&#9;GETINT&#9;INITIAL&#9;PRTXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>INVBETA</B>&#9;BETAI&#9;GAMMLN</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>INVERT</B>&#9;FATAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>IPEDGE</B>&#9;CERROR</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KANGANG</B>&#9;GETTEXT&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KANGLE</B>&#9;GETTEXT&#9;NUMERAL&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KATOM</B>&#9;GETNUMB&#9;GETSTRING&#9;GETTEXT&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KBOND</B>&#9;GETTEXT&#9;KENEG&#9;NUMERAL&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KCHARGE</B>&#9;GETTEXT&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KDIPOLE</B>&#9;GETTEXT&#9;NUMERAL&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KENEG</B>&#9;GETTEXT&#9;NUMERAL&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KEWALD</B>&#9;EWALDCOF&#9;FATAL&#9;FFTSETUP&#9;GETTEXT&#9;MODULI</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KGEOM</B>&#9;FATAL&#9;GEOMETRY&#9;GETTEXT&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KIMPROP</B>&#9;GETTEXT&#9;NUMERAL&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KIMPTOR</B>&#9;GETTEXT&#9;NUMERAL&#9;TORPHASE&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KMPOLE</B>&#9;GETTEXT&#9;NUMBER&#9;NUMERAL&#9;RANDOM&#9;SORT3</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KOPBEND</B>&#9;GETTEXT&#9;NUMBER&#9;NUMERAL&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KOPDIST</B>&#9;GETTEXT&#9;NUMERAL&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KORBIT</B>&#9;GETTEXT&#9;NUMERAL&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KPOLAR</B>&#9;GETTEXT&#9;POLARGRP&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KSOLV</B>&#9;GETTEXT&#9;GETWORD&#9;KANGLE&#9;KBOND&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KSTRBND</B>&#9;GETTEXT&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KSTRTOR</B>&#9;GETTEXT&#9;NUMERAL&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KTORS</B>&#9;GETTEXT&#9;NUMERAL&#9;TORPHASE&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KUREY</B>&#9;GETTEXT&#9;NUMERAL&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>KVDW</B>&#9;GETTEXT&#9;NUMBER&#9;NUMERAL&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>LBFGS</B>&#9;COMMENT'&#9;GETTEXT&#9;SEARCH&#9;UPCASE&#9;WRITEOUT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>LIGASE</B>&#9;FINDATM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>LIGHTS</B>&#9;FATAL&#9;SORT2&#9;SORT5</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>LMSTEP</B>&#9;PRECISE&#9;QRSOLVE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>LOCALMIN</B>&#9;GRADIENT&#9;TNCG</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>LOCALRGD</B>&#9;OCVM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>LOCALROT</B>&#9;OCVM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>LOCALXYZ</B>&#9;TNCG</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MAJORIZE</B>&#9;GETIME&#9;GYRATE&#9;RMSERROR&#9;SETIME</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MAKEINT</B>&#9;ADJACENT&#9;FATAL&#9;GEOMETRY&#9;GETTEXT&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MAKEPDB</B>&#9;ATTACH&#9;FREEUNIT&#9;GETBASE&#9;GETNUCH&#9;GETPROH</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;GETSIDE&#9;NUMERAL&#9;PDBATM&#9;READSEQ&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MAKEXYZ</B>&#9;XYZATM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MAPCHECK</B>&#9;FREEUNIT&#9;NUMERAL&#9;PRTXYZ&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MAXWELL</B>&#9;ERFINV&#9;RANDOM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MDINIT</B>&#9;FREEUNIT&#9;GETTEXT&#9;GETWORD&#9;GRADIENT&#9;GRPLINE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;LATTICE&#9;MAXWELL&#9;MDREST&#9;NUMERAL&#9;RANVEC</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;READDYN&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MDREST</B>&#9;INVERT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MDSAVE</B>&#9;FATAL&#9;FREEUNIT&#9;NUMERAL&#9;OPENEND&#9;PRTDYN</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;PRTXYZ&#9;SUFFIX&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MEASFN</B>&#9;CERROR&#9;TRIPLE&#9;VCROSS&#9;VECANG&#9;VNORM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MEASFP</B>&#9;CERROR&#9;DOT&#9;VCROSS&#9;VECANG&#9;VNORM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MEASFS</B>&#9;CERROR&#9;DOT&#9;VECANG&#9;VNORM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MEASPM</B>&#9;VCROSS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MECHANIC</B>&#9;ACTIVE&#9;ANGLES&#9;ATTACH&#9;BONDS&#9;CLUSTER</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;CUTOFFS&#9;FATAL&#9;FIELD&#9;KANGANG&#9;KANGLE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;KATOM&#9;KBOND&#9;KCHARGE&#9;KDIPOLE&#9;KEWALD</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;KGEOM&#9;KIMPROP&#9;KIMPTOR&#9;KMETAL&#9;KMPOLE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;KOPBEND&#9;KOPDIST&#9;KORBIT&#9;KPOLAR&#9;KSOLV</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;KSTRBND&#9;KSTRTOR&#9;KTORS&#9;KUREY&#9;KVDW</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;LATTICE&#9;MOLECULE&#9;MUTATE&#9;ORBITAL&#9;POLYMER</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;RINGS&#9;SMOOTH&#9;TORSIONS&#9;UNITCELL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MERGE</B>&#9;FATAL&#9;GETREF</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MIDERR</B>&#9;BNDERR&#9;CHIRER&#9;LOCERR&#9;TORSER</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MINIMIZ1</B>&#9;GRADIENT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MINIMIZE</B>&#9;FINAL&#9;FREEUNIT&#9;GETXYZ&#9;GRADIENT&#9;INITIAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;LBFGS&#9;MECHANIC&#9;NEXTARG&#9;PRTXYZ&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MINIROT</B>&#9;FINAL&#9;FREEUNIT&#9;GETINT&#9;GRADROT&#9;INITIAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITROT&#9;LBFGS&#9;MECHANIC&#9;NEXTARG&#9;PRTINT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MINIROT1</B>&#9;GRADROT&#9;MAKEXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MINRIGID</B>&#9;FINAL&#9;FREEUNIT&#9;GETXYZ&#9;GRADRGD&#9;INITIAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;LBFGS&#9;MECHANIC&#9;NEXTARG&#9;ORIENT&#9;PRTXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MINRIGID1</B>&#9;GRADRGD&#9;RIGIDXYZ&#9;XYZRIGID</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MMID</B>&#9;DERIVS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MODECART</B>&#9;CLIMBXYZ&#9;EIGENXYZ&#9;GETREF&#9;IMPOSE&#9;MAKEREF</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MODEROT</B>&#9;CLIMBROT&#9;EIGENROT&#9;MAKEXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MODESRCH</B>&#9;CLIMBER&#9;EIGENROT&#9;MAKEINT&#9;MAKEREF&#9;MAPCHECK</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MODETORS</B>&#9;CLIMBTOR&#9;EIGENTOR&#9;GETREF&#9;IMPOSE&#9;MAKEINT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;MAKEREF</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MODULI</B>&#9;BSPLINE&#9;DFTMOD</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MOLECULE</B>&#9;SORT&#9;SORT3</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MOLUIND</B>&#9;UFIELD</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MOMENTS</B>&#9;INDUCE&#9;JACOBI&#9;ROTMAT&#9;ROTPOLE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>MUTATE</B>&#9;GETTEXT&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>NEIGHBOR</B>&#9;CERROR&#9;DIST2</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>NEWATM</B>&#9;ADDBOND&#9;XYZATM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>NEWTON</B>&#9;FINAL&#9;FREEUNIT&#9;GETTEXT&#9;GETXYZ&#9;GRADIENT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITIAL&#9;MECHANIC&#9;NEXTARG&#9;PRTXYZ&#9;TNCG</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>NEWTON1</B>&#9;GRADIENT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>NEWTON2</B>&#9;HESSIAN</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>NEWTROT</B>&#9;FINAL&#9;FREEUNIT&#9;GETINT&#9;GETTEXT&#9;GRADROT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITIAL&#9;INITROT&#9;MECHANIC&#9;NEXTARG&#9;PRTINT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;TNCG&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>NEWTROT1</B>&#9;GRADROT&#9;MAKEXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>NEWTROT2</B>&#9;HESSROT&#9;MAKEXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>NORMAL</B>&#9;RANDOM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>NUCBASE</B>&#9;OCVM&#9;ORIENT&#9;POTOFF&#9;ZATOM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>NUCCHAIN</B>&#9;NUCBASE&#9;OCVM&#9;ORIENT&#9;ZATOM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>NUCLEIC</B>&#9;BASEFILE&#9;CONNECT&#9;DELETE&#9;FIELD&#9;FREEUNIT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;GETKEY&#9;GETSEQN&#9;INITIAL&#9;MAKEINT&#9;MAKEXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;MOLECULE&#9;NEXTARG&#9;NUCCHAIN&#9;PRTINT&#9;PRTSEQ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;PRTXYZ&#9;TRIMTEXT&#9;VERSION&#9;WATSON</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>NUMBER</B>&#9;TRIMTEXT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>OCVM</B>&#9;GETTEXT&#9;PRECISE&#9;UPCASE&#9;WRITEOUT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>OLDATM</B>&#9;ADDBOND&#9;FATAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>OPTIMIZ1</B>&#9;GRADIENT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>OPTIMIZE</B>&#9;FATAL&#9;FINAL&#9;FREEUNIT&#9;GETXYZ&#9;GRADIENT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITIAL&#9;MECHANIC&#9;NEXTARG&#9;OCVM&#9;PRTXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>OPTIROT</B>&#9;FATAL&#9;FINAL&#9;FREEUNIT&#9;GETINT&#9;GRADROT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITIAL&#9;INITROT&#9;MECHANIC&#9;NEXTARG&#9;OCVM</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;PRTINT&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>OPTIROT1</B>&#9;GRADROT&#9;MAKEXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>OPTRIGID</B>&#9;FATAL&#9;FINAL&#9;FREEUNIT&#9;GETXYZ&#9;GRADRGD</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITIAL&#9;MECHANIC&#9;NEXTARG&#9;OCVM&#9;ORIENT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;PRTXYZ&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>OPTRIGID1</B>&#9;GRADRGD&#9;RIGIDXYZ&#9;XYZRIGID</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ORBITAL</B>&#9;FATAL&#9;GETTEXT&#9;PIPLANE&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ORIENT</B>&#9;XYZRIGID</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>OVERLAP</B>&#9;SLATER</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PATH</B>&#9;FINAL&#9;GETXYZ&#9;IMPOSE&#9;INITIAL&#9;INVERT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;LBFGS&#9;MECHANIC&#9;NEXTARG&#9;ORTHOG&#9;POTNRG</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;WRITEOUT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PATH1</B>&#9;POTNRG</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PATHPNT</B>&#9;OCVM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PATHSCAN</B>&#9;PATHPNT&#9;SADDLE1&#9;TANGENT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PATHVAL</B>&#9;IMPOSE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PDBXYZ</B>&#9;CHKXYZ&#9;DELETE&#9;FIELD&#9;FINAL&#9;FREEUNIT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;GETNUMB&#9;GETPDB&#9;INITIAL&#9;LIGASE&#9;PRTXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;RIBOSOME&#9;SORT&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PIPLANE</B>&#9;FATAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PISCF</B>&#9;NEWATM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PITILT</B>&#9;OLDATM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PLACE</B>&#9;CERROR&#9;DIST2&#9;GETPRB&#9;GETTOR&#9;INEDGE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>POLARGRP</B>&#9;SORT&#9;SORT8</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>POLARIZE</B>&#9;FATAL&#9;GETXYZ&#9;INITIAL&#9;JACOBI&#9;MECHANIC</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;MOLUIND</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>POLYMER</B>&#9;FATAL&#9;GETTEXT&#9;IMAGE&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>POTNRG</B>&#9;GRADIENT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>POWER</B>&#9;RANDOM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PRECOND</B>&#9;CHOLESKY&#9;COLUMN</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PRESSURE</B>&#9;LATTICE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PRMKEY</B>&#9;GETTEXT&#9;GETWORD&#9;POTOFF&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PROCHAIN</B>&#9;GETTEXT&#9;PROSIDE&#9;UPCASE&#9;ZATOM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PROJCT</B>&#9;DOT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PROPYZE</B>&#9;GYRATE&#9;INERTIA&#9;MOMENTS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PROSIDE</B>&#9;FREEUNIT&#9;PRTINT&#9;PRTXYZ&#9;VERSION&#9;ZATOM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PROTEIN</B>&#9;BASEFILE&#9;CHKXYZ&#9;CONNECT&#9;DELETE&#9;FIELD</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;FINAL&#9;FREEUNIT&#9;GETKEY&#9;GETSEQ&#9;INITIAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;MAKEINT&#9;MAKEXYZ&#9;NEXTARG&#9;PROCHAIN&#9;PRTINT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;PRTSEQ&#9;PRTXYZ&#9;TRIMTEXT&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PRTDYN</B>&#9;ZATOM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PRTERR</B>&#9;ZATOM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PRTINT</B>&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PRTMOL2</B>&#9;NUMBER&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PRTMSI</B>&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PRTPDB</B>&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PRTPRM</B>&#9;NUMBER</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PRTSEQ</B>&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PRTXMOL</B>&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PRTXYZ</B>&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PSS</B>&#9;ACTIVE&#9;FINAL&#9;GETTEXT&#9;GETXYZ&#9;IMPOSE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITIAL&#9;INITROT&#9;LOCALXYZ&#9;MAKEINT&#9;MAKEREF</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;MECHANIC&#9;MODECART&#9;MODETORS&#9;NEXTARG&#9;PSSWRITE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SIGMOID&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PSS1</B>&#9;GRADIENT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PSS2</B>&#9;HESSIAN</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PSSRGD1</B>&#9;GRADRGD&#9;RIGIDXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PSSRIGID</B>&#9;FINAL&#9;FREEUNIT&#9;GETTEXT&#9;GETXYZ&#9;IMPOSE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITIAL&#9;MAKEREF&#9;MECHANIC&#9;NEXTARG&#9;NUMERAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;OCVM&#9;ORIENT&#9;PRTXYZ&#9;RGDSRCH&#9;RIGIDXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SIGMOID&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PSSROT</B>&#9;FINAL&#9;FREEUNIT&#9;GETINT&#9;GETTEXT&#9;IMPOSE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITIAL&#9;INITROT&#9;MAKEREF&#9;MAKEXYZ&#9;MECHANIC</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;MODEROT&#9;NEXTARG&#9;NUMERAL&#9;OCVM&#9;PRTXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PSSROT1</B>&#9;GRADROT&#9;MAKEXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PSSWRITE</B>&#9;FREEUNIT&#9;NUMERAL&#9;PRTXYZ&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>PTINCY</B>&#9;DOT&#9;EPUCLC&#9;PROJCT&#9;ROTANG</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>QUATFIT</B>&#9;JACOBI</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>RADIAL</B>&#9;FINAL&#9;FREEUNIT&#9;GETWORD&#9;GETXYZ&#9;IMAGE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITIAL&#9;LATTICE&#9;MOLECULE&#9;NUMERAL&#9;READXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;TRIMTEXT&#9;UNITCELL&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>RANDOM</B>&#9;CALENDAR&#9;GETTEXT&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>RANVEC</B>&#9;RANDOM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>RATTLE</B>&#9;FATAL&#9;IMAGE&#9;PRTERR</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>RATTLE2</B>&#9;FATAL&#9;IMAGE&#9;PRTERR</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>READBLK</B>&#9;FATAL&#9;FREEUNIT&#9;GETWORD&#9;NUMERAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>READDYN</B>&#9;FATAL&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>READINT</B>&#9;FATAL&#9;GETTEXT&#9;GETWORD&#9;NEXTTEXT&#9;TRIMTEXT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>READMOL2</B>&#9;FATAL&#9;GETTEXT&#9;GETWORD&#9;SORT&#9;TRIMTEXT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>READPDB</B>&#9;FATAL&#9;FIXPDB&#9;GETTEXT&#9;TRIMTEXT&#9;UPCASE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>READPRM</B>&#9;FATAL&#9;GETNUMB&#9;GETSTRING&#9;GETTEXT&#9;GETWORD</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;NUMERAL&#9;PRMKEY&#9;TORPHASE&#9;TRIMTEXT&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>READSEQ</B>&#9;FATAL&#9;GETNUMB&#9;GETTEXT&#9;GETWORD&#9;TRIMTEXT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>READXYZ</B>&#9;CHKXYZ&#9;FATAL&#9;GETTEXT&#9;GETWORD&#9;NEXTTEXT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SORT&#9;TRIMTEXT&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>REFINE</B>&#9;LBFGS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>REPLICA</B>&#9;FATAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>RGDSRCH</B>&#9;CLIMBRGD&#9;EIGENRGD&#9;RIGIDXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>RGDSTEP</B>&#9;GRADIENT&#9;LINBODY&#9;MDSAVE&#9;MDSTAT&#9;NEWCRD</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;PRESSURE&#9;REGBODY&#9;ROTRGD&#9;TEMPER</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>RIBOSOME</B>&#9;ADDBOND&#9;ADDSIDE&#9;FATAL&#9;FINDATM&#9;FREEUNIT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;NEWATM&#9;OLDATM&#9;PRTSEQ&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>RINGS</B>&#9;ANGLES&#9;BONDS&#9;FATAL&#9;TORSIONS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>RMSERROR</B>&#9;TRIMTEXT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ROTANG</B>&#9;DOT&#9;VCROSS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ROTCHECK</B>&#9;ROTLIST</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ROTLIST</B>&#9;FATAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SADDLE</B>&#9;COMMENT'&#9;FATAL&#9;FINAL&#9;FREEUNIT&#9;GETTEXT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;GETXYZ&#9;IMPOSE&#9;INITIAL&#9;MAKEINT&#9;MAKEXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;MECHANIC&#9;NEXTARG&#9;PATHPNT&#9;PATHSCAN&#9;PATHVAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;PRTXYZ&#9;READXYZ&#9;SADDLE1&#9;SEARCH&#9;TANGENT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SADDLE1</B>&#9;GRADIENT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SADDLES</B>&#9;CERROR&#9;IPEDGE&#9;TRIPLE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SCAN</B>&#9;ACTIVE&#9;FINAL&#9;FREEUNIT&#9;GETXYZ&#9;INITIAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITROT&#9;LOCALMIN&#9;MAKEINT&#9;MAPCHECK&#9;MECHANIC</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;MODESRCH&#9;NEXTARG&#9;NUMERAL&#9;READXYZ&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SCAN1</B>&#9;GRADIENT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SCAN2</B>&#9;HESSIAN</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SDAREA</B>&#9;SURFATOM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SDSTEP</B>&#9;GRADIENT&#9;MDSAVE&#9;MDSTAT&#9;RATTLE&#9;RATTLE2</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SDTERM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SDTERM</B>&#9;NORMAL&#9;SDAREA</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SETIME</B>&#9;CLOCK</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SHAKEUP</B>&#9;GETNUMB&#9;GETTEXT&#9;GETWORD&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SLATER</B>&#9;ASET&#9;BSET&#9;CJKM&#9;POLYP</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SMOOTH</B>&#9;GETTEXT&#9;NEXTARG&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SNIFFER</B>&#9;FINAL&#9;FREEUNIT&#9;GETREF&#9;GETXYZ&#9;GRADIENT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITIAL&#9;MAKEREF&#9;MECHANIC&#9;NEXTARG&#9;PRTXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SNIFFER1&#9;VERSION&#9;WRITEOUT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SNIFFER1</B>&#9;GRADIENT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SOAK</B>&#9;DELETE&#9;FREEUNIT&#9;IMAGE&#9;LATTICE&#9;MAKEREF</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;MERGE&#9;MOLECULE&#9;READXYZ&#9;SUFFIX&#9;UNITCELL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SPACEFILL</B>&#9;ACTIVE&#9;CONNOLLY&#9;FIELD&#9;FINAL&#9;FREEUNIT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;GETTEXT&#9;GETXYZ&#9;INITIAL&#9;KATOM&#9;KVDW</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;NEXTARG&#9;READXYZ&#9;SUFFIX&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SPECTRUM</B>&#9;BASEFILE&#9;FREEUNIT&#9;INITIAL&#9;NEXTARG&#9;SUFFIX</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SQUARE</B>&#9;GETTEXT&#9;LMSTEP&#9;PRECISE&#9;QRFACT&#9;RSDVALUE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;TRUST&#9;UPCASE&#9;WRITEOUT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SUFFIX</B>&#9;TRIMTEXT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SUPERPOSE</B>&#9;FIELD&#9;FINAL&#9;FREEUNIT&#9;GETTEXT&#9;GETXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;IMPOSE&#9;INITIAL&#9;KATOM&#9;NEXTARG&#9;PRTXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;READXYZ&#9;SUFFIX&#9;TRIMTEXT&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SURFACE</B>&#9;FATAL&#9;SORT2</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SURFATOM</B>&#9;FATAL&#9;SORT2</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SWITCH</B>&#9;REPLICA</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SYBYLXYZ</B>&#9;FINAL&#9;FREEUNIT&#9;GETMOL2&#9;INITIAL&#9;PRTXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>SYMMETRY</B>&#9;CELLATOM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TANGENT</B>&#9;PATHPNT&#9;SADDLE1</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TEMPER</B>&#9;MAXWELL&#9;RANDOM&#9;RANVEC</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TESTGRAD</B>&#9;ENERGY&#9;FINAL&#9;GETTEXT&#9;GETXYZ&#9;GRADIENT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITIAL&#9;MECHANIC&#9;NEXTARG&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TESTHESS</B>&#9;FINAL&#9;FREEUNIT&#9;GETTEXT&#9;GETXYZ&#9;GRADIENT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;HESSIAN&#9;INITIAL&#9;MECHANIC&#9;NEXTARG&#9;NUMGRAD</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;UPCASE&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TESTLIGHT</B>&#9;EBUCK&#9;EBUCK1&#9;ECHARGE&#9;ECHARGE1&#9;EGAUSS</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;EGAUSS1&#9;EHAL&#9;EHAL1&#9;ELJ&#9;ELJ1</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;EMM3HB&#9;EMM3HB1&#9;FINAL&#9;GETIME&#9;GETXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITIAL&#9;LIGHTS&#9;MECHANIC&#9;NEXTARG&#9;SETIME</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TESTROT</B>&#9;ENERGY&#9;FINAL&#9;GETINT&#9;GRADROT&#9;INITIAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITROT&#9;MAKEXYZ&#9;MECHANIC&#9;NEXTARG</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TIMER</B>&#9;ENERGY&#9;FINAL&#9;GETIME&#9;GETTEXT&#9;GETXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;GRADIENT&#9;HESSIAN&#9;INITIAL&#9;MECHANIC&#9;NEXTARG</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SETIME&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TIMEROT</B>&#9;ENERGY&#9;FINAL&#9;GETIME&#9;GETINT&#9;GETTEXT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;GRADROT&#9;HESSROT&#9;INITIAL&#9;INITROT&#9;MECHANIC</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;NEXTARG&#9;SETIME&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TNCG</B>&#9;GETTEXT&#9;HMATRIX&#9;PISCF&#9;SEARCH&#9;TNSOLVE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;UPCASE&#9;WRITEOUT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TNSOLVE</B>&#9;PRECOND</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TORSIONS</B>&#9;FATAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TORUS</B>&#9;CERROR&#9;GETTOR</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TOTERR</B>&#9;BNDERR&#9;CHIRER&#9;LOCERR&#9;TORSER&#9;VDWERR</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TRIANGLE</B>&#9;FATAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TRIPLE</B>&#9;DOT&#9;VCROSS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>TRUST</B>&#9;PRECISE&#9;RSDVALUE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>UDIRECT2</B>&#9;ERFC&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>UMUTUAL2</B>&#9;ERFC&#9;IMAGE&#9;SWITCH</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>UNITCELL</B>&#9;FATAL&#9;GETTEXT&#9;GETWORD&#9;UPCASE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>VAM</B>&#9;CERROR&#9;CIRPLN&#9;DEPTH&#9;DIST2&#9;DOT</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;GENDOT&#9;MEASFN&#9;MEASFP&#9;MEASFS&#9;MEASPM</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;TRIPLE&#9;VCROSS&#9;VNORM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>VDWERR</B>&#9;LIGHTS</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>VECANG</B>&#9;ANORM&#9;DOT&#9;TRIPLE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>VERLET</B>&#9;GRADIENT&#9;MDSAVE&#9;MDSTAT&#9;PRESSURE&#9;RATTLE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;RATTLE2&#9;TEMPER</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>VERSION</B>&#9;LOWCASE&#9;NEXTARG&#9;TRIMTEXT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>VIBRATE</B>&#9;DIAGQ&#9;FATAL&#9;FINAL&#9;FREEUNIT&#9;GETXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;HESSIAN&#9;INITIAL&#9;MECHANIC&#9;NEXTARG&#9;NUMERAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;PRTXYZ&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>VIBROT</B>&#9;DIAGQ&#9;FINAL&#9;GETINT&#9;HESSROT&#9;INITIAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INITROT&#9;MECHANIC</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>VNORM</B>&#9;ANORM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>VOLUME</B>&#9;CONNOLLY</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>VOLUME1</B>&#9;FATAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>VOLUME2</B>&#9;FATAL</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>WATSON</B>&#9;ZATOM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>WATSON1</B>&#9;GRADRGD&#9;RIGIDXYZ</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>WRITEOUT</B>&#9;FREEUNIT&#9;NUMERAL&#9;PRTINT&#9;PRTXYZ&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>XTALERR</B>&#9;ENERGY&#9;XTALMOVE&#9;XTALPRM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>XTALFIT</B>&#9;FINAL&#9;GETXYZ&#9;INITIAL&#9;MECHANIC&#9;NEXTARG</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;SQUARE&#9;XTALPRM</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>XTALLAT1</B>&#9;ENERGY&#9;LATTICE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>XTALMIN</B>&#9;FINAL&#9;FREEUNIT&#9;GETXYZ&#9;GRADIENT&#9;INITIAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;LATTICE&#9;MECHANIC&#9;NEXTARG&#9;OCVM&#9;PRTXYZ</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;TNCG&#9;VERSION&#9;XTALLAT1</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>XTALMOL1</B>&#9;GRADIENT</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>XTALMOL2</B>&#9;HESSIAN</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>XTALMOVE</B>&#9;LATTICE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>XTALPRM</B>&#9;BOUNDS&#9;LATTICE&#9;MOLECULE</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>XYZEDIT</B>&#9;ACTIVE&#9;CUTOFFS&#9;DELETE&#9;FIELD&#9;FINAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;FREEUNIT&#9;GETXYZ&#9;IMAGE&#9;INERTIA&#9;INITIAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;INSERT&#9;KATOM&#9;LATTICE&#9;MAKEREF&#9;MERGE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;PRTXYZ&#9;RANDOM&#9;SOAK&#9;SORT&#9;SORT4</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;UNITCELL&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>XYZINT</B>&#9;FINAL&#9;FREEUNIT&#9;GETTEXT&#9;GETXYZ&#9;INITIAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;MAKEINT&#9;NEXTARG&#9;PRTINT&#9;READINT&#9;UPCASE</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>XYZPDB</B>&#9;FIELD&#9;FINAL&#9;FREEUNIT&#9;GETXYZ&#9;INITIAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;KATOM&#9;MAKEPDB&#9;MOLECULE&#9;PRTPDB&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>XYZRIGID</B>&#9;JACOBI&#9;ROTEULER</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>XYZSYBYL</B>&#9;BONDS&#9;FINAL&#9;FREEUNIT&#9;GETXYZ&#9;INITIAL</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">&#9;PRTMOL2&#9;VERSION</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ZATOM</B>&#9;FATAL</FONT></FONT></P>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>ZVALUE</B>&#9;MAKEXYZ&#9;TRIMTEXT</FONT></FONT></P>
<TABLE WIDTH=450 CELLPADDING=0>
<TR VALIGN=TOP>
<TD COLSPAN=1 WIDTH="12%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">12.</FONT></FONT></U></B></P>
</TD>
<TD COLSPAN=1 WIDTH="88%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">Examples using the
TINKER Package</FONT></FONT></U></B></P>
</TD></TR>
</TABLE>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;This
section contains brief descriptions of the sample calculations found in the
EXAMPLE subdirectory of the TINKER distribution. These examples exercise
several of the current TINKER programs and are intended to provide a flavor of
the capabilities of the package.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">ANION Example</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Computes an
estimation of the free energy of hydration of Cl<SUP>-</SUP> anion vs. Br<SUP>-</SUP>
anion via a 2 picosecond simulation on a ``hybrid'' anion in a box of water
followed by a free energy perturbation calculation</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">ARGON Example</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Performs an
initial energy minimization on a periodic box containing 150 argon atoms
followed by 6 picoseconds of a molecular dynamics using a modified Beeman
integration algorithm and a Bersedsen thermostat</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">CLUSTER Example</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Performs a
set of 10 Gaussian density annealing (GDA) trials on a cluster of 13 argon
atoms in an attempt to locate the global minimum energy structure</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">CRAMBIN Example</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Generates a
TINKER file from a PDB file, followed by a single point energy computation and
determination of the molecular volume and surface area</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">CYCLOHEX Example</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">First
approximately locates the transition state between chair and boat cyclohexane,
followed by subsequent refinement of the transition state and a final
vibrational analysis to show that a single negative frequency is associated
with the saddle point</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">ENKEPHALIN
Example</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Produces
coordinates from the met-enkephalin amino acid sequence and phi/psi angles,
followed by truncated Newton energy minimization and determination of the
lowest frequency normal mode</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">FORMAMIDE Example</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Converts to
a unit cell from fractional coordinates, followed by full crystal energy
minimization and determination of optimal carbonyl oxygen energy parameters
from a fit to lattice energy and structure</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">HELIX Example</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Performs a
rigid-body optimization of the packing of two idealized polyalanine helices
using only van der Waals interactions</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">SALT Example</FONT></FONT></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Converts a
sodium chloride assymetric unit to the corresponding unit cell, then runs a
crystal minimization starting from the initial diffraction structure using
Ewald summation to model the long-range electrostatic interactions.</FONT></FONT></P>
<TABLE WIDTH=450 CELLPADDING=0>
<TR VALIGN=TOP>
<TD COLSPAN=1 WIDTH="12%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">13.</FONT></FONT></U></B></P>
</TD>
<TD COLSPAN=1 WIDTH="88%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">Benchmark Results</FONT></FONT></U></B></P>
</TD></TR>
</TABLE>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The
tables in this section provide CPU benchmarks for basic TINKER energy and
derivative evaluations, vibrational analysis and molecular dynamics. All times
are in seconds and were measured with TINKER executables dimensioned to </FONT><B><TT><FONT FACE="courier, courier new">maxatm</FONT></TT></B><FONT FACE="New Century Schoolbook">
of 10000 and <B>maxhess</B> of 1000000 in the source file </FONT><B><TT><FONT FACE="courier, courier new">sizes.i</FONT></TT></B><FONT FACE="New Century Schoolbook">.
All calculations were run twice in rapid succession on a quiet machine. The
times reported for each benchmark are the results from the second run. If you
have built TINKER on an alternative machine type and are able to run the
benchmarks on the additional machine type, please send the results for
inclusion in a future listing.</FONT></FONT></P>
<P></P>
<CENTER>
<P><B><FONT FACE="Century Schoolbook">BENCHMARK #1:  Calmodulin Energy Evaluation</FONT></B></P>
</CENTER>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">The system is an
isolated molecule of the 148-residue protein calmodulin with 2264 atoms using
the Amber-94 force field. All interactions are computed with no use of
cutoffs. Times listed are for calculation setup followed by a single energy,
energy/gradient and Hessian evaluation.</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">MACHINE TYPE&#9;MHz&#9;SETUP&#9;ENERGY&#9;GRAD&#9;HESS</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq DS10 (Tru64 5.0)&#9;466&#9;0.35&#9;1.28&#9;1.90&#9;8.05</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">DEC Alpha 4100 (Tru64 5.0)&#9;400&#9;0.55&#9;2.22&#9;3.92&#9;15.56</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">SGI IndigoII R10K (Irix 6.5)&#9;195&#9;1.16&#9;3.50&#9;5.93&#9;22.51</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Target-USA Athlon (RH 6.2, PGI)&#9;950&#9;0.35&#9;0.67&#9;1.59&#9;8.04</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Target-USA Athlon (RH 6.2, Absoft)&#9;950&#9;0.33&#9;0.94&#9;1.68&#9;8.58</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Target-USA Athlon (RH 6.2, g77)&#9;950&#9;0.38&#9;0.91&#9;1.95&#9;9.85</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (RH 6.2, PGI)&#9;650&#9;0.46&#9;1.35&#9;2.69&#9;14.20</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (RH 6.2, Absoft)&#9;650&#9;0.39&#9;1.46&#9;2.70&#9;15.45</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (RH 6.2, g77)&#9;650&#9;0.43&#9;1.66&#9;3.65&#9;17.21</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (Win2K, CVF 6.5)&#9;650&#9;0.35&#9;1.29&#9;2.32&#9;12.10</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (Win2K,
LF95 5.5)&#9;650&#9;0.32&#9;1.83&#9;2.88&#9;15.90</FONT></FONT></P>
<P></P>
<P></P>
<CENTER>
<P><B><FONT FACE="Century Schoolbook">BENCHMARK #2:  Crambin Crystal Energy Evaluation</FONT></B></P>
</CENTER>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">The system is a
unit cell of the 46-residue protein crambin containing 2 polypeptide chains, 2
ethanol and 178 water molecules for a total of 1360 atoms using the OPLS-UA
force field. Periodic boundaries are used with particle mesh Ewald for
electrostatics and a 9.0 &Aring; cutoff for vdW interactions. Times listed are
for calculation setup followed by a single energy, energy/ gradient and
Hessian evaluation.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">MACHINE TYPE&#9;MHz&#9;SETUP&#9;ENERGY&#9;GRAD&#9;HESS</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq DS10 (Tru64 5.0)&#9;466&#9;0.28&#9;0.48&#9;0.63&#9;1.78</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">DEC Alpha 4100 (Tru64 5.0)&#9;400&#9;0.46&#9;0.85&#9;1.08&#9;3.46</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">SGI IndigoII R10K (Irix 6.5)&#9;195&#9;0.92&#9;1.00&#9;1.43&#9;4.02</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Target-USA Athlon (RH 6.2, PGI)&#9;950&#9;0.32&#9;0.33&#9;0.48&#9;1.30</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Target-USA Athlon (RH 6.2, Absoft)&#9;950&#9;0.26&#9;0.39&#9;0.53&#9;1.67</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Target-USA Athlon (RH 6.2, g77)&#9;950&#9;0.34&#9;0.46&#9;0.64&#9;1.77</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (RH 6.2, PGI)&#9;650&#9;0.42&#9;0.52&#9;0.76&#9;2.62</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (RH 6.2, Absoft)&#9;650&#9;0.30&#9;0.62&#9;0.89&#9;3.29</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (RH 6.2, g77)&#9;650&#9;0.37&#9;0.75&#9;1.02&#9;3.55</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (Win2K, CVF 6.5)&#9;650&#9;0.29&#9;0.51&#9;0.72&#9;2.43</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (Win2K,
LF95 5.5)&#9;650&#9;0.28&#9;0.61&#9;0.78&#9;3.33</FONT></FONT></P>
<P></P>
<P></P>
<CENTER>
<P><B><FONT FACE="Century Schoolbook">BENCHMARK #3:  Peptide Normal Mode Calculation</FONT></B></P>
</CENTER>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">The system is a
minimum energy conformation of a 20-residue peptide containing one of each of
the standard amino acids for a total of 328 atoms using the OPLS-AA force
field without cutoffs. The time reported is for computation of the Hessian and
calculation of the normal modes of the Hessian matrix and the vibration
frequencies requiring two separate matrix diagonalization steps.</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">MACHINE
TYPE&#9;MHz&#9;NORMAL MODES</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq DS10 (Tru64 5.0)&#9;466&#9;&#9;&#9;41</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">DEC Alpha 4100 (Tru64 5.0)&#9;400&#9;&#9;&#9;76</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">SGI IndigoII R10K (Irix 6.5)&#9;195&#9;&#9;&#9;145</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Target-USA Athlon (RH 6.2, PGI)&#9;950&#9;&#9;&#9;51</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Target-USA Athlon (RH 6.2, Absoft)&#9;950&#9;&#9;&#9;54</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Target-USA Athlon (RH 6.2, g77)&#9;950&#9;&#9;&#9;64</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (RH 6.2, PGI)&#9;650&#9;&#9;&#9;66</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (RH 6.2, Absoft)&#9;650&#9;&#9;&#9;71</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (RH 6.2, g77)&#9;650&#9;&#9;&#9;119</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (Win2K, CVF 6.5)&#9;650&#9;&#9;&#9;63</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (Win2K,
LF95 5.5)&#9;650&#9;&#9;&#9;86</FONT></FONT></P>
<P></P>
<P></P>
<CENTER>
<P><B><FONT FACE="Century Schoolbook"> BENCHMARK #4:  TIP3P Water Box
Molecular Dynamics</FONT></B></P>
</CENTER>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">The system
consists of 216 rigid TIP3P water molecules in a 18.643 &Aring; periodic box,
9.0 &Aring; shifted energy switch cutoffs for nonbonded interactions. The time
reported is for 1000 dynamics steps of 1.0 fs each using the modified Beeman
integrator and Rattle constraints on all bond lengths.</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">MACHINE TYPE&#9;MHz&#9;DYNAMICS</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq DS10 (Tru64 5.0)&#9;466&#9;&#9;&#9;104</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">DEC Alpha 4100 (Tru64 5.0)&#9;400&#9;&#9;&#9;195</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">SGI IndigoII R10K (Irix 6.5)&#9;195&#9;&#9;&#9;278</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Target-USA Athlon (RH 6.2, PGI)&#9;950&#9;&#9;&#9;70</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Target-USA Athlon (RH 6.2, Absoft)&#9;950&#9;&#9;&#9;77</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Target-USA Athlon (RH 6.2, g77)&#9;950&#9;&#9;&#9;125</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (RH 6.2, PGI)&#9;650&#9;&#9;&#9;131</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (RH 6.2, Absoft)&#9;650&#9;&#9;&#9;138</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (RH 6.2, g77)&#9;650&#9;&#9;&#9;226</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (Win2K, CVF 6.5)&#9;650&#9;&#9;&#9;130</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (Win2K,
LF95 5.5)&#9;650&#9;&#9;&#9;141</FONT></FONT></P>
<P></P>
<P></P>
<CENTER>
<P><B><FONT FACE="Century Schoolbook">BENCHMARK #5:  TINKER Water Box
Molecular Dynamics</FONT></B></P>
</CENTER>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook">The system
consists of 216 TINKER flexible polarizable atomic multipole water molecules
in a 18.643 &Aring; periodic box using regular Ewald summation for the
electrostatics and a 12.0 &Aring; switched cutoff for vdW interactions. The
time reported is for 100 dynamics steps of 1.0 fs each using the modified
Beeman integrator and 0.01 Debye rms convergence for induced dipole moments.</FONT></FONT></P>
<P></P>
<P><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">MACHINE TYPE&#9;MHz&#9;DYNAMICS</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq DS10 (Tru64 5.0)&#9;466&#9;&#9;&#9;304</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">DEC Alpha 4100 (Tru64 5.0)&#9;400&#9;&#9;&#9;527</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">SGI IndigoII R10K (Irix 6.5)&#9;195&#9;&#9;&#9;738</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Target-USA Athlon (RH 6.2, PGI)&#9;950&#9;&#9;&#9;243</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Target-USA Athlon (RH 6.2, Absoft)&#9;950&#9;&#9;&#9;281</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Target-USA Athlon (RH 6.2, g77)&#9;950&#9;&#9;&#9;376</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (RH 6.2, PGI)&#9;650&#9;&#9;&#9;443</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (RH 6.2, Absoft)&#9;650&#9;&#9;&#9;525</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (RH 6.2, g77)&#9;650&#9;&#9;&#9;811</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (Win2K, CVF 6.5)&#9;650&#9;&#9;&#9;411</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="Century Schoolbook">Compaq Armada M700 (Win2K,
LF95 5.5)&#9;650&#9;&#9;&#9;535</FONT></FONT></P>
<TABLE WIDTH=450 CELLPADDING=0>
<TR VALIGN=TOP>
<TD COLSPAN=1 WIDTH="12%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">14.</FONT></FONT></U></B></P>
</TD>
<TD COLSPAN=1 WIDTH="88%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">Collaborators &amp; Acknowledgments</FONT></FONT></U></B></P>
</TD></TR>
</TABLE>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;The
TINKER package has developed over a period of many years, very slowly during
the late-1980's, and more rapidly since the mid-1990's in Jay Ponder's
research group at the Washington University School of Medicine in St. Louis.
Many people have played significant roles in the development of the package
into its current form. The major contributors are listed below:</FONT></FONT></P>
<P></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Stew Rubenstein</B>&#9;coordinate
interconversions; original optimization methods</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;and torsional angle manipulation</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Craig Kundrot</B>&#9;molecular
surface area &amp; volume and their derivatives</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Shawn Huston</B>&#9;original
AMBER/OPLS implementation; free energy</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;calculations; time
correlation functions</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Mike Dudek</B>&#9;DMA-derived
multipole models for peptides and proteins</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Yong &quot;Mike&quot; Kong</B>&#9;multipole
electrostatics; dipole polarization; reaction field</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;treatment; TINKER
water model</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Reece Hart</B>&#9;potential
smoothing methodology; Scheraga's DEM,</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;Straub's GDA and extensions</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Mike Hodsdon</B>&#9;extension
of the TINKER distgeom program and its</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;application to NMR NOE
structure determination</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Rohit Pappu</B>&#9;potential
smoothing methodology and PSS algorithms;</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;rigid body
optimization; GB/SA solvation derivatives</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Wijnand Mooij</B>&#9;MM3
directional hydrogen bonding term; crystal lattice</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;minimization code</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Gerald Loeffler</B>&#9;stochastic/Langevin
dynamics implementation</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Marina Vorobieva</B>&#9;nucleic
acid building module and parameter translation</FONT></FONT></P>
<P><B><FONT SIZE=2><FONT FACE="New Century Schoolbook">Nina Sokolova</FONT></FONT></B></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Peter Bagossi&#9;</B>TINKER
force field parameters for alkanes and diatomics</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Pengyu Ren</B>&#9;Ewald
summation for polarizable atomic multipoles;</FONT></FONT></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;TINKER force field for
water, organics and peptides</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Anders Carlsson</B>&#9;ligand
field potential energy term for transition metals</FONT></FONT></P>
<P></P>
<P><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Andrey Kutepov</B>&#9;integrator
for rigid-body dynamics trajectories</FONT></FONT></P>
<P></P>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">In addition,
we would like to thank <B>Tom Darden</B> for making his particle mesh Ewald
code generally available to the simulation community.</FONT></FONT></P>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">It is
critically important that TINKER's distributed force field parameter sets
exactly reproduce the intent of the original force field authors. We would
like to thank <B>Julian Tirado-Rives</B> (OPLS-AA), <B>Alex MacKerell</B>
(CHARMM27), and <B>Adrian Roitberg</B> and <B>Carlos Simmerling</B> (AMBER)
for their help in testing TINKER's results against those given by the
authentic programs and parameter sets. <B>Lou Allinger</B> has provided
updated parameters for MM2 and MM3 on several occasions. His very successful
methods provided the original inspiration for the development of TINKER.</FONT></FONT></P>
<P></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Finally, we
wish to thank the many users of the TINKER package for their suggestions and
comments, praise and criticism, which have resulted in a variety of improvements.</FONT></FONT></P>
<TABLE WIDTH=450 CELLPADDING=0>
<TR VALIGN=TOP>
<TD COLSPAN=1 WIDTH="12%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">15.</FONT></FONT></U></B></P>
</TD>
<TD COLSPAN=1 WIDTH="88%">
<P><B><U><FONT SIZE=5><FONT FACE="New Century Schoolbook">References &amp;
Suggested Reading</FONT></FONT></U></B></P>
</TD></TR>
</TABLE>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">&#9;This
section contains a list of the references to general theory, algorithms and
implementation details which have been of use during the development of the
TINKER package. Methods described in some of the references have been
implemented in detail within the TINKER source code. Other references contain
useful background information although the algorithms themselves are now
obsolete. Still other papers contain ideas or extensions planned for future
inclusion in TINKER. References for specific force field parameter sets are
provided in an earlier section of this User's Guide. This list is heavily
skewed toward biomolecules in general and proteins in particular. This bias
reflects our group's major interests; however an attempt has been made to
include methods which should be generally applicable.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">PARTIAL
LIST OF MOLECULAR MECHANICS SOFTWARE PACKAGES</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">AMBER&#9;Peter
Kollman, University of California, San Francisco</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">AMMP&#9;Rob
Harrison, Thomas Jefferson University, Philadelphia</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ARGOS&#9;Andy
McCammon, University of California, San Diego</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">BOSS&#9;William
Jorgensen, Yale University</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">BRUGEL&#9;Shoshona
Wodak, Free University of Brussels</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">CFF&#9;Shneior
Lifson, Weizmann Institute</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">CHARMM&#9;Martin
Karplus, Harvard University</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">CHARMM/GEMM&#9;Bernard
Brooks, National Institutes of Health, Bethesda</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">DELPHI&#9;Bastian
van de Graaf, Delft University of Technology</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">DISCOVER&#9;Molecular
Simulations Inc., San Diego</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">DL_POLY&#9;W.
Smith &amp; T. Forester, CCP5, Daresbury Laboratory</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ECEPP&#9;Harold
Scheraga, Cornell University</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ENCAD&#9;Michael
Levitt, Stanford University</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">FANTOM&#9;Werner
Braun, University of Texas, Galveston</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">FEDER/2&#9;Nobuhiro
Go, Kyoto University</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">GROMACS&#9;Herman
Berendsen, University of Groningen</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">GROMOS&#9;Wilfred
van Gunsteren, BIOMOS and ETH, Zurich</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">IMPACT&#9;Ronald
Levy, Rutgers University</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">MACROMODEL&#9;Schodinger,
Inc., Jersey City, New Jersey</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">MM2/MM3/MM4&#9;N.
Lou Allinger, University of Georgia</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">MMC&#9;Cliff
Dykstra, Indiana Univ.&#173;Purdue Univ. at Indianapolis</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">MMFF&#9;Tom
Halgren, Merck Research Laboratories, Rahway</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">MMTK&#9;Konrad
Hinsen, Inst. of Structural Biology, Grenoble</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">MOIL&#9;Ron
Elber, Cornell University</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">MOLARIS&#9;Arieh
Warshal, University of Southern California</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">MOLDY&#9;Keith
Refson, Oxford University</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">MOSCITO&#9;Dietmar
Paschek &amp; Alfons Geiger, Universit&auml;t Dortmund</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">NAMD&#9;Klaus
Schulten, University of Illinois, Urbana</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">OOMPAA&#9;Andy
McCammon, University of California, San Diego</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ORAL&#9;Karel
Zimmerman, INRA, Jouy-en-Josas, France</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">ORIENT&#9;Anthony
Stone, Cambridge University</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">PCMODEL&#9;Kevin
Gilbert, Serena Software, Bloomington, Indiana</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">PEFF&#9;Jan
Dillen, University of Pretoria, South Africa</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">Q&#9;Johan
&Aring;qvist, Uppsala University</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">SIBFA&#9;Nohad
Gresh, INSERM, CNRS, Paris</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">SIGMA&#9;Jan
Hermans, University of North Carolina</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">SPASIBA&#9;Gerard
Vergoten, Universit&eacute; de Lille</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">SPASMS&#9;David
Spellmeyer and the Kollman Group, UCSF</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">TINKER&#9;Jay
Ponder, Washington University, St. Louis</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">XPLOR/CNS&#9;Axel
Br&uuml;nger, Stanford University</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">YAMMP&#9;Stephen
Harvey, University of Alabama, Birmingham</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">YASP&#9;Florian
Mueller-Plathe, ETH Zentrum, Zurich</FONT></FONT></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">YETI&#9;Angelo
Vedani, Biografik-Labor 3R, Basel</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>AMBER</B>
    D. A Pearlman, D. A. Case, J. W. Caldwell, W. S. Ross, T. E. Cheatham III,
S. DeBolt, D. Ferguson, G. Seibel and P. Kollman, AMBER, a Package of Computer
Programs for Applying Molecular Mechanics, Normal Mode Analysis, Molecular
Dynamics and Free Energy Calculations to Simulate the Structural and Energetic
Properties of Molecules, <B><I>Comp. Phys. Commun.</I></B>, <I>91</I>, 1-41 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ARGOS</B>
    T. P. Straatsma and J. A. McCammon, ARGOS, a Vectorized General Molecular
Dynamics Program, <B><I>J. Comput. Chem.</I></B>, <I>11</I>, 943-951 (1990)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>CHARMM</B>
    B. R. Brooks, R. E. Bruccoleri, B. D. Olafson, D. J. States, S.
Swaminathan and M. Karplus, CHARMM: A Program for Macromolecular Energy,
Minimization, and Dynamics Calculations, <B><I>J. Comput. Chem.</I></B>, <I>4</I>,
187-217 (1983)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ENCAD</B>
    M. Levitt, M. Hirshberg, R. Sharon and V. Daggett, Potential Energy
Function and Parameters for Simulations for the Molecular Dynamics of Proteins
and Nucleic Acids in Solution, <B><I>Comp. Phys. Commun.</I></B>, <I>91</I>,
215-231 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>FANTOM</B>
    T. Schaumann, W. Braun and K. Wurtrich, The Program FANTOM for Energy
Refinement of Polypeptides and Proteins Using a Newton-Raphson Minimizer in
Torsion Angle Space, <B><I>Biopolymers</I></B>, <I>29</I>, 679-694 (1990)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>FEDER/2</B>
    H. Wako, S. Endo, K. Nagayama and N. Go, FEDER/2: Program for Static and
Dynamic Conformational Energy Analysis of Macro-molecules in Dihedral Angle
Space, <B><I>Comp. Phys. Commun.</I></B>, <I>91</I>, 233-251 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>GROMACS</B>
    H. J. C. Berendsen, D. van der Spoel and R. van Drunen, GROMACS: A
Message-passing Parallel Molecular Dynamics Implementation, <B><I>Comp. Phys. Commun.</I></B>,
<I>91</I>, 43-56 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="Century Schoolbook"><B>GROMOS</B>   
 W. R. P. Scott, P. H. Hunenberger , I. G. Tironi, A. E. Mark, S. R. Billeter,
J. Fennen, A. E. Torda, T. Huber, P. Kruger, W. F. van Gunsteren, The GROMOS
Biomolecular Simulation Program Package, <B><I>J. Phys. Chem. A</I></B>, <I>103</I>,
3596-3607 (1999)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>IMPACT</B>
    D. B. Kitchen, F. Hirata, J. D. Westbrook, R. Levy, D. Kofke and M.
Yarmush, Conserving Energy during Molecular Dynamics Simulations of Water,
Proteins, and Proteins in Water, <B><I>J. Comput. Chem.</I></B>, <I>10</I>,
1169-1180 (1990)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MACROMODEL</B>
    F. Mahamadi, N. G. J. Richards, W. C. Guida, R. Liskamp, M. Lipton, C.
Caufield, G. Chang, T. Hendrickson and W. C. Still, MacroModel&#173;An
Integrated Software System for Modeling Organic and Bioorganic Molecules Using
Molecular Mechanics, <B><I>J. Comput. Chem.</I></B>, <I>11</I>, 440-467 (1990)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MM2</B>  
  N. L. Allinger, Conformational Analysis. 130. MM2. A Hydrocarbon Force Field
Utilizing V</FONT></FONT><FONT FACE="New Century Schoolbook"><SUB><FONT SIZE=1>1</FONT></SUB><FONT SIZE=2>
and V</FONT><SUB><FONT SIZE=1>2</FONT></SUB><FONT SIZE=2> Torsional Terms, <B><I>J.
Am. Chem. Soc.</I></B>, <I>99</I>, 8127-8134 (1977)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MM3</B>  
  N. L. Allinger, Y. H. Yuh and J.-H. Lii, Molecular Mechanics. The MM3 Force
Field for Hydrocarbons, <B><I>J. Am. Chem. Soc.</I></B>, <I>111</I>, 8551-8566 (1989)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MM4</B>  
  N. L. Allinger, K. Chen and J.-H. Lii, An Improved Force Field (MM4) for
Saturated Hydrocarbons, <B><I>J. Comput. Chem.</I></B>, <I>17</I>, 642-668 (1996)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MMC</B>  
  C. E. Dykstra, Molecular Mechanics for Weakly Interacting Assemblies of Rare
Gas Atoms and Small Molecules, <B><I>J. Am. Chem. Soc.</I></B>, <I>111</I>,
6168-6174 (1989)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MMFF</B> 
   T. A. Halgren, Merck Molecular Force Field. I. Basis, Form, Scope,
Parameterization, and Performance of MMFF94, <B><I>J. Comput. Chem.</I></B>, <I>17</I>,
490-516 (1996)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MOIL</B> 
   R. Elber, A. Roitberg, C. Simmerling, R. Goldstein, H. Li, G. Verkhiver, C.
Keasar, J. Zhang and A. Ulitsky, MOIL: A Program for Simulations of
Macromolecules, <B><I>Comp. Phys. Commun.</I></B>, <I>91</I>, 159-189 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>MOSCITO</B>
    See the web site at http:/ganter.chemie.uni-dortmund.de/~pas/moscito.html</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>NAMD</B> 
   L. Kal&eacute;, R. Skeel, M. Bhandarkar, R. Brunner, A. Gursoy, N. Krawetz,
J. Phillips, A. Shinozaki, K. Varadarajan and K. Schulten, NAMD2: Greater
Scalability for Parallel Molecular Dynamics, <B><I>J. Comput. Phys.</I></B>, <I>151</I>,
283-312 (1999)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>OOMPAA</B>
    G. A. Huber and J. A. McCammon, OOMPAA&#173;Object-oriented Model for
Probing Assemblages of Atoms, <B><I>J. Comput. Phys.</I></B>, <I>151</I>,
264-282 (1999)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>ORAL</B> 
   K. Zimmermann, ORAL: All Purpose Molecular Mechanics Simulator and Energy
Minimizer, <B><I>J. Comput. Chem.</I></B>, <I>12</I>, 310-319 (1991)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>PCMODEL</B>
    See the web site at http:/www.serenasoft.com</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>PEFF</B> 
   J. L. M. Dillen, PEFF: A Program for the Development of Empirical Force
Fields, <B><I>J. Comput. Chem.</I></B>, <I>13</I>, 257-267 (1992)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>Q</B>    
See the web site at http://aqvist.bmc.uu.se/Q</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>SIBFA</B>
    N. Gresh, Inter- and Intramolecular Interactions. Inception and
Refinements of the SIBFA, Molecular Mechanics (SMM) Procedure, a Separable,
Polarizable Methodology Grounded on <I>ab Initio</I> SCF/MP2 Computations.
Examples of Applications to Molecular Recognition Problems, <B><I>J. Chim.
Phys. PCB</I></B>, <I>94</I>, 1365-1416 (1997)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>SIGMA</B>
    See the web site at http://femto.med.unc.edu/SIGMA</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>SPASIBA</B>
    P. Derreumaux and G. Vergoten, A New Spectroscopic Molecular Mechanics
Force-Field - Parameters For Proteins, <B><I>J. Chem. Phys.</I></B>, <I>102</I>,
8586-8605 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>TINKER</B>
    See the web site at http://dasher.wustl.edu/tinker</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>YAMMP</B>
    R. K.-Z. Tan and S. C. Harvey, Yammp: Development of a Molecular Mechanics
Program Using the Modular Programming Method, <B><I>J. Comput. Chem.</I></B>, <I>14</I>,
455-470 (1993)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook"><B>YETI</B> 
   A. Vedani, YETI:  An Interactive Molecular Mechanics Program for
Small-Molecule Protein Complexes, <B><I>J. Comput. Chem.</I></B>, <I>9</I>,
269-280 (1988)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">MOLECULAR
MECHANICS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">U. Burkert
and N. L. Allinger, <B>Molecular Mechanics</B>, American Chemical Society,
Washington, D.C., 1982</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">K.
Rasmussen, <B>Potential Energy Functions in Conformational Analysis</B>
(Lecture Notes in Chemistry, Vol. 27), Springer-Verlag, Berlin, 1985</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A. K.
Rapp&eacute; and C. J. Casewit, <B>Molecular Mechanics across Chemistry</B>,
University Science Books, Sausalito, CA, 1997</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">K. Machida, <B>Principles
of Molecular Mechanics</B>, Kodansha/John Wiley &amp; Sons, Tokyo/New York, 1999</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">P. Comba and
T. W. Hambley, <B>Molecular Modeling of Inorganic Compounds</B>, VCH, New
York, 1995</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">COMPUTER
SIMULATION METHODS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. J. Field,
A Practical Introduction to the Simulation of Molecular Systems, Cambridge
Univ. Press, Cambridge, 1999</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A. R. Leach, <B>Molecular
Modelling: Principles and Applications</B>, Addison Wesley Longman, Essex,
England, 1996</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">D. Frankel
and B. Smit, <B>Understanding Molecular Simulation: From Algorithms to Applications</B>,
Academic Press, San Diego, CA, 1996</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">D. C.
Rapaport, <B>The Art of Molecular Dynamics Simulation</B>, Cambridge
University Press, Cambridge, 1995</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. M. Haile, <B>Molecular
Dynamics Simulation</B>, John Wiley and Sons, New York, 1992</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. P. Allen
and D. J. Tildesley, <B>Computer Simulation of Liquids</B>, Oxford University
Press, Oxford, 1987</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. Schlick,
R. D. Skeel, A. T. Br&uuml;nger, L. V. Kale, J. A. Board, J. Hermans and K.
Schulten, Algorithmic Challenges in Computational Molecular Biophysics, <B><I>J.
Comput. Phys.</I></B>, <I>151</I>, 9-48 (1999)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">MODELING
OF BIOLOGICAL MACROMOLECULES</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. A.
McCammon and S. Harvey, <B>Dynamics of Proteins and Nucleic Acids</B>,
Cambridge University Press, Cambridge, 1987</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">C. L. Brooks
III, M. Karplus and B. M. Pettitt, <B>Proteins: A Theoretical Perspective of
Dynamics, Structure, and Thermodynamics</B>, John Wiley and Sons, New York, 1988</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. F. van
Gunsteren, P. K. Weiner and A. J. Wilkinson, <B>Computer Simulation of
Biomolecular Systems, Vol. 1-3</B>, Kluwer Academic Publishers, Dordrecht, 1989-1997</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. E.
Cheatham and B. R. Brooks, Recent Advances in Molecular Dynamics Simulation
towards the Realistic Representation of Biomolecules in Solution, <B><I>Theor.
Chem. Acc.</I></B>, <I>99</I>, 279-288 (1998)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">CONJUGATE
GRADIENT  AND QUASI-NEWTON OPTIMIZATION</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. Nocedal
and S. J. Wright, <B>Numerical Optimization</B>, Springer-Verlag, New York, 1999</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. G. Nash
and A. Sofer, <B>Linear and Nonlinear Programming</B>, McGraw-Hill, New York, 1996</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">R. Fletcher, <B>Practical
Methods of Optimization</B>, John Wiley &amp; Sons Ltd., Chichester, 1987</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">D. G.
Luenberger, <B>Linear and Nonlinear Programming</B>, 2nd Ed., Addison-Wesley,
Reading, MA, 1984</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">P. E. Gill,
W. Murray and M. H. Wright, <B>Practical Optimization</B>, Academic Press, New
York, 1981</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. Nocedal,
Updating Quasi-Newton Matrices with Limited Storage, <B><I>Math. Comp.</I></B>,
773-782 (1980)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. J.
Watowich, E. S. Meyer, R. Hagstrom and R. Josephs, A Stable, Rapidly
Converging Conjugate Gradient Method for Energy Minimization, <B><I>J. Comput. Chem.</I></B>,
<I>9</I>, 650-661 (1988)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. C.
Davidon, Optimally Conditioned Optimization Algorithms without Line Searches, <B><I>Math.
Prog.</I></B>, <I>9</I>, 1-30 (1975)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">TRUNCATED
NEWTON OPTIMIZATION</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. W. Ponder
and F. M. Richards, An Efficient Newton-like Method for Molecular Mechanics
Energy Minimization of Large Molecules, <B><I>J. Comput. Chem.</I></B>, <I>8</I>,
1016-1024 (1987)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">R. S. Dembo
and T. Steihaug, Truncated-Newton Algorithms for Large-Scale Unconstrained
Optimization, <B><I>Math. Prog.</I></B>, <I>26</I>, 190-212 (1983)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. C.
Eisenstat and H. F. Walker, Choosing the Forcing Terms in an Inexact Newton
Method, <B><I>SIAM J. Sci. Comput.</I></B>, <I>17</I>, 16-32 (1996)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. Schlick
and M. Overton, A Powerful Truncated Newton Method for Potential Energy
Minimization, <B><I>J. Comput. Chem.</I></B>, <I>8</I>, 1025-1039 (1987)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">D. S.
Kershaw, The Incomplete Cholesky-Conjugate Gradient Method for the Iterative
Solution of Systems of Linear Equations, <B><I>J. Comput. Phys.</I></B>, <I>26</I>,
43-65 (1978)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. A.
Manteuffel, An Incomplete Factorization Technique for Positive Definite Linear
Systems, <B><I>Math. Comp.</I></B>, <I>34</I>, 473-497 (1980)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">P.
Derreumaux, G. Zhang and T. Schlick and B. R. Brooks, A Truncated Newton
Minimizer Adapted for CHARMM and Biomolecular Applications, <B><I>J. Comput. Chem.</I></B>,
<I>15</I>, 532-552 (1994)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">I. S. Duff,
A. M. Erisman and J. K. Reid, <B>Direct Methods for Sparse Matrices</B>,
Oxford University Press, Oxford, 1986</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">POTENTIAL
ENERGY SMOOTHING</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">R. V. Pappu,
R. K. Hart and J. W. Ponder, Analysis and Application of Potential Energy
Smoothing Methods for Global Optimization, <B><I>J. Phys. Chem. B</I></B>, <I>102</I>,
9725-9742 (1998)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">L. Piela, J.
Kostrowicki and H. A. Scheraga, The Multiple-Minima Problem in the
Conformational Analysis of Molecules. Deformation of the Potential Energy
Hypersurface by the Diffusion Equation Method, <B><I>J. Phys. Chem.</I></B>, <I>93</I>,
3339-3346 (1989)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. Ma and J.
E. Straub, Simulated Annealing Using the Classical Density Distribution, <B><I>J.
Chem. Phys.</I></B>, <I>101</I>, 533-541 (1994)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">C. Tsoo and
C. L. Brooks, Cluster Structure Determination Using Gaussian Density
Distribution Global Minimization Methods, <B><I>J. Chem. Phys.</I></B>, <I>101</I>,
6405-6411 (1994)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. Nakamura,
H. Hirose, M. Ikeguchi and J. Doi, Conformational Energy Minimization Using a
Two-Stage Method, <B><I>J. Phys. Chem.</I></B>, <I>99</I>, 8374-8378 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. Huber, A.
E. Torda and W. F. van Gunsteren, Structure Optimization Combining Soft-Core
Interaction Functions, the Diffusion Equation Method, and Molecular Dynamics, <B><I>J.
Phys. Chem. A</I></B>, <I>101</I>, 5926-5930 (1997)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S.
Schelstraete and H. Verschelde, Finding Minimum-Energy Configurations of
Lennard-Jones Clusters Using an Effective Potential, <B><I>J. Phys. Chem. A</I></B>,
<I>101</I>, 310-315 (1998)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">I.
Andricioaei and J. E. Straub, Global Optimization Using Bad Derivatives:
Derivative-Free Method for Molecular Energy Minimization, <B><I>J. Comput. Chem.</I></B>,
<I>19</I>, 1445-1455 (1998)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">L. Piela,
Search for the Most Stable Structures on Potential Energy Surfaces, <B><I>Coll.
Czech. Chem. Commun.</I></B>, <I>63</I>, 1368-1380 (1998)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">``SNIFFER''
GLOBAL OPTIMIZATION</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A. O.
Griewank, Generalized Descent for Global Optimization, <B><I>J. Opt. Theor. Appl.</I></B>,
<I>34</I>, 11-39 (1981)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">R. A. R.
Butler and E. E. Slaminka, An Evaluation of the Sniffer Global Optimization
Algorithm Using Standard Test Functions, <B><I>J. Comput. Phys.</I></B>, <I>99</I>,
28-32 (1993)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. W. Rogers
and R. A. Donnelly, Potential Transformation Methods for Large-Scale Global
Optimization, <B><I>SIAM J. Optim.</I></B>, <I>5</I>, 871-891 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">INTEGRATION
METHODS FOR MOLECULAR DYNAMICS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">D. Beeman,
Some Multistep Methods for Use in Molecular Dynamics Calculations, <B><I>J.
Comput. Phys.</I></B>, <I>20</I>, 130-139 (1976)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. Levitt
and H. Meirovitch, Integrating the Equations of Motion, <B><I>J. Mol. Biol.</I></B>,
<I>168</I>, 617-620 (1983)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. Aqvist,
W. F. van Gunsteren, M. Leijonmarck and O. Tapia, A Molecular Dynamics Study
of the C-Terminal Fragment of the L7/L12 Ribosomal Protein, <B><I>J. Mol. Biol.</I></B>,
<I>183</I>, 461-477 (1985)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. C. Swope,
H. C. Andersen, P. H. Berens and K. R. Wilson, A Computer Simulation Method
for the Calculation of Equilibrium Constants for the Formation of Physical
Clusters of Molecules: Application to Small Water Clusters, <B><I>J. Chem. Phys.</I></B>,
<I>76</I>, 637-649 (1982)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">CONSTRAINT
DYNAMICS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. F. van
Gunsteren and H. J. C. Berendsen, Algorithms for Macromolecular Dynamics and
Constraint Dynamics, <B><I>Mol. Phys.</I></B>, <I>34</I>, 1311-1327 (1977)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">G. Ciccotti,
M. Ferrario and J.-P. Ryckaert, Molecular Dynamics of Rigid Systems in
Cartesian Coordinates: A General Formulation, <B><I>Mol. Phys.</I></B>, <I>47</I>,
1253-1264 (1982)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">H. C.
Andersen, Rattle: A ``Velocity'' Version of the Shake Algorithm for Molecular
Dynamics Calculations, <B><I>J. Comput. Phys.</I></B>, <I>52</I>, 24-34 (1983)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">R. Kutteh,
RATTLE Recipe for General Holonomic Constraints: Angle and Torsion
Constraints, <B><I>CCP5 Newsletter</I></B>, <I>46</I>, 9-17 (1998) [available
from the web site at http://www.dl.ac.uk/CCP/CCP5/newsletter_index.html]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">B. J.
Palmer, Direct Application of SHAKE to the Velocity Verlet Algorithm, <B><I>J.
Comput. Phys.</I></B>, <I>104</I>, 470-472 (1993)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. Miyamoto
and P. A. Kollman, SETTLE: An Analytical Version of the SHAKE and RATTLE
Algorithm for Rigid Water Models, <B><I>J. Comput. Chem.</I></B>, <I>13</I>,
952-962 (1992)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">B. Hess, H.
Bekker, H. J. C. Berendsen and J. G. E. M. Fraaije, LINCS: A Linear Constraint
Solver for Molecular Simulations, <B><I>J. Comput. Chem.</I></B>, <I>18</I>,
1463-1472 (1997)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. T.
Slusher and P. T. Cummings, Non-Iterative Constraint Dynamics using
Velocity-Explicit Verlet Methods, <B><I>Mol. Simul.</I></B>, <I>18</I>,
213-224 (1996)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">LANGEVIN,
BROWNIAN AND STOCHASTIC DYNAMICS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. P. Allen,
Brownian Dynamics Simulation of a Chemical Reaction in Solution, <B><I>Mol. Phys.</I></B>,
<I>40</I>, 1073-1087 (1980)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. F. van
Gunsteren and H. J. C. Berendsen, Algorithms for Brownian Dynamics, <B><I>Mol. Phys.</I></B>,
<I>45</I>, 637-647 (1982)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">F. Guarnieri
and W. C. Still, A Rapidly Convergent Simulation Method: Mixed Monte
Carlo/Stochastic Dynamics, <B><I>J. Comput. Chem.</I></B>, <I>15</I>,
1302-1310 (1994)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. G.
Paterlini and D. M. Ferguson, Constant Temperature Simulations using the
Langevin Equation with Velocity Verlet Integration, <B><I>Chem. Phys.</I></B>, <I>236</I>,
243-252 (1998)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">CONSTANT
TEMPERATURE AND PRESSURE DYNAMICS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">H. J. C.
Berendsen, J. P. M. Postma, W. F. van Gunsteren, A. DiNola and J. R. Haak,
Molecular Dynamics with Coupling to an External Bath, <B><I>J. Chem. Phys.</I></B>,
<I>81</I>, 3684-3690 (1984)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. G.
Hoover, Canonical Dynamics: Equilibrium Phase-space Distributions, <B><I>Phys.
Rev. A</I></B>, <I>31</I>, 1695-1697 (1985)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. J.
Morales, S. Toxvaerd and L. F. Rull, Computer Simulation of a Phase Transition
at Constant Temperature and Pressure, <B><I>Phys. Rev. A</I></B>, <I>34</I>,
1495-1498 (1986)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">B. R.
Brooks, Algorithms for Molecular Dynamics at Constant Temperature and
Pressure, Internal Report of Division of Computer Research and Technology,
National Institutes of Health, 1988.</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. Levitt,
Molecular Dynamics of Native Protein: Computer Simulation of Trajectories, <B><I>J.
Mol. Biol.</I></B>, <I>168</I>, 595-620 (1983)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">OUT-OF-PLANE
DEFORMATION TERMS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. R. Maple,
U. Dinar and A. T. Hagler, Derivation of Force Fields for Molecular Mechanics
and Dynamics from <I>ab initio</I> Energy Surfaces, <B><I>Proc. Natl. Acad.
Sci. USA</I></B>, <I>85</I>, 5350-5354 (1988)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S.-H. Lee,
K. Palmo and S. Krimm, New Out-of-Plane Angle and Bond Angle Internal
Coordinates and Related Potential Energy Functions for Molecular Mechanics and
Dynamics Simulations, <B><I>J. Comput. Chem.</I></B>, <I>20</I>, 1067-1084 (1999)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">ANALYTICAL
DERIVATIVES OF POTENTIAL FUNCTIONS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">K. J.
Miller, R. J. Hinde and J. Anderson, First and Second Derivative Matrix
Elements for the Stretching, Bending, and Torsional Energy, <B><I>J. Comput. Chem.</I></B>,
<I>10</I>, 63-76 (1989)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">D. H. Faber
and C. Altona, UTAH5: A Versatile Programme Package for the Calculation of
Molecular Properties by Force Field Methods, <B><I>Computers &amp; Chemistry</I></B>,
<I>1</I>, 203-213 (1977)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. C. Swope
and D. M. Ferguson, Alternative Expressions for Energies and Forces Due to
Angle Bending and Torsional Energy, Report G320-3561, <B><I>J. Comput. Chem.</I></B>,
<I>13</I>, 585-594 (1992)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A. Blondel
and M. Karplus, New Formulation for Derivatives of Torsion Angles and Improper
Torsion Angles in Molecular Mechanics: Elimination of Singularities, <B><I>J.
Comput. Chem.</I></B>, <I>17</I>, 1132-1141 (1996)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">R. E. Tuzun,
D. W. Noid and B. G. Sumpter, Efficient Treatment of Out-of-Plane Bend and
Improper Torsion Interactions in MM2, MM3, and MM4 Molecular Mechanics Calculations,<B><I>
J. Comput. Chem.</I></B>, <I>18</I>, 1804-1811 (1997)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">TORSIONAL
SPACE DERIVATIVES AND NORMAL MODES</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. Levitt,
C. Sander and P. S. Stern, Protein Normal-mode Dynamics:  Trypsin Inhibitor,
Crambin, Ribonuclease and Lysozyme, <B><I>J. Mol. Biol.</I></B>, <I>181</I>,
423-447 (1985)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. Levitt,
Protein Folding by Restrained Energy Minimization and Molecular Dynamics, <B><I>J.
Mol. Biol.</I></B>, <I>170</I>, 723-764 (1983)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">H. Wako and
N. Go, Algorithm for Rapid Calculation of Hessian of Conformational Energy
Function of Proteins by Supercomputer, <B><I>J. Comput. Chem.</I></B>, <I>8</I>,
625-635 (1987)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">H. Abe, W.
Braun, T. Noguti and N. Go, Rapid Calculation of First and Second Derivatives
of Conformational Energy with Respect to Dihedral Angles for Proteins: General
Recurrent Equations, <B><I>Computers &amp; Chemistry</I></B>, <I>8</I>,
239-247 (1984)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. Noguti
and N. Go, A Method of Rapid Calculation of a Second Derivative Matrix of
Conformational Energy for Large Molecules, <B><I>J. Phys. Soc. Japan</I></B>, <I>52</I>,
3685-3690 (1983)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">ANALYTICAL
SURFACE AREA AND VOLUME</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. L.
Connolly, Analytical Molecular Surface Calculation, <B><I>J. Appl. Cryst.</I></B>,
<I>16</I>, 548-558 (1983)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. L.
Connolly, Computation of Molecular Volume, <B><I>J. Am. Chem. Soc.</I></B>, <I>107</I>,
1118-1124 (1985)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. L.
Connolly, Molecular Surfaces: A Review, available from the web site at http://www.netsci.org/Science/Compchem/feature14.html</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">C. E.
Kundrot, J. W. Ponder and F. M. Richards, Algorithms for Calculating Excluded
Volume and Its Derivatives as a Function of Molecular Conformation and Their
Use in Energy Minimization, <B><I>J. Comput. Chem.</I></B>, <I>12</I>, 402-409 (1991)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. J.
Richmond, Solvent Accessible Surface Area and Excluded Volume in Proteins, <B><I>J.
Mol. Biol.</I></B>, <I>178</I>, 63-89 (1984)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">L. Wesson
and D. Eisenberg, Atomic Solvation Parameters Applied to Molecular Dynamics of
Proteins in Solution, <B><I>Protein Science</I></B>, <I>1</I>, 227-235 (1992)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">V. Gononea
and E. Osawa, Implementation of Solvent Effect in Molecular Mechanics, Part 3.
The First- and Second-order Analytical Derivatives of Excluded Volume, <B><I>J.
Mol. Struct. (Theochem)</I></B>, <I>311</I> 305-324 (1994)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">K. D. Gibson
and H. A. Scheraga, Exact Calculation of the Volume and Surface Area of Fused
Hard-sphere Molecules with Unequal Atomic Radii, <B><I>Mol. Phys.</I></B>, <I>62</I>,
1247-1265 (1987)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">K. D. Gibson
and H. A. Scheraga, Surface Area of the Intersection of Three Spheres with
Unequal Radii: A Simplified Analytical Formula, <B><I>Mol. Phys.</I></B>, <I>64</I>,
641-644 (1988)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S.
Sridharan, A. Nichols and K. A. Sharp, A Rapid Method for Calculating
Derivatives of Solvent Accessible Surface Areas of Molecules, <B><I>J. Comput, Chem.</I></B>,
<I>16</I>, 1038-1044 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">APPROXIMATE
SURFACE AREA AND VOLUME</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. J. Wodak
and J. Janin, Analytical Approximation to the Accessible Surface Area of
Proteins, <B><I>Proc. Natl. Acad. Sci. USA</I></B>, <I>77</I>, 1736-1740 (1980)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. Hasel, T.
F. Hendrickson and W. C. Still, A Rapid Approximation to the Solvent
Accessible Surface Areas of Atoms, <B><I>Tetrahedron Comput. Method.</I></B>, <I>1</I>,
103-116 (1988)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. Weiser,
P. S. Shenkin and W. C. Still, Approximate Solvent-Accessible Surface Areas
from Tetrahedrally Directed Neighber Densities, <B><I>Biopolymers</I></B>, <I>50</I>,
373-380 (1999)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">BOUNDARY
CONDITIONS AND NEIGHBOR METHODS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. F. van
Gunsteren, H. J. C. Berendsen, F. Colonna, D. Perahia, J. P. Hollenberg and D.
Lellouch, On Searching Neighbors in Computer Simulations of Macromolecular
Systems, <B><I>J. Comput. Chem.</I></B>, <I>5</I>, 272-279  (1984)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">F. Sullivan,
R. D. Mountain and J. O'Connell, Molecular Dynamics on Vector Computers, <B><I>J.
Comput. Phys.</I></B>, <I>61</I>, 138-153 (1985)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. Boris, A
Vectorized ``Near Neighbors'' Algorithm of Order N Using a Monotonic Logical
Grid, <B><I>J. Comput. Phys.</I></B>, <I>66</I>, 1-20 (1986)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. G.
Lambrakos and J. P. Boris, Geometric Properties of the Monotonic Lagrangian
Grid Algorithm for Near Neighbors Calculations, <B><I>J. Comput. Phys.</I></B>,
<I>73</I>, 183-202 (1987)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. A.
Andrea, W. C. Swope and H. C. Andersen, The Role of Long Ranged Forces in
Determining the Structure and Properties of Liquid Water, <B><I>J. Chem. Phys.</I></B>,
<I>79</I>, 4576-4584 (1983)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">D. N.
Theodorou and U. W. Suter, Geometrical Considerations in Model Systems with
Periodic Boundary Conditions, <B><I>J. Chem. Phys.</I></B>, <I>82</I>, 955-966 (1985)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. Barnes
and P. Hut, A Hierarchical O(NlogN) Force-calculation Algorithm, <B><I>Nature</I></B>,
<I>234</I>, 446-449 (1986)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">CUTOFF
AND TRUNCATION METHODS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">P. J.
Steinbach and B. R. Brooks, New Spherical-Cutoff Methods for Long-Range Forces
in Macromolecular Simulation, <B><I>J. Comput. Chem.</I></B>, <I>15</I>,
667-683 (1993)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">R. J.
Loncharich and B. R. Brooks, The Effects of Truncating Long-Range Forces on
Protein Dynamics, <B><I>Proteins</I></B>, <I>6</I>, 32-45 (1989)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">C. L. Brooks
III, B. M. Pettitt and M. Karplus, Structural and Energetic Effects of
Truncating Long Ranged Interactions in Ionic and Polar Fluids, <B><I>J. Chem. Phys.</I></B>,
<I>83</I>, 5897-5908 (1985)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">EWALD
SUMMATION TECHNIQUES</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A. Y.
Toukmaji and J. A. Board, Jr., Ewald Summation Techniques in Perspective: A
Survey, <B><I>Comp. Phys. Commun.</I></B>, <I>95</I>, 73-92 (1996)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. Darden,
L. Perera, L. Li and L. Pedersen, New Tricks for Modelers from the
Crystallography Toolkit: The Particle Mesh Ewald Algorithm and its Use in
Nucleic Acid Simulations, <B><I>Structure</I></B>, <I>7</I>, R550-R60 (1999)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. Darden,
D. York and L. G. Pedersen, Particle Mesh Ewald: An N&#183;log(<I>N</I>)
Method for Ewald Sums in Large Systems, <B><I>J. Chem. Phys.</I></B>, <I>98</I>,
10089-10092 (1993)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">U. Essmann,
L. Perera, M. L. Berkowitz, T. Darden, H. Lee and L. G. Pedersen, A Smooth
Particle Mesh Ewald Method, <B><I>J. Chem. Phys.</I></B>, <I>103</I>,
8577-8593 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. Smith,
Point Multipoles in the Ewald Summation (Revisited), <B><I>CCP5 Newsletter</I></B>,
<I>46</I>, 18-30 (1998)  [available from http://www.dl.ac.uk/CCP/CCP5/newsletter_index.html]</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. E.
Feller, R. W. Pastor, A. Rojnuckarin, S. Bogusz and B. R. Brooks, Effect of
Electrostatic Force Truncation on Interfacial and Transport Properties of
Water, <B><I>J. Phys. Chem.</I></B>, <I>100</I>, 17011-17020 (1996)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. Weber, P.
H. H&uuml;nenberger and J. A. McCammon, Molecular Dynamics Simulations of a
Polyalanine Octapeptide under Ewald Boundary Conditions: Influence of
Artificial Periodicity on Peptide Conformation, <B><I>J. Phys. Chem. B</I></B>,
<I>104</I>, 3668-3675 (2000)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">CONJUGATED
AND AROMATIC SYSTEMS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">N. L.
Allinger, F. Li, L. Yan and J. C. Tai, Molecular Mechanics (MM3) Calculations
on Conjugated Hydrocarbons, <B><I>J. Comput. Chem.</I></B>, <I>11</I>, 868-895 (1990)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. T.
Sprague, J. C. Tai, Y. Yuh and N. L. Allinger, The MMP2 Calculational Method, <B><I>J.
Comput. Chem.</I></B>, <I>8</I>, 581-603 (1987)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. Kao, A
Molecular Orbital Based Molecular Mechanics Approach to Study Conjugated
Hydrocarbons, <B><I>J. Am. Chem. Soc.</I></B>, <I>109</I>, 3818-3829 (1987)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. Kao and
N. L. Allinger, Conformational Analysis: Heats of Formation of Conjugated
Hydrocarbons by the Force Field Method, <B><I>J. Am. Chem. Soc.</I></B>, <I>99</I>,
975-986 (1977)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">D. H. Lo and
M. A. Whitehead, Accurate Heats of Atomization and Accurate Bond Lengths:
Benzenoid Hydrocarbons, <B><I>Can. J. Chem.</I></B>, <I>46</I>, 2027-2040 (1968)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">G. D. Zeiss
and M. A. Whitehead, Hetero-atomic Molecules: Semi-empirical Molecular Orbital
Calculations and Prediction of Physical Properties, <B><I>J. Chem. Soc. A</I></B>,
1727-1738 (1971)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">FREE
ENERGY SIMULATION METHODS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">P. Kollman,
Free Energy Calculations: Applications to Chemical and Biochemical Phenomena, <B><I>Chem.
Rev.</I></B>, <I>93</I>, 2395-2417 (1993)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">B. L. Tembe
and J. A. McCammon, Ligand-Receptor Interactions, <B><I>Computers &amp; Chemistry</I></B>,
<I>8</I>, 281-283 (1984)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. L.
Jorgensen and C. Ravimohan, Monte Carlo Simulation of Differences in Free
Energy of Hydration, <B><I>J. Chem. Phys.</I></B>, <I>83</I>, 3050-3054 (1985)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. L.
Jorgensen, J. K. Buckner, S. Boudon and J. Tirado-Rives, Efficient Computation
of Absolute Free Energies of Binding by Computer Simulations:  Application to
the Methane Dimer in Water, <B><I>J. Chem. Phys.</I></B>, <I>89</I>, 3742-3746 (1988)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. H.
Fleischman and C. L. Brooks III, Thermodynamics of Aqueous Solvation: 
Solution Properties of Alcohols and Alkanes, <B><I>J. Chem. Phys.</I></B>, <I>87</I>,
3029-3037 (1987)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">U. C. Singh,
F. K. Brown, P. A. Bash and P. A. Kollman, An Approach to the Application of
Free Energy Perturbation Methods Using Molecular Dynamics, <B><I>J. Am. Chem. Soc.</I></B>,
<I>109</I>, 1607-1614 (1987)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">D. A.
Pearlman and P. A. Kollman, A New Method for Carrying out Free Energy
Perturbation Calculations: Dynamically Modified Windows, <B><I>J. Chem. Phys.</I></B>,
<I>90</I>, 2460-2470 (1989)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. P.
Straatsma, H. J. C. Berendsen and J. P. M. Postma, Free Energy of Hydrophobic
Hydration:  A Molecular Dynamics Study of Noble Gases in Water, <B><I>J. Chem. Phys.</I></B>,
<I>85</I>, 6720-6727 (1986)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. P.
Straatsma and H. J. C. Berendsen, Free Energy of Ionic Hydration:  Analysis of
a Thermodynamic Integration Technique to Evaluate Free Energy Differences by
Molecular Dynamics Simulations, <B><I>J. Chem. Phys.</I></B>, <I>89</I>,
5876-5886 (1988)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. Mezei,
The Finite Difference Thermodynamic Integration, Tested on Calculating the
Hydration Free Energy Difference between Acetone and Dimethylamine in Water, <B><I>J.
Chem. Phys.</I></B>, <I>86</I>, 7084-7088 (1987)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A. E. Mark
and W. F. van Gunsteren, Decomposition of the Free Energy of a System in Terms
of Specific Interactions, <B><I>J. Mol. Biol.</I></B>, <I>240</I>, 167-176 (1994)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. Boresch
and M. Karplus, The Meaning of Copmponent Analysis: Decomposition of the Free
Energy in Terms of Specific Interactions, <B><I>J. Mol. Biol.</I></B>, <I>254</I>,
801-807 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">METHODS
FOR PARAMETER DETERMINATION</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">N. L.
Allinger, X. Zhou and J. Bergsma, Molecular Mechanics Parameters, <B><I>J.
Mol. Struct. (THEOCHEM)</I></B>, <I>312</I>, 69-83 (1994)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A. J.
Pertsin and A. I. Kitaigorodsky, <B>The Atom-Atom Potential Method:
Application to Organic Molecular Solids</B>, Springer-Verlag, Berlin, 1987</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">D. E.
Williams, Transferable Empirical Nonbonded Potential Functions, in <B>Crystal
Cohesion and Conformational Energies</B>, Ed. by R. M. Metzger,
Springer-Verlag, Berlin, 1981</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A. T. Hagler
and S. Lifson, A Procedure for Obtaining Energy Parameters from Crystal
Packing, <B><I>Acta Cryst.</I></B>, <I>B30</I>, 1336-1341 (1974)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A. T.
Hagler, S. Lifson and P. Dauber, Consistent Force Field Studies of
Intermolecular Forces in Hydrogen-Bonded Crystals:  A Benchmark for the
Objective Comparison of Alternative Force Fields, <B><I>J. Am. Chem. Soc.</I></B>,
<I>101</I>, 5122-5130 (1979)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. L.
Jorgensen, J. D. Madura and C. J. Swenson, Optimized Intermolecular Potential
Functions for Liquid Hydrocarbons, <B><I>J. Am. Chem. Soc.</I></B>, <I>106</I>,
6638-6646 (1984)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. L.
Jorgensen and C. J. Swenson, Optimized Intermolecular Potential Functions for
Amides and Peptides: Structure and Properties of Liquid Amides, <B><I>J. Am.
Chem. Soc.</I></B>, <I>107</I>, 569-578 (1985)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. R. Maple,
U. Dinur and A. T. Hagler, Derivation of Force Fields for Molecular Mechanics
and Dynamics from <I>ab Initio</I> Surfaces, <B><I>Proc. Nat. Acad. Sci. USA</I></B>,
<I>85</I>, 5350-5354 (1988)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">U. Dinur and
A. T. Hagler, Direct Evaluation of Nonbonding Interactions from <I>ab Initio</I>
Calculations, <B><I>J. Am. Chem. Soc.</I></B>, <I>111</I>, 5149-5151 (1989)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">ELECTROSTATIC
INTERACTIONS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. L. Price,
Towards More Accurate Model Intermolecular Potentials for Organic Molecules, <B><I>Rev.
Comput. Chem.</I></B>, <I>14</I>, 225-289 (2000)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">C. H.
Faerman and S. L. Price, A Transferable Distributed Multipole Model for the
Electrostatic Interactions of Peptides and Amides, <B><I>J. Am. Chem. Soc.</I></B>,
<I>112</I>, 4915-4926 (1990)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">C. E.
Dykstra, Electrostatic Interaction Potentials in Molecular Force Fields, <B><I>Chem.
Rev.</I></B>, <I>93</I>, 2339-2353 (1993)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. J. Dudek
and J. W. Ponder, Accurate Modeling of the Intramolecular Electrostatic Energy
of Proteins, <B><I>J. Comput. Chem.</I></B>, <I>16</I>, 791-816 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">U. Koch and
E. Egert, An Improved Description of the Molecular Charge Density in Force
Fields with Atomic Multipole Moments, <B><I>J. Comput. Chem.</I></B>, <I>16</I>,
937-944 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">D. E.
Williams, Representation of the Molecular Electrostatic Potential by Atomic
Multipole and Bond Dipole Models, <B><I>J. Comput. Chem.</I></B>, <I>9</I>,
745-763 (1988)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">F. Colonna,
E. Evleth and J. G. Angyan, Critical Analysis of Electric Field Modeling:
Formamide, <B><I>J. Comput. Chem.</I></B>, <I>13</I>, 1234-1245 (1992)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">POLARIZATION
EFFECTS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. Kuwajima
and A. Warshel, Incorporating Electric Polarizabilities in Water-Water
Interaction Potentials, <B><I>J. Phys. Chem.</I></B>, <I>94</I>, 460-466 (1990)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. W.
Caldwell and P. A. Kollman, Structure and Properties of Neat Liquids Using
Nonadditive Molecular Dynamics: Water, Methanol, and N-Methylacetamide, <B><I>J.
Phys. Chem.</I></B>, <I>99</I>, 6208-6219 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">D. N.
Bernardo, Y. Ding, K. Kroegh-Jespersen and R. M. Levy, An Anisotropic
Polarizable Water Model: Incorporation of All-Atom Polarizabilities into
Molecular Mechanics Force Fields, <B><I>J. Phys. Chem.</I></B>, <I>98</I>,
4180-4187 (1994)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">P. T. van
Duijnen and M. Swart, Molecular and Atomic Polarizabilities: Thole's Model
Revisited, <B><I>J. Phys. Chem. A</I></B>, <I>102</I>, 2399-2407 (1998)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">K. J.
Miller, Calculation of the Molecular Polarizability Tensor, <B><I>J. Am. Chem. Soc.</I></B>,
<I>112</I>, 8543-8551 (1990)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J.
Applequist, J. R. Carl and K.-K. Fung, An Atom Dipole Interaction Model for
Molecular Polarizability. Application to Polyatomic Molecules and
Determination of Atom Polarizabilities, <B><I>J. Am. Chem. Soc.</I></B>, <I>94</I>,
2952-2960 (1972)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J.
Applequist, Atom Charge Transfer in Molecular Polarizabilities. Application of
the Olson-Sundberg Model to Aliphatic and Aromatic Hydrocarbons, <B><I>J.
Phys. Chem.</I></B>, <I>97</I>, 6016-6023 (1993)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A. J. Stone,
Distributed Polarizabilities, <B><I>Mol. Phys.</I></B>, <I>56</I>, 1065-1082 (1985)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. M. Stout
and C. E. Dykstra, A Distributed Model of the Electrical Response of Organic
Molecules, <B><I>J. Phys. Chem. A</I></B>, <I>102</I>, 1576-1582 (1998)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">MACROSCOPIC
TREATMENT OF SOLVENT</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">C. J. Cramer
and D. G. Truhlar, Continuum Solvation Models: Classical and Quantum
Mechanical Implementations, <B><I>Rev. Comput. Chem.</I></B>, <I>6</I>, 1-72 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">B.Roux and
T. Simonson, Implicit Solvation Models, <B><I>Biophys. Chem.</I></B>, <I>78</I>,
1-20 (1999)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. K.
Gilson, Introduction to Continuum Electrostatics with Molecular Applications,
available from http://gilsonlab.umbi.umd.edu</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">SURFACE
AREA-BASED SOLVATION MODELS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">D. Eisenberg
and A. D. McLachlan, Solvation Energy in Protein Folding and Binding, <B><I>Nature</I></B>,
<I>319</I>, 199-203 (1986)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">L. Wesson
and D. Eisenberg, Atomic Solvation Parameters Applied to Molecular Dynamics of
Proteins in Solution, <B><I>Prot. Sci.</I></B>, <I>1</I>, 227-235 (1992)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. Ooi, M.
Oobatake, G. Nemethy and H. A. Scheraga, Accessible Surface Areas as a Measure
of the Thermodynamic Parameters of Hydration of Peptides, <B><I>Proc. Natl.
Acad. Sci. USA</I></B>, <I>84</I>, 3086-3090 (1987)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. D.
Augspurger and H. A. Scheraga, An Efficient, Differentiable Hydration
Potential for Peptides and Proteins, <B><I>J. Comput. Chem.</I></B>, <I>17</I>,
1549-1558 (1996)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">GENERALIZED
BORN SOLVATION MODELS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">W. C. Still,
A. Tempczyk, R. C. Hawley and T. Hendrickson, A Semiempirical Treatment of
Solvation for Molecular Mechanics and Dynamics, <B><I>J. Am. Chem. Soc.</I></B>,
<I>112</I>, 6127-6129 (1990)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">D. Qiu, P.
S. Shenkin, F. P. Hollinger and W. C. Still, The GB/SA Continuum Model for
Solvation. A Fast Analytical Method for the Calculation of Approximate Born
Radii, <B><I>J. Phys. Chem. A</I></B>, <I>101</I>, 3005-3014 (1997)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">G. D.
Hawkins, C. J. Cramer and D. G. Truhlar, Pairwise Solute Descreening of Solute
Charges from a Dielectric Medium, <B><I>Chem. Phys. Lett.</I></B>, <I>246</I>,
122-129 (1995)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">G. D.
Hawkins, C. J. Cramer and D. G. Truhlar, Parametrized Models of Aqueous Free
Energies of Solvation Based on Pairwise Descreening of Solute Atomic Charges
from a Dielectric Medium, <B><I>J. Phys. Chem.</I></B>, <I>100</I>,
19824-19839 (1996)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A. Onufriev,
D. Bashford and D. A. Case, Modification of the Generalized Born Model
Suitable for Macromolecules, <B><I>J. Phys. Chem. B</I></B>, <I>104</I>,
3712-3720 (2000)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. Schaefer
and M. Karplus, A Comprehensive Analytical Treatment of Continuum
Electrostatics, <B><I>J. Phys. Chem.</I></B>, <I>100</I>, 1578-1599 (1996)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">M. Schaefer,
C. Bartels and M. Karplus, Solution Conformations and Thermodynamics of
Structured Peptides: Molecular Dynamics Simulation with an Implicit Solvation
Model, <B><I>J. Mol. Biol.</I></B>, <I>284</I>, 835-848 (1998)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">SUPERPOSITION
OF COORDINATE SETS</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. J.
Kearsley, An Algorithm for the Simultaneous Superposition of a Structural
Series, <B><I>J. Comput. Chem.</I></B>, <I>11</I>, 1187-1192 (1990)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">R. Diamond,
A Note on the Rotational Superposition Problem, <B><I>Acta Cryst.</I></B>, <I>A44</I>,
211-216 (1988)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">A. D.
McLachlan, Rapid Comparison of Protein Structures, <B><I>Acta Cryst.</I></B>, <I>A38</I>,
871-873 (1982)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. C.
Nyburg, Some Uses of a Best Molecular Fit Routine, <B><I>Acta Cryst.</I></B>, <I>B30</I>,
251-253 (1974)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><B><U><FONT SIZE=2><FONT FACE="New Century Schoolbook">LOCATION
OF TRANSITION STATES</FONT></FONT></U></B></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">R.
Czerminski and R. Elber, Reaction Path Study of Conformational Transitions and
Helix Formation in a Tetrapeptide, <B><I>Proc. Nat. Acad. Sci. USA</I></B>, <I>86</I>,
6963 (1989)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">R. S. Berry,
H. L. Davis and T. L. Beck, Finding Saddles on Multidimensional Potential
Surfaces, <B><I>Chem. Phys. Lett.</I></B>, <I>147</I>, 13 (1988)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">K. Muller,
Reaction Paths on Multidimensional Energy Hypersurfaces, <B><I>Ang. Chem. Int.
Ed. Engl.</I></B>, <I>19</I>, 1-13 (1980)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. Bell and
J. S. Crighton, Locating Transition States, <B><I>J. Chem. Phys.</I></B>, <I>80</I>,
2464-2475 (1984)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">S. Fischer
and M. Karplus, Conjugate Peak Refinement: An Algorithm for Finding Reaction
Paths and Accurate Transition States in Systems with Many Degrees of Freedom, <B><I>Chem.
Phys. Lett.</I></B>, <I>194</I>, 252-261 (1992)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">J. E.
Sinclair and R. Fletcher, A New Method of Saddle-Point Location for the
Calculation of Defect Migration Energies, <B><I>J. Phys. C</I></B>, <I>7</I>,
864-870 (1974)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">R. Elber and
M. Karplus, A Method for Determining Reaction Paths in Large Molecules: 
Application to Myoglobin, <B><I>Chem. Phys. Lett.</I></B>, <I>139</I>, 375-380 (1987)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">D. T. Nguyen
and D. A. Case, On Finding Stationary States on Large-Molecule Potential
Energy Surfaces, <B><I>J. Phys. Chem.</I></B>, <I>89</I>, 4020-4026 (1985)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">T. A.
Halgren and W. N. Lipscomb, The Synchronous-Transit Method for Determining
Reaction Pathways and Locating Molecular Transition States, <B><I>Chem. Phys. Lett.</I></B>,
<I>49</I>, 225-232 (1977)</FONT></FONT></P>
<P ALIGN=JUSTIFY></P>
<P ALIGN=JUSTIFY><FONT SIZE=2><FONT FACE="New Century Schoolbook">G. T.
Barkema and N. Mousseau, Event-Based Relaxation of Continuous Disordered
Systems, <B><I>Phys. Rev. Lett.</I></B>, <I>77</I>, 4358-4361 (1996)</FONT></FONT></P>
<P></P>
</BODY>
</HTML>
